{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyspark.conf import SparkConf\n",
    "from pyspark.sql import SparkSession\n",
    "from pyspark.sql.types import *\n",
    "from pyspark.sql import functions as Funct\n",
    "from pyspark.sql.window import Window\n",
    "import csv\n",
    "import re\n",
    "\n",
    "sparkSession = SparkSession.builder.appName(\"myApp\").getOrCreate()\n",
    "#Get default configurations\n",
    "sparkSession.sparkContext._conf.getAll()\n",
    "#Update default configurations\n",
    "conf = sparkSession.sparkContext._conf.setAll([('spark.executor.memory', '16g')\\\n",
    "                                        , ('spark.app.name', 'Spark Updated Conf')\\\n",
    "                                        , ('spark.executor.cores', '12')\\\n",
    "                                        , ('spark.cores.max', '12')\\\n",
    "                                        , ('spark.driver.memory','16g')\\\n",
    "                                        ,('spark.driver.maxResultSize','16g')])\n",
    "#Stop the current Spark Session\n",
    "sparkSession.sparkContext.stop()\n",
    "#Create a Spark Session\n",
    "sparkSession = SparkSession.builder.config(conf=conf).getOrCreate()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.2 (Loading the dataset into an RDD)\n",
    "Loading the dataset in the following RDD:<br>\n",
    "a) userRatingsRDD: pair RDD from **user_libraries.txt** using the user hash as the key and the liked paper(s) as value(s).<br>\n",
    "b) paperTermsRDD: pair RDD from **papers.csv** using the paper_id as the key and the words contained in the abstract as the value(s).<br>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.2\n",
    "#Reading txt and csv files into RDD\n",
    "userLibRDD = sparkSession.sparkContext.textFile(\"./mod_users_libraries.txt\")\n",
    "stopWordRDD = sparkSession.sparkContext.textFile(\"./stopwords_en.txt\")\n",
    "papersRDD = sparkSession.sparkContext.textFile(\"./papers.csv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('28d3f81251d94b09735497477a5e4e02',\n",
       "  '3929762,503574,5819422,4238883,5788061,462949,635215,635216,4810441,3481823,4165233,3366480,5984302,4238942,5490453,4636156,5996865,4194836,5828780,4450195'),\n",
       " ('d0c9aaa788153daeaf1f1538b3d46bbb',\n",
       "  '2080631,6343346,5184704,7756088,2653863,6607628,4236212,1277953,226864,3140015,8806369,311570,5687747,767516,4781370,2841637,2445106,1959511,2688186,2363430,6614346,853030,5336762,4226226,239571,4089758,4140337,913868,7562861,3190274,2782576,12571584,2049617,5761055,5441098,3466838,2080691,1805577,7570111,5760287,2855355,3281547,1012525,3512183,678653')]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.2\n",
    "#Creating a pair RDD from user_libraries.txt using the user hash as key and liked paper(s) as value(s)\n",
    "userRatingsRDD = userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\n",
    "#Displaying first 'n' elements of dataset\n",
    "userLibRDD.unpersist()\n",
    "userRatingsRDD.take(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('28d3f81251d94b09735497477a5e4e02', '3929762'),\n",
       " ('28d3f81251d94b09735497477a5e4e02', '503574'),\n",
       " ('28d3f81251d94b09735497477a5e4e02', '5819422'),\n",
       " ('28d3f81251d94b09735497477a5e4e02', '4238883'),\n",
       " ('28d3f81251d94b09735497477a5e4e02', '5788061')]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.2\n",
    "#converting the userRatingsRDD into key (user_hash_id) value (paper_id) pairs RDD\n",
    "userRatingsList = []\n",
    "for keyListVal in userRatingsRDD.collect():\n",
    "    for val in keyListVal[1].split(','):\n",
    "        userRatingsList.append((keyListVal[0],val))\n",
    "\n",
    "userRatingsKVPairRDD = sparkSession.sparkContext.parallelize(userRatingsList)\n",
    "userRatingsKVPairRDD.take(5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('80546',\n",
       "  '\"the genetic code has been regarded as arbitrary in the sense that the codon-amino acid assignments could be different than they actually are. this general idea has been spelled out differently by previous, often rather implicit accounts of arbitrariness. they have drawn on the frozen accident theory, on evolutionary contingency, on alternative causal pathways, and on the absence of direct stereochemical interactions between codons and amino acids. it has also been suggested that the arbitrariness of the genetic code justifies attributing semantic information to macromolecules, notably to {dna}. i argue that these accounts of arbitrariness are unsatisfactory. i propose that the code is arbitrary in the sense of jacques monod\\'s concept of chemical arbitrariness: the genetic code is arbitrary in that any codon requires certain chemical and structural properties to specify a particular amino acid, but these properties are not required in virtue of a principle of chemistry. this notion of arbitrariness is compatible with several recent hypotheses about code evolution. i maintain that the code\\'s chemical arbitrariness is neither sufficient nor necessary for attributing semantic information to nucleic acids.\"'),\n",
       " ('5842862',\n",
       "  '\"choosing good problems is essential for being a good scientist. but what is a good problem, and how do you choose one? the subject is not usually discussed explicitly within our profession. scientists are expected to be smart enough to figure it out on their own and through the observation of their teachers. this lack of explicit discussion leaves a vacuum that can lead to approaches such as choosing problems that can give results that merit publication in valued journals, resulting in a job and tenure.\"')]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.2\n",
    "#Creating a pair RDD from papers.csv using paper_id as key and abstract as value(s)\n",
    "paperTermsRDD = papersRDD.map(lambda line: (line.split(',')[0], \",\".join(line.split(',')[14:])))\n",
    "papersRDD.unpersist()\n",
    "paperTermsRDD.take(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.3 (Joining Collections)\n",
    "Compute for each user the top-10 most frequent words appearing in the papers she likes. Exclude the stop words listed in **stopwords_en.txt**.<br>\n",
    "Store the results into a file which contains in each line the user hash and the list of her retrieved words sorted by frequency (top 1 is the most frequent)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['a', 'able', 'about', 'above', 'according', 'accordingly', 'across', 'actually', 'after', 'afterwards']\n"
     ]
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#Creating a broadcast variable holding stop words\n",
    "stopWordBrdcast = sparkSession.sparkContext.broadcast(stopWordRDD.collect())\n",
    "print(stopWordBrdcast.value[0:10])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('503574', ('28d3f81251d94b09735497477a5e4e02', '')),\n",
       " ('5788061',\n",
       "  ('28d3f81251d94b09735497477a5e4e02',\n",
       "   ' and its impact on {cmb} anomalies\",')),\n",
       " ('4238942', ('28d3f81251d94b09735497477a5e4e02', '')),\n",
       " ('5490453', ('28d3f81251d94b09735497477a5e4e02', '')),\n",
       " ('5828780', ('28d3f81251d94b09735497477a5e4e02', ''))]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#Join the userRatingsKVPairRDD with the paperTermsRDD\n",
    "#userRatingsKVPairRDD.paper_id = paperTermsRDD.paper_id\n",
    "joinResultRDD = userRatingsKVPairRDD.map(lambda x: (x[1], x[0])).join(paperTermsRDD)\n",
    "userRatingsKVPairRDD.unpersist()\n",
    "joinResultRDD.take(5)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('f05bcffe7951de9e5a32fff4a42eb088',\n",
       "  'a protein domain database for functional characterization and annotation.\",\"{prosite} consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. it is complemented by {prorule}, a collection of rules based on profiles and patterns, which increases the discriminatory power of these profiles and patterns by providing additional information about functionally and/or structurally critical amino acids. {prosite} is largely used for the annotation of domain features of {uniprotkb}/{swiss-prot} entries. among the 983 ({dna}-binding) domains, repeats and zinc fingers present in {swiss-prot} (release 57.8 of 22 september 2009), 696 ( approximately 70\\\\%) are annotated with {prosite} descriptors using information from {prorule}. in order to allow better functional characterization of domains, {prosite} developments focus on subfamily specific profiles and a new profile building method giving more weight to functionally important residues. here, we describe {amsa}, an annotated multiple sequence alignment format used to build a new generation of generalized profiles, the migration of {scanprosite} to {vital-it}, a cluster of 633 {cpus}, and the adoption of the distributed annotation system ({das}) to facilitate {prosite} data integration and interchange with other sources. the latest version of {prosite} (release 20.54, of 22 september 2009) contains 1308 patterns, 863 profiles and 869 {prorules}. {prosite} is accessible at: http://www.expasy.org/prosite/.\"\"impressive advances in {ngs} have enabled an immense diversity of novel applications. the barrier of the \\\\$1000 genome has recently been broken. important novel tools for clinical diagnostics based on {ngs} are appearing. third-generation technologies may further revolutionize genomics research. significant challenges for {ngs} remain, in particular data storage and processing. ten years ago next-generation sequencing ({ngs}) technologies appeared on the market. during the past decade, tremendous progress has been made in terms of speed, read length, and throughput, along with a sharp reduction in per-base cost. together, these advances democratized {ngs} and paved the way for the development of a large number of novel {ngs} applications in basic science as well as in translational research areas such as clinical diagnostics, agrigenomics, and forensic science. here we provide an overview of the evolution of {ngs} and discuss the most significant improvements in sequencing technologies and library preparation protocols. we also explore the current landscape of {ngs} applications and provide a perspective for future developments.\"{dna} methylation: roles in mammalian development,\"{dna} methylation is among the best studied epigenetic modifications and is essential to mammalian development. although the methylation status of most {cpg} dinucleotides in the genome is stably propagated through mitosis, improvements to methods for measuring methylation have identified numerous regions in which it is dynamically regulated. in this review, we discuss key concepts in the function of {dna} methylation in mammals, stemming from more than two decades of research, including many recent studies that have elucidated when and where {dna} methylation has a regulatory role in the genome. we include insights from early development, embryonic stem cells and adult lineages, particularly haematopoiesis, to highlight the general features of this modification as it participates in both global and localized epigenetic regulation.\"storing, and querying biological pathways.\",\"biological pathways, including metabolic pathways, protein interaction networks, signal transduction pathways, and gene regulatory networks, are currently represented in over 220 diverse databases. these data are crucial for the study of specific biological processes, including human diseases. standard exchange formats for pathway information, such as {biopax}, {cellml}, {sbml} and {psi}-{mi}, enable convenient collection of this data for biological research, but mechanisms for common storage and communication are required. we have developed {cpath}, an open source database and web application for collecting, storing, and querying biological pathway data. {cpath} makes it easy to aggregate custom pathway data sets available in standard exchange formats from multiple databases, present pathway data to biologists via a customizable web interface, and export pathway data via a web service to third-party software, such as cytoscape, for visualization and analysis. {cpath} is software only, and does not include new pathway information. key features include: a built-in identifier mapping service for linking identical interactors and linking to external resources; built-in support for {psi}-{mi} and {biopax} standard pathway exchange formats; a web service interface for searching and retrieving pathway data sets; and thorough documentation. the {cpath} software is freely available under the {lgpl} open source license for academic and commercial use. {cpath} is a robust, scalable, modular, professional-grade software platform for collecting, storing, and querying biological pathways. it can serve as the core data handling component in information systems for pathway visualization, analysis and modeling.\"new haven, connecticut 06520, usa.\",annotation transfer between genomes: protein-protein interologs and {protein-dna} regulogs.,texts, and trees.\",\"{bionames} is a web database of taxonomic names for animals, linked to the primary literature and, wherever possible, to phylogenetic trees. it aims to provide a taxonomic \"\"dashboard\"\" where at a glance we can see a summary of the taxonomic and phylogenetic information we have for a given taxon and hence provide a quick answer to the basic question \"\"what is this taxon?\"\" {bionames} combines classifications from the global biodiversity information facility ({gbif}) and {genbank}, images from the encyclopedia of life ({eol}), animal names from the index of organism names ({ion}), and bibliographic data from multiple sources including the biodiversity heritage library ({bhl}) and {crossref}. the user interface includes display of full text articles, interactive timelines of taxonomic publications, and zoomable phylogenies. it is available at http://bionames.org.\"\"the prediction of regulatory elements is a problem where computational methods offer great hope. over the past few years, numerous tools have become available for this task. the purpose of the current assessment is twofold: to provide some guidance to users regarding the accuracy of currently available tools in various settings, and to provide a benchmark of data sets for assessing future tools.\"\"unraveling the mechanisms that regulate gene expression is a major challenge in biology. an important task in this challenge is to identify regulatory elements, especially the binding sites in deoxyribonucleic acid ({dna}) for transcription factors. these binding sites are short {dna} segments that are called motifs. recent advances in genome sequence availability and in high-throughput gene expression analysis technologies have allowed for the development of computational methods for motif finding. as a result, a large number of motif finding algorithms have been implemented and applied to various motif models over the past decade. this survey reviews the latest developments in {dna} motif finding algorithms. earlier algorithms use promoter sequences of coregulated genes from single genome and search for statistically overrepresented motifs. recent algorithms are designed to use phylogenetic footprinting or orthologous sequences and also an integrated approach where promoter sequences of coregulated genes and phylogenetic footprinting are used. all the algorithms studied have been reported to correctly detect the motifs that have been previously detected by laboratory experimental approaches, and some algorithms were able to find novel motifs. however, most of these motif finding algorithms have been shown to work successfully in yeast and other lower organisms, but perform significantly worse in higher organisms. despite considerable efforts to date, {dna} motif finding remains a complex challenge for biologists and computer scientists. researchers have taken many different approaches in developing motif discovery tools and the progress made in this area of research is very encouraging. performance comparison of different motif finding tools and identification of the best tools have proven to be a difficult task because tools are designed based on algorithms and motif models that are diverse and complex and our incomplete understanding of the biology of regulatory mechanism does not always provide adequate evaluation of underlying algorithms over motif models.\"\"the gibbs motif sampler is a software package for locating common elements in collections of biopolymer sequences. in this paper we describe a new variation of the gibbs motif sampler, the gibbs recursive sampler, which has been developed specifically for locating multiple transcription factor binding sites for multiple transcription factors simultaneously in unaligned {dna} sequences that may be heterogeneous in {dna} composition. here we describe the basic operation of the web-based version of this sampler. the sampler may be acces-sed at http://bayesweb.wadsworth.org/gibbs/gibbs.html and at http://www.bioinfo.rpi.edu/applications/bayesian/gibbs/gibbs.html. an online user guide is available at http://bayesweb.wadsworth.org/gibbs/bernoulli.html and at http://www.bioinfo.rpi.edu/applications/bayesian/gibbs/manual/bernoulli.html. solaris, solaris.x86 and linux versions of the sampler are available as stand-alone programs for academic and not-for-profit users. commercial licenses are also available. the gibbs recursive sampler is distributed in accordance with the {iscb} level 0 guidelines and a requirement for citation of use in scientific publications.\"wellcome trust genome campus, hinxton, cambridge, cb10 1sd, uk.\",the ontology lookup service: more data and better tools for controlled vocabulary queries,\"the ontology lookup service ({ols}) (http://www.ebi.ac.uk/ols) provides interactive and programmatic interfaces to query, browse and navigate an ever increasing number of biomedical ontologies and controlled vocabularies. the volume of data available for querying has more than quadrupled since it went into production and {ols} functionality has been integrated into several high-usage databases and data entry tools. improvements have been made to both {ols} query interfaces, based on user feedback and requirements, to improve usability and service interoperability and provide novel ways to perform queries.\"\"in a chromatin immunoprecipitation followed by high-throughput sequencing ({chip}-seq) experiment, an important consideration in experimental design is the minimum number of sequenced reads required to obtain statistically significant results. we present an extensive evaluation of the impact of sequencing depth on identification of enriched regions for key histone modifications ({h3k4me3}, {h3k36me3}, {h3k27me3} and {h3k9me2}/me3) using deep-sequenced datasets in human and fly. we propose to define sufficient sequencing depth as the number of reads at which detected enrichment regions increase <1\\\\% for an additional million reads. although the required depth depends on the nature of the mark and the state of the cell in each experiment, we observe that sufficient depth is often reached at <20 million reads for fly. for human, there are no clear saturation points for the examined datasets, but our analysis suggests 40–50 million reads as a practical minimum for most marks. we also devise a mathematical model to estimate the sufficient depth and total genomic coverage of a mark. lastly, we find that the five algorithms tested do not agree well for broad enrichment profiles, especially at lower depths. our findings suggest that sufficient sequencing depth and an appropriate peak-calling algorithm are essential for ensuring robustness of conclusions derived from {chip}-seq data.\"\"all inferences in comparative biology depend on accurate estimates of evolutionary relationships. recent phylogenetic analyses have turned away from maximum parsimony towards the probabilistic techniques of maximum likelihood and bayesian markov chain monte carlo ({bmcmc}). these probabilistic techniques represent a parametric approach to statistical phylogenetics, because their criterion for evaluating a topology—the probability of the data, given the tree—is calculated with reference to an explicit evolutionary model from which the data are assumed to be identically distributed. maximum parsimony can be considered nonparametric, because trees are evaluated on the basis of a general metric—the minimum number of character state changes required to generate the data on a given tree—without assuming a specific distribution1. the shift to parametric methods was spurred, in large part, by studies showing that although both approaches perform well most of the time2, maximum parsimony is strongly biased towards recovering an incorrect tree under certain combinations of branch lengths, whereas maximum likelihood is not3, 4, 5, 6. all these evaluations simulated sequences by a largely homogeneous evolutionary process in which data are identically distributed. there is ample evidence, however, that real-world gene sequences evolve heterogeneously and are not identically distributed7, 8, 9, 10, 11, 12, 13, 14, 15, 16. here we show that maximum likelihood and {bmcmc} can become strongly biased and statistically inconsistent when the rates at which sequence sites evolve change non-identically over time. maximum parsimony performs substantially better than current parametric methods over a wide range of conditions tested, including moderate heterogeneity and phylogenetic problems not normally considered difficult.\"java library, r library, and {tool-chain} for mass spectrometry data analysis\",\"the recent proliferation of high-resolution mass spectrometers has generated a wealth of new data analysis methods. however, flexible integration of these methods into configurations best suited to the research question is hampered by heterogeneous file formats and monolithic software development. the {mzxml}, {mzdata}, and {mzml} file formats have enabled uniform access to unprocessed raw data. in this paper we present our efforts to produce an equally simple and powerful format, {peakml}, to uniformly exchange processed intermediary and result data. to demonstrate the versatility of {peakml}, we have developed an open source java toolkit for processing, filtering, and annotating mass spectra in a customizable pipeline ({mzmatch}), as well as a user-friendly data visualization environment ({peakml} viewer). the {peakml} format in particular enables the flexible exchange of processed data between software created by different groups or companies, as we illustrate by providing a {peakml}-based integration of the widely used {xcms} package with {mzmatch} data processing tools. as an added advantage, downstream analysis can benefit from direct access to the full mass trace information underlying summarized mass spectrometry results, providing the user with the means to rapidly verify results. the {peakml}/{mzmatch} software is freely available at http://mzmatch.sourceforge.net, with documentation, tutorials, and a community forum.\"braunschweig, germany.\",{transfac}: a database on transcription factors and their {dna} binding sites,{transfac} is a database about eukaryotic transcription regulating {dna} sequence elements and the transcription factors binding to and acting through them. this report summarizes the present status of this database and accompanying retrieval tools.hinxton, cambridge cb10 1sd and department of genetics, university of cambridge, cambridge cb2 3eh, uk.\",{chebi}: a database and ontology for chemical entities of biological interest,\"chemical entities of biological interest ({chebi}) is a freely available dictionary of molecular entities focused on \\'small\\' chemical compounds. the molecular entities in question are either natural products or synthetic products used to intervene in the processes of living organisms. genome-encoded macromolecules (nucleic acids, proteins and peptides derived from proteins by cleavage) are not as a rule included in {chebi}. in addition to molecular entities, {chebi} contains groups (parts of molecular entities) and classes of entities. {chebi} includes an ontological classification, whereby the relationships between molecular entities or classes of entities and their parents and/or children are specified. {chebi} is available online at http://www.ebi.ac.uk/chebi/\"\"we offer a publicly available service that can interactively map protein identifiers and protein sequences to the majority of commonly used protein databases. programmatic access is available through a standards-compliant {soap} interface or a lightweight {rest} interface. the {picr} interface, documentation and code examples are available at {http://www.ebi.ac.uk/tools}/picr.\"\"differential gene expression is the fundamental mechanism underlying animal development and cell differentiation. however, it is a challenge to identify comprehensively and accurately the {dna} sequences that are required to regulate gene expression: namely, cis-regulatory modules ({crms}). three major features, either singly or in combination, are used to predict {crms}: clusters of transcription factor binding site motifs, non-coding {dna} that is under evolutionary constraint and biochemical marks associated with {crms}, such as histone modifications and protein occupancy. the validation rates for predictions indicate that identifying diagnostic biochemical marks is the most reliable method, and understanding is enhanced by the analysis of motifs and conservation patterns within those predicted {crms}.\"\"reactome (http://www.reactome.org) is a manually curated open-source open-data resource of human pathways and reactions. the current version 46 describes 7088 human proteins (34\\\\% of the predicted human proteome), participating in 6744 reactions based on data extracted from 15 107 research publications with {pubmed} links. the reactome web site and analysis tool set have been completely redesigned to increase speed, flexibility and user friendliness. the data model has been extended to support annotation of disease processes due to infectious agents and to mutation.\"{piana}: protein interactions and network analysis.,\"we present a software framework and tool called protein interactions and network analysis ({piana}) that facilitates working with protein interaction networks by (1) integrating data from multiple sources, (2) providing a library that handles graph-related tasks and (3) automating the analysis of protein-protein interaction networks. {piana} can also be used as a stand-alone application to create protein interaction networks and perform tasks such as predicting protein interactions and helping to identify spots in a {2d} electrophoresis gel. availability: {piana} is under the {gnu} {gpl}. source code, database and detailed documentation may be freely downloaded from http://sbi.imim.es/piana.\"university of california, berkeley, usa. amoses@ocf.berkeley.edu\",phylogenetic motif detection by expectation-maximization on evolutionary mixtures.,\"the preferential conservation of transcription factor binding sites implies that non-coding sequence data from related species will prove a powerful asset to motif discovery. we present a unified probabilistic framework for motif discovery that incorporates evolutionary information. we treat aligned {dna} sequence as a mixture of evolutionary models, for motif and background, and, following the example of the {meme} program, provide an algorithm to estimate the parameters by {expectation-maximization}. we examine a variety of evolutionary models and show that our approach can take advantage of phylogenic information to avoid false positives and discover motifs upstream of groups of characterized target genes. we compare our method to traditional motif finding on only conserved regions. an implementation will be made available at http://rana.lbl.gov.\"harpenden, uk. jacob.koehler@bbsrc.ac.uk\",\"linking experimental results, biological networks and sequence analysis methods using ontologies and generalised data structures.\",\"the structure of a closely integrated data warehouse is described that is designed to link different types and varying numbers of biological networks, sequence analysis methods and experimental results such as those coming from microarrays. the data schema is inspired by a combination of graph based methods and generalised data structures and makes use of ontologies and meta-data. the core idea is to consider and store biological networks as graphs, and to use generalised data structures ({gds}) for the storage of further relevant information. this is possible because many biological networks can be stored as graphs: protein interactions, signal transduction networks, metabolic pathways, gene regulatory networks etc. nodes in biological graphs represent entities such as promoters, proteins, genes and transcripts whereas the edges of such graphs specify how the nodes are related. the semantics of the nodes and edges are defined using ontologies of node and relation types. besides generic attributes that most biological entities possess (name, attribute description), further information is stored using generalised data structures. by directly linking to underlying sequences (exons, introns, promoters, amino acid sequences) in a systematic way, close interoperability to sequence analysis methods can be achieved. this approach allows us to store, query and update a wide variety of biological information in a way that is semantically compact without requiring changes at the database schema level when new kinds of biological information is added. we describe how this datawarehouse is being implemented by extending the text-mining framework {ondex} to link, support and complement different bioinformatics applications and research activities such as microarray analysis, sequence analysis and modelling/simulation of biological systems. the system is developed under the {gpl} license and can be downloaded from http://sourceforge.net/projects/ondex/\"national library of medicine, national institutes of health, bethesda, md 20894.\",detecting subtle sequence signals: a gibbs sampling strategy for multiple alignment.,\"a wealth of protein and {dna} sequence data is being generated by genome projects and other sequencing efforts. a crucial barrier to deciphering these sequences and understanding the relations among them is the difficulty of detecting subtle local residue patterns common to multiple sequences. such patterns frequently reflect similar molecular structures and biological properties. a mathematical definition of this \"\"local multiple alignment\"\" problem suitable for full computer automation has been used to develop a new and sensitive algorithm, based on the statistical method of iterative sampling. this algorithm finds an optimized local alignment model for n sequences in n-linear time, requiring only seconds on current workstations, and allows the simultaneous detection and optimization of multiple patterns and pattern repeats. the method is illustrated as applied to helix-turn-helix proteins, lipocalins, and prenyltransferases.\"usa\",readings in speech recognition,an abstract is not available.\"{background}:the {kegg} {pathway} database provides a plethora of pathways for a diversity of organisms. all pathway components are directly linked to other {kegg} databases, such as {kegg} {compound} or {kegg} {reaction}. therefore, the pathways can be extended with an enormous amount of information and provide a foundation for initial structural modeling approaches. as a drawback, {kgml}-formatted {kegg} pathways are primarily designed for visualization purposes and often omit important details for the sake of a clear arrangement of its entries. thus, a direct conversion into systems biology models would produce incomplete and erroneous {models.results}:here, we present a precise method for processing and converting {kegg} pathways into initial metabolic and signaling models encoded in the standardized community pathway formats {sbml} (levels 2 and 3) and {biopax} (levels 2 and 3). this method involves correcting invalid or incomplete {kgml} content, creating complete and valid stoichiometric reactions, translating relations to signaling models and augmenting the pathway content with various information, such as cross-references to entrez gene, {omim}, {uniprot} {chebi}, and many {more.finally}, we compare several existing conversion tools for {kegg} pathways and show that the conversion from {kegg} to {biopax} does not involve a loss of information, whilst lossless translations to {sbml} can only be performed using {sbml} level 3, including its recently proposed qualitative models and groups extension {packages.conclusions}:building correct {biopax} and {sbml} signaling models from the {kegg} database is a unique characteristic of the proposed method. further, there is no other approach that is able to appropriately construct metabolic models from {kegg} pathways, including correct reactions with stoichiometry. the resulting initial models, which contain valid and comprehensive {sbml} or {biopax} code and a multitude of cross-references, lay the foundation to facilitate further modeling steps.\"ondex web: web-based visualization and exploration of heterogeneous biological networks.,\"ondex web is a new web-based implementation of the network visualization and exploration tools from the ondex data integration platform. new features such as context-sensitive menus and annotation tools provide users with intuitive ways to explore and manipulate the appearance of heterogeneous biological networks. ondex web is open source, written in java and can be easily embedded into web sites as an applet. ondex web supports loading data from a variety of network formats, such as {xgmml}, {nwb}, pajek and {oxl}. {http://ondex.rothamsted.ac.uk/ondexweb}.\"\"{kegg} (kyoto encyclopedia of genes and genomes) is a knowledge base for systematic analysis of gene functions, linking genomic information with higher order functional information. the genomic information is stored in the {genes} database, which is a collection of gene catalogs for all the completely sequenced genomes and some partial genomes with up-to-date annotation of gene functions. the higher order functional information is stored in the {pathway} database, which contains graphical representations of cellular processes, such as metabolism, membrane transport, signal transduction and cell cycle. the {pathway} database is supplemented by a set of ortholog group tables for the information about conserved subpathways (pathway motifs), which are often encoded by positionally coupled genes on the chromosome and which are especially useful in predicting gene functions. a third database in {kegg} is {ligand} for the information about chemical compounds, enzyme molecules and enzymatic reactions. {kegg} provides java graphics tools for browsing genome maps, comparing two genome maps and manipulating expression maps, as well as computational tools for sequence comparison, graph comparison and path computation. the {kegg} databases are daily updated and made freely available (http://www. genome.ad.jp/kegg/).\"44 binney st, boston, ma 02115, usa. rgentlem@jimmy.harvard.edu\",bioconductor: open software development for computational biology and bioinformatics.,\"the bioconductor project is an initiative for the collaborative creation of extensible software for computational biology and bioinformatics. the goals of the project include: fostering collaborative development and widespread use of innovative software, reducing barriers to entry into interdisciplinary scientific research, and promoting the achievement of remote reproducibility of research results. we describe details of our aims and methods, identify current challenges, compare bioconductor to other open bioinformatics projects, and provide working examples.\"\"{metabolights} (http://www.ebi.ac.uk/metabolights) is the first general-purpose, open-access repository for metabolomics studies, their raw experimental data and associated metadata, maintained by one of the major open-access data providers in molecular biology. metabolomic profiling is an important tool for research into biological functioning and into the systemic perturbations caused by diseases, diet and the environment. the effectiveness of such methods depends on the availability of public open data across a broad range of experimental methods and conditions. the {metabolights} repository, powered by the open source {isa} framework, is cross-species and cross-technique. it will cover metabolite structures and their reference spectra as well as their biological roles, locations, concentrations and raw data from metabolic experiments. studies automatically receive a stable unique accession number that can be used as a publication reference (e.g. {mtbls1}). at present, the repository includes 15 submitted studies, encompassing 93 protocols for 714 assays, and span over 8 different species including human, caenorhabditis elegans, mus musculus and arabidopsis thaliana. eight hundred twenty-seven of the metabolites identified in these studies have been mapped to {chebi}. these studies cover a variety of techniques, including {nmr} spectroscopy and mass spectrometry.\"with increased coverage and integration.\",\"complete knowledge of all direct and indirect interactions between proteins in a given cell would represent an important milestone towards a comprehensive description of cellular mechanisms and functions. although this goal is still elusive, considerable progress has been made-particularly for certain model organisms and functional systems. currently, protein interactions and associations are annotated at various levels of detail in online resources, ranging from raw data repositories to highly formalized pathway databases. for many applications, a global view of all the available interaction data is desirable, including lower-quality data and/or computational predictions. the {string} database (http://string-db.org/) aims to provide such a global perspective for as many organisms as feasible. known and predicted associations are scored and integrated, resulting in comprehensive protein networks covering >1100 organisms. here, we describe the update to version 9.1 of {string}, introducing several improvements: (i) we extend the automated mining of scientific texts for interaction information, to now also include full-text articles; (ii) we entirely re-designed the algorithm for transferring interactions from one model organism to the other; and (iii) we provide users with statistical information on any functional enrichment observed in their networks.\"\"{background}:biologically active sequence motifs often have positional preferences with respect to a genomic landmark. for example, many known transcription factor binding sites ({tfbss}) occur within an interval [-300, 0] bases upstream of a transcription start site ({tss}). although some programs for identifying sequence motifs exploit positional information, most of them model it only implicitly and with ad hoc methods, making them unsuitable for general motif {searches.results}:{a-glam}, a user-friendly computer program for identifying sequence motifs, now incorporates a bayesian model systematically combining sequence and positional information. {a-glam}\\'s predictions with and without positional information were compared on two human {tfbs} datasets, each containing sequences corresponding to the interval [-2000, 0] bases upstream of a known {tss}. a rigorous statistical analysis showed that positional information significantly improved the prediction of sequence motifs, and an extensive cross-validation study showed that {a-glam}\\'s model was robust against mild misspecification of its parameters. as expected, when sequences in the datasets were successively truncated to the intervals [-1000, 0], [-500, 0] and [-250, 0], positional information aided motif prediction less and less, but never hurt it {significantly.conclusion}:although sequence truncation is a viable strategy when searching for biologically active motifs with a positional preference, a probabilistic model (used reasonably) generally provides a superior and more robust strategy, particularly when the sequence motifs\\' positional preferences are not well characterized.\"\"sumary: two sample logo is a web-based tool that detects and displays statistically significant differences in position-specific symbol compositions between two sets of multiple sequence alignments. in a typical scenario, two groups of aligned sequences will share a common motif but will differ in their functional annotation. the inclusion of the background alignment provides an appropriate underlying amino acid or nucleotide distribution and addresses intersite symbol correlations. in addition, the difference detection process is sensitive to the sizes of the aligned groups. two sample logo extends {weblogo}, a widely-used sequence logo generator. the source code is distributed under the {mit} open source license agreement and is available for download free of charge.\"new york, ny 10021, usa. saurabh@lonnrot.rockefeller.edu\",{phyme}: a probabilistic algorithm for finding motifs in sets of orthologous sequences.,\"this paper addresses the problem of discovering transcription factor binding sites in heterogeneous sequence data, which includes regulatory sequences of one or more genes, as well as their orthologs in other species. we propose an algorithm that integrates two important aspects of a motif\\'s significance - overrepresentation and cross-species conservation - into one probabilistic score. the algorithm allows the input orthologous sequences to be related by any user-specified phylogenetic tree. it is based on the {expectation-maximization} technique, and scales well with the number of species and the length of input sequences. we evaluate the algorithm on synthetic data, and also present results for data sets from yeast, fly, and human. the results demonstrate that the new approach improves motif discovery by exploiting multiple species information.\"\"despite its increasing role in communication, the {world-wide} web remains uncontrolled: any individual or institution can create a website with any number of documents and links. this unregulated growth leads to a huge and complex web, which becomes a large directed graph whose vertices are documents and whose edges are links ({urls}) that point from one document to another. the topology of this graph determines the web\\'s connectivity and consequently how effectively we can locate information on it. but its enormous size (estimated to be at least 8108 documents1) and the continual changing of documents and links make it impossible to catalogue all the vertices and edges.\"pittsburgh usa.\",penalized and weighted k-means for clustering with scattered objects and prior information in high-throughput biological data,\"motivation: cluster analysis is one of the most important data mining tools for investigating high-throughput biological data. the existence of many scattered objects that should not be clustered has been found to hinder performance of most traditional clustering algorithms in such a high-dimensional complex situation. very often, additional prior knowledge from databases or previous experiments is also available in the analysis. excluding scattered objects and incorporating existing prior information are desirable to enhance the clustering performance.  results: in this article, a class of loss functions is proposed for cluster analysis and applied in high-throughput genomic and proteomic data. two major extensions from k-means are involved: penalization and weighting. the additive penalty term is used to allow a set of scattered objects without being clustered. weights are introduced to account for prior information of preferred or prohibited cluster patterns to be identified. their relationship with the classification likelihood of gaussian mixture models is explored. incorporation of good prior information is also shown to improve the global optimization issue in clustering. applications of the proposed method on simulated data as well as high-throughput data sets from tandem mass spectrometry ({ms}/{ms}) and microarray experiments are presented. our results demonstrate its superior performance over most existing methods and its computational simplicity and extensibility in the application of large complex biological data sets.  availability: http://www.pitt.edu/\\\\~{}ctseng/research/software.html  contact: ctseng@pitt.edu  supplementary information: supplementary data are available at bioinformatics online. 10.1093/bioinformatics/btm320\"\"a method for studying the sequence-specific binding of proteins to {dba} is described. the technique is a simple conjoining of the {maxam-gilbert} {dna}-sequencing method and the technique of {dnaase}-protected fragment isolation. fragments of a 5\\' end-labelled, double-stranded {dna} segment, partially degraded by {dnaase} in the presence and absence of the binding protein, are visualized by electrophoresis and autoradiography alongside the base-specific reaction products of the {maxam-gilbert} sequencing method. it is then possible to see the protective \"\"footprint\"\" of the binding protein on the {dna} sequence. the binding of lac repressor to lac operator is visualized by \"\"footprinting\"\" as an example. equillibrium estimates indicate that 10-fold sequence-specificity (differential binding constant) could be studied easily using this technique.\"regulation of cytoplasmic {mrna} decay.,\"discoveries made over the past 20 years highlight the importance of {mrna} decay as a means of modulating gene expression and thereby protein production. up until recently, studies largely focused on identifying cis-acting sequences that serve as {mrna} stability or instability elements, the proteins that bind these elements, how the process of translation influences {mrna} decay and the ribonucleases that catalyse decay. now, current studies have begun to elucidate how the decay process is regulated. this review examines our current understanding of how mammalian cell {mrna} decay is controlled by different signalling pathways and lays out a framework for future research.\"431 state hall, detroit, mi-48202, us.\",babel\\'s tower revisited: a universal resource for cross-referencing across annotation databases,\"motivation: annotation databases are widely used as public repositories of biological knowledge. however, most of these resources have been developed by independent groups which used different designs and different identifiers for the same biological entities. as we show in this article, incoherent name spaces between various databases represent a serious impediment to using the existing annotations at their full potential. navigating between various such name spaces by mapping {ids} from one database to another is a very important issue which is not properly addressed at the {moment.results}: we have developed a web-based resource, {onto-translate} ({ot}), which effectively addresses this problem. {ot} is able to map onto each other different types of biological entities from the following annotation databases: {swiss-prot}, {trembl}, {nref}, {pir}, gene ontology, {kegg}, entrez gene, {genbank}, {genpept}, {image}, {refseq}, {unigene}, {omim}, {pdb}, eukaryotic promoter database, {hugo} gene nomenclature committee and {netaffx}. currently, {ot} is able to perform 462 types of mappings between 29 different types of {ids} from 17 databases concerning 53 organisms. among these, over 300 types of translations and 15 types of {ids} are not currently supported by any other tool or resource. on average, {ot} is able to correctly map between 96 and 99\\\\% of the biological entities provided as input. in terms of speed, sets of ∼20 000 {ids} can be translated in <30 s, in most {cases.availability}: {ot} is a part of {onto-tools}, which is freely available at {http://vortex.cs.wayne.edu/projects}.{htmlcontact}:sorin@wayne.edu\"\"motivation: a significant and stubbornly intractable problem in genome sequence analysis has been the de novo identification of transcription factor binding sites in promoter regions. although theoretically pleasing, probabilistic methods have faced difficulties due to model mismatch and the nature of the biological sequence. these problems result in inference in a high dimensional, highly multimodal space, and consequently often display only local convergence and hence unsatisfactory performance.  algorithm: in this article, we derive and demonstrate a novel method utilizing a sequential monte carlo-based expectation-maximization ({em}) optimization to improve performance in this scenario. the monte carlo element should increase the robustness of the algorithm compared to classical {em}. furthermore, the parallel nature of the sequential monte carlo algorithm should be more robust than gibbs sampling approaches to multimodality problems.  results: we demonstrate the superior performance of this algorithm on both semi-synthetic and real data from escherichia coli.  availability: http://sigproc-eng.cam.ac.uk/[\\\\~{}]ej230/smc\\\\_em\\\\_tfbsid.tar.gz  contact: ej230@cam.ac.uk  supplementary information: supplementary data are available at bioinformatics online. 10.1093/bioinformatics/btm054\"ny 11274, usa.\",identifying tissue-selective transcription factor binding sites in vertebrate promoters,\"we present a computational method aimed at systematically identifying tissue-selective transcription factor binding sites. our method focuses on the differences between sets of promoters that are associated with differentially expressed genes, and it is effective at identifying the highly degenerate motifs that characterize vertebrate transcription factor binding sites. results on simulated data indicate that our method detects motifs with greater accuracy than the leading methods, and its detection of strongly overrepresented motifs is nearly perfect. we present motifs identified by our method as the most overrepresented in promoters of liver- and muscle-selective genes, demonstrating that our method accurately identifies known transcription factor binding sites and previously uncharacterized motifs.\"berzelius v\\\\\"\"{a}g 35, s-17177 stockholm, sweden.\",{jaspar}: an open‐access database for eukaryotic transcription factor binding profiles,\"the analysis of regulatory regions in genome sequences is strongly based on the detection of potential transcription factor binding sites. the preferred models for representation of transcription factor binding specificity have been termed position‐specific scoring matrices. {jaspar} is an open‐access database of annotated, high‐quality, matrix‐based transcription factor binding site profiles for multicellular eukaryotes. the profiles were derived exclusively from sets of nucleotide sequences experimentally demonstrated to bind transcription factors. the database is complemented by a web interface for browsing, searching and subset selection, an online sequence analysis utility and a suite of programming tools for genome‐wide and comparative genomic analysis of regulatory regions. {jaspar} is available at http://jaspar. cgb.ki.se.\"notre dame, in 46556, usa.\",functional and topological characterization of protein interaction networks,\"the elucidation of the cell\\'s large-scale organization is a primary challenge for post-genomic biology, and understanding the structure of protein interaction networks offers an important starting point for such studies. we compare four available databases that approximate the protein interaction network of the yeast, saccharomyces cerevisiae, aiming to uncover the network\\'s generic large-scale properties and the impact of the proteins\\' function and cellular localization on the network topology. we show how each database supports a scale-free, topology with hierarchical modularity, indicating that these features represent a robust and generic property of the protein interactions network. we also find strong correlations between the network\\'s structure and the functional role and subcellular localization of its protein constituents, concluding that most functional and/or localization classes appear as relatively segregated subnetworks of the full protein interaction network. the uncovered systematic differences between the four protein interaction databases reflect their relative coverage for different functional and localization classes and provide a guide for their utility in various bioinformatics studies.\"\"{enologos} is a web-based tool that generates sequence logos from various input sources. sequence logos have become a popular way to graphically represent {dna} and amino acid sequence patterns from a set of aligned sequences. each position of the alignment is represented by a column of stacked symbols with its total height reflecting the information content in this position. currently, the available web servers are able to create logo images from a set of aligned sequences, but none of them generates weighted sequence logos directly from energy measurements or other sources. with the advent of high-throughput technologies for estimating the contact energy of different {dna} sequences, tools that can create logos directly from binding affinity data are useful to researchers. {enologos} generates sequence logos from a variety of input data, including energy measurements, probability matrices, alignment matrices, count matrices and aligned sequences. furthermore, {enologos} can represent the mutual information of different positions of the consensus sequence, a unique feature of this tool. another web interface for our software, {c2h2}-{enologos}, generates logos for the {dna}-binding preferences of the {c2h2} zinc-finger transcription factor family members. {enologos} and {c2h2}-{enologos} are accessible over the web at http://biodev.hgen.pitt.edu/enologos/.\"where to publish and find ontologies? a survey of ontology libraries,\"one of the key promises of the semantic web is its potential to enable and facilitate data interoperability. the ability of data providers and application developers to share and reuse ontologies is a critical component of this data interoperability: if different applications and data sources use the same set of well defined terms for describing their domain and data, it will be much easier for them to  ” talk” to one another. ontology libraries are the systems that collect ontologies from different sources and facilitate the tasks of finding, exploring, and using these ontologies. thus ontology libraries can serve as a link in enabling diverse users and applications to discover, evaluate, use, and publish ontologies. in this paper, we provide a survey of the growing—and surprisingly diverse—landscape of ontology libraries. we highlight how the varying scope and intended use of the libraries affects their features, content, and potential exploitation in applications. from reviewing 11 ontology libraries, we identify a core set of questions that ontology practitioners and users should consider in choosing an ontology library for finding ontologies or publishing their own. we also discuss the research challenges that emerge from this survey, for the developers of ontology libraries to address.\"\"the {pazar} database unites independently created and maintained data collections of transcription factor and regulatory sequence annotation. the flexible {pazar} schema permits the representation of diverse information derived from experiments ranging from biochemical {protein–dna} binding to cellular reporter gene assays. data collections can be made available to the public, or restricted to specific system users. the data \\'boutiques\\' within the shopping-mall-inspired system facilitate the analysis of genomics data and the creation of predictive models of gene regulation. since its initial release, {pazar} has grown in terms of data, features and through the addition of an associated package of software tools called the {orca} toolkit ({orcatk}). {orcatk} allows users to rapidly develop analyses based on the information stored in the {pazar} system. {pazar} is available at http://www.pazar.info. {orcatk} can be accessed through convenient buttons located in the {pazar} pages or via our website at {http://www.cisreg.ca/orcatk}.\"\"proteins are traditionally identified on the basis of their individual actions as catalysts, signalling molecules, or building blocks in cells and microorganisms. but our post-genomic view is expanding the protein\\'s role into an element in a network of protein–protein interactions as well, in which it has a contextual or cellular function within functional modules1, 2. here we provide quantitative support for this idea by demonstrating that the phenotypic consequence of a single gene deletion in the yeast saccharomyces cerevisiae is affected to a large extent by the topological position of its protein product in the complex hierarchical web of molecular interactions.\"250 longwood avenue, boston, ma 02115, usa.\",\"the synergizer service for translating gene, protein and other biological identifiers\",summary: the synergizer is a database and web service that provides translations of biological database identifiers. it is accessible both programmatically and {interactively.availability}: the synergizer is freely available to all users inter-actively via a web application (http://llama.med.harvard.edu/synergizer/translate) and programmatically via a web service. clients implementing the synergizer application programming interface ({api}) are also freely available. please visit http://llama.med.harvard.edu/synergizer/doc for {details.contact}: fritz\\\\_roth@hms.harvard.edula jolla, california 92037, usa.\",\"{xcms}: processing mass spectrometry data for metabolite profiling using nonlinear peak alignment, matching, and identification.\",\"metabolite profiling in biomarker discovery, enzyme substrate assignment, drug activity/specificity determination, and basic metabolic research requires new data preprocessing approaches to correlate specific metabolites to their biological origin. here we introduce an {lc}/{ms}-based data analysis approach, {xcms}, which incorporates novel nonlinear retention time alignment, matched filtration, peak detection, and peak matching. without using internal standards, the method dynamically identifies hundreds of endogenous metabolites for use as standards, calculating a nonlinear retention time correction profile for each sample. following retention time correction, the relative metabolite ion intensities are directly compared to identify changes in specific endogenous metabolites, such as potential biomarkers. the software is demonstrated using data sets from a previously reported enzyme knockout study and a large-scale study of plasma samples. {xcms} is freely available under an open-source license at http://metlin.scripps.edu/download/.\"\"mapping the chromosomal locations of transcription factors, nucleosomes, histone modifications, chromatin remodeling enzymes, chaperones, and polymerases is one of the key tasks of modern biology, as evidenced by the encyclopedia of {dna} elements ({encode}) project. to this end, chromatin immunoprecipitation followed by high-throughput sequencing ({chip}-seq) is the standard methodology. mapping such {protein-dna} interactions in vivo using {chip}-seq presents multiple challenges not only in sample preparation and sequencing but also for computational analysis. here, we present step-by-step guidelines for the computational analysis of {chip}-seq data. we address all the major steps in the analysis of {chip}-seq data: sequencing depth selection, quality checking, mapping, data normalization, assessment of reproducibility, peak calling, differential binding analysis, controlling the false discovery rate, peak annotation, visualization, and motif analysis. at each step in our guidelines we discuss some of the software tools most frequently used. we also highlight the challenges and problems associated with each step in {chip}-seq data analysis. we present a concise workflow for the analysis of {chip}-seq data in figure 1 that complements and expands on the recommendations of the {encode} and {modencode} projects. each step in the workflow is described in detail in the following sections.\"divergence measures based on the shannon entropy,\"a novel class of information-theoretic divergence measures based on the shannon entropy is introduced. unlike the well-known kullback divergences, the new measures do not require the condition of absolute continuity to be satisfied by the probability distributions involved. more importantly, their close relationship with the variational distance and the probability of misclassification error are established in terms of bounds. these bounds are crucial in many applications of divergence measures. the measures are also well characterized by the properties of nonnegativity, finiteness, semiboundedness, and boundedness\"liivi 2, 50409 tartu, estonia.\",{g:profiler}--a web-based toolset for functional profiling of gene lists from large-scale experiments.,\"{g:profiler} (http://biit.cs.ut.ee/gprofiler/) is a public web server for characterising and manipulating gene lists resulting from mining high-throughput genomic data. {g:profiler} has a simple, user-friendly web interface with powerful visualisation for capturing gene ontology ({go}), pathway, or transcription factor binding site enrichments down to individual gene levels. besides standard multiple testing corrections, a new improved method for estimating the true effect of multiple testing over complex structures like {go} has been introduced. interpreting ranked gene lists is supported from the same interface with very efficient algorithms. such ordered lists may arise when studying the most significantly affected genes from high-throughput data or genes co-expressed with the query gene. other important aspects of practical data analysis are supported by modules tightly integrated with {g:profiler}. these are: {g:convert} for converting between different database identifiers; {g:orth} for finding orthologous genes from other species; and {g:sorter} for searching a large body of public gene expression data for co-expression. {g:profiler} supports 31 different species, and underlying data is updated regularly from sources like the ensembl database. bioinformatics communities wishing to integrate with {g:profiler} can use alternative simple textual outputs.\"\"we present model-based analysis of {chip}-seq data, {macs}, which analyzes data generated by short read sequencers such as solexa\\'s genome analyzer. {macs} empirically models the shift size of {chip}-seq tags, and uses it to improve the spatial resolution of predicted binding sites. {macs} also uses a dynamic poisson distribution to effectively capture local biases in the genome, allowing for more robust predictions. {macs} compares favorably to existing {chip}-seq peak-finding algorithms, and is freely available.\"protein and metabolite identifier mapping services.\",\"many complementary solutions are available for the identifier mapping problem. this creates an opportunity for bioinformatics tool developers. tools can be made to flexibly support multiple mapping services or mapping services could be combined to get broader coverage. this approach requires an interface layer between tools and mapping services. here we present {bridgedb}, a software framework for gene, protein and metabolite identifier mapping. this framework provides a standardized interface layer through which bioinformatics tools can be connected to different identifier mapping services. this approach makes it easier for tool developers to support identifier mapping. mapping services can be combined or merged to support multi-omics experiments or to integrate custom microarray annotations. {bridgedb} provides its own ready-to-go mapping services, both in webservice and local database forms. however, the framework is intended for customization and adaptation to any identifier mapping service. {bridgedb} has already been integrated into several bioinformatics applications. by uncoupling bioinformatics tools from mapping services, {bridgedb} improves capability and flexibility of those tools. all described software is open source and available at http://www.bridgedb.org.\"\"motivation: probabilistic approaches for inferring transcription factor binding sites ({tfbss}) and regulatory motifs from {dna} sequences have been developed for over two decades. previous work has shown that prediction accuracy can be significantly improved by incorporating features such as the competition of multiple transcription factors ({tfs}) for binding to nearby sites, the tendency of {tfbss} for co-regulated {tfs} to cluster and form cis-regulatory modules and explicit evolutionary modeling of conservation of {tfbss} across orthologous sequences. however, currently available tools only incorporate some of these features, and significant methodological hurdles hampered their synthesis into a single consistent probabilistic framework.\"\"we have developed {netpath} as a resource of curated human signaling pathways. as an initial step, {netpath} provides detailed maps of a number of immune signaling pathways, which include approximately 1,600 reactions annotated from the literature and more than 2,800 instances of transcriptionally regulated genes - all linked to over 5,500 published articles. we anticipate {netpath} to become a consolidated resource for human signaling pathways that should enable systems biology approaches.\"\"we describe a set of best practices for scientific software development, based on research and experience, that will improve scientists\\' productivity and the reliability of their software.\"\"the {uniprot} consortium was formed in 2002 by groups from the swiss institute of bioinformatics ({sib}), the european bioinformatics institute ({ebi}) and the protein information resource ({pir}) at georgetown university, and soon afterwards the website http://www.uniprot.org was set up as a central entry point to {uniprot} resources. requests to this address were redirected to one of the three organisations\\' websites. while these sites shared a set of static pages with general information about {uniprot}, their pages for searching and viewing data were different. to provide users with a consistent view and to cut the cost of maintaining three separate sites, the consortium decided to develop a common website for {uniprot}. following several years of intense development and a year of public beta testing, the http://www.uniprot.org domain was switched to the newly developed site described in this paper in july 2008.\"\"summary: pathview is a novel tool set for pathway-based data integration and visualization. it maps and renders user data on relevant pathway graphs. users only need to supply their data and specify the target pathway. pathview automatically downloads the pathway graph data, parses the data file, maps and integrates user data onto the pathway and renders pathway graphs with the mapped data. although built as a stand-alone program, pathview may seamlessly integrate with pathway and functional analysis tools for large-scale and fully automated analysis pipelines.\"\"motivation: there is accumulating evidence that the chromatin environment of transcription factor ({tf}) binding sites in promoter regions has a critical influence on their regulatory potential. recent studies have mapped {tf} binding sites and nucleosome positions throughout the yeast genome; however, there is a lack of computation tools to integrate these data types.  results: we have developed the ceres software to facilitate the integrated analysis of {tf} binding sites and nucleosome positions in the model eukaryote s.cerevisiae. ceres enables users to dynamically display the spatial organization of {tf} binding sites and nucleosome positions of individual genes, or the average profiles for large gene sets. ceres provides novel statistical tools to test for the enrichment of {tf} binding sites and chromatin environments for user-selected gene sets. ceres also enables users to search the genome for combinations of {tf} binding sites that are associated with specific chromatin environments. preliminary analysis using the ceres software indicates that functional and conserved {tf} binding sites are often associated with specific chromatin environments.  availability: {http://bioinformatics1.smb.wsu.edu/ceres}  contact: jwyrick@wsu.edu  supplementary information: supplementary data are available at bioinformatics online. 10.1093/bioinformatics/btp657\"\"promiscuous individuals are the vulnerable nodes to target in safe-sex campaigns. unlike clearly defined \\'real-world\\' networks1, social networks tend to be subjective to some extent2, 3 because the perception of what constitutes a social link may differ between individuals. one unambiguous type of connection, however, is sexual contact, and here we analyse the sexual behaviour of a random sample of individuals4 to reveal the mathematical features of a sexual-contact network.\"new york state department of health, albany, new york 12208, usa. thompson@wadsworth.org\",decoding human regulatory circuits.,\"clusters of transcription factor binding sites ({tfbss}) which direct gene expression constitute cis-regulatory modules ({crms}). we present a novel algorithm, based on gibbs sampling, which locates, de novo, the cis features of these {crms}, their component {tfbss}, and the properties of their spatial distribution. the algorithm finds 69\\\\% of experimentally reported {tfbss} and 85\\\\% of the {crms} in a reference data set of regions upstream of genes differentially expressed in skeletal muscle cells. a discriminant procedure based on the output of the model specifically discriminated regulatory sequences in muscle-specific genes in an independent test set. application of the method to the analysis of 2710 10-kb fragments upstream of annotated human genes identified 17 novel candidate modules with a false discovery rate </=0.05, demonstrating the applicability of the method to genome-scale data.\"link communities reveal multiscale complexity in networks.,\"networks have become a key approach to understanding systems of interacting objects, unifying the study of diverse phenomena including biological organisms and human society. one crucial step when studying the structure and dynamics of networks is to identify communities: groups of related nodes that correspond to functional subunits such as protein complexes or social spheres. communities in networks often overlap such that nodes simultaneously belong to several groups. meanwhile, many networks are known to possess hierarchical organization, where communities are recursively grouped into a hierarchical structure. however, the fact that many real networks have communities with pervasive overlap, where each and every node belongs to more than one group, has the consequence that a global hierarchy of nodes cannot capture the relationships between overlapping groups. here we reinvent communities as groups of links rather than nodes and show that this unorthodox approach successfully reconciles the antagonistic organizing principles of overlapping communities and hierarchy. in contrast to the existing literature, which has entirely focused on grouping nodes, link communities naturally incorporate overlap while revealing hierarchical organization. we find relevant link communities in many networks, including major biological networks such as protein-protein interaction and metabolic networks, and show that a large social network contains hierarchically organized community structures spanning inner-city to regional scales while maintaining pervasive overlap. our results imply that link communities are fundamental building blocks that reveal overlap and hierarchical organization in networks to be two aspects of the same phenomenon.\"recent advances in genomic {dna} sequencing of microbial species from single cells,\"the vast majority of microbial species remain uncultivated and, until recently, about half of all known bacterial phyla were identified only from their {16s} ribosomal {rna} gene sequence. with the advent of single-cell sequencing, genomes of uncultivated species are rapidly filling in unsequenced branches of\"wellcome trust genome campus, hinxton, cambridge cb10 1sd, uk. brazma@ebi.ac.uk\",minimum information about a microarray experiment ({miame})—toward standards for microarray data,\"microarray analysis has become a widely used tool for the generation of gene expression data on a genomic scale. although many significant results have been derived from microarray studies, one limitation has been the lack of standards for presenting and exchanging such data. here we present a proposal, the minimum information about a microarray experiment ({miame}), that describes the minimum information required to ensure that microarray data can be easily interpreted and that results derived from its analysis can be independently verified. the ultimate goal of this work is to establish a standard for recording and reporting microarray-based gene expression data, which will in turn facilitate the establishment of databases and public repositories and enable the development of data analysis tools. with respect to {miame}, we concentrate on defining the content and structure of the necessary information rather than the technical format for capturing it.\"uji, kyoto 611-0011, japan.\",{kegg}: kyoto encyclopedia of genes and genomes,\"kyoto encyclopedia of genes and genomes ({kegg}) is a knowledge base for systematic analysis of gene functions in terms of the networks of genes and molecules. the major component of {kegg} is the {pathway} database that consists of graphical diagrams of biochemical pathways including most of the known metabolic pathways and some of the known regulatory pathways. the pathway information is also represented by the ortholog group tables summarizing orthologous and paralogous gene groups among different organisms. {kegg} maintains the {genes} database for the gene catalogs of all organisms with complete genomes and selected organisms with partial genomes, which are continuously re-annotated, as well as the {ligand} database for chemical compounds and enzymes. each gene catalog is associated with the graphical genome map for chromosomal locations that is represented by java applet. in addition to the data collection efforts, {kegg} develops and provides various computational tools, such as for reconstructing biochemical pathways from the complete genome sequence and for predicting gene regulatory networks from the gene expression profiles. the {kegg} databases are daily updated and made freely available (http://www.genome.ad.jp/kegg/).\"harvard school of public health, and channing laboratory, boston, ma, usa. jzhang@jimmy.harvard.edu\",an extensible application for assembling annotation for genomic data,\"summary: {annbuilder} is an r package for assembling genomic annotation data. the system currently provides parsers to process annotation data from {locuslink}, gene ontology consortium, and human gene project and can be extended to new data sources via user defined parsers. {annbuilder} differs from other existing systems in that it provides users with unlimited ability to assemble data from user selected sources. the products of {annbuilder} are files in {xml} format that can be easily used by different {systems.availability}: (http://www.bioconductor.org). open {source.contact}: jzhang@jimmy.harvard.edu\"{plogo}: a probabilistic approach to visualizing sequence motifs,\"here, we describe the development of {wikipathways} (http://www.wikipathways.org), a public wiki for pathway curation, since it was first published in 2008. new features are discussed, as well as developments in the community of contributors. new features include a zoomable pathway viewer, support for pathway ontology annotations, the ability to mark pathways as private for a limited time and the availability of stable hyperlinks to pathways and the elements therein. {wikipathways} content is freely available in a variety of formats such as the {biopax} standard, and the content is increasingly adopted by external databases and tools, including wikipedia. a recent development is the use of {wikipathways} as a staging ground for centrally curated databases such as reactome. {wikipathways} is seeing steady growth in the number of users, page views and edits for each pathway. to assess whether the community curation experiment can be considered successful, here we analyze the relation between use and contribution, which gives results in line with other wiki projects. the novel use of pathway pages as supplementary material to publications, as well as the addition of tailored content for research domains, is expected to stimulate growth further.\"glasgow g12 8qw, uk. raya@stats.gla.ac.uk\",how scale-free are biological networks.,\"the concept of scale-free network has emerged as a powerful unifying paradigm in the study of complex systems in biology and in physical and social studies. metabolic, protein, and gene interaction networks have been reported to exhibit scale-free behavior based on the analysis of the distribution of the number of connections of the network nodes. here we study 10 published datasets of various biological interactions and perform goodness-of-fit tests to determine whether the given data is drawn from the power-law distribution. our analysis did not identify a single interaction network that has a nonzero probability of being drawn from the power-law distribution.\"\"liquid chromatography coupled to mass spectrometry ({lc}/{ms}) is an important analytical technology for e.g. metabolomics experiments. determining the boundaries, centres and intensities of the two-dimensional signals in the {lc}/{ms} raw data is called feature detection. for the subsequent analysis of complex samples such as plant extracts, which may contain hundreds of compounds, corresponding to thousands of features – a reliable feature detection is mandatory.\"ankara 06533, turkey.\",patika: an integrated visual environment for collaborative construction and analysis of cellular pathways,\"motivation: availability of the sequences of entire genomes shifts the scientific curiosity towards the identification of function of the genomes in large scale as in genome studies. in the near future, data produced about cellular processes at molecular level will accumulate with an accelerating rate as a result of proteomics studies. in this regard, it is essential to develop tools for storing, integrating, accessing, and analyzing this data effectively.\"\"biometrics laboratory, wadsworth center for laboratories and research, new york state department of health, albany, new york 12201\",an expectation maximization ({em}) algorithm for the identification and characterization of common sites in unaligned biopolymer sequences,\"statistical methodology for the identification and characterization of protein binding sites in a set of unaligned {dna} fragments is presented. each sequence must contain at least one common site. no alignment of the sites is required. instead, the uncertainty in the location of the sites is handled by employing the missing information principle to develop an  ” expectation maximization” ({em}) algorithm. this approach allows for the simultaneous identification of the sites and characterization of the binding motifs. the reliability of the algorithm increases with the number of fragments, but the computations increase only linearly. the method is illustrated with an example, using known cyclic adenosine monophophate receptor protein ({crp}) binding sites. the final motif is utilized in a search for undiscovered {crp} binding sites.\"\"{background}:protein-protein interaction ({ppi}) databases have become a major resource for investigating biological networks and pathways in cells. a number of publicly available repositories for human {ppis} are currently available. each of these databases has their own unique features with a large variation in the type and depth of their {annotations.results}:we analyzed the major publicly available primary databases that contain literature curated {ppi} information for human proteins. this included {bind}, {dip}, {hprd}, {intact}, {mint}, {mips}, {pdzbase} and reactome databases. the number of binary non-redundant human {ppis} ranged from 101 in {pdzbase} and 346 in {mips} to 11,367 in {mint} and 36,617 in {hprd}. the number of genes annotated with at least one interactor was 9,427 in {hprd}, 4,975 in {mint}, 4,614 in {intact}, 3,887 in {bind} and <1,000 in the remaining databases. the number of literature citations for the {ppis} included in the databases was 43,634 in {hprd}, 11,480 in {mint}, 10,331 in {intact}, 8,020 in {bind} and <2,100 in the remaining {databases.conclusion}:given the importance of {ppis}, we suggest that submission of {ppis} to repositories be made mandatory by scientific journals at the time of manuscript submission as this will minimize annotation errors, promote standardization and help keep the information up to date. we hope that our analysis will help guide biomedical scientists in selecting the most appropriate database for their needs especially in light of the dramatic differences in their content.\"\"motivation: transcriptome analysis allows detection and clustering of genes that are coexpressed under various biological circumstances. under the assumption that coregulated genes share cis-acting regulatory elements, it is important to investigate the upstream sequences controlling the transcription of these genes. to improve the robustness of the gibbs sampling algorithm to noisy data sets we propose an extension of this algorithm for motif finding with a higher-order background model.\"\"an international, peer-reviewed genome sciences journal featuring outstanding original research that offers novel insights into the biology of all organisms\"department of molecular virology, immunology and medical genetics , the ohio state university, columbus, 43210, usa.\",{agris} and {atregnet}. a platform to link {cis-regulatory} elements and transcription factors into regulatory networks,\"gene regulatory pathways converge at the level of transcription, where interactions among regulatory genes and between regulators and target genes result in the establishment of spatiotemporal patterns of gene expression. the growing identification of direct target genes for key transcription factors ({tfs}) through traditional and high-throughput experimental approaches has facilitated the elucidation of regulatory networks at the genome level. to integrate this information into a web-based knowledgebase, we have developed the arabidopsis gene regulatory information server ({agris}). {agris}, which contains all arabidopsis (arabidopsis thaliana) promoter sequences, {tfs}, and their target genes and functions, provides the scientific community with a platform to establish regulatory networks. {agris} currently houses three linked databases: {atcisdb} (arabidopsis thaliana cis-regulatory database), {attfdb} (arabidopsis thaliana transcription factor database), and {atregnet} (arabidopsis thaliana regulatory network). {attfdb} contains 1,690 arabidopsis {tfs} and their sequences (protein and {dna}) grouped into 50 (october 2005) families with information on available mutants in the corresponding genes. {atcisdb} consists of 25,806 (september 2005) promoter sequences of annotated arabidopsis genes with a description of putative cis-regulatory elements. {atregnet} links, in direct interactions, several hundred genes with the {tfs} that control their expression. the current release of {atregnet} contains a total of 187 (september 2005) direct targets for 66 {tfs}. {agris} can be accessed at {http://arabidopsis}.med.ohio-state.edu.\"dept of molecular virology, immunology and medical genetics, the ohio state university, columbus, oh 43210, usa. davuluri-1@medctr.osu.edu\",\"{agris}: arabidopsis gene regulatory information server, an information resource of arabidopsis cis-regulatory elements and transcription factors\",\"{background}:the gene regulatory information is hardwired in the promoter regions formed by cis-regulatory elements that bind specific transcription factors ({tfs}). hence, establishing the architecture of plant promoters is fundamental to understanding gene expression. the determination of the regulatory circuits controlled by each {tf} and the identification of the cis-regulatory sequences for all genes have been identified as two of the goals of the multinational coordinated arabidopsis thaliana functional genomics project by the multinational arabidopsis steering committee (june {2002).results}:{agris} is an information resource of arabidopsis promoter sequences, transcription factors and their target genes. {agris} currently contains two databases, {attfdb} (arabidopsis thaliana transcription factor database) and {atcisdb} (arabidopsis thaliana cis-regulatory database). {attfdb} contains information on approximately 1,400 transcription factors identified through motif searches and grouped into 34 families. {attfdb} links the sequence of the transcription factors with available mutants and, when known, with the possible genes they may regulate. {atcisdb} consists of the 5\\' regulatory sequences of all 29,388 annotated genes with a description of the corresponding cis-regulatory elements. users can search the databases for (i) promoter sequences, (ii) a transcription factor, (iii) a direct target genes for a specific transcription factor, or (vi) a regulatory network that consists of transcription factors and their target {genes.conclusion}:{agris} provides the necessary software tools on arabidopsis transcription factors and their putative binding sites on all genes to initiate the identification of transcriptional regulatory networks in the model dicotyledoneous plant arabidopsis thaliana. {agris} can be accessed from http://arabidopsis.med.ohio-state.edu webcite.\"\"the study of the metabolite complement of biological samples, known as metabolomics, is creating large amounts of data, and support for handling these data sets is required to facilitate meaningful analyses that will answer biological questions. we present a data model for plant metabolomics known as {armet} (architecture for metabolomics). it encompasses the entire experimental time line from experiment definition and description of biological source material, through sample growth and preparation to the results of chemical analysis. such formal data descriptions, which specify the full experimental context, enable principled comparison of data sets, allow proper interpretation of experimental results, permit the repetition of experiments and provide a basis for the design of systems for data storage and transmission. the current design and example implementations are freely available (http://www.armet.org/). we seek to advance discussion and community adoption of a standard for metabolomics, which would promote principled collection, storage and transmission of experiment data.\"washington 98103, usa.\",cytoscape: a software environment for integrated models of biomolecular interaction networks,\"an international, peer-reviewed genome sciences journal featuring outstanding original research that offers novel insights into the biology of all organisms\"md 20850, usa\",{resourcerer}: a database for annotating and linking microarray resources within and across species.,\"microarray expression analysis is providing unprecedented data on gene expression in humans and mammalian model systems. although such studies provide a tremendous resource for understanding human disease states, one of the significant challenges is cross-referencing the data derived from different species, across diverse expression analysis platforms, in order to properly derive inferences regarding gene expression and disease state. to address this problem, we have developed {resourcerer}, a microarray-resource annotation and cross-reference database built using the analysis of expressed sequence tags ({ests}) and gene sequences provided by the {tigr} gene index ({tgi}) and {tigr} orthologous gene alignment ({toga}) databases [now called eukaryotic gene orthologs ({ego})].\"mn, usa\",toward principles for the design of ontologies used for knowledge sharing?,an abstract is not available.london, nw7 1aa, uk. cedric.notredame@europe.com\",t-coffee: a novel method for fast and accurate multiple sequence alignment,\"we describe a new method ({t-coffee}) for multiple sequence alignment that provides a dramatic improvement in accuracy with a modest sacrifice in speed as compared to the most commonly used alternatives. the method is broadly based on the popular progressive approach to multiple alignment but avoids the most serious pitfalls caused by the greedy nature of this algorithm. with {t-coffee} we pre-process a data set of all pair-wise alignments between the sequences. this provides us with a library of alignment information that can be used to guide the progressive alignment. intermediate alignments are then based not only on the sequences to be aligned next but also on how all of the sequences align with each other. this alignment information can be derived from heterogeneous sources such as a mixture of alignment programs and/or structure superposition. here, we illustrate the power of the approach by using a combination of local and global pair-wise alignments to generate the library. the resulting alignments are significantly more reliable, as determined by comparison with a set of 141 test cases, than any of the popular alternatives that we tried. the improvement, especially clear with the more difficult test cases, is always visible, regardless of the phylogenetic spread of the sequences in the tests.\"kanazawa university, kanazawa 920-0934, japan. titolab@kenroku.kanazawa-u.ac.jp\",a comprehensive two-hybrid analysis to explore the yeast protein interactome,\"protein–protein interactions play crucial roles in the execution of various biological functions. accordingly, their comprehensive description would contribute considerably to the functional interpretation of fully sequenced genomes, which are flooded with novel genes of unpredictable functions. we previously developed a system to examine two-hybrid interactions in all possible combinations between the ≈6,000 proteins of the budding yeast saccharomyces cerevisiae. here we have completed the comprehensive analysis using this system to identify 4,549 two-hybrid interactions among 3,278 proteins. unexpectedly, these data do not largely overlap with those obtained by the other project [uetz, p., et al. (2000) nature (london) 403, 623–627] and hence have substantially expanded our knowledge on the protein interaction space or interactome of the yeast. cumulative connection of these binary interactions generates a single huge network linking the vast majority of the proteins. bioinformatics-aided selection of biologically relevant interactions highlights various intriguing subnetworks. they include, for instance, the one that had successfully foreseen the involvement of a novel protein in spindle pole body function as well as the one that may uncover a hitherto unidentified multiprotein complex potentially participating in the process of vesicular transport. our data would thus significantly expand and improve the protein interaction map for the exploration of genome functions that eventually leads to thorough understanding of the cell as a molecular system.\"3900 reservoir road, nw, box 571414, washington, dc 20057-1414, usa. pirmail@georgetown.edu\",the protein information resource.,\"the protein information resource ({pir}) is an integrated public resource of protein informatics that supports genomic and proteomic research and scientific discovery. {pir} maintains the protein sequence database ({psd}), an annotated protein database containing over 283 000 sequences covering the entire taxonomic range. family classification is used for sensitive identification, consistent annotation, and detection of annotation errors. the superfamily curation defines signature domain architecture and categorizes memberships to improve automated classification. to increase the amount of experimental annotation, the {pir} has developed a bibliography system for literature searching, mapping, and user submission, and has conducted retrospective attribution of citations for experimental features. {pir} also maintains {nref}, a non-redundant reference database, and {iproclass}, an integrated database of protein family, function, and structure information. {pir}-{nref} provides a timely and comprehensive collection of protein sequences, currently consisting of more than 1 000 000 entries from {pir}-{psd}, {swiss}-{prot}, {trembl}, {refseq}, {genpept}, and {pdb}. the {pir} web site (http://pir.georgetown.edu) connects data analysis tools to underlying databases for information retrieval and knowledge discovery, with functionalities for interactive queries, combinations of sequence and text searches, and sorting and visual exploration of search results. the {ftp} site provides free download for {psd} and {nref} biweekly releases and auxiliary databases and files.\"ithaca, new york 14853, usa. djw24@columbia.edu\",collective dynamics of \\'small-world\\' networks,\"networks of coupled dynamical systems have been used to model biological oscillators1, 2, 3, 4, josephson junction arrays5,6, excitable media7, neural networks8, 9, 10, spatial games11, genetic control networks12 and many other self-organizing systems. ordinarily, the connection topology is assumed to be either completely regular or completely random. but many biological, technological and social networks lie somewhere between these two extremes. here we explore simple models of networks that can be tuned through this middle ground: regular networks \\'rewired\\' to introduce increasing amounts of disorder. we find that these systems can be highly clustered, like regular lattices, yet have small characteristic path lengths, like random graphs. we call them \\'small-world\\' networks, by analogy with the small-world phenomenon13,14 (popularly known as six degrees of separation15). the neural network of the worm caenorhabditis elegans, the power grid of the western united states, and the collaboration graph of film actors are shown to be small-world networks. models of dynamical systems with small-world coupling display enhanced signal-propagation speed, computational power, and synchronizability. in particular, infectious diseases spread more easily in small-world networks than in regular lattices.\"\"gene ontology ({go}) has established itself as the undisputed standard for protein function annotation. most annotations are inferred electronically, i.e. without individual curator supervision, but they are widely considered unreliable. at the same time, we crucially depend on those automated annotations, as most newly sequenced genomes are non-model organisms. here, we introduce a methodology to systematically and quantitatively evaluate electronic annotations. by exploiting changes in successive releases of the {uniprot} gene ontology annotation database, we assessed the quality of electronic annotations in terms of specificity, reliability, and coverage. overall, we not only found that electronic annotations have significantly improved in recent years, but also that their reliability now rivals that of annotations inferred by curators when they use evidence other than experiments from primary literature. this work provides the means to identify the subset of electronic annotations that can be relied upon—an important outcome given that >98\\\\% of all annotations are inferred without direct curation. in the {uniprot} gene ontology annotation database, the largest repository of functional annotations, over 98\\\\% of all function annotations are inferred in silico, without curator oversight. yet these  ” electronic {go} annotations” are generally perceived as unreliable; they are disregarded in many studies. in this article, we introduce novel methodology to systematically evaluate the quality of electronic annotations. we then provide the first comprehensive assessment of the reliability of electronic {go} annotations. overall, we found that electronic annotations are more reliable than generally believed, to an extent that they are competitive with annotations inferred by curators when they use evidence other than experiments from primary literature. but we also report significant variations among inference methods, types of annotations, and organisms. this work provides guidance for gene ontology users and lays the foundations for improving computational approaches to {go} function inference.\"\"signaling networks in eukaryotes are made up of upstream and downstream subnetworks. the upstream subnetwork contains the intertwined network of signaling pathways, while the downstream regulatory part contains transcription factors and their binding sites on the {dna} as well as {micrornas} and their {mrna} targets. currently, most signaling and regulatory databases contain only a subsection of this network, making comprehensive analyses highly time-consuming and dependent on specific data handling expertise. the need for detailed mapping of signaling systems is also supported by the fact that several drug development failures were caused by undiscovered cross-talk or regulatory effects of drug targets. we previously created a uniformly curated signaling pathway resource, {signalink}, to facilitate the analysis of pathway cross-talks. here, we present {signalink} 2, which significantly extends the coverage and applications of its predecessor. we developed a novel concept to integrate and utilize different subsections (i.e., layers) of the signaling network. the multi-layered (onion-like) database structure is made up of signaling pathways, their pathway regulators (e.g., scaffold and endocytotic proteins) and modifier enzymes (e.g., phosphatases, ubiquitin ligases), as well as transcriptional and post-transcriptional regulators of all of these components. the user-friendly website allows the interactive exploration of how each signaling protein is regulated. the customizable download page enables the analysis of any user-specified part of the signaling network. compared to other signaling resources, distinctive features of {signalink} 2 are the following: 1) it involves experimental data not only from humans but from two invertebrate model organisms, c. elegans and d. melanogaster; 2) combines manual curation with large-scale datasets; 3) provides confidence scores for each interaction; 4) operates a customizable download page with multiple file formats (e.g., {biopax}, cytoscape, {sbml}). non-profit users can access {signalink} 2 free of charge at {http://signalink}.org. with {signalink} 2 as a single resource, users can effectively analyze signaling pathways, scaffold proteins, modifier enzymes, transcription factors and {mirnas} that are important in the regulation of signaling processes. this integrated resource allows the systems-level examination of how cross-talks and signaling flow are regulated, as well as provide data for cross-species comparisons and drug discovery analyses.\"\"{biomart} central portal (www.biomart.org) offers a one-stop shop solution to access a wide array of biological databases. these include major biomolecular sequence, pathway and annotation databases such as ensembl, uniprot, reactome, {hgnc}, wormbase and {pride}; for a complete list, visit, http://www.biomart.org/biomart/martview. moreover, the web server features seamless data federation making cross querying of these data sources in a user friendly and unified way. the web server not only provides access through a web interface ({martview}), it also supports programmatic access through a perl {api} as well as {restful} and {soap} oriented web services. the website is free and open to all users and there is no login requirement.\"\"{background}:cellular processes and pathways, whose deregulation may contribute to the development of cancers, are often represented as cascades of proteins transmitting a signal from the cell surface to the nucleus. however, recent functional genomic experiments have identified thousands of interactions for the signalling canonical proteins, challenging the traditional view of pathways as independent functional entities. combining information from pathway databases and interaction networks obtained from functional genomic experiments is therefore a promising strategy to obtain more robust pathway and process representations, facilitating the study of cancer-related {pathways.results}:we present a methodology for extending pre-defined protein sets representing cellular pathways and processes by mapping them onto a protein-protein interaction network, and extending them to include densely interconnected interaction partners. the added proteins display distinctive network topological features and molecular function annotations, and can be proposed as putative new components, and/or as regulators of the communication between the different cellular processes. finally, these extended pathways and processes are used to analyse their enrichment in pancreatic mutated genes. significant associations between mutated genes and certain processes are identified, enabling an analysis of the influence of previously non-annotated cancer mutated {genes.conclusions}:the proposed method for extending cellular pathways helps to explain the functions of cancer mutated genes by exploiting the synergies of canonical knowledge and large-scale interaction data.\"\"the ensembl (http://www.ensembl.org/) database project provides a bioinformatics framework to organise biology around the sequences of large genomes. it is a comprehensive source of stable automatic annotation of the human genome sequence, with confirmed gene predictions that have been integrated with external data sources, and is available as either an interactive web site or as flat files. it is also an open source software engineering project to develop a portable system able to handle very large genomes and associated requirements from sequence analysis to data storage and visualisation. the ensembl site is one of the leading sources of human genome sequence annotation and provided much of the analysis for publication by the international human genome project of the draft genome. the ensembl system is being installed around the world in both companies and academic sites on machines ranging from supercomputers to laptops.\"new york, new york, united states of america.\",{phylogibbs}: a gibbs sampling motif finder that incorporates phylogeny,\"a central problem in the bioinformatics of gene regulation is to find the binding sites for regulatory proteins. one of the most promising approaches toward identifying these short and fuzzy sequence patterns is the comparative analysis of orthologous intergenic regions of related species. this analysis is complicated by various factors. first, one needs to take the phylogenetic relationship between the species into account in order to distinguish conservation that is due to the occurrence of functional sites from spurious conservation that is due to evolutionary proximity. second, one has to deal with the complexities of multiple alignments of orthologous intergenic regions, and one has to consider the possibility that functional sites may occur outside of conserved segments. here we present a new motif sampling algorithm, {phylogibbs}, that runs on arbitrary collections of multiple local sequence alignments of orthologous sequences. the algorithm searches over all ways in which an arbitrary number of binding sites for an arbitrary number of transcription factors ({tfs}) can be assigned to the multiple sequence alignments. these binding site configurations are scored by a bayesian probabilistic model that treats aligned sequences by a model for the evolution of binding sites and  ” background” intergenic {dna}. this model takes the phylogenetic relationship between the species in the alignment explicitly into account. the algorithm uses simulated annealing and monte carlo markov-chain sampling to rigorously assign posterior probabilities to all the binding sites that it reports. in tests on synthetic data and real data from five saccharomyces species our algorithm performs significantly better than four other motif-finding algorithms, including algorithms that also take phylogeny into account. our results also show that, in contrast to the other algorithms, {phylogibbs} can make realistic estimates of the reliability of its predictions. our tests suggest that, running on the five-species multiple alignment of a single gene\\'s upstream region, {phylogibbs} on average recovers over 50\\\\% of all binding sites in s. cerevisiae at a specificity of about 50\\\\%, and 33\\\\% of all binding sites at a specificity of about 85\\\\%. we also tested {phylogibbs} on collections of multiple alignments of intergenic regions that were recently annotated, based on {chip}-on-chip data, to contain binding sites for the same {tf}. we compared {phylogibbs}\\'s results with the previous analysis of these data using six other motif-finding algorithms. for 16 of 21 {tfs} for which all other motif-finding methods failed to find a significant motif, {phylogibbs} did recover a motif that matches the literature consensus. in 11 cases where there was disagreement in the results we compiled lists of known target genes from the literature, and found that running {phylogibbs} on their regulatory regions yielded a binding motif matching the literature consensus in all but one of the cases. interestingly, these literature gene lists had little overlap with the targets annotated based on the {chip}-on-chip data. the {phylogibbs} code can be downloaded from http://www.biozentrum.unibas.ch/\\\\~{}nimwege\\u200bn/cgi-bin/phylogibbs.cgi or http://www.imsc.res.in/\\\\~{}rsidd/phylogibbs. the full set of predicted sites from our tests on yeast are available at http://www.swissregulon.unibas.ch. computational discovery of regulatory sites in intergenic {dna} is one of the central problems in bioinformatics. up until recently motif finders would typically take one of the following two general approaches. given a known set of co-regulated genes, one searches their promoter regions for significantly overrepresented sequence motifs. alternatively, in a  ” phylogenetic footprinting” approach one searches multiple alignments of orthologous intergenic regions for short segments that are significantly more conserved than expected based on the phylogeny of the species. in this work the authors present an algorithm, {phylogibbs}, that combines these two approaches into one integrated bayesian framework. the algorithm searches over all ways in which an arbitrary number of binding sites for an arbitrary number of transcription factors can be assigned to arbitrary collections of multiple sequence alignments while taking into account the phylogenetic relations between the sequences. the authors perform a number of tests on synthetic data and real data from saccharomyces genomes in which {phylogibbs} significantly outperforms other existing methods. finally, a novel anneal-and-track strategy allows {phylogibbs} to make accurate estimates of the reliability of its predictions.\"ontologies, and gene expression data\",\"the explosion in the number of functional genomic datasets generated with tools such as {dna} microarrays has created a critical need for resources that facilitate the interpretation of large-scale biological data. {source} is a web-based database that brings together information from a broad range of resources, and provides it in manner particularly useful for genome-scale analyses. {source}\\'s {genereports} include aliases, chromosomal location, functional descriptions, {geneontology} annotations, gene expression data, and links to external databases. we curate published microarray gene expression datasets and allow users to rapidly identify sets of co-regulated genes across a variety of tissues and a large number of conditions using a simple and intuitive interface. {source} provides content both in gene and {cdna} clone-centric pages, and thus simplifies analysis of datasets generated using {cdna} microarrays. {source} is continuously updated and contains the most recent and accurate information available for human, mouse, and rat genes. by allowing dynamic linking to individual gene or clone reports, {source} facilitates browsing of large genomic datasets. finally, {sources} batch interface allows rapid extraction of data for thousands of genes or clones at once and thus facilitates statistical analyses such as assessing the enrichment of functional attributes within clusters of genes. {source} is available at http://source.stanford.edu.\"\"the purpose of this article is to provide a brief history of the development and application of computer algorithms for the analysis and prediction of {dna} binding sites. this problem can be conveniently divided into two subproblems. the first is, given a collection of known binding sites, develop a representation of those sites that can be used to search new sequences and reliably predict where additional binding sites occur. the second is, given a set of sequences known to contain binding sites for a common factor, but not knowing where the sites are, discover the location of the sites in each sequence and a representation for the specificity of the protein.\"\"motivation: we propose a new class of variable-order bayesian network ({vobn}) models for the identification of transcription factor binding sites ({tfbss}). the proposed models generalize the widely used position weight matrix ({pwm}) models, markov models and bayesian network models. in contrast to these models, where for each position a fixed subset of the remaining positions is used to model dependencies, in {vobn} models, these subsets may vary based on the specific nucleotides observed, which are called the context. this flexibility turns out to be of advantage for the classification and analysis of {tfbss}, as statistical dependencies between nucleotides in different {tfbs} positions (not necessarily adjacent) may be taken into account efficiently--in a position-specific and context-specific manner.  results: we apply the {vobn} model to a set of 238 experimentally verified sigma-70 binding sites in escherichia coli. we find that the {vobn} model can distinguish these 238 sites from a set of 472 intergenic  non-promoter\\' sequences with a higher accuracy than fixed-order markov models or bayesian trees. we use a replicated stratified-holdout experiment having a fixed true-negative rate of 99.9\\\\%. we find that for a foreground inhomogeneous {vobn} model of order 1 and a background homogeneous variable-order markov ({vom}) model of order 5, the obtained mean true-positive ({tp}) rate is 47.56\\\\%. in comparison, the best {tp} rate for the conventional models is 44.39\\\\%, obtained from a foreground {pwm} model and a background 2nd-order markov model. as the standard deviation of the estimated {tp} rate is [\\\\~{}=]0.01\\\\%, this improvement is highly significant.  availability: all datasets are available upon request from the authors. a web server for utilizing the {vobn} and {vom} models is available at http://www.eng.tau.ac.il/\\\\~{}bengal/  contact: bengal@eng.tau.ac.il 10.1093/bioinformatics/bti410\"beyond regression: new tools for prediction and analysis in the behavioral sciences,\"the use of ontologies has become a mainstream activity within bioinformatics. in a largely descriptive science such as biology, the need to have a common understanding of things described is obvious. the need to be able to apply computational methods to the large quantities of data being produced also suggests a computational requirement to standardise descriptions in biology.   as a mechanism for describing the categories of entities and their characteristics, ontologies offer many of the features that can support a descriptive science. the main use of ontologies in bioinformatics has been the delivery of controlled vocabularies. in this chapter we explore this use of ontology, but also other uses, especially those that have a deeper computational aspect. we take a broad view of ontology to include many ontology-like resources and classify the uses of ontology and ontology-like artifacts. we present a series of case studies and conclude by describing the current state and future directions for bio-ontologies.\"national research council canada, saskatoon, saskatchewan. mwilkinson@gene.pbi.nrc.ca\",{biomoby}: an open source biological web services proposal.,\"{biomoby} is an open source research project which aims to generate an architecture for the discovery and distribution of biological data through web services; data and services are decentralised, but the availability of these resources, and the instructions for interacting with them, are registered in a central location called {moby} central. {biomoby} adds to the web services paradigm, as exemplified by universal data discovery and integration ({uddi}), by having an object-driven registry query system with object and service ontologies. this allows users to traverse expansive and disparate data sets where each possible next step is presented based on the data object currently in-hand. moreover, a path from the current data object to a desired final data object could be automatically discovered using the registry. native {biomoby} objects are lightweight {xml}, and make up both the query and the response of a simple object access protocol ({soap}) transaction.\"\"the {biocyc} database collection is a set of 160 pathway/genome databases ({pgdbs}) for most eukaryotic and prokaryotic species whose genomes have been completely sequenced to date. each {pgdb} in the {biocyc} collection describes the genome and predicted metabolic network of a single organism, inferred from the {metacyc} database, which is a reference source on metabolic pathways from multiple organisms. in addition, each bacterial {pgdb} includes predicted operons for the corresponding species. the {biocyc} collection provides a unique resource for computational systems biology, namely global and comparative analyses of genomes and metabolic networks, and a supplement to the {biocyc} resource of curated {pgdbs}. the omics viewer available through the {biocyc} website allows scientists to visualize combinations of gene expression, proteomics and metabolomics data on the metabolic maps of these organisms. this paper discusses the computational methodology by which the {biocyc} collection has been expanded, and presents an aggregate analysis of the collection that includes the range of number of pathways present in these organisms, and the most frequently observed pathways. we seek scientists to adopt and curate individual {pgdbs} within the {biocyc} collection. only by harnessing the expertise of many scientists we can hope to produce biological databases, which accurately reflect the depth and breadth of knowledge that the biomedical research community is producing.\"are graph databases ready for bioinformatics?,contact: lars.juhl.jensen{at}gmail.comusa\",{wordnet}: a lexical database for english,\"because meaningful sentences are composed of meaningful words, any system that hopes to process natural languages as people do must have information about words and their meanings. this information is traditionally provided through dictionaries, and machine-readable dictionaries are now widely available. but dictionary entries evolved for the convenience of human readers, not for machines. {wordnet1} provides a more effective combination of traditional lexicographic information and modern computing. {wordnet} is an online lexical database designed for use under program control. english nouns, verbs, adjectives, and adverbs are organized into sets of synonyms, each representing a lexicalized concept. semantic relations link the synonym sets [4].\"systems and communication, university of milan-bicocca, via bicocca degli arcimboldi 8, milan, i-20126, italy. pavesi@disco.unimib.it\",an algorithm for finding signals of unknown length in {dna} sequences.,\"pattern discovery in unaligned {dna} sequences is a challenging problem in both computer science and molecular biology. several different methods and techniques have been proposed so far, but in most of the cases signals in {dna} sequences are very complicated and avoid detection. exact exhaustive methods can solve the problem only for short signals with a limited number of mutations. in this work, we extend exhaustive enumeration also to longer patterns. more in detail, the basic version of algorithm presented in this paper, given as input a set of sequences and an error ratio epsilon < 1, finds all patterns that occur in at least q sequences of the set with at most epsilonm mutations, where m is the length of the pattern. the only restriction is imposed on the location of mutations along the signal. that is, a valid occurrence of a pattern can present at most [epsiloni] mismatches in the first i nucleotides, and so on. however, we show how the algorithm can be used also when no assumption can be made on the position of mutations. in this case, it is also possible to have an estimate of the probability of finding a signal according to the signal length, the error ratio, and the input parameters. finally, we discuss some significance measures that can be used to sort the patterns output by the algorithm.\"\"valuable binding-site annotation data are stored in databases. however, several types of errors can, and do, occur in the process of manually incorporating annotation data from the scientific literature into these databases. here, we introduce {motifadjuster} {http://dig.ipk-gatersleben.de/motifadjuster}.html webcite, a tool that helps to detect these errors, and we demonstrate its efficacy on public data sets.\"an update\",\"the arabidopsis gene regulatory information server ({agris}; http://arabidopsis.med.ohio-state.edu/) provides a comprehensive resource for gene regulatory studies in the model plant arabidopsis thaliana. three interlinked databases, {attfdb}, {atcisdb} and {atregnet}, furnish comprehensive and updated information on transcription factors ({tfs}), predicted and experimentally verified cis-regulatory elements ({cres}) and their interactions, respectively. in addition to significant contributions in the identification of the entire set of {tf}–{dna} interactions, which are the key to understand the gene regulatory networks that govern arabidopsis gene expression, tools recently incorporated into {agris} include the complete set of words length 5–15 present in the arabidopsis genome and the integration of {atregnet} with visualization tools, such as the recently developed {rein} application. all the information in {agris} is publicly available and downloadable upon registration.\"oxford road, manchester m13 9pl, uk. garwood@cs.man.ac.uk\",\"{pedro}: a database for storing, searching and disseminating experimental proteomics data\",\"{background}:proteomics is rapidly evolving into a high-throughput technology, in which substantial and systematic studies are conducted on samples from a wide range of physiological, developmental, or pathological conditions. reference maps from {2d} gels are widely circulated. however, there is, as yet, no formally accepted standard representation to support the sharing of proteomics data, and little systematic dissemination of comprehensive proteomic data {sets.results}:this paper describes the design, implementation and use of a proteome experimental data repository ({pedro}), which makes comprehensive proteomics data sets available for browsing, searching and downloading. it is also serves to extend the debate on the level of detail at which proteomics data should be captured, the sorts of facilities that should be provided by proteome data management systems, and the techniques by which such facilities can be made {available.conclusions}:the {pedro} database provides access to a collection of comprehensive descriptions of experimental data sets in proteomics. not only are these data sets interesting in and of themselves, they also provide a useful early validation of the {pedro} data model, which has served as a starting point for the ongoing standardisation activity through the proteome standards initiative of the human proteome organisation.\"globally integrated and scored.\",\"an essential prerequisite for any systems-level understanding of cellular functions is to correctly uncover and annotate all functional interactions among proteins in the cell. toward this goal, remarkable progress has been made in recent years, both in terms of experimental measurements and computational prediction techniques. however, public efforts to collect and present protein interaction information have struggled to keep up with the pace of interaction discovery, partly because protein-protein interaction information can be error-prone and require considerable effort to annotate. here, we present an update on the online database resource search tool for the retrieval of interacting genes ({string}); it provides uniquely comprehensive coverage and ease of access to both experimental as well as predicted interaction information. interactions in {string} are provided with a confidence score, and accessory information such as protein domains and {3d} structures is made available, all within a stable and consistent identifier space. new features in {string} include an interactive network viewer that can cluster networks on demand, updated on-screen previews of structural information including homology models, extensive data updates and strongly improved connectivity and integration with third-party resources. version 9.0 of {string} covers more than 1100 completely sequenced organisms; the resource can be reached at http://string-db.org.\"\"we present an online server that generates a {3d} representation of properties of user-submitted {rna} or {dna} alignments. the visualized properties are information of single alignment columns, mutual information of two alignment positions as well as the position-specific fraction of gaps. the nucleotide composition of both single columns and column pairs is visualized with the help of color-coded {3d} bars labeled with letters. the server generates both {vrml} and {jvx} output that can be viewed with a {vrml} viewer or the {javaview} applet, respectively. we show that combining these different features of an alignment into one {3d} representation is helpful in identifying correlations between bases and potential {rna} and {dna} base pairs. significant known correlations between the {trna} 3\\' anticodon cardinal nucleotide and the extended anticodon were observed, as were correlations within the amino acid acceptor stem and between the cardinal nucleotide and the acceptor stem. the online server can be accessed using the {url} http://correlogo.abcc.ncifcrf.gov.\"\"a genetic interaction network containing ∼1000 genes and ∼4000 interactions was mapped by crossing mutations in 132 different query genes into a set of ∼4700 viable gene yeast deletion mutants and scoring the double mutant progeny for fitness defects. network connectivity was predictive of function because interactions often occurred among functionally related genes, and similar patterns of interactions tended to identify components of the same pathway. the genetic network exhibited dense local neighborhoods; therefore, the position of a gene on a partially mapped network is predictive of other genetic interactions. because digenic interactions are common in yeast, similar networks may underlie the complex genetics associated with inherited phenotypes in other organisms.\"friedrich-schiller-university jena, jena, germany.\",towards de novo identification of metabolites by analyzing tandem mass spectra.,\"mass spectrometry is among the most widely used technologies in proteomics and metabolomics. being a high-throughput method, it produces large amounts of data that necessitates an automated analysis of the spectra. clearly, database search methods for protein analysis can easily be adopted to analyze metabolite mass spectra. but for metabolites, de novo interpretation of spectra is even more important than for protein data, because metabolite spectra databases cover only a small fraction of naturally occurring metabolites: even the model plant arabidopsis thaliana has a large number of enzymes whose substrates and products remain unknown. the field of bio-prospection searches biologically diverse areas for metabolites which might serve as pharmaceuticals. de novo identification of metabolite mass spectra requires new concepts and methods since, unlike proteins, metabolites possess a non-linear molecular structure. in this work, we introduce a method for fully automated de novo identification of metabolites from tandem mass spectra. mass spectrometry data is usually assumed to be insufficient for identification of molecular structures, so we want to estimate the molecular formula of the unknown metabolite, a crucial step for its identification. the method first calculates all molecular formulas that explain the parent peak mass. then, a graph is build where vertices correspond to molecular formulas of all peaks in the fragmentation mass spectra, whereas edges correspond to hypothetical fragmentation steps. our algorithm afterwards calculates the maximum scoring subtree of this graph: each peak in the spectra must be scored at most once, so the subtree shall contain only one explanation per peak. unfortunately, finding this subtree is {np}-hard. we suggest three exact algorithms (including one fixed parameter tractable algorithm) as well as two heuristics to solve the problem. tests on real mass spectra show that the {fpt} algorithm and the heuristics solve the problem suitably fast and provide excellent results: for all 32 test compounds the correct solution was among the top five suggestions, for 26 compounds the first suggestion of the exact algorithm was correct. http://www.bio.inf.uni-jena.de/tandemms\"167a castetter hall, albuquerque, nm 87131-1091, usa.\",the small world inside large metabolic networks.,\"the metabolic network of the catabolic, energy and biosynthetic metabolism of escherichia coli is a paradigmatic case for the large genetic and metabolic networks that functional genomics efforts are beginning to elucidate. to analyse the structure of previously unknown networks involving hundreds or thousands of components by simple visual inspection is impossible, and quantitative approaches are needed to analyse them. we have undertaken a graph theoretical analysis of the e. coli metabolic network and find that this network is a small-world graph, a type of graph distinct from both regular and random networks and observed in a variety of seemingly unrelated areas, such as friendship networks in sociology, the structure of electrical power grids, and the nervous system of caenorhabditis elegans. moreover, the connectivity of the metabolites follows a power law, another unusual but by no means rare statistical distribution. this provides an objective criterion for the centrality of the tricarboxylic acid cycle to metabolism. the small-world architecture may serve to minimize transition times between metabolic states, and contains evidence about the evolutionary history of metabolism.\"usa.\",the value of prior knowledge in discovering motifs with {meme}.,\"{meme} is a tool for discovering motifs in sets of protein or {dna} sequences. this paper describes several extensions to {meme} which increase its ability to find motifs in a totally unsupervised fashion, but which also allow it to benefit when prior knowledge is available. when no background knowledge is asserted. {meme} obtains increased robustness from a method for determining motif widths automatically, and from probabilistic models that allow motifs to be absent in some input sequences. on the other hand, {meme} can exploit prior knowledge about a motif being present in all input sequences, about the length of a motif and whether it is a palindrome, and (using dirichlet mixtures) about expected patterns in individual motif positions. extensive experiments are reported which support the claim that {meme} benefits from, but does not require, background knowledge. the experiments use seven previously studied {dna} and protein sequence families and 75 of the protein families documented in the prosite database of sites and patterns, release 11.1.\"\"{genemania} (http://www.genemania.org) is a flexible, user-friendly web interface for generating hypotheses about gene function, analyzing gene lists and prioritizing genes for functional assays. given a query list, {genemania} extends the list with functionally similar genes that it identifies using available genomics and proteomics data. {genemania} also reports weights that indicate the predictive value of each selected data set for the query. six organisms are currently supported (arabidopsis thaliana, caenorhabditis elegans, drosophila melanogaster, mus musculus, homo sapiens and saccharomyces cerevisiae) and hundreds of data sets have been collected from {geo}, {biogrid}, pathway commons and {i2d}, as well as organism-specific functional genomics data sets. users can select arbitrary subsets of the data sets associated with an organism to perform their analyses and can upload their own data sets to analyze. the {genemania} algorithm performs as well or better than other gene function prediction methods on yeast and mouse benchmarks. the high accuracy of the {genemania} prediction algorithm, an intuitive user interface and large database make {genemania} a useful tool for any biologist.\"information, knowledge and principle: back to metabolism in {kegg}.\",\"in the hierarchy of data, information and knowledge, computational methods play a major role in the initial processing of data to extract information, but they alone become less effective to compile knowledge from information. the kyoto encyclopedia of genes and genomes ({kegg}) resource (http://www.kegg.jp/ or http://www.genome.jp/kegg/) has been developed as a reference knowledge base to assist this latter process. in particular, the {kegg} pathway maps are widely used for biological interpretation of genome sequences and other high-throughput data. the link from genomes to pathways is made through the {kegg} orthology system, a collection of manually defined ortholog groups identified by k numbers. to better automate this interpretation process the {kegg} modules defined by boolean expressions of k numbers have been expanded and improved. once genes in a genome are annotated with k numbers, the {kegg} modules can be computationally evaluated revealing metabolic capacities and other phenotypic features. the reaction modules, which represent chemical units of reactions, have been used to analyze design principles of metabolic networks and also to improve the definition of k numbers and associated annotations. for translational bioinformatics, the {kegg} {medicus} resource has been developed by integrating drug labels (package inserts) used in society.\"corrensstr, 3, 06466 gatersleben, germany. junker@ipk-gatersleben.de\",{vanted}: a system for advanced data analysis and visualization in the context of biological networks.,\"recent advances with high-throughput methods in life-science research have increased the need for automatized data analysis and visual exploration techniques. sophisticated bioinformatics tools are essential to deduct biologically meaningful interpretations from the large amount of experimental data, and help to understand biological processes. we present {vanted}, a tool for the visualization and analysis of networks with related experimental data. data from large-scale biochemical experiments is uploaded into the software via a microsoft excel-based form. then it can be mapped on a network that is either drawn with the tool itself, downloaded from the {kegg} pathway database, or imported using standard network exchange formats. transcript, enzyme, and metabolite data can be presented in the context of their underlying networks, e. g. metabolic pathways or classification hierarchies. visualization and navigation methods support the visual exploration of the data-enriched networks. statistical methods allow analysis and comparison of multiple data sets such as different developmental stages or genetically different lines. correlation networks can be automatically generated from the data and substances can be clustered according to similar behavior over time. as examples, metabolite profiling and enzyme activity data sets have been visualized in different metabolic maps, correlation networks have been generated and similar time patterns detected. some relationships between different metabolites were discovered which are in close accordance with the literature. {vanted} greatly helps researchers in the analysis and interpretation of biochemical data, and thus is a useful tool for modern biological research. {vanted} as a java web start application including a user guide and example data sets is available free of charge at http://vanted.ipk-gatersleben.de.\"\"the human genome encodes the blueprint of life, but the function of the vast majority of its nearly three billion bases is unknown. the encyclopedia of {dna} elements ({encode}) project has systematically mapped regions of transcription, transcription factor association, chromatin structure and histone modification. these data enabled us to assign biochemical functions for 80\\\\% of the genome, in particular outside of the well-studied protein-coding regions. many discovered candidate regulatory elements are physically associated with one another and with expressed genes, providing new insights into the mechanisms of gene regulation. the newly identified elements also show a statistical correspondence to sequence variants linked to human disease, and can thereby guide interpretation of this variation. overall, the project provides new insights into the organization and regulation of our genes and genome, and is an expansive resource of functional annotations for biomedical research.\"summary: arcadia translates text-based descriptions of biological networks ({sbml} files) into standardized diagrams ({sbgn} {pd} maps). users can view the same model from different perspectives and easily alter the layout to emulate traditional textbook representations.\"motivation: the complexity of cancer is prompting researchers to find new ways to synthesize information from diverse data sources and to carry out coordinated research efforts that span multiple institutions. there is a need for standard applications, common data models, and software infrastructure to enable more efficient access to and sharing of distributed computational resources in cancer research. to address this need the national cancer institute ({nci}) has initiated a national-scale effort, called the cancer biomedical informatics grid ({cabig}™), to develop a federation of interoperable research information {systems.results}: at the heart of the {cabig} approach to federated interoperability effort is a grid middleware infrastructure, called {cagrid}. in this paper we describe the {cagrid} framework and its current implementation, {cagrid} version 0.5. {cagrid} is a model-driven and service-oriented architecture that synthesizes and extends a number of technologies to provide a standardized framework for the advertising, discovery, and invocation of data and analytical resources. we expect {cagrid} to greatly facilitate the launch and ongoing management of coordinated cancer research studies involving multiple institutions, to provide the ability to manage and securely share information and analytic resources, and to spur a new generation of research applications that empower researchers to take a more integrative, trans-domain approach to data mining and {analysis.availability}: the {cagrid} version 0.5 release can be downloaded from {https://cabig.nci.nih.gov/workspaces/architecture}/{cagrid}/. the operational test bed grid can be accessed through the client included in the release, or through the {cagrid}-browser web application {http://cagrid-browser.nci.nih.gov.contact}:joel.saltz@osumc.edu\"conservation and condition-specificity of single and alternative transcription start sites in the drosophila genome\",\"{background}:transcription initiation is a key component in the regulation of gene expression. {mrna} 5\\' full-length sequencing techniques have enhanced our understanding of mammalian transcription start sites ({tsss}), revealing different initiation patterns on a genomic {scale.results}:to identify {tsss} in drosophila melanogaster, we applied a hierarchical clustering strategy on available 5\\' expressed sequence tags ({ests}) and identified a high quality set of 5,665 {tsss} for approximately 4,000 genes. we distinguished two initiation patterns: \\'peaked\\' {tsss}, and \\'broad\\' {tss} cluster groups. peaked promoters were found to contain location-specific sequence elements; conversely, broad promoters were associated with non-location-specific elements. in alignments across other drosophila genomes, conservation levels of sequence elements exceeded 90 within the melanogaster subgroup, but dropped considerably for distal species. elements in broad promoters had lower levels of conservation than those in peaked promoters. when characterizing the distributions of {ests}, 64 of {tsss} showed distinct associations to one out of eight different spatiotemporal conditions. available whole-genome tiling array time series data revealed different temporal patterns of embryonic activity across the majority of genes with distinct alternative promoters. many genes with maternally inherited transcripts were found to have alternative promoters utilized later in development. core promoters of maternally inherited transcripts showed differences in motif composition compared to zygotically active {promoters.conclusions}:our study provides a comprehensive map of drosophila {tsss} and the conditions under which they are utilized. distinct differences in motif associations with initiation pattern and spatiotemporal utilization illustrate the complex regulatory code of transcription initiation.\"\"the use of computational modeling to describe and analyze biological systems is at the heart of systems biology. model structures, simulation descriptions and numerical results can be encoded in structured formats, but there is an increasing need to provide an additional semantic layer. semantic information adds meaning to components of structured descriptions to help identify and interpret them unambiguously. ontologies are one of the tools frequently used for this purpose. we describe here three ontologies created specifically to address the needs of the systems biology community. the systems biology ontology ({sbo}) provides semantic information about the model components. the kinetic simulation algorithm ontology ({kisao}) supplies information about existing algorithms available for the simulation of systems biology models, their characterization and interrelationships. the terminology for the description of dynamics ({teddy}) categorizes dynamical features of the simulation results and general systems behavior. the provision of semantic information extends a model\\'s longevity and facilitates its reuse. it provides useful insight into the biology of modeled processes, and may be used to make informed decisions on subsequent simulation experiments.\"visualizing, and analyzing mass spectrometry-based molecular profile data\",\"mass spectrometry ({ms}) coupled with online separation methods is commonly applied for differential and quantitative profiling of biological samples in metabolomic as well as proteomic research. such approaches are used for systems biology, functional genomics, and biomarker discovery, among others. an ongoing challenge of these molecular profiling approaches, however, is the development of better data processing methods. here we introduce a new generation of a popular open-source data processing toolbox, {mzmine} 2. a key concept of the {mzmine} 2 software design is the strict separation of core functionality and data processing modules, with emphasis on easy usability and support for high-resolution spectra processing. data processing modules take advantage of embedded visualization tools, allowing for immediate previews of parameter settings. newly introduced functionality includes the identification of peaks using online databases, {msn} data support, improved isotope pattern support, scatter plot visualization, and a new method for peak list alignment based on the random sample consensus ({ransac}) algorithm. the performance of the {ransac} alignment was evaluated using synthetic datasets as well as actual experimental data, and the results were compared to those obtained using other alignment algorithms. {mzmine} 2 is freely available under a {gnu} {gpl} license and can be obtained from the project website at: http://mzmine.sourceforge.net/. the current version of {mzmine} 2 is suitable for processing large batches of data and has been applied to both targeted and non-targeted metabolomic analyses.\"\"we have previously combined statistical alignment and phylogenetic footprinting to detect conserved functional elements without assuming a fixed alignment. considering a probability-weighted distribution of alignments removes sensitivity to alignment errors, properly accommodates regions of alignment uncertainty, and increases the accuracy of functional element prediction. our method utilized standard dynamic programming hidden markov model algorithms to analyze up to four sequences.\"histone core modifications regulating nucleosome structure and dynamics,\"post-translational modifications of histones regulate all {dna}-templated processes, including replication, transcription and repair. these modifications function as platforms for the recruitment of specific effector proteins, such as transcriptional regulators or chromatin remodellers. recent data suggest that histone modifications also have a direct effect on nucleosomal\"\"motivation: protein-protein interactions ({ppis}), though extremely valuable towards a better understanding of protein functions and cellular processes, do not provide any direct information about the regions/domains within the proteins that mediate the interaction. most often, it is only a fraction of a protein that directly interacts with its biological partners. thus, understanding interaction at the domain level is a critical step towards (i) thorough understanding of {ppi} networks; (ii) precise identification of binding sites; (iii) acquisition of insights into the causes of deleterious mutations at interaction sites; and (iv) most importantly, development of drugs to inhibit pathological protein interactions. in addition, knowledge derived from known domain–domain interactions ({ddis}) can be used to understand binding interfaces, which in turn can help discover unknown {ppis}.\"\"qualitative frameworks, especially those based on the logical discrete formalism, are increasingly used to model regulatory and signalling networks. a major advantage of these frameworks is that they do not require precise quantitative data, and that they are well-suited for studies of large networks. while numerous groups have developed specific computational tools that provide original methods to analyse qualitative models, a standard format to exchange qualitative models has been missing.\"\"the minimum information required in the annotation of models registry (http://www.ebi.ac.uk/miriam) provides unique, perennial and location-independent identifiers for data used in the biomedical domain. at its core is a shared catalogue of data collections, for each of which an individual namespace is created, and extensive metadata recorded. this namespace allows the generation of uniform resource identifiers ({uris}) to uniquely identify any record in a collection. moreover, various services are provided to facilitate the creation and resolution of the identifiers. since its launch in 2005, the system has evolved in terms of the structure of the identifiers provided, the software infrastructure, the number of data collections recorded, as well as the scope of the registry itself. we describe here the new parallel identification scheme and the updated supporting software infrastructure. we also introduce the new identifiers.org service (http://identifiers.org) that is built upon the information stored in the registry and which provides directly resolvable identifiers, in the form of uniform resource locators ({urls}). the flexibility of the identification scheme and resolving system allows its use in many different fields, where unambiguous and perennial identification of data entities are necessary.\"\"most of the published quantitative models in biology are lost for the community because they are either not made available or they are insufficiently characterized to allow them to be reused. the lack of a standard description format, lack of stringent reviewing and authors\\' carelessness are the main causes for incomplete model descriptions. with today\\'s increased interest in detailed biochemical models, it is necessary to define a minimum quality standard for the encoding of those models. we propose a set of rules for curating quantitative models of biological systems. these rules define procedures for encoding and annotating models represented in machine-readable form. we believe their application will enable users to (i) have confidence that curated models are an accurate reflection of their associated reference descriptions, (ii) search collections of curated models with precision, (iii) quickly identify the biological phenomena that a given curated model or model constituent represents and (iv) facilitate model reuse and composition into large subcellular models.\"{cronos}: the cross-reference navigation server.,piscataway, nj 08854-8087, usa. berman@rcsb.rutgers.edu\",the protein data bank,\"the protein data bank ({pdb}; http://www.rcsb.org/pdb/ ) is the single worldwide archive of structural data of biological macromolecules. this paper describes the goals of the {pdb}, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.\"software for systems biology: from tools to integrated platforms,environmentally induced foregut remodeling by {pha}-{4/foxa} and {daf}-{12/nhr}.,\"growth and development of the caenorhabditis elegans foregut (pharynx) depends on coordinated gene expression, mediated by pharynx defective ({pha})-{4/foxa} in combination with additional, largely unidentified transcription factors. here, we used whole genome analysis to establish clusters of genes expressed in different pharyngeal cell types. we created an expectation maximization algorithm to identify cis-regulatory elements that activate expression within the pharyngeal gene clusters. one of these elements mediates the response to environmental conditions within pharyngeal muscles and is recognized by the nuclear hormone receptor ({nhr}) {daf}-12. our data suggest that {pha}-4 and {daf}-12 endow the pharynx with transcriptional plasticity to respond to diverse developmental and physiological cues. our combination of bioinformatics and in vivo analysis has provided a powerful means for genome-wide investigation of transcriptional control.\"\"metabolomic studies are targeted at identifying and quantifying all metabolites in a given biological context. among the tools used for metabolomic research, mass spectrometry is one of the most powerful tools. however, metabolomics by mass spectrometry always reveals a high number of unknown compounds which complicate in depth mechanistic or biochemical understanding. in principle, mass spectrometry can be utilized within strategies of de novo structure elucidation of small molecules, starting with the computation of the elemental composition of an unknown metabolite using accurate masses with errors <5 ppm (parts per million). however even with very high mass accuracy (<1 ppm) many chemically possible formulae are obtained in higher mass regions. in automatic routines an additional orthogonal filter therefore needs to be applied in order to reduce the number of potential elemental compositions. this report demonstrates the necessity of isotope abundance information by mathematical confirmation of the concept. high mass accuracy (<1 ppm) alone is not enough to exclude enough candidates with complex elemental compositions (c, h, n, s, o, p, and potentially f, cl, br and si). use of isotopic abundance patterns as a single further constraint removes >95\\\\% of false candidates. this orthogonal filter can condense several thousand candidates down to only a small number of molecular formulas. example calculations for 10, 5, 3, 1 and 0.1 ppm mass accuracy are given. corresponding software scripts can be downloaded from http://fiehnlab.ucdavis.edu. a comparison of eight chemical databases revealed that {pubchem} and the dictionary of natural products can be recommended for automatic queries using molecular formulae. more than 1.6 million molecular formulae in the range 0-500 da were generated in an exhaustive manner under strict observation of mathematical and chemical rules. assuming that ion species are fully resolved (either by chromatography or by high resolution mass spectrometry), we conclude that a mass spectrometer capable of 3 ppm mass accuracy and 2\\\\% error for isotopic abundance patterns outperforms mass spectrometers with less than 1 ppm mass accuracy or even hypothetical mass spectrometers with 0.1 ppm mass accuracy that do not include isotope information in the calculation of molecular formulae.\"\"summary: {biodbnet} is an online web resource that provides interconnected access to many types of biological databases. it has integrated many of the most commonly used biological databases and in its current state has 153 database identifiers (nodes) covering all aspects of biology including genes, proteins, pathways and other biological concepts. {biodbnet} offers various ways to work with these databases including conversions, extensive database reports, custom navigation and has various tools to enhance the quality of the results. importantly, the access to {biodbnet} is updated regularly, providing access to the most recent releases of each individual {database.availability}: {http://biodbnet.abcc.ncifcrf.govcontact}: {stephensr@mail.nih.govsupplementary} information: supplementary data are available at bioinformatics online\"\"{background}:the goal of information integration in systems biology is to combine information from a number of databases and data sets, which are obtained from both high and low throughput experiments, under one data management scheme such that the cumulative information provides greater biological insight than is possible with individual information sources considered {separately.results}:here we present {pathsys}, a graph-based system for creating a combined database of networks of interaction for generating integrated view of biological mechanisms. we used {pathsys} to integrate over 14 curated and publicly contributed data sources for the budding yeast (s. cerevisiae) and gene ontology. a number of exploratory questions were formulated as a combination of relational and graph-based queries to the integrated database. thus, {pathsys} is a general-purpose, scalable, graph-data warehouse of biological information, complete with a graph manipulation and a query language, a storage mechanism and a generic data-importing mechanism through {schema-mapping.conclusion}:results from several test studies demonstrate the effectiveness of the approach in retrieving biologically interesting relations between genes and proteins, the networks connecting them, and of the utility of {pathsys} as a scalable graph-based warehouse for interaction-network integration and a hypothesis generator system. the {pathsys}\\'s client software, named {biologicalnetworks}, developed for navigation and analyses of molecular networks, is available as a java web start application at {http://brak.sdsc.edu/pub/biologicalnetworks} webcite.\"wa, usa.\",{bionetbuilder}: automatic integration of biological networks,\"{bionetbuilder} is an open-source client-server cytoscape plugin that offers a user-friendly interface to create biological networks integrated from several databases. users can create networks for ∼1500 organisms, including common model organisms and human. currently supported databases include: {dip}, {bind}, prolinks, {kegg}, {hprd}, the {biogrid} and {go}, among others. the {bionetbuilder} plugin client is available as a java webstart, providing a platform-independent network interface to these public {databases.availability}:{http://err.bio.nyu.edu/cytoscape/bionetbuilder/contact}:iliana\\\\_avila-campillo@merck.com\"\"{jaspar} (http://jaspar.genereg.net) is the largest open-access database of matrix-based nucleotide profiles describing the binding preference of transcription factors from multiple species. the fifth major release greatly expands the heart of {jaspar}-the {jaspar} {core} subcollection, which contains curated, non-redundant profiles-with 135 new curated profiles (74 in vertebrates, 8 in drosophila melanogaster, 10 in caenorhabditis elegans and 43 in arabidopsis thaliana; a 30\\\\% increase in total) and 43 older updated profiles (36 in vertebrates, 3 in d. melanogaster and 4 in a. thaliana; a 9\\\\% update in total). the new and updated profiles are mainly derived from published chromatin immunoprecipitation-seq experimental datasets. in addition, the web interface has been enhanced with advanced capabilities in browsing, searching and subsetting. finally, the new {jaspar} release is accompanied by a new {biopython} package, a new r tool package and a new {r/bioconductor} data package to facilitate access for both manual and automated methods.\"\"the analysis and usage of biological data is hindered by the spread of information across multiple repositories and the difficulties posed by different nomenclature systems and storage formats. in particular, there is an important need for data unification in the study and use of protein-protein interactions. without good integration strategies, it is difficult to analyze the whole set of available data and its properties. we introduce {biana} (biologic interactions and network analysis), a tool for biological information integration and network management. {biana} is a python framework designed to achieve two major goals: i) the integration of multiple sources of biological information, including biological entities and their relationships, and ii) the management of biological information as a network where entities are nodes and relationships are edges. moreover, {biana} uses properties of proteins and genes to infer latent biomolecular relationships by transferring edges to entities sharing similar properties. {biana} is also provided as a plugin for cytoscape, which allows users to visualize and interactively manage the data. a web interface to {biana} providing basic functionalities is also available. the software can be downloaded under {gnu} {gpl} license from {http://sbi.imim.es/web/biana}.php. {biana}\\'s approach to data unification solves many of the nomenclature issues common to systems dealing with biological data. {biana} can easily be extended to handle new specific data repositories and new specific data types. the unification protocol allows {biana} to be a flexible tool suitable for different user requirements: non-expert users can use a suggested unification protocol while expert users can define their own specific unification rules.\"santa cruz, ca 95064, usa. donnak@soe.ucsc.edu\",the {ucsc} genome browser database: 2008 update,\"the university of california, santa cruz, genome browser database ({gbd}) provides integrated sequence and annotation data for a large collection of vertebrate and model organism genomes. seventeen new assemblies have been added to the database in the past year, for a total coverage of 19 vertebrate and 21 invertebrate species as of september 2007. for each assembly, the {gbd} contains a collection of annotation data aligned to the genomic sequence. highlights of this year\\'s additions include a 28-species human-based vertebrate conservation annotation, an enhanced {ucsc} genes set, and more human variation, {mgc}, and {encode} data. the database is optimized for fast interactive performance with a set of web-based tools that may be used to view, manipulate, filter and download the annotation data. new toolset features include the genome graphs tool for displaying genome-wide data sets, session saving and sharing, better custom track management, expanded genome browser configuration options and a genome browser wiki site. the downloadable {gbd} data, the companion genome browser toolset and links to documentation and related information can be found at: http://genome.ucsc.edu/.\"\"{background}:a significant problem in the study of mechanisms of an organism\\'s development is the elucidation of interrelated factors which are making an impact on the different levels of the organism, such as genes, biological molecules, cells, and cell systems. numerous sources of heterogeneous data which exist for these subsystems are still not integrated sufficiently enough to give researchers a straightforward opportunity to analyze them together in the same frame of study. systematic application of data integration methods is also hampered by a multitude of such factors as the orthogonal nature of the integrated data and naming {problems.results}:here we report on a new version of {biologicalnetworks}, a research environment for the integral visualization and analysis of heterogeneous biological data. {biologicalnetworks} can be queried for properties of thousands of different types of biological entities (genes/proteins, promoters, {cogs}, pathways, binding sites, and other) and their relations (interactions, co-expression, co-citations, and other). the system includes the build-pathways infrastructure for molecular interactions/relations and module discovery in high-throughput experiments. also implemented in {biologicalnetworks} are the integrated genome viewer and comparative genomics browser applications, which allow for the search and analysis of gene regulatory regions and their conservation in multiple species in conjunction with molecular pathways/networks, experimental data and functional {annotations.conclusions}:the new release of {biologicalnetworks} together with its back-end database introduces extensive functionality for a more efficient integrated multi-level analysis of microarray, sequence, regulatory, and other data. {biologicalnetworks} is freely available at http://www.biologicalnetworks.org webcite.\"hinxton, cambridge cb10 1sd, uk. apweiler@ebi.ac.uk\",{uniprot}: the universal protein knowledgebase,\"to provide the scientific community with a single, centralized, authoritative resource for protein sequences and functional information, the {swiss‐prot}, {trembl} and {pir} protein database activities have united to form the universal protein knowledgebase ({uniprot}) consortium. our mission is to provide a comprehensive, fully classified, richly and accurately annotated protein sequence knowledgebase, with extensive cross‐references and query interfaces. the central database will have two sections, corresponding to the familiar {swiss‐prot} (fully manually curated entries) and {trembl} (enriched with automated classification, annotation and extensive cross‐references). for convenient sequence searches, {uniprot} also provides several non‐redundant sequence databases. the {uniprot} {nref} ({uniref}) databases provide representative subsets of the knowledgebase suitable for efficient searching. the comprehensive {uniprot} archive ({uniparc}) is updated daily from many public source databases. the {uniprot} databases can be accessed online (http://www.uniprot.org) or downloaded in several formats (ftp://ftp.uniprot.org/pub). the scientific community is encouraged to submit data for inclusion in {uniprot}.\"\"we have used a \"\"perceptron\"\" algorithm to find a weighting function which distinguishes e. coli translational initiation sites from all other sites in a library of over 78,000 nucleotides of {mrna} sequence. the \"\"perceptron\"\" examined sequences as linear representations. the \"\"perceptron\"\" is more successful at finding gene beginnings than our previous searches using \"\"rules\"\" (see previous paper). we note that the weighting function can find translational initiation sites within sequences that were not included in the training set.\"boston, ma 02215, usa. amaral@buphy.bu.edu\",classes of small-world networks,\"we study the statistical properties of a variety of diverse real-world networks. we present evidence of the occurrence of three classes of small-world networks: (a) scale-free networks, characterized by a vertex connectivity distribution that decays as a power law; (b) broad-scale networks, characterized by a connectivity distribution that has a power law regime followed by a sharp cutoff; and (c) single-scale networks, characterized by a connectivity distribution with a fast decaying tail. moreover, we note for the classes of broad-scale and single-scale networks that there are constraints limiting the addition of new links. our results suggest that the nature of such constraints may be the controlling factor for the emergence of different classes of networks.\"90095-1570, usa.\",the database of interacting proteins: 2004 update.,\"the database of interacting proteins (http://dip.doe-mbi.ucla.edu) aims to integrate the diverse body of experimental evidence on protein-protein interactions into a single, easily accessible online database. because the reliability of experimental evidence varies widely, methods of quality assessment have been developed and utilized to identify the most reliable subset of the interactions. this {core} set can be used as a reference when evaluating the reliability of high-throughput protein-protein interaction data sets, for development of prediction methods, as well as in the studies of the properties of protein interaction networks.\"\"motivation: spatio-temporal regulation of gene expression is an indispensable characteristic in the development processes of all animals. \\'master switches\\', a central set of regulatory genes whose states (on/off or activated/deactivated) determine specific developmental fate or cell-fate specification, play a pivotal role for whole developmental processes. in this study on genome-wide integrative network analysis the underlying design principles of developmental gene regulatory networks are examined.\"\"{weblogo} generates sequence logos, graphical representations of the patterns within a multiple sequence alignment. sequence logos provide a richer and more precise description of sequence similarity than consensus sequences and can rapidly reveal significant features of the alignment otherwise difficult to perceive. each logo consists of stacks of letters, one stack for each position in the sequence. the overall height of each stack indicates the sequence conservation at that position (measured in bits), whereas the height of symbols within the stack reflects the relative frequency of the corresponding amino or nucleic acid at that position. {weblogo} has been enhanced recently with additional features and options, to provide a convenient and highly configurable sequence logo generator. a command line interface and the complete, open {weblogo} source code are available for local installation and customization. copyright 2004 cold spring harbor laboratory press\"\"ontologies have proven very useful for capturing knowledge as a hierarchy of terms and their interrelationships. in biology a major challenge has been to construct ontologies of gene function given incomplete biological knowledge and inconsistencies in how this knowledge is manually curated. here we show that large networks of gene and protein interactions in saccharomyces cerevisiae can be used to infer an ontology whose coverage and power are equivalent to those of the manually curated gene ontology ({go}). the network-extracted ontology ({nexo}) contains 4,123 biological terms and 5,766 term-term relations, capturing 58\\\\% of known cellular components. we also explore robust {nexo} terms and term relations that were initially not cataloged in {go}, a number of which have now been added based on our analysis. using quantitative genetic interaction profiling and chemogenomics, we find further support for many of the uncharacterized terms identified by {nexo}, including multisubunit structures related to protein trafficking or mitochondrial function. this work enables a shift from using ontologies to evaluate data to using data to construct and evaluate ontologies.\"csic/usal), 37007 salamanca, spain.\",{apid}: agile protein interaction {dataanalyzer}.,\"agile protein interaction {dataanalyzer} ({apid}) is an interactive bioinformatics web tool developed to integrate and analyze in a unified and comparative platform main currently known information about protein-protein interactions demonstrated by specific small-scale or large-scale experimental methods. at present, the application includes information coming from five main source databases enclosing an unified sever to explore >35 000 different proteins and 111 000 different proven interactions. the web includes search tools to query and browse upon the data, allowing selection of the interaction pairs based in calculated parameters that weight and qualify the reliability of each given protein interaction. such parameters are for the \\'proteins\\': connectivity, cluster coefficient, gene ontology ({go}) functional environment, {go} environment enrichment; and for the \\'interactions\\': number of methods, {go} overlapping, {ipfam} domain-domain interaction. {apid} also includes a graphic interactive tool to visualize selected sub-networks and to navigate on them or along the whole interaction network. the application is available open access at http://bioinfow.dep.usal.es/apid/.\"445 henry mall, madison, wi 53706, usa. ecoli@genetics.wisc.edu\",the complete genome sequence of escherichia coli k-12,\"the 4,639,221–base pair sequence of escherichia {colik}-12 is presented. of 4288 protein-coding genes annotated, 38 percent have no attributed function. comparison with five other sequenced microbes reveals ubiquitous as well as narrowly distributed gene families; many families of similar genes within e. coli are also evident. the largest family of paralogous proteins contains 80 {abc} transporters. the genome as a whole is strikingly organized with respect to the local direction of replication; guanines, oligonucleotides possibly related to replication and recombination, and most genes are so oriented. the genome also contains insertion sequence ({is}) elements, phage remnants, and many other patches of unusual composition indicating genome plasticity through horizontal transfer.\"large sequence-based datasets are often scanned for conserved sequence patterns to extract useful biological information1. sequence logos2 have been developed to visualize conserved patterns in oligonucleotide and protein sequences and rely on shannon\\'s information theory to calculate conservation among all positions in a multiple-sequence alignment.\"{background}:motif discovery aims to detect short, highly conserved patterns in a collection of unaligned {dna} or protein sequences. discriminative motif finding algorithms aim to increase the sensitivity and selectivity of motif discovery by utilizing a second set of sequences, and searching only for patterns that can differentiate the two sets of sequences. potential applications of discriminative motif discovery include discovering transcription factor binding site motifs in {chip}-chip data and finding protein motifs involved in thermal stability using sets of orthologous proteins from thermophilic and mesophilic {organisms.results}:we describe {deme}, a discriminative motif discovery algorithm for use with protein and {dna} sequences. input to {deme} is two sets of sequences; a \"\"positive\"\" set and a \"\"negative\"\" set. {deme} represents motifs using a probabilistic model, and uses a novel combination of global and local search to find the motif that optimally discriminates between the two sets of sequences. {deme} is unique among discriminative motif finders in that it uses an informative bayesian prior on protein motif columns, allowing it to incorporate prior knowledge of residue characteristics. we also introduce four, synthetic, discriminative motif discovery problems that are designed for evaluating discriminative motif finders in various biologically motivated contexts. we test {deme} using these synthetic problems and on two biological problems: finding yeast transcription factor binding motifs in {chip}-chip data, and finding motifs that discriminate between groups of thermophilic and mesophilic orthologous {proteins.conclusion}:using artificial data, we show that {deme} is more effective than a non-discriminative approach when there are \"\"decoy\"\" motifs or when a variant of the motif is present in the \"\"negative\"\" sequences. with real data, we show that {deme} is as good, but not better than non-discriminative algorithms at discovering yeast transcription factor binding motifs. we also show that {deme} can find highly informative thermal-stability protein motifs. binaries for the stand-alone program {deme} is free for academic use and is available at http://bioinformatics.org.au/deme/ webcite\"albany, ny 12201.\",a phylogenetic gibbs sampler that yields centroid solutions for cis-regulatory site prediction,\"motivation: identification of functionally conserved regulatory elements in sequence data from closely related organisms is becoming feasible, due to the rapid growth of public sequence databases. closely related organisms are most likely to have common regulatory motifs; however, the recent speciation of such organisms results in the high degree of correlation in their genome sequences, confounding the detection of functional elements. additionally, alignment algorithms that use optimization techniques are limited to the detection of a single alignment that may not be representative. comparative-genomics studies must be able to address the phylogenetic correlation in the data and efficiently explore the alignment space, in order to make specific and biologically relevant {predictions.results}: we describe here a gibbs sampler that employs a full phylogenetic model and reports an ensemble centroid solution. we describe regulatory motif detection using both simulated and real data, and demonstrate that this approach achieves improved specificity, sensitivity, and positive predictive value over non-phylogenetic algorithms, and over phylogenetic algorithms that report a maximum likelihood {solution.availability}: the software is freely available at {http://bayesweb.wadsworth.org/gibbs/gibbs.htmlcontact}: {william\\\\_thompson\\\\_1}@{brown.edusupplementary} information: supplementary data are available at bioinformatics online.\"university of pittsburgh and department of human genetics, graduate school of public health, and university of pittsburgh cancer institute, school of medicine, university of pittsburgh, pittsburgh, pa, usa.\",{stamp}: a web tool for exploring {dna}-binding motif similarities.,\"{stamp} is a newly developed web server that is designed to support the study of {dna}-binding motifs. {stamp} may be used to query motifs against databases of known motifs; the software aligns input motifs against the chosen database (or alternatively against a user-provided dataset), and lists of the highest-scoring matches are returned. such similarity-search functionality is expected to facilitate the identification of transcription factors that potentially interact with newly discovered motifs. {stamp} also automatically builds multiple alignments, familial binding profiles and similarity trees when more than one motif is inputted. these functions are expected to enable evolutionary studies on sets of related motifs and fixed-order regulatory modules, as well as illustrating similarities and redundancies within the input motif collection. {stamp} is a highly flexible alignment platform, allowing users to \\'mix-and-match\\' between various implemented comparison metrics, alignment methods (local or global, gapped or ungapped), multiple alignment strategies and tree-building methods. motifs may be inputted as frequency matrices (in many of the commonly used formats), consensus sequences, or alignments of known binding sites. {stamp} also directly accepts the output files from 12 supported motif-finders, enabling quick interpretation of motif-discovery analyses. {stamp} is available at http://www.benoslab.pitt.edu/stamp.\"a lightweight cross-platform tool for controlled vocabulary queries\",\"{background}:with the vast amounts of biomedical data being generated by high-throughput analysis methods, controlled vocabularies and ontologies are becoming increasingly important to annotate units of information for ease of search and retrieval. each scientific community tends to create its own locally available ontology. the interfaces to query these ontologies tend to vary from group to group. we saw the need for a centralized location to perform controlled vocabulary queries that would offer both a lightweight web-accessible user interface as well as a consistent, unified {soap} interface for automated {queries.results}:the ontology lookup service ({ols}) was created to integrate publicly available biomedical ontologies into a single database. all modified ontologies are updated daily. a list of currently loaded ontologies is available online. the database can be queried to obtain information on a single term or to browse a complete ontology using {ajax}. auto-completion provides a user-friendly search mechanism. an {ajax}-based ontology viewer is available to browse a complete ontology or subsets of it. a programmatic interface is available to query the webservice using {soap}. the service is described by a {wsdl} descriptor file available online. a sample java client to connect to the webservice using {soap} is available for download from {sourceforge}. all {ols} source code is publicly available under the open source apache {licence.conclusion}:the {ols} provides a user-friendly single entry point for publicly available ontologies in the open biomedical ontology ({obo}) format. it can be accessed interactively or programmatically at http://www.ebi.ac.uk/ontology-lookup/.\"\"recent progress in massively parallel sequencing platforms has enabled genome-wide characterization of {dna}-associated proteins using the combination of chromatin immunoprecipitation and sequencing ({chip}-seq). although a variety of methods exist for analysis of the established alternative {chip} microarray ({chip}-chip), few approaches have been described for processing {chip}-seq data. to fill this gap, we propose an analysis pipeline specifically designed to detect protein-binding positions with high accuracy. using previously reported data sets for three transcription factors, we illustrate methods for improving tag alignment and correcting for background signals. we compare the sensitivity and spatial precision of three peak detection algorithms with published methods, demonstrating gains in spatial precision when an asymmetric distribution of tags on positive and negative strands is considered. we also analyze the relationship between the depth of sequencing and characteristics of the detected binding positions, and provide a method for estimating the sequencing depth necessary for a desired coverage of protein binding sites.\"graph-based analysis and visualization of experimental results with {ondex}.,\"assembling the relevant information needed to interpret the output from high-throughput, genome scale, experiments such as gene expression microarrays is challenging. analysis reveals genes that show statistically significant changes in expression levels, but more information is needed to determine their biological relevance. the challenge is to bring these genes together with biological information distributed across hundreds of databases or buried in the scientific literature (millions of articles). software tools are needed to automate this task which at present is labor-intensive and requires considerable informatics and biological expertise. this article describes {ondex} and how it can be applied to the task of interpreting gene expression results. {ondex} is a database system that combines the features of semantic database integration and text mining with methods for graph-based analysis. an overview of the {ondex} system is presented, concentrating on recently developed features for graph-based analysis and visualization. a case study is used to show how {ondex} can help to identify causal relationships between stress response genes and metabolic pathways from gene expression data. {ondex} also discovered functional annotations for most of the genes that emerged as significant in the microarray experiment, but were previously of unknown function.\"\"chromatin immunoprecipitation experiments followed by sequencing ({chip}–seq) detect {protein–dna} binding events and chemical modifications of histone proteins. challenges in the standard {chip}–seq protocol have motivated recent enhancements in this approach, such as reducing the number of cells that are required and increasing the resolution. complementary experimental approaches — for example, {dnasei} hypersensitive site mapping and analysis of chromatin interactions that are mediated by particular proteins — provide additional information about {dna}-binding proteins and their function. these data are now being used to identify variability in the functions of {dna}-binding proteins across genomes and individuals. in this review, i describe the latest advances in methods to detect and functionally characterize {dna}-bound proteins.\"ihnestr. 73, 14195 berlin, germany.\",predicting transcription factor affinities to {dna} from a biophysical model.,\"theoretical efforts to understand the regulation of gene expression are traditionally centered around the identification of transcription factor binding sites at specific {dna} positions. more recently these efforts have been supplemented by experimental data for relative binding affinities of proteins to longer intergenic sequences. the question arises to what extent these two approaches converge. in this paper, we adopt a physical binding model to predict the relative binding affinity of a transcription factor for a given sequence. we find that a significant fraction of genome-wide binding data in yeast can be accounted for by simple count matrices and a physical model with only two parameters. we demonstrate that our approach is both conceptually and practically more powerful than traditional methods, which require selection of a cutoff. our analysis yields biologically meaningful parameters, suitable for predicting relative binding affinities in the absence of experimental binding data. the c source code for our {trap} program is freely available for non-commercial use at {http://www.molgen.mpg.de/\\\\~{}manke/papers/tfaffinities}/\"five-vertebrate {chip}-seq reveals the evolutionary dynamics of transcription factor binding.,\"transcription factors ({tfs}) direct gene expression by binding to {dna} regulatory regions. to explore the evolution of gene regulation, we used chromatin immunoprecipitation with high-throughput sequencing ({chip}-seq) to determine experimentally the genome-wide occupancy of two {tfs}, {ccaat}/enhancer-binding protein alpha and hepatocyte nuclear factor 4 alpha, in the livers of five vertebrates. although each {tf} displays highly conserved {dna} binding preferences, most binding is species-specific, and aligned binding events present in all five species are rare. regions near genes with expression levels that are dependent on a {tf} are often bound by the {tf} in multiple species yet show no enhanced {dna} sequence constraint. binding divergence between species can be largely explained by sequence changes to the bound motifs. among the binding events lost in one lineage, only half are recovered by another binding event within 10 kilobases. our results reveal large interspecies differences in transcriptional regulation and provide insight into regulatory evolution.\"'),\n",
       " ('beaca02b21b7cad6cb738c5e2682af8d',\n",
       "  'new haven, connecticut 06520, usa.\",annotation transfer between genomes: protein-protein interologs and {protein-dna} regulogs.,\"bioinformatics program, boston university, boston, massachusetts; department of biomedical engineering, boston university, boston, massachusetts\",{zdock}: an initial-stage protein-docking algorithm,\"the development of scoring functions is of great importance to protein docking. here we present a new scoring function for the initial stage of unbound docking. it combines our recently developed pairwise shape complementarity with desolvation and electrostatics. we compare this scoring function with three other functions on a large benchmark of 49 nonredundant test cases and show its superior performance, especially for the antibody-antigen category of test cases. for 44 test cases (90\\\\% of the benchmark), we can retain at least one near-native structure within the top 2000 predictions at the 6° rotational sampling density, with an average of 52 near-native structures per test case. the remaining five difficult test cases can be explained by a combination of poor binding affinity, large backbone conformational changes, and our algorithm\\'s strong tendency for identifying large concave binding pockets. all four scoring functions have been integrated into our fast fourier transform based docking algorithm {zdock}, which is freely available to academic users at http://zlab.bu.edu/∼rong/dock. proteins 2003;52:80–87. {\\\\copyright} 2003 {wiley-liss}, inc.\"\"human protein reference database ({hprd}--http://www.hprd.org/), initially described in 2003, is a database of curated proteomic information pertaining to human proteins. we have recently added a number of new features in {hprd}. these include {phosphomotif} finder, which allows users to find the presence of over 320 experimentally verified phosphorylation motifs in proteins of interest. another new feature is a protein distributed annotation {system--human} proteinpedia (http://www.humanproteinpedia.org/)--through which laboratories can submit their data, which is mapped onto protein entries in {hprd}. over 75 laboratories involved in proteomics research have already participated in this effort by submitting data for over 15,000 human proteins. the submitted data includes mass spectrometry and protein microarray-derived data, among other data types. finally, {hprd} is also linked to a compendium of human signaling pathways developed by our group, {netpath} (http://www.netpath.org/), which currently contains annotations for several cancer and immune signaling pathways. since the last update, more than 5500 new protein sequences have been added, making {hprd} a comprehensive resource for studying the human proteome.\"\"{background}:microarrays revolutionized biological research by enabling gene expression comparisons on a transcriptome-wide scale. microarrays, however, do not estimate absolute expression level accurately. at present, high throughput sequencing is emerging as an alternative methodology for transcriptome studies. although free of many limitations imposed by microarray design, its potential to estimate absolute transcript levels is {unknown.results}:in this study, we evaluate relative accuracy of microarrays and transcriptome sequencing ({rna}-seq) using third methodology: proteomics. we find that {rna}-seq provides a better estimate of absolute expression {levels.conclusion}:our result shows that in terms of overall technical performance, {rna}-seq is the technique of choice for studies that require accurate estimation of absolute transcript levels.\"\"cancers evolve by a reiterative process of clonal expansion, genetic diversification and clonal selection within the adaptive landscapes of tissue ecosystems. the dynamics are complex, with highly variable patterns of genetic diversity and resulting clonal architecture. therapeutic intervention may destroy cancer clones and erode their habitats, but it can also inadvertently provide a potent selective pressure for the expansion of resistant variants. the inherently darwinian character of cancer is the primary reason for this therapeutic failure, but it may also hold the key to more effective control.\"\"intratumoral heterogeneity arises through evolution of genetically diverse subclones during tumor progression. however, whether cells within single genetic clones are functionally equivalent remains unknown. by combining {dna} copy number alteration ({cna}) profiling, sequencing, and lentiviral lineage tracking, we followed the repopulation dynamics of 150 single lentivirus-marked lineages from 10 human colorectal cancers through serial xenograft passages in mice. {cna} and mutational analysis distinguished individual clones and showed that clones remained stable on serial transplantation. despite this stability, the proliferation, persistence, and chemotherapy tolerance of lentivirally marked lineages were variable within each clone. chemotherapy promoted dominance of previously minor or dormant lineages. thus, apart from genetic diversity, tumor cells display inherent functional variability in tumor propagation potential, a mechanism that contributes both to cancer growth and therapy tolerance.\"\"recent advances in functional genomics have helped generate large-scale high-throughput protein interaction data. such networks, though extremely valuable towards molecular level understanding of cells, do not provide any direct information about the regions (domains) in the proteins that mediate the interaction. here, we performed co-evolutionary analysis of domains in interacting proteins in order to understand the degree of co-evolution of interacting and non-interacting domains. using a combination of sequence and structural analysis, we analyzed protein–protein interactions in {f1-atpase}, {sec23p/sec24p}, {dna}-directed {rna} polymerase and nuclear pore complexes, and found that interacting domain pair(s) for a given interaction exhibits higher level of co-evolution than the non-interacting domain pairs. motivated by this finding, we developed a computational method to test the generality of the observed trend, and to predict large-scale domain–domain interactions. given a protein–protein interaction, the proposed method predicts the domain pair(s) that is most likely to mediate the protein interaction. we applied this method on the yeast interactome to predict domain–domain interactions, and used known domain–domain interactions found in {pdb} crystal structures to validate our predictions. our results show that the prediction accuracy of the proposed method is statistically significant. comparison of our prediction results with those from two other methods reveals that only a fraction of predictions are shared by all the three methods, indicating that the proposed method can detect known interactions missed by other methods. we believe that the proposed method can be used with other methods to help identify previously unrecognized domain–domain interactions on a genome scale, and could potentially help reduce the search space for identifying interaction sites.\"boston, ma 02215.\",\"structure, function, and evolution of transient and obligate protein–protein interactions\",\"recent analyses of high-throughput protein interaction data coupled with large-scale investigations of evolutionary properties of interaction networks have left some unanswered questions. to what extent do protein interactions act as constraints during evolution of the protein sequence? how does the type of interaction, specifically transient or obligate, play into these constraints? are the mutations in the binding site of an interacting protein correlated with mutations in the binding site of its partner? we address these and other questions by relying on a carefully curated dataset of protein complex structures. results point to the importance of distinguishing between transient and obligate interactions. we conclude that residues in the interfaces of obligate complexes tend to evolve at a relatively slower rate, allowing them to coevolve with their interacting partners. in contrast, the plasticity inherent in transient interactions leads to an increased rate of substitution for the interface residues and leaves little or no evidence of correlated mutations across the interface.\"\"motivation: structural genomics projects are beginning to produce protein structures with unknown function, therefore, accurate, automated predictors of protein function are required if all these structures are to be properly annotated in reasonable time. identifying the interface between two interacting proteins provides important clues to the function of a protein and can reduce the search space required by docking algorithms to predict the structures of complexes.\"\"biomolecular structure and modelling unit, department of biochemistry and molecular biology, university college london, london, united kingdom\",scoring residue conservation,\"the importance of a residue for maintaining the structure and function of a protein can usually be inferred from how conserved it appears in a multiple sequence alignment of that protein and its homologues. a reliable metric for quantifying residue conservation is desirable. over the last two decades many such scores have been proposed, but none has emerged as a generally accepted standard. this work surveys the range of scores that biologists, biochemists, and, more recently, bioinformatics workers have developed, and reviews the intrinsic problems associated with developing and evaluating such a score. a general formula is proposed that may be used to compare the properties of different particular conservation scores or as a measure of conservation in its own right. proteins 2002;48:227–241. {\\\\copyright} 2002 {wiley-liss}, inc.\"state university of new york at buffalo, 124 sherman hall, buffalo, ny 14214, usa.\",protein binding site prediction using an empirical scoring function,\"most biological processes are mediated by interactions between proteins and their interacting partners including proteins, nucleic acids and small molecules. this work establishes a method called {pinup} for binding site prediction of monomeric proteins. with only two weight parameters to optimize, {pinup} produces not only 42.2\\\\% coverage of actual interfaces (percentage of correctly predicted interface residues in actual interface residues) but also 44.5\\\\% accuracy in predicted interfaces (percentage of correctly predicted interface residues in the predicted interface residues) in a cross validation using a 57-protein dataset. by comparison, the expected accuracy via random prediction (percentage of actual interface residues in surface residues) is only 15\\\\%. the binding sites of the 57-protein set are found to be easier to predict than that of an independent test set of 68 proteins. the average coverage and accuracy for this independent test set are 30.5 and 29.4\\\\%, respectively. the significant gain of {pinup} over expected random prediction is attributed to (i) effective residue-energy score and accessible-surface-area-dependent interface-propensity, (ii) isolation of functional constraints contained in the conservation score from the structural constraints through the combination of residue-energy score (for structural constraints) and conservation score and (iii) a consensus region built on top-ranked initial patches.\"gower street, london, wc1e 6bt, england.\",prediction of protein-protein interaction sites using patch analysis,\"a method for defining and analysing a series of residue patches on the surface of protein structures is used to predict the location of protein-protein interaction sites. each residue patch is analysed for six parameters; solvation potential, residue interface propensity, hydrophobicity, planarity, protrusion and accessible surface area. the method involves the calculation of a relative combined score that gives the probability of a surface patch forming protein-protein interactions. predictions are made for the known structures of protomers from 28 homo-dimers, large protomers from 11 hetero-complexes, small protomers from 14 hetero-complexes, and antigens from six antibody-antigen complexes. the predictions are successful for 66\\\\% (39/59) of the structures and the remainder can usually be rationalized in terms of additional interaction sites.\"\"the sequence of the human genome encodes the genetic instructions for human physiology, as well as rich information about human evolution. in 2001, the international human genome sequencing consortium reported a draft sequence of the euchromatic portion of the human genome. since then, the international collaboration has worked to convert this draft into a genome sequence with high accuracy and nearly complete coverage. here, we report the result of this finishing process. the current genome sequence (build 35) contains 2.85 billion nucleotides interrupted by only 341 gaps. it covers \\\\~{}99\\\\% of the euchromatic genome and is accurate to an error rate of \\\\~{}1 event per 100,000 bases. many of the remaining euchromatic gaps are associated with segmental duplications and will require focused work with new methods. the near-complete sequence, the first for a vertebrate, greatly improves the precision of biological analyses of the human genome including studies of gene number, birth and death. notably, the human genome seems to encode only 20,000–25,000 protein-coding genes. the genome sequence reported here should serve as a firm foundation for biomedical research in the decades ahead.\"lab of biophysical chemistry, university of groningen, nijenborgh 4, 9747 ag groningen, the netherlands; department of computer science, university of groningen, groningen, the netherlands\",{lincs}: a linear constraint solver for molecular simulations,\"in this article, we present a new {linear} constraint solver ({lincs}) for molecular simulations with bond constraints. the algorithm is inherently stable, as the constraints themselves are reset instead of derivatives of the constraints, thereby eliminating drift. although the derivation of the algorithm is presented in terms of matrices, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large molecules. at the same accuracy, the {lincs} algorithm is three to four times faster than the {shake} algorithm. parallelization of the algorithm is straightforward.\\xa0{\\\\copyright} 1997 john wiley \\\\& sons, inc.\\xa0j comput chem18: 1463–1472, 1997\"distant metastasis occurs late during the genetic evolution of pancreatic cancer,exploring the genomes of cancer cells: progress and promise.,exome sequencing identifies frequent mutation of the {swi}/{snf} complex gene {pbrm1} in renal carcinoma,\"the genetics of renal cancer is dominated by inactivation of the {vhl} tumour suppressor gene in clear cell carcinoma ({ccrcc}), the commonest histological subtype. a recent large-scale screen of ∼3,500 genes by {pcr}-based exon re-sequencing identified several new cancer genes in {ccrcc} including {utx} (also known as {kdm6a}), {jarid1c} (also known as {kdm5c}) and {setd2} (ref. 2). these genes encode enzymes that demethylate ({utx}, {jarid1c}) or methylate ({setd2}) key lysine residues of histone h3. modification of the methylation state of these lysine residues of histone h3 regulates chromatin structure and is implicated in transcriptional control. however, together these mutations are present in fewer than 15\\\\% of {ccrcc}, suggesting the existence of additional, currently unidentified cancer genes. here, we have sequenced the protein coding exome in a series of primary {ccrcc} and report the identification of the {swi}/{snf} chromatin remodelling complex gene {pbrm1} (ref. 4) as a second major {ccrcc} cancer gene, with truncating mutations in 41\\\\% (92/227) of cases. these data further elucidate the somatic genetic architecture of {ccrcc} and emphasize the marked contribution of aberrant chromatin biology.\"florida state university, tallahassee, florida 32306, usa.\",{meta-ppisp}: a meta web server for protein-protein interaction site prediction,\"summary: a number of complementary methods have been developed for predicting protein-protein interaction sites. we sought to increase prediction robustness and accuracy by combining results from different predictors, and report here a meta web server, {meta-ppisp}, that is built on three individual web servers: {cons-ppisp} (http://pipe.scs.fsu.edu/ppisp.html), promate (http://bioportal.weizmann.ac.il/promate), and {pinup} ({http://sparks.informatics.iupui.edu/pinup}/). a linear regression method, using the raw scores of the three servers as input, was trained on a set of 35 nonhomologous proteins. cross validation showed that {meta-ppisp} outperforms all the three individual servers. at coverages identical to those of the individual methods, the accuracy of {meta-ppisp} is higher by 4.8 to 18.2 percentage points. similar improvements in accuracy are also seen on {capri} and other targets.\"\"it is informative to detect highly conserved positions in proteins and nucleic acid sequence/structure since they are often indicative of structural and/or functional importance. {consurf} (http://consurf.tau.ac.il) and {conseq} (http://conseq.tau.ac.il) are two well-established web servers for calculating the evolutionary conservation of amino acid positions in proteins using an empirical bayesian inference, starting from protein structure and sequence, respectively. here, we present the new version of the {consurf} web server that combines the two independent servers, providing an easier and more intuitive step-by-step interface, while offering the user more flexibility during the process. in addition, the new version of {consurf} calculates the evolutionary rates for nucleic acid sequences. the new version is freely available at: http://consurf.tau.ac.il/.\"rumelifeneri yolu, 34450 sariyer istanbul, turkey; basic research program, saic-frederick, inc., center for cancer research nanobiology program, nci-frederick, frederick, maryland 21702; and sackler institute of molecular medicine, department of human genetics and molecular medicine, sackler school of medicine, tel aviv university, tel aviv 69978, israel\",principles of {protein−protein} interactions: what are the preferred ways for proteins to interact?,pmid: 18355092\"10.1093/bioinformatics/btm215 motivation: mutual information can be used to explore covarying positions in biological sequences. in the past, it has been successfully used to infer {rna} secondary structure conformations from multiple sequence alignments. in this study, we show that the same principles allow the discovery of transcription factor amino acids that are coevolving with nucleotides in their {dna}-binding {targets.results}: given an alignment of transcription factor binding domains, and a separate alignment of their {dna} target motifs, we demonstrate that mutually covarying base-amino acid positions may indicate possible {protein\\\\^{a}\\x80\\x93dna} contacts. examples explored in this study include {c2h2} zinc finger, homeodomain and {bhlh} {dna}-binding motif families, where a number of known base-amino acid contacting positions are identified. mutual information analyses may aid the prediction of base-amino acid contacting pairs for particular transcription factor families, thereby yielding structural insights from sequence information alone. such inference of {protein\\\\^{a}\\x80\\x93dna} contacting positions may guide future experimental studies of {dna} {recognition.contact}: shaun.mahony@ccbb.pitt.edu or benos@pitt.edu\"unil, ch-101 lausanne, conway institute, university college dublin, belfield, dublin 4, ireland and information g\\\\\\'{e}nomique et structurale, cnrs upr2589, institute for structural biology and microbiology (ibsm), parc scientifique de luminy, 163 avenue de luminy, fr- 13288, marseille cedex 09, france.\",the {m-coffee} web server: a meta-method for computing multiple sequence alignments by combining alternative alignment methods.,\"the {m-coffee} server is a web server that makes it possible to compute multiple sequence alignments ({msas}) by running several {msa} methods and combining their output into one single model. this allows the user to simultaneously run all his methods of choice without having to arbitrarily choose one of them. the {msa} is delivered along with a local estimation of its consistency with the individual {msas} it was derived from. the computation of the consensus multiple alignment is carried out using a special mode of the {t-coffee} package [notredame, higgins and heringa ({t-coffee}: a novel method for fast and accurate multiple sequence alignment. j. mol. biol. 2000; 302: 205-217); wallace, {o\\'sullivan}, higgins and notredame ({m-coffee}: combining multiple sequence alignment methods with {t-coffee}. nucleic acids res. 2006; 34: 1692-1699)] given a set of sequences ({dna} or proteins) in {fasta} format, {m-coffee} delivers a multiple alignment in the most common formats. {m-coffee} is a freeware open source package distributed under a {gpl} license and it is available either as a standalone package or as a web service from www.tcoffee.org.\"\"protein evolution depends on intramolecular coevolutionary networks whose complexity is proportional to the underlying functional and structural interactions among sites. here we present a novel approach that vastly improves the sensitivity of previous methods for detecting coevolution through a weighted comparison of divergence between amino acid sites. the analysis of the {hiv}-1 gag protein detected convergent adaptive coevolutionary events responsible for the selective variability emerging between subtypes. coevolution analysis and functional data for heat-shock proteins, hsp90 and {groel}, highlight that almost all detected coevolving sites are functionally or structurally important. the results support previous suggestions pinpointing the complex interdomain functional interactions within these proteins and we propose new amino acid sites as important for interdomain functional communication. three-dimensional information sheds light on the functional and structural constraints governing the coevolution between sites. our covariation analyses propose two types of coevolving sites in agreement with previous reports: pairs of sites spatially proximal, where compensatory mutations could maintain the local structure stability, and clusters of distant sites located in functional domains, suggesting a functional dependency between them. all sites detected under adaptive evolution in these proteins belong to coevolution groups, further underlining the importance of testing for coevolution in selective constraints analyses.\"houston, tx 77030, usa.\",rapid detection of similarity in protein structure and function through contact metric distances,\"the characterization of biological function among newly determined protein structures is a central challenge in structural genomics. one class of computational solutions to this problem is based on the similarity of protein structure. here, we implement a simple yet efficient measure of protein structure similarity, the contact metric. even though its computation avoids structural alignments and is therefore nearly instantaneous, we find that small values correlate with geometrical root mean square deviations obtained from structural alignments. to test whether the contact metric detects functional similarity, as defined by gene ontology ({go}) terms, it was compared in large-scale computational experiments to four other measures of structural similarity, including alignment algorithms as well as alignment independent approaches. the contact metric was the fastest method and its sensitivity, at any given specificity level, was a close second only to fast alignment and search tool—a structural alignment method that is slower by three orders of magnitude. critically, nearly 40\\\\% of correct functional inferences by the contact metric were not identified by any other approach, which shows that the contact metric is complementary and computationally efficient in detecting functional relationships between proteins. a public \\'contact metric internet server\\' is provided.\"69117 heidelberg, germany.\",protein complexes: structure prediction challenges for the 21st century,\"only a tiny fraction of the many hundreds of known protein complexes are also of known three-dimensional structure. the experimental difficulties surrounding structure determination of complexes make methods that are able to predict structures paramount. the challenge of predicting complex structures is daunting and raises many issues that need to be addressed. to produce the best models, new prediction methods have to somehow combine partial structures with a mixed bag of experimental data, including interactions and low-resolution electron microscopy images.\"1775 north ursula street, mail stop b140, po box 6511, denver, co, 80045, usa.\",{genespeed}: protein domain organization of the transcriptome.,\"the {genespeed} database (http://genespeed.uchsc.edu/) is an online database and resource tool facilitating the detailed study of protein domain homology in the transcriptomes of homo sapiens, mus musculus, drosophila melanogaster and caenorhabditis elegans. the population schema for the {genespeed} database takes advantage of {howard} parallel cluster technology (http://www.massivelyparallel.com/) and performs exhaustive {tblastn} searches covering all pre-assigned {pfam} domain classes in all species (currently 7973 domain families) against the respective unigene {est} databases of the selected four transcriptomes. the resulting database provides a complete annotation of presumed protein domain presence for each unigene cluster. to complement this domain annotation we have also performed a custom transcription factor-family curation of all pfam domains, incorporated the gene ontology classifications for these domains as well as integrated the novartis {symatlas2} dataset for both human and mouse which provides rapid and easy access to tissue-based expression analysis. consequently, the {genespeed} database provides the user with the capability to browse or search the database by any of these specialized criteria as well as more traditional means (gene identifier, gene symbol, etc.), thereby enabling a supervised analysis of gene families through a top-down hierarchical basis defined by domain content, all directly linked to an optimized gene expression dataset.\"\"the pyruvate kinase isoforms {pkm1} and {pkm2} are alternatively spliced products of the {pkm2} gene. {pkm2}, but not {pkm1}, alters glucose metabolism in cancer cells and contributes to tumorigenesis by mechanisms that are not explained by its known biochemical activity. we show that {pkm2} gene transcription is activated by hypoxia-inducible factor 1 ({hif}-1). {pkm2} interacts directly with the {hif}-1± subunit and promotes transactivation of {hif}-1 target genes by enhancing {hif}-1 binding and p300 recruitment to hypoxia response elements, whereas {pkm1} fails to regulate {hif}-1 activity. interaction of {pkm2} with prolyl hydroxylase 3 ({phd3}) enhances {pkm2} binding to {hif}-1± and {pkm2} coactivator function. mass spectrometry and anti-hydroxyproline antibody assays demonstrate {pkm2} hydroxylation on proline-403/408. {phd3} knockdown inhibits {pkm2} coactivator function, reduces glucose uptake and lactate production, and increases o2 consumption in cancer cells. thus, {pkm2} participates in a positive feedback loop that promotes {hif}-1 transactivation and reprograms glucose metabolism in cancer cells. º {pkm2} is a direct {hif}-1 target gene º {pkm2} functions as a coactivator for {hif}-1-dependent gene transcription º {pkm2} coactivator function is stimulated by the prolyl hydroxylase {phd3} º {phd3} knockdown reverses the warburg effect in cancer cells.\"\"{background}:the majority of human non-protein-coding {dna} is made up of repetitive sequences, mainly transposable elements ({tes}). it is becoming increasingly apparent that many of these repetitive {dna} sequence elements encode gene regulatory functions. this fact has important evolutionary implications, since repetitive {dna} is the most dynamic part of the genome. we set out to assess the evolutionary rate and pattern of experimentally characterized human transcription factor binding sites ({tfbs}) that are derived from repetitive versus non-repetitive {dna} to test whether repeat-derived {tfbs} are in fact rapidly evolving. we also evaluated the position-specific patterns of variation among {tfbs} to look for signs of functional constraint on {tfbs} derived from repetitive and non-repetitive {dna}.{results}:we found numerous experimentally characterized {tfbs} in the human genome, 7-10\\\\% of all mapped sites, which are derived from repetitive {dna} sequences including simple sequence repeats ({ssrs}) and {tes}. {te}-derived {tfbs} sequences are far less conserved between species than {tfbs} derived from {ssrs} and non-repetitive {dna}. despite their rapid evolution, several lines of evidence indicate that {te}-derived {tfbs} are functionally constrained. first of all, ancient {te} families, such as {mir} and l2, are enriched for {tfbs} relative to younger families like alu and l1. secondly, functionally important positions in {te}-derived {tfbs}, specifically those residues thought to physically interact with their cognate protein binding factors ({tf}), are more evolutionarily conserved than adjacent {tfbs} positions. finally, {te}-derived {tfbs} show position-specific patterns of sequence variation that are highly distinct from random patterns and similar to the variation seen for non-repeat derived sequences of the same {tfbs}.{conclusion}:the abundance of experimentally characterized human {tfbs} that are derived from repetitive {dna} speaks to the substantial regulatory effects that this class of sequence has on the human genome. the unique evolutionary properties of repeat-derived {tfbs} are perhaps even more intriguing. {te}-derived {tfbs} in particular, while clearly functionally constrained, evolve extremely rapidly relative to non-repeat derived sites. such rapidly evolving {tfbs} are likely to confer species-specific regulatory phenotypes, i.e. divergent expression patterns, on the human evolutionary lineage. this result has practical implications with respect to the widespread use of evolutionary conservation as a surrogate for functionally relevant non-coding {dna}. most {te}-derived {tfbs} would be missed using the kinds of sequence conservation-based screens, such as phylogenetic footprinting, that are used to help characterize non-coding {dna}. thus, the very {tfbs} that are most likely to yield human-specific characteristics will be neglected by the comparative genomic techniques that are currently de rigeur for the identification of novel regulatory sites.\"\"comprehensive sequencing of human cancers has identified recurrent mutations in genes encoding chromatin regulatory proteins. for clear cell renal cell carcinoma ({ccrcc}), three of the five commonly mutated genes encode the chromatin regulators {pbrm1}, {setd2}, and {bap1}. how these mutations alter the chromatin landscape and transcriptional program in {ccrcc} or other cancers is not understood. here, we identified alterations in chromatin organization and transcript profiles associated with mutations in chromatin regulators in a large cohort of primary human kidney tumors. by associating variation in chromatin organization with mutations in {setd2}, which encodes the enzyme responsible for {h3k36} trimethylation, we found that changes in chromatin accessibility occurred primarily within actively transcribed genes. this increase in chromatin accessibility was linked with widespread alterations in {rna} processing, including intron retention and aberrant splicing, affecting ∼25\\\\% of all expressed genes. furthermore, decreased nucleosome occupancy proximal to misspliced exons was observed in tumors lacking {h3k36me3}. these results directly link mutations in {setd2} to chromatin accessibility changes and {rna} processing defects in cancer. detecting the functional consequences of specific mutations in chromatin regulatory proteins in primary human samples could ultimately inform the therapeutic application of an emerging class of chromatin-targeted compounds.\"german cancer research center, inf 280, heidelberg, 69120, germany. w.huber@dkfz.de\",variance stabilization applied to microarray data calibration and to the quantification of differential expression.,\"we introduce a statistical model for microarray gene expression data that comprises data calibration, the quantification of differential expression, and the quantification of measurement error. in particular, we derive a transformation h for intensity measurements, and a difference statistic deltah whose variance is approximately constant along the whole intensity range. this forms a basis for statistical inference from microarray data, and provides a rational data pre-processing strategy for multivariate analyses. for the transformation h, the parametric form h(x)=arsinh(a+bx) is derived from a model of the variance-versus-mean dependence for microarray intensity data, using the method of variance stabilizing transformations. for large intensities, h coincides with the logarithmic transformation, and deltah with the log-ratio. the parameters of h together with those of the calibration between experiments are estimated with a robust variant of maximum-likelihood estimation. we demonstrate our approach on data sets from different experimental platforms, including two-colour {cdna} arrays and a series of affymetrix oligonucleotide arrays.\"\"the development of noninvasive methods to detect and monitor tumors continues to be a major challenge in oncology. we used digital polymerase chain reaction-based technologies to evaluate the ability of circulating tumor {dna} ({ctdna}) to detect tumors in 640 patients with various cancer types. we found that {ctdna} was detectable in >75\\\\% of patients with advanced pancreatic, ovarian, colorectal, bladder, gastroesophageal, breast, melanoma, hepatocellular, and head and neck cancers, but in less than 50\\\\% of primary brain, renal, prostate, or thyroid cancers. in patients with localized tumors, {ctdna} was detected in 73, 57, 48, and 50\\\\% of patients with colorectal cancer, gastroesophageal cancer, pancreatic cancer, and breast adenocarcinoma, respectively. {ctdna} was often present in patients without detectable circulating tumor cells, suggesting that these two biomarkers are distinct entities. in a separate panel of 206 patients with metastatic colorectal cancers, we showed that the sensitivity of {ctdna} for detection of clinically relevant {kras} gene mutations was 87.2\\\\% and its specificity was 99.2\\\\%. finally, we assessed whether {ctdna} could provide clues into the mechanisms underlying resistance to epidermal growth factor receptor blockade in 24 patients who objectively responded to therapy but subsequently relapsed. twenty-three (96\\\\%) of these patients developed one or more mutations in genes involved in the mitogen-activated protein kinase pathway. together, these data suggest that {ctdna} is a broadly applicable, sensitive, and specific biomarker that can be used for a variety of clinical and research purposes in patients with multiple different types of cancer.\"baltimore, maryland 21231.\",a genetic model for colorectal tumorigenesis.,identification of computational hot spots in protein interfaces: combining solvent accessibility and inter-residue potentials improves the accuracy.,uji, kyoto 611-0011, japan. kanehisa@kuicr.kyoto-u.ac.jp\",{kegg}: kyoto encyclopedia of genes and genomes.,\"{kegg} (kyoto encyclopedia of genes and genomes) is a knowledge base for systematic analysis of gene functions, linking genomic information with higher order functional information. the genomic information is stored in the {genes} database, which is a collection of gene catalogs for all the completely sequenced genomes and some partial genomes with up-to-date annotation of gene functions. the higher order functional information is stored in the {pathway} database, which contains graphical representations of cellular processes, such as metabolism, membrane transport, signal transduction and cell cycle. the {pathway} database is supplemented by a set of ortholog group tables for the information about conserved subpathways (pathway motifs), which are often encoded by positionally coupled genes on the chromosome and which are especially useful in predicting gene functions. a third database in {kegg} is {ligand} for the information about chemical compounds, enzyme molecules and enzymatic reactions. {kegg} provides java graphics tools for browsing genome maps, comparing two genome maps and manipulating expression maps, as well as computational tools for sequence comparison, graph comparison and path computation. the {kegg} databases are daily updated and made freely available (http://www. genome.ad.jp/kegg/).\"d-38124 braunschweig, germany. ewi@gbf.de\",{transfac}: an integrated system for gene expression regulation,\"{transfac} is a database on transcription factors, their genomic binding sites and {dna}-binding profiles ({http://transfac.gbf.de/transfac}/ ). its content has been enhanced, in particular by information about training sequences used for the construction of nucleotide matrices as well as by data on plant sites and factors. moreover, {transfac} has been extended by two new modules: {pathodb} provides data on pathologically relevant mutations in regulatory regions and transcription factor genes, whereas {s/mart} {db} compiles features of scaffold/matrix attached regions ({s/mars}) and the proteins binding to them. additionally, the databases {transpath}, about signal transduction, and {cytomer}, about organs and cell types, have been extended and are increasingly integrated with the {transfac} data sources.\"\"the heterogeneity of cancer genomes in terms of acquired mutations complicates the identification of genes whose modification may exert a driver role in tumorigenesis. in this study, we present a novel method that integrates expression profiles, mutation effects, and systemic properties of mutated genes to identify novel cancer drivers. we applied our method to ovarian cancer samples and were able to identify putative drivers in the majority of carcinomas without mutations in known cancer genes, thus suggesting that it can be used as a complementary approach to find rare driver mutations that cannot be detected using frequency-based approaches.\"9500 gilman drive, la jolla, ca 92093, usa. nbaker@mccammon.ucsd.edu\",electrostatics of nanosystems: application to microtubules and the ribosome,\"evaluation of the electrostatic properties of biomolecules has become a standard practice in molecular biophysics. foremost among the models used to elucidate the electrostatic potential is the {poisson-boltzmann} equation; however, existing methods for solving this equation have limited the scope of accurate electrostatic calculations to relatively small biomolecular systems. here we present the application of numerical methods to enable the trivially parallel solution of the {poisson-boltzmann} equation for supramolecular structures that are orders of magnitude larger in size. as a demonstration of this methodology, electrostatic potentials have been calculated for large microtubule and ribosome structures. the results point to the likely role of electrostatics in a variety of activities of these structures.\"boston university, boston, ma 02152, usa.\",extracting conserved gene expression motifs from gene expression data.,\"we propose a representation for gene expression data called conserved gene expression motifs or {xmotifs}. a gene\\'s expression level is conserved across a set of samples if the gene is expressed with the same abundance in all the samples. a conserved gene expression motif is a subset of genes that is simultaneously conserved across a subset of samples. we present a computational technique to discover large conserved gene motifs that cover all the samples and classes in the data. when applied to published data sets representing different cancers or disease outcomes, our algorithm constructs {xmotifs} that distinguish between the various classes.\"\"as whole-genome sequencing becomes commoditized and we begin to sequence and analyze personal genomes for clinical and diagnostic purposes, it is necessary to understand what constitutes a complete sequencing experiment for determining genotypes and detecting single-nucleotide variants. here, we show that the current recommendation of ∼30× coverage is not adequate to produce genotype calls across a large fraction of the genome with acceptably low error rates. our results are based on analyses of a clinical sample sequenced on two related illumina platforms, {gaiix} and {hiseq} 2000, to a very high depth (126×). we used these data to establish genotype-calling filters that dramatically increase accuracy. we also empirically determined how the callable portion of the genome varies as a function of the amount of sequence data used. these results help provide a  ” sequencing guide” for future whole-genome sequencing decisions and metrics by which coverage statistics should be reported.\"\"increasing knowledge about the organization of proteins into complexes, systems, and pathways has led to a flowering of theoretical approaches for exploiting this knowledge in order to better learn the functions of proteins and their roles underlying phenotypic traits and diseases. much of this body of theory has been developed and tested in model organisms, relying on their relative simplicity and genetic and biochemical tractability to accelerate the research. in this review, we discuss several of the major approaches for computationally integrating proteomics and genomics observations into integrated protein networks, then applying guilt-by-association in these networks in order to identify genes underlying traits. recent trends in this field include a rising appreciation of the modular network organization of proteins underlying traits or mutational phenotypes, and how to exploit such protein modularity using computational approaches related to the internet search algorithm {pagerank}. many protein network-based predictions have recently been experimentally confirmed in yeast, worms, plants, and mice, and several successful approaches in model organisms have been directly translated to analyze human disease, with notable recent applications to glioma and breast cancer prognosis.\"\"massively parallel approaches to nucleic acid sequencing have matured from proof-of-concept to commercial products during the past 5 years. these technologies are now widely accessible, increasingly affordable, and have already exerted a transformative influence on the study of human cancer. here, we review new features of cancer genomes that are being revealed by large-scale applications of these technologies. we focus on those insights most likely to affect future clinical practice. foremost among these lessons, we summarize the formidable genetic heterogeneity within given cancer types that is appreciable with higher resolution profiling and larger sample sets. we discuss the inherent challenges of defining driving genomic events in a given cancer genome amidst thousands of other somatic events. finally, we explore the organizational, regulatory and societal challenges impeding precision cancer medicine based on genomic profiling from assuming its place as standard-of-care.\",statistical analysis and prediction of protein–protein interfaces,\"predicting protein–protein interfaces from a three-dimensional structure is a key task of computational structural proteomics. in contrast to geometrically distinct small molecule binding sites, protein–protein interface are notoriously difficult to predict. we generated a large nonredundant data set of 1494 true protein–protein interfaces using biological symmetry annotation where necessary. the data set was carefully analyzed and a support vector machine was trained on a combination of a new robust evolutionary conservation signal with the local surface properties to predict protein–protein interfaces. fivefold cross validation verifies the high sensitivity and selectivity of the model. as much as 97\\\\% of the predicted patches had an overlap with the true interface patch while only 22\\\\% of the surface residues were included in an average predicted patch. the model allowed the identification of potential new interfaces and the correction of mislabeled oligomeric states. proteins 2005. {\\\\copyright} 2005 {wiley-liss}, inc.\"swiss federal institute of technology zurich eth zentrum, 8092 zurich, switzerland. barkow@tik.ee.ethz.ch\",{bicat}: a biclustering analysis toolbox.,\"{summary}: besides classical clustering methods such as hierarchical clustering, in recent years biclustering has become a popular approach to analyze biological data sets, e.g. gene expression data. the biclustering analysis toolbox ({bicat}) is a software platform for clustering-based data analysis that integrates various biclustering and clustering techniques in terms of a common graphical user interface. furthermore, {bicat} provides different facilities for data preparation, inspection and postprocessing such as discretization, filtering of biclusters according to specific criteria or gene pair analysis for constructing gene interconnection graphs. the possibility to use different biclustering algorithms inside a single graphical tool allows the user to compare clustering results and choose the algorithm that best fits a specific biological scenario. the toolbox is described in the context of gene expression analysis, but is also applicable to other types of data, e.g. data from proteomics or synthetic lethal experiments. {availability}: the {bicat} toolbox is freely available at http://www.tik.ee.ethz.ch/sop/bicat and runs on all operating systems. the java source code of the program and a developer\\'s guide is provided on the website as well. therefore, users may modify the program and add further algorithms or extensions.\"\"{background}:an important goal in bioinformatics is to unravel the network of transcription factors ({tfs}) and their targets. this is important in the human genome, where many {tfs} are involved in disease progression. here, classification methods are applied to identify new targets for 152 transcriptional regulators using publicly-available targets as training examples. three types of sequence information are used: composition, conservation, and {overrepresentation.results}:starting with 8817 {tf}-target interactions we predict an additional 9333 targets for 152 {tfs}. randomized classifiers make few predictions (\\\\~{}2/18660) indicating that our predictions for many {tfs} are significantly enriched for true targets. an enrichment score is calculated and used to filter new {predictions.two} case-studies for the {tfs} {oct4} and {wt1} illustrate the usefulness of our predictions: * many predicted {oct4} targets fall into the wnt-pathway. this is consistent with known biology as {oct4} is developmentally related and wnt pathway plays a role in early development. * beginning with 15 known targets, 354 predictions are made for {wt1}. {wt1} has a role in formation of wilms\\' tumor. chromosomal regions previously implicated in wilms\\' tumor by cytological evidence are statistically enriched in predicted {wt1} targets. these findings may shed light on wilms\\' tumor progression, suggesting that the tumor progresses either by loss of {wt1} or by loss of regions harbouring its targets. * targets of {wt1} are statistically enriched for cancer related functions including metastasis and apoptosis. among new targets are {bax} and {pde4b}, which may help mediate the established anti-apoptotic effects of {wt1}. * of the thirteen {tfs} found which co-regulate genes with {wt1} (p [less than or equal to] 0.02), 8 have been previously implicated in cancer. the regulatory-network for {wt1} targets in genomic regions relevant to wilms\\' tumor is {provided.conclusion}:we have assembled a set of features for the targets of human {tfs} and used them to develop classifiers for the determination of new regulatory targets. many predicted targets are consistent with the known biology of their regulators, and new targets for the wilms\\' tumor regulator, {wt1}, are proposed. we speculate that wilms\\' tumor development is mediated by chromosomal rearrangements in the location of {wt1} {targets.reviewers}:this article was reviewed by trey ideker, vladimir a. kuznetsov(nominated by frank eisenhaber), and tzachi pilpel.\"\"conserved residues in protein-protein interfaces correlate with residue hot-spots. to obtain insight into their roles, we have studied their mobility. we have performed 39 explicit solvent simulations of 15 complexes and their monomers, with the interfaces varying in size, shape, and function. the dynamic behavior of conserved residues in unbound monomers illustrates significantly lower flexibility as compared to their environment, suggesting that already prior to binding they are constrained in a bound-like configuration. to understand this behavior, we have analyzed the inter- and intra-chain hydrogen-bond residence-time in the interfaces. we find that conserved residues are not involved significantly in hydrogen bonds across the interface as compared to non-conserved. however, the monomer simulations reveal that conserved residues contribute dominantly to hydrogen bond formation prior to binding. packing of conserved residues across the trajectories is significantly higher prior and following the binding, rationalizing their lower mobility. backbone torsional angle distributions show that conserved residues assume restricted regions of space and the most visited conformations in the bound and unbound trajectories are similar, suggesting that conserved residues are pre-organized. combined with previous studies, we conclude that conserved residues, hot spots, anchor and interface-buried residues may be similar residues, fulfilling similar roles.\"\"european molecular biology laboratory, 69012 heidelberg, germany\",knowledge-based protein secondary structure assignment.,\"we have developed an automatic algorithm {stride} for protein secondary structure assignment from atomic coordinates based on the combined use of hydrogen bond energy and statistically derived backbone torsional angle information. parameters of the pattern recognition procedure were optimized using designations provided by the crystallographers as a standard-of-truth. comparison to the currently most widely used technique {dssp} by kabsch and sander (biopolymers 22:2577-2637, 1983) shows that {stride} and {dssp} assign secondary structural states in 58 and 31\\\\% of 226 protein chains in our data sample, respectively, in greater agreement with the specific residue-by-residue definitions provided by the discoverers of the structures while in 11\\\\% of the chains, the assignments are the same. {stride} delineates every 11th helix and every 32nd strand more in accord with published assignments.\"kharagpur, 721 302, india. subhajyoti\\\\_de@yahoo.com\",interaction preferences across protein-protein interfaces of obligatory and non-obligatory components are different.,\"{background}: a polypeptide chain of a protein-protein complex is said to be obligatory if it is bound to another chain throughout its functional lifetime. such a chain might not adopt the native fold in the unbound form. a non-obligatory polypeptide chain associates with another chain and dissociates upon molecular stimulus. although conformational changes at the interaction interface are expected, the overall {3-d} structure of the non-obligatory chain is unaltered. the present study focuses on protein-protein complexes to understand further the differences between obligatory and non-obligatory interfaces. {results}: a non-obligatory chain in a complex of known {3-d} structure is recognized by its stable existence with same fold in the bound and unbound forms. on the contrary, an obligatory chain is detected by its existence only in the bound form with no evidence for the native-like fold of the chain in the unbound form. various interfacial properties of a large number of complexes of known {3-d} structures thus classified are comparatively analyzed with an aim to identify structural descriptors that distinguish these two types of interfaces. we report that the interaction patterns across the interfaces of obligatory and non-obligatory components are different and contacts made by obligatory chains are predominantly non-polar. the obligatory chains have a higher number of contacts per interface (20 +/- 14 contacts per interface) than non-obligatory chains (13 +/- 6 contacts per interface). the involvement of main chain atoms is higher in the case of obligatory chains (16.9 \\\\%) compared to non-obligatory chains (11.2 \\\\%). the beta-sheet formation across the subunits is observed only among obligatory protein chains in the dataset. apart from these, other features like residue preferences and interface area produce marginal differences and they may be considered collectively while distinguishing the two types of interfaces. {conclusion}: these results can be useful in distinguishing the two types of interfaces observed in structures determined in large-scale in the structural genomics initiatives, especially for those multi-component protein assemblies for which the biochemical characterization is incomplete.\"may,2006-02-17 16:55:08,\"bijvoet center for biomolecular research, utrecht university, utrecht, the netherlands.\",{whiscy}: what information does surface conservation yield? application to data-driven docking,\"protein–protein interactions play a key role in biological processes. identifying the interacting residues is a first step toward understanding these interactions at a structural level. in this study, the interface prediction program {whiscy} is presented. it combines surface conservation and structural information to predict protein–protein interfaces. the accuracy of the predictions is more than three times higher than a random prediction. these predictions have been combined with another interface prediction program, {promate} [neuvirth et al. j mol biol 2004;338:181–199], resulting in an even more accurate predictor. the usefulness of the predictions was tested using the data-driven docking program {haddock} [dominguez et al. j am chem soc 2003;125:1731–1737] in an unbound docking experiment, with the goal of generating as many near-native structures as possible. unrefined rigid body docking solutions within 10 \\\\aa{} ligand {rmsd} from the true structure were generated for 22 out of 25 docked complexes. for 18 complexes, more than 100 of the 8000 generated models were correct. our results demonstrates the potential of using interface predictions to drive protein–protein docking. proteins 2006. {\\\\copyright} 2006 {wiley-liss}, inc.\"de boelelaan 1081a, 1081 hv amsterdam, the netherlands. wvanwie@few.vu.nl.\",weighted clustering of called array {cgh} data,\"array comparative genomic hybridization ({acgh}) is a laboratory technique to measure chromosomal copy number changes. a clear biological interpretation of the measurements is obtained by mapping these onto an ordinal scale with categories loss/normal/gain of a copy. the pattern of gains and losses harbors a level of tumor specificity. here, we present {wecca} (weighted clustering of called {acgh} data), a method for weighted clustering of samples on the basis of the ordinal {acgh} data. two similarities to be used in the clustering and particularly suited for ordinal data are proposed, which are generalized to deal with weighted observations. in addition, a new form of linkage, especially suited for ordinal data, is introduced. in a simulation study, we show that the proposed cluster method is competitive to clustering using the continuous data. we illustrate {wecca} using an application to a breast cancer data set, where {wecca} finds a clustering that relates better with survival than the original one.\"\"small cell lung carcinoma ({sclc}) is a highly lethal, smoking-associated cancer with few known targetable genetic alterations. using genome sequencing, we characterized the somatic evolution of a genetically engineered mouse model ({gemm}) of {sclc} initiated by loss of trp53 and rb1. we identified alterations in {dna} copy number and complex genomic rearrangements and demonstrated a low somatic point mutation frequency in the absence of tobacco mutagens. alterations targeting the tumor suppressor pten occurred in the majority of murine {sclc} studied, and engineered pten deletion accelerated murine {sclc} and abrogated loss of chr19 in trp53; rb1; pten compound mutant tumors. finally, we found evidence for polyclonal and sequential metastatic spread of murine {sclc} by comparative sequencing of families of related primary tumors and metastases. we propose a temporal model of {sclc} tumorigenesis with implications for human {sclc} therapeutics and the nature of cancer-genome evolution in {gemms}. \"\"murine {sclcs} acquire few point mutations in the absence of tobacco {mutagens\"\"pten} is recurrently mutated, and engineered deletion accelerates tumor {progression\"\"mycl1} amplifications play an early, central role in {msclc} {tumorigenesis\"\"analysis} of related primary and metastatic {msclc} suggests complex clonal evolution comprehensive genomic analysis in a mouse model of small-cell lung carcinoma delineates metastatic progression in this tumor type, showing that selection for specific cancer mutations can drive large genomic rearrangements and that distant metastases likely arise from a common metastatic seeding step involving the lymph nodes.\"vancouver, bc, canada\",combining probability from independent tests: the weighted z-method is superior to fisher\\'s approach.,\"the most commonly used method in evolutionary biology for combining information across multiple tests of the same null hypothesis is fisher\\'s combined probability test. this note shows that an alternative method called the weighted z-test has more power and more precision than does fisher\\'s test. furthermore, in contrast to some statements in the literature, the weighted z-method is superior to the unweighted z-transform approach. the results in this note show that, when combining p-values from multiple tests of the same hypothesis, the weighted z-method should be preferred.\"\"a major obstacle towards elucidating the molecular basis of transcriptional regulation is the lack of a detailed understanding of the interplay between non-specific and specific {protein-dna} interactions. based on molecular dynamics simulations of {c(2)h}(2) zinc fingers ({zfs}) and engrailed homeodomain transcription factors ({tfs}), we show that each of the studied {dna}-binding domains has a set of highly constrained side chains in preset configurations ready to form hydrogen bonds with the {dna} backbone. interestingly, those domains that bury their recognition helix into the major groove are found to have an electrostatic hot spot for cl(-) ions located on the same binding cavity as the most buried {dna} phosphate. the spot is characterized by three protein hydrogen bond donors, often including two basic side chains. if bound, cl(-) ions, likely mimicking phosphates, steer side chains that end up forming specific contacts with bases into bound-like conformations. these findings are consistent with a multi-step {dna}-binding mechanism in which a pre-organized set of {tf} side chains assist in the desolvation of phosphates into well defined sites, prompting the re-organization of specificity determining side chains into conformations suitable for the recognition of their cognate sequence.\"and department of biochemistry and molecular biology, indiana university medical school, 714 north senate avenue, indianapolis, in 46202, usa. afernan@iupui.edu\",the nonconserved wrapping of conserved protein folds reveals a trend toward increasing connectivity in proteomic networks,\"although protein folding domains are generally conserved for function across distant homologous sequences, one crucial structural feature is not conserved: the wrapping of backbone hydrogen bonds, that is, the extent to which they are intramolecularly desolvated and thereby protected from water attack. extensive data on protein complex interfaces led us to postulate that insufficiently wrapped backbone hydrogen bonds in monomeric domains must be adhesive, and therefore determinants of interactivity, a result that has been experimentally confirmed. here, we show that the wrapping of certain conserved folds becomes progressively poorer as species diverge in some lineages. this trend is thus concurrent with a progressive enhancement of the interactivity of individual domains sharing the conserved fold. such increase in interactivity is predicted to impose an  ” evolutionary brake” on the overall speed of sequence divergence. this phenomenon follows when more and more residues become engaged in protein associations and thus become functionally indispensable. for complete proteomes for which statistically significant structural data are available, scale-free network statistics based solely on the distribution of folding domains, catalogued by their number of wrapping defects, best describe the proteomic connectivity. thus, the intermolecular connectivity may be effectively used as a measure of species complexity. our results might contribute to explaining how interactome complexity may be achieved without a dramatic increase in genome size.\"evolution of human {bcr}-{abl1} lymphoblastic leukaemia-initiating cells,\"many tumours are composed of genetically diverse cells; however, little is known about how diversity evolves or the impact that diversity has on functional properties. here, using xenografting and {dna} copy number alteration ({cna}) profiling of human {bcr}-{abl1} lymphoblastic leukaemia, we demonstrate that genetic diversity occurs in functionally defined leukaemia-initiating cells and that many diagnostic patient samples contain multiple genetically distinct leukaemia-initiating cell subclones. reconstructing the subclonal genetic ancestry of several samples by {cna} profiling demonstrated a branching multi-clonal evolution model of leukaemogenesis, rather than linear succession. for some patient samples, the predominant diagnostic clone repopulated xenografts, whereas in others it was outcompeted by minor subclones. reconstitution with the predominant diagnosis clone was associated with more aggressive growth properties in xenografts, deletion of {cdkn2a} and {cdkn2b}, and a trend towards poorer patient outcome. our findings link clonal diversity with leukaemia-initiating-cell function and underscore the importance of developing therapies that eradicate all intratumoral subclones.\"santa cruz, 1156 high st, santa cruz, ca 95064, usa.\",{footprinter}: a program designed for phylogenetic footprinting,\"phylogenetic footprinting is a method for the discovery of regulatory elements in a set of homologous regulatory regions, usually collected from multiple species. it does so by identifying the best conserved motifs in those homologous regions. this note describes web software that has been designed specifically for this purpose, making use of the phylogenetic relationships among the homologous sequences in order to make more accurate predictions. the software is called {footprinter} and is available at http://bio.cs.washington.edu/software.html.\"\"the basic region leucine zipper ({bzip}) proteins form one of the largest families of transcription factors in eukaryotic cells. despite relatively high homology between the amino acid sequences of the {bzip} motifs, these proteins recognize diverse {dna} sequences. here we report the 2.0 \\\\aa{} resolution crystal structure of the {bzip} motif of one such transcription factor, {pap1}, a fission yeast {ap}-1-like transcription factor that binds {dna} containing the novel consensus sequence {ttacgtaa}. the structure reveals how the pap1-specific residues of the {bzip} basic region recognize the target sequence and shows that the side chain of the invariant asn in the {bzip} motif adopts an alternative conformation in pap1. this conformation, which is stabilized by a pap1-specific residue and its associated water molecule, recognizes a different base in the target sequence from that in other {bzip} subfamilies.\"convergent loss of {pten} leads to clinical resistance to a {pi}({3)k}[agr] inhibitor,\"broad and deep tumour genome sequencing has shed new light on tumour heterogeneity and provided important insights into the evolution of metastases arising from different clones. there is an additional layer of complexity, in that tumour evolution may be influenced by selective pressure provided by therapy, in a similar fashion to that occurring in infectious diseases. here we studied tumour genomic evolution in a patient (index patient) with metastatic breast cancer bearing an activating {pik3ca} (phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha, {pi}({3)k}\\\\&\\\\#177;) mutation. the patient was treated with the {pi}({3)k}\\\\&\\\\#177; inhibitor {byl719}, which achieved a lasting clinical response, but the patient eventually became resistant to this drug (emergence of lung metastases) and died shortly thereafter. a rapid autopsy was performed and material from a total of 14 metastatic sites was collected and sequenced. all metastatic lesions, when compared to the pre-treatment tumour, had a copy loss of {pten} (phosphatase and tensin homolog) and those lesions that became refractory to {byl719} had additional and different {pten} genetic alterations, resulting in the loss of {pten} expression. to put these results in context, we examined six other patients also treated with {byl719}. acquired bi-allelic loss of {pten} was found in one of these patients, whereas in two others {pik3ca} mutations present in the primary tumour were no longer detected at the time of progression. to characterize our findings functionally, we examined the effects of {pten} knockdown in several preclinical models (both in cell lines intrinsically sensitive to {byl719} and in {pten}-null xenografts derived from our index patient), which we found resulted in resistance to {byl719}, whereas simultaneous {pi}({3)k} p110\\\\&\\\\#178; blockade reverted this resistance phenotype. we conclude that parallel genetic evolution of separate metastatic sites with different {pten} genomic alterations leads to a convergent {pten}-null phenotype resistant to {pi}({3)k}\\\\&\\\\#177; inhibition.\"\"researchers have been studying metastasis for more than 100 years, and only recently have we gained insight into the mechanisms by which metastatic cells arise from primary tumours and the reasons that certain tumour types tend to metastasize to specific organs. stephen paget\\'s 1889 proposal that metastasis depends on cross-talk between selected cancer cells (the \\'seeds\\') and specific organ microenvironments (the \\'soil\\') still holds forth today. it is now known that the potential of a tumour cell to metastasize depends on its interactions with the homeostatic factors that promote tumour-cell growth, survival, angiogenesis, invasion and metastasis. how has this field developed over the past century, and what major breakthroughs are most likely to lead to effective therapeutic approaches?\"712 medical center drive, rockville, maryland 20850, usa. johnq@tigr.org\",computational analysis of microarray data,\"{background}:comparison of metabolic networks across species is a key to understanding how evolutionary pressures shape these networks. by selecting taxa representative of different lineages or lifestyles and using a comprehensive set of descriptors of the structure and complexity of their metabolic networks, one can highlight both qualitative and quantitative differences in the metabolic organization of species subject to distinct evolutionary paths or environmental {constraints.results}:we used a novel representation of metabolic networks, termed network of interacting pathways or {nip}, to focus on the modular, high-level organization of the metabolic capabilities of the cell. using machine learning techniques we identified the most relevant aspects of cellular organization that change under evolutionary pressures. we considered the transitions from prokarya to eukarya (with a focus on the transitions among the archaea, bacteria and eukarya), from unicellular to multicellular eukarya, from free living to host-associated bacteria, from anaerobic to aerobic, as well as the acquisition of cell motility or growth in an environment of various levels of salinity or temperature. intuitively, we expect organisms with more complex lifestyles to have more complex and robust metabolic networks. here we demonstrate for the first time that such organisms are not only characterized by larger, denser networks of metabolic pathways but also have more efficiently organized cross communications, as revealed by subtle changes in network topology. these changes are unevenly distributed among metabolic pathways, with specific categories of pathways being promoted to more central locations as an answer to environmental {constraints.conclusions}:combining methods from graph theory and machine learning, we have shown here that evolutionary pressures not only affects gene and protein sequences, but also specific details of the complex wiring of functional modules in the cell. this approach allows the identification and quantification of those changes, and provides an overview of the evolution of intracellular systems.\"\"we have presented a new gapped {pwm} model for variable length {dna} binding sites that is not too restrictive nor over-parameterised. our comparison with existing tools shows that on average it does not have better predictive accuracy than existing methods. however, it does provide more interpretable models of motifs of variable structure that are suitable for follow-up structural studies. to our knowledge, we are the first to apply variable length motif models to eukaryotic {chip}-seq data sets and consequently the first to show their value in this domain. the results include a novel motif for the ubiquitous transcription factor sp1.\"\"motivation: although metabolic reactions are unquestionably shaped by evolutionary processes, the degree to which the overall structure and complexity of their interconnections are linked to the phylogeny of species has not been evaluated in depth. here, we apply an original metabolome representation, termed network of interacting pathways or {nip}, with a combination of graph theoretical and machine learning strategies, to address this question. {nips} compress the information of the metabolic network exhibited by a species into much smaller networks of overlapping metabolic pathways, where nodes are pathways and links are the metabolites they exchange.\"44 binney st, boston, ma 02115, usa. rgentlem@jimmy.harvard.edu\",bioconductor: open software development for computational biology and bioinformatics.,\"the bioconductor project is an initiative for the collaborative creation of extensible software for computational biology and bioinformatics. the goals of the project include: fostering collaborative development and widespread use of innovative software, reducing barriers to entry into interdisciplinary scientific research, and promoting the achievement of remote reproducibility of research results. we describe details of our aims and methods, identify current challenges, compare bioconductor to other open bioinformatics projects, and provide working examples.\"\"although genomewide {rna} expression analysis has become a routine tool in biomedical research, extracting biological insight from such information remains a major challenge. here, we describe a powerful analytical method called gene set enrichment analysis ({gsea}) for interpreting gene expression data. the method derives its power by focusing on gene sets, that is, groups of genes that share common biological function, chromosomal location, or regulation. we demonstrate how {gsea} yields insights into several cancer-related data sets, including leukemia and lung cancer. notably, where single-gene analysis finds little similarity between two independent studies of patient survival in lung cancer, {gsea} reveals many biological pathways in common. the {gsea} method is embodied in a freely available software package, together with an initial database of 1,325 biologically defined gene sets.\"\"the naked mole-rat is the longest living rodent with a maximum lifespan exceeding 28 years. in addition to its longevity, naked mole-rats have an extraordinary resistance to cancer as tumors have never been observed in these rodents. furthermore, we show that a combination of activated ras and {sv40} {lt} fails to induce robust anchorage-independent growth in naked mole-rat cells, while it readily transforms mouse fibroblasts. the mechanisms responsible for the cancer resistance of naked mole-rats were unknown. here we show that naked mole-rat fibroblasts display hypersensitivity to contact inhibition, a phenomenon we termed \"\"early contact inhibition.\"\" contact inhibition is a key anticancer mechanism that arrests cell division when cells reach a high density. in cell culture, naked mole-rat fibroblasts arrest at a much lower density than those from a mouse. we demonstrate that early contact inhibition requires the activity of p53 and {prb} tumor suppressor pathways. inactivation of both p53 and {prb} attenuates early contact inhibition. contact inhibition in human and mouse is triggered by the induction of {p27(kip1}). in contrast, early contact inhibition in naked mole-rat is associated with the induction of {p16(ink4a}). furthermore, we show that the roles of {p16(ink4a}) and {p27(kip1}) in the control of contact inhibition became temporally separated in this species: the early contact inhibition is controlled by {p16(ink4a}), and regular contact inhibition is controlled by {p27(kip1}). we propose that the additional layer of protection conferred by two-tiered contact inhibition contributes to the remarkable tumor resistance of the naked mole-rat.\"\"genetic sequence alignment is the basis of many evolutionary and comparative studies, and errors in alignments lead to errors in the interpretation of evolutionary information in genomes. traditional multiple sequence alignment methods disregard the phylogenetic implications of gap patterns that they create and infer systematically biased alignments with excess deletions and substitutions, too few insertions, and implausible insertion-deletion–event histories. we present a method that prevents these systematic errors by recognizing insertions and deletions as distinct evolutionary events. we show theoretically and practically that this improves the quality of sequence alignments and downstream analyses over a wide range of realistic alignment problems. these results suggest that insertions and sequence turnover are more common than is currently thought and challenge the conventional picture of sequence evolution and mechanisms of functional and structural changes.\"\"in molecular dynamics ({md}) simulations the need often arises to maintain such parameters as temperature or pressure rather than energy and volume, or to impose gradients for studying transport properties in nonequilibrium {md}. a method is described to realize coupling to an external bath with constant temperature or pressure with adjustable time constants for the coupling. the method is easily extendable to other variables and to gradients, and can be applied also to polyatomic molecules involving internal constraints. the influence of coupling time constants on dynamical variables is evaluated. a leap‐frog algorithm is presented for the general case involving constraints with coupling to both a constant temperature and a constant pressure bath.\"illinois 60637, usa. ariel@uchicago.edu\",dehydron: a structurally encoded signal for protein interaction.,\"we introduce a quantifiable structural motif, called dehydron, that is shown to be central to protein-protein interactions. a dehydron is a defectively packed backbone hydrogen bond suggesting preformed monomeric structure whose coulomb energy is highly sensitive to binding-induced water exclusion. such preformed hydrogen bonds are effectively adhesive, since water removal from their vicinity contributes to their stability. at the structural level, a significant correlation is established between dehydrons and sites for protein complexation, with the {hiv}-1 capsid protein p24 complexed with antibody light-chain {fab25}.3 providing the most dramatic correlation. furthermore, the number of dehydrons in homologous similar-fold proteins from different species is shown to be a signature of proteomic complexity. the techniques are then applied to higher levels of organization: the formation of the capsid and its organization in picornaviruses correlates strongly with the distribution of dehydrons on the rim of the virus unit. furthermore, antibody contacts and crystal contacts may be assigned to dehydrons still prevalent after the capsid has been assembled. the implications of the dehydron as an encoded signal in proteomics, bioinformatics, and inhibitor drug design are emphasized.\"\"although a few cancer genes are mutated in a high proportion of tumours of a given type (>20\\\\%), most are mutated at intermediate frequencies (2-20\\\\%). to explore the feasibility of creating a comprehensive catalogue of cancer genes, we analysed somatic point mutations in exome sequences from 4,742 human cancers and their matched normal-tissue samples across 21 cancer types. we found that large-scale genomic analysis can identify nearly all known cancer genes in these tumour types. our analysis also identified 33 genes that were not previously known to be significantly mutated in cancer, including genes related to proliferation, apoptosis, genome stability, chromatin regulation, immune evasion, {rna} processing and protein homeostasis. down-sampling analysis indicates that larger sample sizes will reveal many more genes mutated at clinically important frequencies. we estimate that near-saturation may be achieved with 600-5,000 samples per tumour type, depending on background mutation frequency. the results may help to guide the next stage of cancer genomics.\"\"we describe a statistical method for the characterization of genomic aberrations in single nucleotide polymorphism microarray data acquired from cancer genomes. our approach allows us to model the joint effect of polyploidy, normal {dna} contamination and intra-tumour heterogeneity within a single unified bayesian framework. we demonstrate the efficacy of our method on numerous datasets including laboratory generated mixtures of normal-cancer cell lines and real primary tumours.\"\"crosstalk and complexity within signaling pathways and their perturbation by oncogenes limit component-by-component approaches to understanding human disease. network analysis of how normal and oncogenic signaling can be rewired by drugs may provide opportunities to target tumors with high specificity and efficacy. using targeted inhibition of oncogenic signaling pathways, combined with {dna}-damaging chemotherapy, we report that time-staggered {egfr} inhibition, but not simultaneous coadministration, dramatically sensitizes a subset of triple-negative breast cancer cells to genotoxic drugs. systems-level analysis-using high-density time-dependent measurements of signaling networks, gene expression profiles, and cell phenotypic responses in combination with mathematical modeling-revealed an approach for altering the intrinsic state of the cell through dynamic rewiring of oncogenic signaling pathways. this process converts these cells to a less tumorigenic state that is more susceptible to {dna} damage-induced cell death by reactivation of an extrinsic apoptotic pathway whose function is suppressed in the oncogene-addicted state. copyright {\\\\copyright} 2012 elsevier inc. all rights reserved.\"emergence of {kras} mutations and acquired resistance to {anti-egfr} therapy in colorectal cancer.,\"a main limitation of therapies that selectively target kinase signalling pathways is the emergence of secondary drug resistance. cetuximab, a monoclonal antibody that binds the extracellular domain of epidermal growth factor receptor ({egfr}), is effective in a subset of {kras} wild-type metastatic colorectal cancers. after an initial response, secondary resistance invariably ensues, thereby limiting the clinical benefit of this drug. the molecular bases of secondary resistance to cetuximab in colorectal cancer are poorly understood. here we show that molecular alterations (in most instances point mutations) of {kras} are causally associated with the onset of acquired resistance to {anti-egfr} treatment in colorectal cancers. expression of mutant {kras} under the control of its endogenous gene promoter was sufficient to confer cetuximab resistance, but resistant cells remained sensitive to combinatorial inhibition of {egfr} and mitogen-activated protein-kinase kinase ({mek}). analysis of metastases from patients who developed resistance to cetuximab or panitumumab showed the emergence of {kras} amplification in one sample and acquisition of secondary {kras} mutations in 60\\\\% (6 out of 10) of the cases. {kras} mutant alleles were detectable in the blood of cetuximab-treated patients as early as 10 months before radiographic documentation of disease progression. in summary, the results identify {kras} mutations as frequent drivers of acquired resistance to cetuximab in colorectal cancers, indicate that the emergence of {kras} mutant clones can be detected non-invasively months before radiographic progression and suggest early initiation of a {mek} inhibitor as a rational strategy for delaying or reversing drug resistance.\"\"intra-tumour genetic heterogeneity is the result of ongoing evolutionary change within each cancer. the expansion of genetically distinct sub-clonal populations may explain the emergence of drug resistance, and if so, would have prognostic and predictive utility. however, methods for objectively quantifying tumour heterogeneity have been missing and are particularly difficult to establish in cancers where predominant copy number variation prevents accurate phylogenetic reconstruction owing to horizontal dependencies caused by long and cascading genomic rearrangements. to address these challenges, we present {medicc}, a method for phylogenetic reconstruction and heterogeneity quantification based on a minimum event distance for intra-tumour copy-number comparisons. using a transducer-based pairwise comparison function, we determine optimal phasing of major and minor alleles, as well as evolutionary distances between samples, and are able to reconstruct ancestral genomes. rigorous simulations and an extensive clinical study show the power of our method, which outperforms state-of-the-art competitors in reconstruction accuracy, and additionally allows unbiased numerical quantification of tumour heterogeneity. accurate quantification and evolutionary inference are essential to understand the functional consequences of tumour heterogeneity. the {medicc} algorithms are independent of the experimental techniques used and are applicable to both next-generation sequencing and array {cgh} data. cancer is a disease of random mutation and selection within the cellular genomes of an organism. as a result, when advanced disease is diagnosed, the cells comprising the tumour show a great amount of variability on the genomic level, a phenomenon termed intra-tumour genetic heterogeneity. heterogeneity is thought to be one of the main reasons why tumors become resistant to therapy, and thus hinders personalised medicine approaches. if we want to understand tumour heterogeneity and its connection to resistance development we need to quantify it, which implies reconstructing the evolutionary history of cancer within the patient. unfortunately, so far, methods for accurate reconstructions of these particular evolutionary trees and for quantification of heterogeneity have been missing. we here present {medicc}, a method that uses a minimum evolution criterion to compare cancer genomes based on genomic profiles of {dna} content (copy-number profiles). it enables accurate reconstruction of the history of the disease and quantifies heterogeneity. it is specifically designed to deal with diploid human genomes, in that it disentangles genomic events on both parental alleles and includes a variety of accompanying algorithms to test for shapes of the evolutionary trees as well as the rate at which the cancer evolves.\"boston, ma 02115, usa. yizong.cheng@uc.edu\",biclustering of expression data.,\"an efficient node-deletion algorithm is introduced to find submatrices in expression data that have low mean squared residue scores and it is shown to perform well in finding co-regulation patterns in yeast and human. this introduces \"\"biclustering\"\", or simultaneous clustering of both genes and conditions, to knowledge discovery from expression data. this approach overcomes some problems associated with traditional clustering methods, by allowing automatic discovery of similarity based on a subset of attributes, simultaneous clustering of genes and conditions, and overlapped grouping that provides a better representation for genes with multiple functions or regulated by many factors.\"new york, new york, usa\",theory and limitations of genetic network inference from microarray data.,\"since the advent of gene expression microarray technology more than 10 years ago, many computational approaches have been developed aimed at using statistical associations between {mrna} abundance profiles to predict transcriptional regulatory interactions. the ultimate goal is to develop causal network models describing the transcriptional influences that genes exert on each other (via their protein products), which can be used to predict network disruptions (e.g., mutations) leading to a disease phenotype, as well as the appropriate therapeutic intervention. however, microarray data measure only a small component of the interacting variables in a genetic regulatory network, as cells are known to regulate gene expression via many diverse mechanisms. although many researchers have acknowledged the questionable interpretation of statistical dependencies between {mrna} profiles, very little work has been done on theoretically characterizing the nature of inferred dependencies using models that account for unobserved interacting variables. in this work, we review the theory behind reverse engineering algorithms derived from three separate disciplines-system control theory, graphical models, and information theory-and highlight several mathematical relationships between the various methods. we then apply recent theoretical work on constructing graphical models with latent variables to the context of reverse engineering genetic networks. we demonstrate that even the addition of simple latent variables induces statistical dependencies between non-directly interacting (e.g., co-regulated) genes that cannot be eliminated by conditioning on any observed variables.\"\"high-throughput sequencing platforms are generating massive amounts of genetic variation data for diverse genomes, but it remains a challenge to pinpoint a small subset of functionally important variants. to fill these unmet needs, we developed the {annovar} tool to annotate single nucleotide variants ({snvs}) and insertions/deletions, such as examining their functional consequence on genes, inferring cytogenetic bands, reporting functional importance scores, finding variants in conserved regions, or identifying variants reported in the 1000 genomes project and {dbsnp}. {annovar} can utilize annotation databases from the {ucsc} genome browser or any annotation data set conforming to generic feature format version 3 ({gff3}). we also illustrate a \\'variants reduction\\' protocol on 4.7\\u2009million {snvs} and indels from a human genome, including two causal mutations for miller syndrome, a rare recessive disease. through a stepwise procedure, we excluded variants that are unlikely to be causal, and identified 20 candidate genes including the causal gene. using a desktop computer, {annovar} requires ∼4\\u2009min to perform gene-based annotation and ∼15\\u2009min to perform variants reduction on 4.7\\u2009million variants, making it practical to handle hundreds of human genomes in a day. {annovar} is freely available at http://www.openbioinformatics.org/annovar/.\"\"the characterization of protein interactions is essential for understanding biological systems. while genome-scale methods are available for identifying interacting proteins, they do not pinpoint the interacting motifs (e.g., a domain, sequence segments, a binding site, or a set of residues). here, we develop and apply a method for delineating the interacting motifs of hub proteins (i.e., highly connected proteins). the method relies on the observation that proteins with common interaction partners tend to interact with these partners through a common interacting motif. the sole input for the method are binary protein interactions; neither sequence nor structure information is needed. the approach is evaluated by comparing the inferred interacting motifs with domain families defined for 368 proteins in the structural classification of proteins ({scop}). the positive predictive value of the method for detecting proteins with common {scop} families is 75\\\\% at sensitivity of 10\\\\%. most of the inferred interacting motifs were significantly associated with sequence patterns, which could be responsible for the common interactions. we find that yeast hubs with multiple interacting motifs are more likely to be essential than hubs with one or two interacting motifs, thus rationalizing the previously observed correlation between essentiality and the number of interacting partners of a protein. we also find that yeast hubs with multiple interacting motifs evolve slower than the average protein, contrary to the hubs with one or two interacting motifs. the proposed method will help us discover unknown interacting motifs and provide biological insights about protein hubs and their roles in interaction networks.\"\"{background}: biclustering of gene expression data searches for local patterns of gene expression. a bicluster (or a two-way cluster) is defined as a set of genes whose expression profiles are mutually similar within a subset of experimental conditions/samples. although several biclustering algorithms have been studied, few are based on rigorous statistical models. {results}: we developed a bayesian biclustering model ({bbc}), and implemented a gibbs sampling procedure for its statistical inference. we showed that bayesian biclustering model can correctly identify multiple clusters of gene expression data. using simulated data both from the model and with realistic characters, we demonstrated the {bbc} algorithm outperforms other methods in both robustness and accuracy. we also showed that the model is stable for two normalization methods, the interquartile range normalization and the smallest quartile range normalization. applying the {bbc} algorithm to the yeast expression data, we observed that majority of the biclusters we found are supported by significant biological evidences, such as enrichments of gene functions and transcription factor binding sites in the corresponding promoter sequences. {conclusions}: the {bbc} algorithm is shown to be a robust model-based biclustering method that can discover biologically significant gene-condition clusters in microarray data. the {bbc} model can easily handle missing data via monte carlo imputation and has the potential to be extended to integrated study of gene transcription networks.\"\"{background}: the arrangement of regulatory motifs in gene promoters, or promoter architecture, is the result of mutation and selection processes that have operated over many millions of years. in mammals, tissue-specific transcriptional regulation is related to the presence of specific protein-interacting {dna} motifs in gene promoters. however, little is known about the relative location and spacing of these motifs. to fill this gap, we have performed a systematic search for motifs that show significant bias at specific promoter locations in a large collection of housekeeping and tissue-specific genes. {results}: we observe that promoters driving housekeeping gene expression are enriched in particular motifs with strong positional bias, such as {yy1}, which are of little relevance in promoters driving tissue-specific expression. we also identify a large number of motifs that show positional bias in genes expressed in a highly tissue-specific manner. they include well-known tissue-specific motifs, such as {hnf1} and {hnf4} motifs in liver, kidney and small intestine, or {rfx} motifs in testis, as well as many potentially novel regulatory motifs. based on this analysis, we provide predictions for 559 tissue-specific motifs in mouse gene promoters. {conclusion}: the study shows that motif positional bias is an important feature of mammalian proximal promoters and that it affects both general and tissue-specific motifs. motif positional constraints define very distinct promoter architectures depending on breadth of expression and type of tissue.\"\"understanding how genes are expressed and regulated in different tissues is a fundamental and challenging question. however, most of currently available biological databases do not focus on tissue-specific gene regulation.\"\"we propose a novel approach to predict domain-domain interactions from a protein-protein interaction network. in our method we apply a parsimony-driven explanation of the network, where the domain interactions are inferred using linear programming optimization, and false positives in the protein network are handled by a probabilistic construction. this method outperforms previous approaches by a considerable margin. the results indicate that the parsimony principle provides a correct approach for detecting domain-domain contacts.\"philadelphia, pennsylvania 19104, usa. hxzhou@einstein.drexel.edu\",prediction of protein interaction sites from sequence profile and residue neighbor list.,\"protein-protein interaction sites are predicted from a neural network with sequence profiles of neighboring residues and solvent exposure as input. the network was trained on 615 pairs of nonhomologous complex-forming proteins. tested on a different set of 129 pairs of nonhomologous complex-forming proteins, 70\\\\% of the 11,004 predicted interface residues are actually located in the interfaces. these 7732 correctly predicted residues account for 65\\\\% of the 11,805 residues making up the 129 interfaces. the main strength of the network predictor lies in the fact that neighbor lists and solvent exposure are relatively insensitive to structural changes accompanying complex formation. as such, it performs equally well with bound or unbound structures of the proteins. for a set of 35 test proteins, when the input was calculated from the bound and unbound structures, the correct fractions of the predicted interface residues were 69 and 70\\\\%, respectively.\"\"next-generation {dna} sequencing technologies are enabling genome-wide measurements of somatic mutations in large numbers of cancer patients. a major challenge in the interpretation of these data is to distinguish functional \"\"driver mutations\"\" important for cancer development from random \"\"passenger mutations.\"\" a common approach for identifying driver mutations is to find genes that are mutated at significant frequency in a large cohort of cancer genomes. this approach is confounded by the observation that driver mutations target multiple cellular signaling and regulatory pathways. thus, each cancer patient may exhibit a different combination of mutations that are sufficient to perturb these pathways. this mutational heterogeneity presents a problem for predicting driver mutations solely from their frequency of occurrence. we introduce two combinatorial properties, coverage and exclusivity, that distinguish driver pathways, or groups of genes containing driver mutations, from groups of genes with passenger mutations. we derive two algorithms, called dendrix, to find driver pathways de novo from somatic mutation data. we apply dendrix to analyze somatic mutation data from 623 genes in 188 lung adenocarcinoma patients, 601 genes in 84 glioblastoma patients, and 238 known mutations in 1000 patients with various cancers. in all data sets, we find groups of genes that are mutated in large subsets of patients and whose mutations are approximately exclusive. our dendrix algorithms scale to whole-genome analysis of thousands of patients and thus will prove useful for larger data sets to come from the cancer genome atlas ({tcga}) and other large-scale cancer genome sequencing projects.\"\"cancer evolves dynamically as clonal expansions supersede one another driven by shifting selective pressures, mutational processes, and disrupted cancer genes. these processes mark the genome, such that a cancer\\'s life history is encrypted in the somatic mutations present. we developed algorithms to decipher this narrative and applied them to 21 breast cancers. mutational processes evolve across a cancer\\'s lifespan, with many emerging late but contributing extensive genetic variation. subclonal diversification is prominent, and most mutations are found in just a fraction of tumor cells. every tumor has a dominant subclonal lineage, representing more than 50\\\\% of tumor cells. minimal expansion of these subclones occurs until many hundreds to thousands of mutations have accumulated, implying the existence of long-lived, quiescent cell lineages capable of substantial proliferation upon acquisition of enabling genomic changes. expansion of the dominant subclone to an appreciable mass may therefore represent the final rate-limiting step in a breast cancer\\'s development, triggering diagnosis. \\\\^{a}\\x96º genome-wide analyses of mutations emerging through time in 21 breast cancers \\\\^{a}\\x96º minimal expansion of subclones occurs until thousands of mutations have accumulated \\\\^{a}\\x96º cancer-specific signatures of point mutations and genomic instability emerge late \\\\^{a}\\x96º {erbb2} amplification begins early but continues to evolve over long molecular time\"\"spatial and temporal dissection of the genomic changes occurring during the evolution of human non–small cell lung cancer ({nsclc}) may help elucidate the basis for its dismal prognosis. we sequenced 25 spatially distinct regions from seven operable {nsclcs} and found evidence of branched evolution, with driver mutations arising before and after subclonal diversification. there was pronounced intratumor heterogeneity in copy number alterations, translocations, and mutations associated with {apobec} cytidine deaminase activity. despite maintained carcinogen exposure, tumors from smokers showed a relative decrease in smoking-related mutations over time, accompanied by an increase in {apobec}-associated mutations. in tumors from former smokers, genome-doubling occurred within a smoking-signature context before subclonal diversification, which suggested that a long period of tumor latency had preceded clinical detection. the regionally separated driver mutations, coupled with the relentless and heterogeneous nature of the genome instability processes, are likely to confound treatment success in {nsclc}.\"\"somatic alterations in cellular {dna} underlie almost all human cancers. the prospect of targeted therapies and the development of high-resolution, genome-wide approaches are now spurring systematic efforts to characterize cancer genomes. here we report a large-scale project to characterize copy-number alterations in primary lung adenocarcinomas. by analysis of a large collection of tumours (n = 371) using dense single nucleotide polymorphism arrays, we identify a total of 57 significantly recurrent events. we find that 26 of 39 autosomal chromosome arms show consistent large-scale copy-number gain or loss, of which only a handful have been linked to a specific gene. we also identify 31 recurrent focal events, including 24 amplifications and 7 homozygous deletions. only six of these focal events are currently associated with known mutations in lung carcinomas. the most common event, amplification of chromosome 14q13.3, is found in approximately 12\\\\% of samples. on the basis of genomic and functional analyses, we identify {nkx2}-1 ({nk2} homeobox 1, also called {titf1}), which lies in the minimal 14q13.3 amplification interval and encodes a lineage-specific transcription factor, as a novel candidate proto-oncogene involved in a significant fraction of lung adenocarcinomas. more generally, our results indicate that many of the genes that are involved in lung adenocarcinoma remain to be discovered.\"\"the elucidation of the human genome sequence has made it possible to identify genetic alterations in cancers in unprecedented detail. to begin a systematic analysis of such alterations, we determined the sequence of well-annotated human protein-coding genes in two common tumor types. analysis of 13,023 genes in 11 breast and 11 colorectal cancers revealed that individual tumors accumulate an average of ∼90 mutant genes but that only a subset of these contribute to the neoplastic process. using stringent criteria to delineate this subset, we identified 189 genes (average of 11 per tumor) that were mutated at significant frequency. the vast majority of these genes were not known to be genetically altered in tumors and are predicted to affect a wide range of cellular functions, including transcription, adhesion, and invasion. these data define the genetic landscape of two human cancer types, provide new targets for diagnostic and therapeutic intervention, and open fertile avenues for basic research in tumor biology.\"university park, pa 16802, usa. djmiller@engr.psu.edu\",emergent unsupervised clustering paradigms with potential application to bioinformatics.,\"in recent years, there has been a great upsurge in the application of data clustering, statistical classification, and related machine learning techniques to the field of molecular biology, in particular analysis of {dna} microarray expression data. clustering methods can be used to group co-expressed genes, shedding light on gene function and co-regulation. alternatively, they can group samples or conditions to identify phenotypical groups, disease subgroups, or to help identify disease pathways. a rich variety of unsupervised techniques have been applied, including partitional, hierarchical, graph-based, model-based, and biclustering methods. while a number of machine learning problems and tools have found mainstream applications in bioinformatics, in this article we identify some challenging problems which, though clearly relevant to bioinformatics, have not been extensively investigated in this domain. these include i) unsupervised clustering with unsupervised feature selection, ii) semisupervised learning, iii) unsupervised learning (and supervised learning) in the presence of confounding variables, and iv) stability of clustering solutions. we review recent methods which address these problems and take the position that these methods are well-suited to addressing some common scenarios that occur in bioinformatics.\"understanding the contribution of synonymous mutations to human disease,\"synonymous mutations - sometimes called \\'silent\\' mutations - are now widely acknowledged to be able to cause changes in protein expression, conformation and function. the recent increase in knowledge about the association of genetic variants with disease, particularly through genome-wide association studies, has revealed a substantial contribution of synonymous {snps} to human disease risk and other complex traits. here we review current understanding of the extent to which synonymous mutations influence disease, the various molecular mechanisms that underlie these effects and the implications for future research and biomedical applications.\"united kingdom. elevy@mrc-lmb.cam.ac.uk\",{3d} complex: a structural classification of protein complexes,\"most of the proteins in a cell assemble into complexes to carry out their function. it is therefore crucial to understand the physicochemical properties as well as the evolution of interactions between proteins. the protein data bank represents an important source of information for such studies, because more than half of the structures are homo- or heteromeric protein complexes. here we propose the first hierarchical classification of whole protein complexes of known {3-d} structure, based on representing their fundamental structural features as a graph. this classification provides the first overview of all the complexes in the protein data bank and allows nonredundant sets to be derived at different levels of detail. this reveals that between one-half and two-thirds of known structures are multimeric, depending on the level of redundancy accepted. we also analyse the structures in terms of the topological arrangement of their subunits and find that they form a small number of arrangements compared with all theoretically possible ones. this is because most complexes contain four subunits or less, and the large majority are homomeric. in addition, there is a strong tendency for symmetry in complexes, even for heteromeric complexes. finally, through comparison of biological units in the protein data bank with the protein quaternary structure database, we identified many possible errors in quaternary structure assignments. our classification, available as a database and web server at {http://www.3dcomplex}.org, will be a starting point for future work aimed at understanding the structure and evolution of protein complexes.\"\"a catalogue of molecular aberrations that cause ovarian cancer is critical for developing and deploying therapies that will improve patients\\' lives. the cancer genome atlas project has analysed messenger {rna} expression, {microrna} expression, promoter methylation and {dna} copy number in 489 high-grade serous ovarian adenocarcinomas and the {dna} sequences of exons from coding genes in 316 of these tumours. here we report that high-grade serous ovarian cancer is characterized by {tp53} mutations in almost all tumours (96\\\\%); low prevalence but statistically recurrent somatic mutations in nine further genes including {nf1}, {brca1}, {brca2}, {rb1} and {cdk12}; 113 significant focal {dna} copy number aberrations; and promoter methylation events involving 168 genes. analyses delineated four ovarian cancer transcriptional subtypes, three {microrna} subtypes, four promoter methylation subtypes and a transcriptional signature associated with survival duration, and shed new light on the impact that tumours with {brca1}/2 ({brca1} or {brca2}) and {ccne1} aberrations have on survival. pathway analyses suggested that homologous recombination is defective in about half of the tumours analysed, and that {notch} and {foxm1} signalling are involved in serous ovarian cancer pathophysiology.\"\"many cancers have substantial genomic heterogeneity within a given tumor, and to fully understand that diversity requires the ability to perform single cell analysis. we performed targeted sequencing of a panel of single nucleotide variants ({snvs}), deletions, and {igh} sequences in 1,479 single tumor cells from six acute lymphoblastic leukemia ({all}) patients. by accurately segregating groups of cooccurring mutations into distinct clonal populations, we identified codominant clones in the majority of patients. evaluation of intraclonal mutation patterns identified clone-specific punctuated cytosine mutagenesis events, showed that most structural variants are acquired before {snvs}, determined that {kras} mutations occur late in disease development but are not sufficient for clonal dominance, and identified clones within the same patient that are arrested at varied stages in b-cell development. taken together, these data order the sequence of genetic events that underlie childhood {all} and provide a framework for understanding the development of the disease at single-cell resolution.\"\"darwinian evolution favours genotypes with high replication rates, a process called \\'survival of the fittest\\'. however, knowing the replication rate of each individual genotype may not suffice to predict the eventual survivor, even in an asexual population. according to quasi-species theory, selection favours the cloud of genotypes, interconnected by mutation, whose average replication rate is highest1, 2, 3, 4, 5. here we confirm this prediction using digital organisms that self-replicate, mutate and evolve6, 7, 8, 9. forty pairs of populations were derived from 40 different ancestors in identical selective environments, except that one of each pair experienced a 4-fold higher mutation rate. in 12 cases, the dominant genotype that evolved at the lower mutation rate achieved a replication rate >1.5-fold faster than its counterpart. we allowed each of these disparate pairs to compete across a range of mutation rates. in each case, as mutation rate was increased, the\\xa0outcome of competition switched to favour the genotype with the lower replication rate. these genotypes, although they occupied lower fitness peaks, were located in flatter regions of the fitness surface and were therefore more robust with respect to mutations.\"\"advances in structural genomics and protein structure prediction require the design of automatic, fast, objective, and well benchmarked methods capable of comparing and assessing the similarity of low-resolution three-dimensional structures, via experimental or theoretical approaches. here, a new method for sequence-independent structural alignment is presented that allows comparison of an experimental protein structure with an arbitrary low-resolution protein tertiary model. the heuristic algorithm is given and then used to show that it can describe random structural alignments of proteins with different folds with good accuracy by an extreme value distribution. from this observation, a structural similarity score between two proteins or two different conformations of the same protein is derived from the likelihood of obtaining a given structural alignment by chance. the performance of the derived score is then compared with well established, consensus manual-based scores and data sets. we found that the new approach correlates better than other tools with the gold standard provided by a human evaluator. timings indicate that the algorithm is fast enough for routine use with large databases of protein models. overall, our results indicate that the new program ({mammoth}) will be a good tool for protein structure comparisons in structural genomics applications. {mammoth} is available from our web site at http://physbio.mssm.edu/\\\\~{}ortizg/.\"\"experiment and simulation are now conspiring to give atomic-level descriptions of protein folding relevant to folding, misfolding, trafficking, and degradation in the cell. we are on the threshold of predicting those protein folding events using simulation that has been carefully benchmarked by experiment.\"lyngby, denmark. gorm@cbs.dtu.dk\",the biology of eukaryotic promoter prediction--a review.,\"computational prediction of eukaryotic promoters from the nucleotide sequence is one of the most attractive problems in sequence analysis today, but it is also a very difficult one. thus, current methods predict in the order of one promoter per kilobase in human {dna}, while the average distance between functional promoters has been estimated to be in the range of 30-40 kilobases. although it is conceivable that some of these predicted promoters correspond to cryptic initiation sites that are used in vivo, it is likely that most are false positives. this suggests that it is important to carefully reconsider the biological data that forms the basis of current algorithms, and we here present a review of data that may be useful in this regard. the review covers the following topics: (1) basal transcription and core promoters, (2) activated transcription and transcription factor binding sites, (3) {cpg} islands and {dna} methylation, (4) chromosomal structure and nucleosome modification, and (5) chromosomal domains and domain boundaries. we discuss the possible lessons that may be learned, especially with respect to the wealth of information about epigenetic regulation of transcription that has been appearing in recent years.\"\"aligning and comparing genomic sequences enables the identification of conserved sequence signatures and can enrich for coding and noncoding functional regions. in vertebrates, the comparison of human and rodent genomes and the comparison of evolutionarily distant genomes, such as human and pufferfish, have identified specific sets of \\'ultraconserved\\' sequence elements associated with the control of early development. however, is this just the tip of a \\'conservation iceberg\\' or do these sequences represent a specific class of regulatory element? studies on the zebrafish phox2b gene region and the {encode} project suggest that many regulatory elements are not highly conserved, posing intriguing questions about the relationship between noncoding sequence conservation and function and the evolution of regulatory sequences.\"\"plasma of cancer patients contains cell-free tumor {dna} that carries information on tumor mutations and tumor burden. individual mutations have been probed using allele-specific assays, but sequencing of entire genes to detect cancer mutations in circulating {dna} has not been demonstrated. we developed a method for tagged-amplicon deep sequencing ({tam}-seq) and screened 5995 genomic bases for low-frequency mutations. using this method, we identified cancer mutations present in circulating {dna} at allele frequencies as low as 2\\\\%, with sensitivity and specificity of >97\\\\%. we identified mutations throughout the tumor suppressor gene {tp53} in circulating {dna} from 46 plasma samples of advanced ovarian cancer patients. we demonstrated use of {tam}-seq to noninvasively identify the origin of metastatic relapse in a patient with multiple primary tumors. in another case, we identified in plasma an {egfr} mutation not found in an initial ovarian biopsy. we further used {tam}-seq to monitor tumor dynamics, and tracked 10 concomitant mutations in plasma of a metastatic breast cancer patient over 16 months. this low-cost, high-throughput method could facilitate analysis of circulating {dna} as a noninvasive  ” liquid biopsy” for personalized cancer genomics.\"\"structural bioinformatics, biotec tu dresden, tatzberg 47-51, 01307 dresden, germany\",characterization of interfacial solvent in protein complexes and contribution of <i>wet {spots</i}> to the interface description,\"water networks in protein interfaces can complement direct interactions contributing significantly to molecular recognition, function, and stability of protein association. thus, water can be seen as an extension or addition of protein structural features, which may add plenty of information to protein interfacial definition. however, solvent is frequently neglected in protein interaction studies. analysis of the interfacial information contained in the {pdb} is essential to achieve more accurate descriptions of protein interfaces. with this aim, we have used the {scowlp} database () and applied computational geometry methods to extract and analyze interfacial information of a high-resolution nonredundant dataset of 176 protein complexes containing obligate and transient interfaces. we have identified all interfacial residues and characterized them in terms of temperature factors, secondary structure, residue composition, and pairing preferences to understand their contribution to the interface description. we have paid special attention to water-bridged residues; focusing on those that interact only mediated by a water molecule called wet spots. our results show that 40.1\\\\% of the interfacial residues are interacting through water and that wet spots represent a 14.5\\\\% of the total, emphasizing the importance of the inclusion of solvent in protein interaction studies, and the contribution of wet spots to interfacial description. wet spots present similar characteristics to residues binding buried water molecules in the core or cavities of proteins; being preferably located in nonregular secondary structures and establishing hydrogen bonds by their main-chains. we observe that obligate and transient interfaces present a comparable amount of solvent. moreover, the role of solvent in both complex types differs according to the different nature of their interfaces. the information obtained in our studies will assist in the process of accomplishing more accurate descriptions of protein interfaces and may be helpful to improve comparison of protein family interfaces, to facilitate rational ligand design, and to guide protein docking. proteins 2007. {\\\\copyright} 2007 {wiley-liss}, inc.\"\"the identification of protein–protein interaction sites is an essential intermediate step for mutant design and the prediction of protein networks. in recent years a significant number of methods have been developed to predict these interface residues and here we review the current status of the field. progress in this area requires a clear view of the methodology applied, the data sets used for training and testing the systems, and the evaluation procedures. we have analysed the impact of a representative set of features and algorithms and highlighted the problems inherent in generating reliable protein data sets and in the posterior analysis of the results. although it is clear that there have been some improvements in methods for predicting interacting sites, several major bottlenecks remain. proteins in complexes are still under-represented in the structural databases and in particular many proteins involved in transient complexes are still to be crystallized. we provide suggestions for effective feature selection, and make it clear that community standards for testing, training and performance measures are necessary for progress in the field.\"p-1/12 cit scheme viim, calcutta 700 054, india.\",conservation and relative importance of residues across protein-protein interfaces,\"a core region surrounded by a rim characterizes biological interfaces. we ascertain the importance of the core by showing the sequence entropies of the residues comprising the core to be smaller than those in the rim. such a distinction is not seen in the 2-fold-related, nonphysiological interfaces formed in crystal lattices of monomeric proteins, thereby providing a procedure for characterizing the oligomeric state from crystal structures of protein molecules. this method is better than those that rely on the comparison of the sequence entropies in the interface and the rest of the protein surface, especially in cases where the surface harbors additional binding sites. to a good approximation there is a correlation between the accessible surface area lost because of complexation and {δδg} values obtained through alanine-scanning mutagenesis (26-38 cal per \\\\aa{}2 of the surface buried) for residues located in the core, a relationship that is not discernable for rim residues. if, however, a residue participates in hydrogen bonding across the interface, the extent of stabilization is 52 cal/mol per 1 \\\\aa{}2 of the nonpolar surface area buried by the residue. as opposed to an amino acid classification used earlier, an environment-based grouping of residues yields a better discrimination in the sequence entropy between the core and the rim.\"co-adaptation and interactions.\",\"co-evolution has an important function in the evolution of species and it is clearly manifested in certain scenarios such as host-parasite and predator-prey interactions, symbiosis and mutualism. the extrapolation of the concepts and methodologies developed for the study of species co-evolution at the molecular level has prompted the development of a variety of computational methods able to predict protein interactions through the characteristics of co-evolution. particularly successful have been those methods that predict interactions at the genomic level based on the detection of pairs of protein families with similar evolutionary histories (similarity of phylogenetic trees: mirrortree). future advances in this field will require a better understanding of the molecular basis of the co-evolution of protein families. thus, it will be important to decipher the molecular mechanisms underlying the similarity observed in phylogenetic trees of interacting proteins, distinguishing direct specific molecular interactions from other general functional constraints. in particular, it will be important to separate the effects of physical interactions within protein complexes (\\'co-adaptation\\') from other forces that, in a less specific way, can also create general patterns of co-evolution.\"\"humans differ from other primates by marked differences in cognitive abilities and a significantly larger brain. these differences correlate with metabolic changes, as evidenced by the relative up-regulation of energy-related genes and metabolites in human brain. while the mechanisms underlying these evolutionary changes have not been elucidated, altered activities of key transcription factors ({tfs}) could play a pivotal role. to assess this possibility, we analyzed microarray data from five tissues from humans and chimpanzees. we identified 90 {tf} genes with significantly different expression levels in human and chimpanzee brain among which the rapidly evolving {krab}-zinc finger genes are markedly over-represented. the differentially expressed {tfs} cluster within a robust regulatory network consisting of two distinct but interlinked modules, one strongly associated with energy metabolism functions, and the other with transcription, vesicular transport, and ubiquitination. our results suggest that concerted changes in a relatively small number of interacting {tfs} may coordinate major gene expression differences in human and chimpanzee brain.\"\"the recognition of transcription factor binding sites ({tfbss}) is the first step on the way to deciphering the {dna} regulatory code. there is a large variety of experimental approaches providing information on {tfbs} location in genomic sequences. many computational approaches to {tfbs} recognition based on the experimental data obtained are available, each having its own advantages and shortcomings. this article provides short review of approaches to computational recognition of {tfbs} in genomic sequences and methods of experimental verification of predicted sites. we also present a case study of the interplay between experimental and theoretical approaches to the successful prediction of steroidogenic factor 1 ({sf1}).\"\"breast cancer is the most common cancer and the leading cause of cancer-related death in women worldwide.1 metastatic breast cancer remains an incurable disease but is treatable by means of serial administration of endocrine, cytotoxic, or biologic therapies. the monitoring of treatment response is essential to avoid continuing ineffective therapies, to prevent unnecessary side effects, and to determine the benefit of new therapeutics. treatment response is generally assessed with the use of serial imaging, but radiographic measurements often fail to detect changes in tumor burden. therefore, there is an urgent need for biomarkers that measure tumor burden with high sensitivity .\\xa0.\\xa0.\"genomic architecture and evolution of clear cell renal cell carcinomas defined by multiregion sequencing,\"clear cell renal carcinomas ({ccrccs}) can display intratumor heterogeneity ({ith}). we applied multiregion exome sequencing (m-seq) to resolve the genetic architecture and evolutionary histories of ten {ccrccs}. ultra-deep sequencing identified {ith} in all cases. we found that 73-75\\\\% of identified {ccrcc} driver aberrations were subclonal, confounding estimates of driver mutation prevalence. {ith} increased with the number of biopsies analyzed, without evidence of saturation in most tumors. chromosome 3p loss and {vhl} aberrations were the only ubiquitous events. the proportion of {c>t} transitions at {cpg} sites increased during tumor progression. m-seq permits the temporal resolution of {ccrcc} evolution and refines mutational signatures occurring during tumor development.\"aneuploidy drives genomic instability in yeast.,\"aneuploidy decreases cellular fitness, yet it is also associated with cancer, a disease of enhanced proliferative capacity. to investigate one mechanism by which aneuploidy could contribute to tumorigenesis, we examined the effects of aneuploidy on genomic stability. we analyzed 13 budding yeast strains that carry extra copies of single chromosomes and found that all aneuploid strains exhibited one or more forms of genomic instability. most strains displayed increased chromosome loss and mitotic recombination, as well as defective {dna} damage repair. aneuploid fission yeast strains also exhibited defects in mitotic recombination. aneuploidy-induced genomic instability could facilitate the development of genetic alterations that drive malignant growth in cancer.\"\"proteins that can interact with multiple partners play central roles in the network of protein–protein interactions. they are called hub proteins, and recently it was suggested that an abundance of intrinsically disordered regions on their surfaces facilitates their binding to multiple partners. however, in those studies, the hub proteins were identified as proteins with multiple partners, regardless of whether the interactions were transient or permanent. as a result, a certain number of hub proteins are subunits of stable multi-subunit proteins, such as supramolecules. it is well known that stable complexes and transient complexes have different structural features, and thus the statistics based on the current definition of hub proteins will hide the true nature of hub proteins. therefore, in this paper, we first describe a new approach to identify proteins with multiple partners dynamically, using the protein data bank, and then we performed statistical analyses of the structural features of these proteins. we refer to the proteins as transient hub proteins or sociable proteins, to clarify the difference with hub proteins. as a result, we found that the main difference between sociable and nonsociable proteins is not the abundance of disordered regions, in contrast to the previous studies, but rather the structural flexibility of the entire protein. we also found greater predominance of charged and polar residues in sociable proteins than previously reported.\"\"domains are the building blocks of proteins and play a crucial role in protein\\\\&\\\\#8211;protein interactions. here, we propose a new approach for the analysis and prediction of domain\\\\&\\\\#8211;domain interfaces. our method, which relies on the representation of domains as residue-interacting networks, finds an optimal decomposition of domain structures into modules. the resulting modules comprise highly cooperative residues, which exhibit few connections with other modules. we found that non-overlapping binding sites in a domain, involved in different domain\\\\&\\\\#8211;domain interactions, are generally contained in different modules. this observation indicates that our modular decomposition is able to separate protein domains into regions with specialized functions. our results show that modules with high modularity values identify binding site regions, demonstrating the predictive character of modularity. furthermore, the combination of modularity with other characteristics, such as sequence conservation or surface patches, was found to improve our predictions. in an attempt to give a physical interpretation to the modular architecture of domains, we analyzed in detail six examples of protein domains with available experimental binding data. the modular configuration of the {tem1}-\\\\&\\\\#946;-lactamase binding site illustrates the energetic independence of hotspots located in different modules and the cooperativity of those sited within the same modules. the energetic and structural cooperativity between intramodular residues is also clearly shown in the example of the chymotrypsin inhibitor, where non\\\\&\\\\#8211;binding site residues have a synergistic effect on binding. interestingly, the binding site of the t cell receptor \\\\&\\\\#946; chain variable domain 2.1 is contained in one module, which includes structurally distant hot regions displaying positive cooperativity. these findings support the idea that modules possess certain functional and energetic independence. a modular organization of binding sites confers robustness and flexibility to the performance of the functional activity, and facilitates the evolution of protein interactions.\"\"division of biomedical informatics, children\\'s hospital research foundation, cincinnati, ohio 45229; department of informatics, nicholas copernicus university, 87-100 toru\\\\&nacute;, poland\",prediction-based fingerprints of protein-protein interactions,\"the recognition of protein interaction sites is an important intermediate step toward identification of functionally relevant residues and understanding protein function, facilitating experimental efforts in that regard. toward that goal, the authors propose a novel representation for the recognition of protein-protein interaction sites that integrates enhanced relative solvent accessibility ({rsa}) predictions with high resolution structural data. an observation that {rsa} predictions are biased toward the level of surface exposure consistent with protein complexes led the authors to investigate the difference between the predicted and actual (i.e., observed in an unbound structure) {rsa} of an amino acid residue as a fingerprint of interaction sites. the authors demonstrate that {rsa} prediction-based fingerprints of protein interactions significantly improve the discrimination between interacting and noninteracting sites, compared with evolutionary conservation, physicochemical characteristics, structure-derived and other features considered before. on the basis of these observations, the authors developed a new method for the prediction of protein-protein interaction sites, using machine learning approaches to combine the most informative features into the final predictor. for training and validation, the authors used several large sets of protein complexes and derived from them nonredundant representative chains, with interaction sites mapped from multiple complexes. alternative machine learning techniques are used, including support vector machines and neural networks, so as to evaluate the relative effects of the choice of a representation and a specific learning algorithm. the effects of induced fit and uncertainty of the negative (noninteracting) class assignment are also evaluated. several representative methods from the literature are reimplemented to enable direct comparison of the results. using rigorous validation protocols, the authors estimated that the new method yields the overall classification accuracy of about 74\\\\% and matthews correlation coefficients of 0.42, as opposed to up to 70\\\\% classification accuracy and up to 0.3 matthews correlation coefficient for methods that do not utilize {rsa} prediction-based fingerprints. the new method is available at . proteins 2007. {\\\\copyright} 2006 {wiley-liss}, inc.\"\"clonal evolution is a key feature of cancer progression and relapse. we studied intratumoral heterogeneity in 149 chronic lymphocytic leukemia ({cll}) cases by integrating whole-exome sequence and copy number to measure the fraction of cancer cells harboring each somatic mutation. we identified driver mutations as predominantly clonal (e.g., {myd88}, trisomy 12, and del(13q)) or subclonal (e.g., {sf3b1} and {tp53}), corresponding to earlier and later events in {cll} evolution. we sampled leukemia cells from 18 patients at two time points. ten of twelve {cll} cases treated with chemotherapy (but only one of six without treatment) underwent clonal evolution, predominantly involving subclones with driver mutations (e.g., {sf3b1} and {tp53}) that expanded over time. furthermore, presence of a subclonal driver mutation was an independent risk factor for rapid disease progression. our study thus uncovers patterns of clonal evolution in {cll}, providing insights into its stepwise transformation, and links the presence of subclones with adverse clinical outcomes. copyright {\\\\copyright} 2013 elsevier inc. all rights reserved.\"\"aneuploidy has been recognized as a hallmark of cancer for more than 100 years, yet no general theory to explain the recurring patterns of aneuploidy in cancer has emerged. here, we develop tumor suppressor and oncogene ({tuson}) explorer, a computational method that analyzes the patterns of mutational signatures in tumors and predicts the likelihood that any individual gene functions as a tumor suppressor ({tsg}) or oncogene ({og}). by analyzing >8,200 tumor-normal pairs, we provide statistical evidence suggesting that many more genes possess cancer driver properties than anticipated, forming a continuum of oncogenic potential. integrating our driver predictions with information on somatic copy number alterations, we find that the distribution and potency of {tsgs} ({stop} genes), {ogs}, and essential genes ({go} genes) on chromosomes can predict the complex patterns of aneuploidy and copy number variation characteristic of cancer genomes. we propose that the cancer genome is shaped through a process of cumulative haploinsufficiency and triplosensitivity. copyright {\\\\copyright} 2013 elsevier inc. all rights reserved.\"iowa city, ia 52242-1109, usa. adrain-elcock@uiowa.edu\",identification of protein oligomerization states by analysis of interface conservation,\"the discrimination of true oligomeric protein–protein contacts from nonspecific crystal contacts remains problematic. criteria that have been used previously base the assignment of oligomeric state on consideration of the area of the interface and/or the results of scoring functions based on statistical potentials. both techniques have a high success rate but fail in more than 10\\\\% of cases. more importantly, the oligomeric states of several proteins are incorrectly assigned by both methods. here we test the hypothesis that true oligomeric contacts should be identifiable on the basis of an increased degree of conservation of the residues involved in the interface. by quantifying the degree of conservation of the interface and comparing it with that of the remainder of the protein surface, we develop a new criterion that provides a highly effective complement to existing methods.\"national institutes of health, bethesda, md 20894, usa. altschul@ncbi.nlm.nih.gov\",gapped {blast} and {psi}-{blast}: a new generation of protein database search programs.,\"the {blast} programs are widely used tools for searching protein and {dna} databases for sequence similarities. for protein comparisons, a variety of definitional, algorithmic and statistical refinements described here permits the execution time of the {blast} programs to be decreased substantially while enhancing their sensitivity to weak similarities. a new criterion for triggering the extension of word hits, combined with a new heuristic for generating gapped alignments, yields a gapped {blast} program that runs at approximately three times the speed of the original. in addition, a method is introduced for automatically combining statistically significant alignments produced by {blast} into a position-specific score matrix, and searching the database using this matrix. the resulting {position-specific} iterated {blast} ({psi}-{blast}) program runs at approximately the same speed per iteration as gapped {blast}, but in many cases is much more sensitive to weak but biologically relevant sequence similarities. {psi}-{blast} is used to uncover several new and interesting members of the {brct} superfamily.\"california, usa. cherry@stanford.edu\",gene ontology: tool for the unification of biology. the gene ontology consortium.,\"genomic sequencing has made it clear that a large fraction of the genes specifying the core biological functions are shared by all eukaryotes. knowledge of the biological role of such shared proteins in one organism can often be transferred to other organisms. the goal of the gene ontology consortium is to produce a dynamic, controlled vocabulary that can be applied to all eukaryotes even as knowledge of gene and protein roles in cells is accumulating and changing. to this end, three independent ontologies accessible on the {world-wide} web (http://www.geneontology.org) are being constructed: biological process, molecular function and cellular component.\"the university of manchester, michael smith building, oxford road, manchester m13 9pt, uk.\",protein interactions from complexes: a structural perspective.,san diego, ca 92121, usa.\",a gene atlas of the mouse and human protein-encoding transcriptomes,\"the tissue-specific pattern of {mrna} expression can indicate important clues about gene function. high-density oligonucleotide arrays offer the opportunity to examine patterns of gene expression on a genome scale. toward this end, we have designed custom arrays that interrogate the expression of the vast majority of protein-encoding human and mouse genes and have used them to profile a panel of 79 human and 61 mouse tissues. the resulting data set provides the expression patterns for thousands of predicted genes, as well as known and poorly characterized genes, from mice and humans. we have explored this data set for global trends in gene expression, evaluated commonly used lines of evidence in gene prediction methodologies, and investigated patterns indicative of chromosomal organization of transcription. we describe hundreds of regions of correlated transcription and show that some are subject to both tissue and parental allele-specific expression, suggesting a link between spatial expression and imprinting.\"\"exhaustive methods of sequence alignment are accurate but slow, whereas heuristic approaches run quickly, but their complexity makes them more difficult to implement. we introduce bounded sparse dynamic programming ({bsdp}) to allow rapid approximation to exhaustive alignment. this is used within a framework whereby the alignment algorithms are described in terms of their underlying model, to allow automated development of efficient heuristic implementations which may be applied to a general set of sequence comparison problems. the speed and accuracy of this approach compares favourably with existing methods. examples of its use in the context of genome annotation are given. this system allows rapid implementation of heuristics approximating to many complex alignment models, and has been incorporated into the freely available sequence alignment program, exonerate.\"and college of engineering, rumelifeneri yolu, 34450 sariyer istanbul, turkey. okeskin@ku.edu.tr\",hot regions in protein--protein interactions: the organization and contribution of structurally conserved hot spot residues.,seattle 98195, usa.\",assembly of protein tertiary structures from fragments with similar local sequences using simulated annealing and bayesian scoring functions,\"we explore the ability of a simple simulated annealing procedure to assemble native-like structures from fragments of unrelated protein structures with similar local sequences using bayesian scoring functions. environment and residue pair specific contributions to the scoring functions appear as the first two terms in a series expansion for the residue probability distributions in the protein database; the decoupling of the distance and environment dependencies of the distributions resolves the major problems with current database-derived scoring functions noted by thomas and dill. the simulated annealing procedure rapidly and frequently generates native-like structures for small helical proteins and better than random structures for small \\\\^{i}² sheet containing proteins. most of the simulated structures have native-like solvent accessibility and secondary structure patterns, and thus ensembles of these structures provide a particularly challenging set of decoys for evaluating scoring functions. we investigate the effects of multiple sequence information and different types of conformational constraints on the overall performance of the method, and the ability of a variety of recently developed scoring functions to recognize the native-like conformations in the ensembles of simulated structures.\"london, u.k.\",protein-protein interactions: a review of protein dimer structures.,rehovot 76100 israel.\",{promate}: a structure based prediction program to identify the location of protein-protein binding sites.,se-10691 stockholm, sweden\",\"{gromacs} 4:\\u2009 algorithms for highly efficient, {load-balanced}, and scalable molecular simulation\",\"molecular simulation is an extremely useful, but computationally very expensive tool for studies of chemical and biomolecular systems. here, we present a new implementation of our molecular simulation toolkit {gromacs} which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines. the code encompasses a minimal-communication domain decomposition algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel. to improve the scaling properties of the common particle mesh ewald electrostatics algorithms, we have in addition used a {multiple-program}, {multiple-data} approach, with separate node domains responsible for direct and reciprocal space interactions. not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest numbers of standard cluster nodes.\"000 human genes?\",10.1126/science.1058969\"vertebrate {cpg} islands ({cgis}) are short interspersed {dna} sequences that deviate significantly from the average genomic pattern by being {gc}-rich, {cpg}-rich, and predominantly nonmethylated. most, perhaps all, {cgis} are sites of transcription initiation, including thousands that are remote from currently annotated promoters. shared {dna} sequence features adapt {cgis} for promoter function by destabilizing nucleosomes and attracting proteins that create a transcriptionally permissive chromatin state. silencing of {cgi} promoters is achieved through dense {cpg} methylation or polycomb recruitment, again using their distinctive {dna} sequence composition. {cgis} are therefore generically equipped to influence local chromatin structure and simplify regulation of gene activity.\"replication stress links structural and numerical cancer chromosomal instability,\"essentially all the biological functions of {dna} depend on site‐specific {dna}‐binding proteins finding their targets, and therefore \\'searching\\' through megabases of non‐target {dna}. in this article, we review current understanding of how this sequence searching is done. we review how simple diffusion through solution may be unable to account for the rapid rates of association observed in experiments on some model systems, primarily the lac repressor. we then present a simplified version of the \\'facilitated diffusion\\' model of berg, winter and von hippel, showing how non‐specific {dna}–protein interactions may account for accelerated targeting, by permitting the protein to sample many binding sites per {dna} encounter. we discuss the 1‐dimensional \\'sliding\\' motion of protein along non‐specific {dna}, often proposed to be the mechanism of this multiple site sampling, and we discuss the role of short‐range diffusive \\'hopping\\' motions. we then derive the optimal range of sliding for a few physical situations, including simple models of chromosomes in vivo, showing that a sliding range of ∼100\\xa0bp before dissociation optimizes targeting in vivo. going beyond first‐order binding kinetics, we discuss how processivity, the interaction of a protein with two or more targets on the same {dna}, can reveal the extent of sliding and we review recent experiments studying processivity using the restriction enzyme {ecorv}. finally, we discuss how single molecule techniques might be used to study the dynamics of {dna} site‐specific targeting of proteins.\"a comprehensive transcriptional portrait of human cancer cell lines,hinxton, cambridge cb10 1sd, uk.\",inference of macromolecular assemblies from crystalline state.,\"we discuss basic physical-chemical principles underlying the formation of stable macromolecular complexes, which in many cases are likely to be the biological units performing a certain physiological function. we also consider available theoretical approaches to the calculation of macromolecular affinity and entropy of complexation. the latter is shown to play an important role and make a major effect on complex size and symmetry. we develop a new method, based on chemical thermodynamics, for automatic detection of macromolecular assemblies in the protein data bank ({pdb}) entries that are the results of x-ray diffraction experiments. as found, biological units may be recovered at 80-90\\\\% success rate, which makes x-ray crystallography an important source of experimental data on macromolecular complexes and protein-protein interactions. the method is implemented as a public {www} service.\"repitools: an r package for the analysis of enrichment-based epigenomic data.,,estimates of worldwide burden of cancer in 2008: {globocan} 2008,\"estimates of the worldwide incidence and mortality from 27 cancers in 2008 have been prepared for 182 countries as part of the {globocan} series published by the international agency for research on cancer. in this article, we present the results for 20 world regions, summarizing the global patterns for the eight most common cancers. overall, an estimated 12.7 million new cancer cases and 7.6 million cancer deaths occur in 2008, with 56\\\\% of new cancer cases and 63\\\\% of the cancer deaths occurring in the less developed regions of the world. the most commonly diagnosed cancers worldwide are lung (1.61 million, 12.7\\\\% of the total), breast (1.38 million, 10.9\\\\%) and colorectal cancers (1.23 million, 9.7\\\\%). the most common causes of cancer death are lung cancer (1.38 million, 18.2\\\\% of the total), stomach cancer (738,000 deaths, 9.7\\\\%) and liver cancer (696,000 deaths, 9.2\\\\%). cancer is neither rare anywhere in the world, nor mainly confined to high-resource countries. striking differences in the patterns of cancer from region to region are observed.\"intra-tumour heterogeneity: a looking glass for cancer?,\"cancers arise owing to mutations in a subset of genes that confer growth advantage. the availability of the human genome sequence led us to propose that systematic resequencing of cancer genomes for mutations would lead to the discovery of many additional cancer genes. here we report more than 1,000 somatic mutations found in 274\\u2009megabases (mb) of {dna} corresponding to the coding exons of 518 protein kinase genes in 210 diverse human cancers. there was substantial variation in the number and pattern of mutations in individual cancers reflecting different exposures, {dna} repair defects and cellular origins. most somatic mutations are likely to be \\'passengers\\' that do not contribute to oncogenesis. however, there was evidence for \\'driver\\' mutations contributing to the development of the cancers studied in approximately 120 genes. systematic sequencing of cancer genomes therefore reveals the evolutionary diversity of cancers and implicates a larger repertoire of cancer genes than previously anticipated.\"\"{formoleculardynamicssimulationsofhydratedproteinsasimpleyetreliablemodelfortheintermolecularpotentialforwaterisrequired.suchamodelmustbeaneffectivepairpotentialvalidforliquiddensitiesthattakesaveragemany}—{bodyinteractionsintoaccount.wehavedevelopedathree}—{pointchargemodel(onhydrogenandoxygenpositions)withalennard}—{jones6—12potentialontheoxygenpositionsonly.parametersforthemodelweredeterminedfrom12moleculardynamicsrunscoveringthetwo}—{dimensionalparameterspaceofchargeandoxygenrepulsion.bothpotentialenergyandpressurewererequiredtocoincidewithexperimentalvalues}.{themodelhasverysatisfactoryproperties,iseasilyincorporatedintoprotein—waterpotentials,andrequiresonly0.25seccomputertimeperdynamicsstep(for216molecules)onacray}—1computer.\"\"author {summarya} historically difficult problem in computational biology is the identification of transcription factor binding sites ({tfbs}) in the promoters of co-regulated genes. with increasing emphasis on research in transcriptional regulation, this problem is also uniquely relevant to emerging results from recent experiments in high-throughput and systems biology. despite extensive research in the area, recent evaluations of previously published techniques show much room for improvement. in this paper, we introduce a fundamentally new approach to the identification of {tfbs}. first, we start by representing nucleotides in promoters as an undirected, weighted graph. given this representation of a binding site graph ({bsg}), we employ relatively simple graph clustering techniques to identify functional {tfbs}. we show that {bsg} predictions significantly outperform all previously evaluated methods in nearly every performance measure using a standardized assessment benchmark. we also find that this approach is more robust than traditional gibbs sampling to selection of input promoters, and thus more likely to perform well under noisy experimental conditions. finally, {bsgs} are very good at predicting specificity determining nucleotides. using {bsg} predictions, we were able to confirm recent experimental results on binding specificity of e-box {tfs} {cbf1} and {pho4} and predict novel specificity determining nucleotides for {tye7}.\"\"while genetic mutation is a hallmark of cancer, many cancers also acquire epigenetic alterations during tumorigenesis including aberrant {dna} hypermethylation of tumor suppressors, as well as changes in chromatin modifications as caused by genetic mutations of the chromatin-modifying machinery. however, the extent of epigenetic alterations in cancer cells has not been fully characterized. here, we describe complete methylome maps at single nucleotide resolution of a low-passage breast cancer cell line and primary human mammary epithelial cells. we find widespread {dna} hypomethylation in the cancer cell, primarily at partially methylated domains ({pmds}) in normal breast cells. unexpectedly, genes within these regions are largely silenced in cancer cells. the loss of {dna} methylation in these regions is accompanied by formation of repressive chromatin, with a significant fraction displaying allelic {dna} methylation where one allele is {dna} methylated while the other allele is occupied by histone modifications {h3k9me3} or {h3k27me3}. our results show a mutually exclusive relationship between {dna} methylation and {h3k9me3} or {h3k27me3}. these results suggest that global {dna} hypomethylation in breast cancer is tightly linked to the formation of repressive chromatin domains and gene silencing, thus identifying a potential epigenetic pathway for gene regulation in cancer cells.\"\"regulation of gene expression at the transcriptional level is a fundamental mechanism that is well conserved in all cellular systems. due to advances in large-scale experimental analyses, we now have a wealth of information on gene regulation such as {mrna} expression level across multiple conditions, genome-wide location data of transcription factors and data on transcription factor binding sites. this knowledge can be used to reconstruct transcriptional regulatory networks. such networks are usually represented as directed graphs where regulatory interactions are depicted as directed edges from the transcription factor nodes to the target gene nodes. this abstract representation allows us to apply graph theory to study transcriptional regulation at global and local levels, to predict regulatory motifs and regulatory modules such as regulons and to compare the regulatory network of different genomes. here we review some of the available computational methodologies for studying transcriptional regulatory networks as well as their interpretation.\"germany.\",statistical modeling of transcription factor binding affinities predicts regulatory interactions,\"recent experimental and theoretical efforts have highlighted the fact that binding of transcription factors to {dna} can be more accurately described by continuous measures of their binding affinities, rather than a discrete description in terms of binding sites. while the binding affinities can be predicted from a physical model, it is often desirable to know the distribution of binding affinities for specific sequence backgrounds. in this paper, we present a statistical approach to derive the exact distribution for sequence models with fixed {gc} content. we demonstrate that the affinity distribution of almost all known transcription factors can be effectively parametrized by a class of generalized extreme value distributions. moreover, this parameterization also describes the affinity distribution for sequence backgrounds with variable {gc} content, such as human promoter sequences. our approach is applicable to arbitrary sequences and all transcription factors with known binding preferences that can be described in terms of a motif matrix. the statistical treatment also provides a proper framework to directly compare transcription factors with very different affinity distributions. this is illustrated by our analysis of human promoters with known binding sites, for many of which we could identify the known regulators as those with the highest affinity. the combination of physical model and statistical normalization provides a quantitative measure which ranks transcription factors for a given sequence, and which can be compared directly with large-scale binding data. its successful application to human promoter sequences serves as an encouraging example of how the method can be applied to other sequences. the binding of proteins to {dna} is a key molecular mechanism, which can regulate the expression of genes in response to different cellular and environmental conditions. the extensive research on gene regulation has generated binding models for many transcription factors, but the prediction of new binding sites is still challenging and difficult to improve in any systematic way. recent experimental advances, notably high throughput binding assays, have shifted the theoretical focus from the prediction of new binding sites towards more quantitative models for the binding affinities of transcription factors, which can now be measured across whole genomes. therefore we have developed a biophysical model which accounts for much of the observed variation in binding strength. here we extend this framework to model not just the binding affinity, but also its distribution in various sequence backgrounds. this enables us to compare predicted affinities from different transcription factors, and to rank them according to their normalized affinity. what are the biological implications of such a ranking? we have demonstrated that many known associations between transcription factors and their respective targets appear as strong interactions. this provides a rationale to predict, for any given promoter region, those transcription factors which are most likely to be involved in its regulation.\"assessing phylogenetic motif models for predicting transcription factor binding sites.,stanford, ca 94305, usa.\",repeated observation of breast tumor subtypes in independent gene expression data sets,\"characteristic patterns of gene expression measured by {dna} microarrays have been used to classify tumors into clinically relevant subgroups. in this study, we have refined the previously defined subtypes of breast tumors that could be distinguished by their distinct patterns of gene expression. a total of 115 malignant breast tumors were analyzed by hierarchical clustering based on patterns of expression of 534  ” intrinsic” genes and shown to subdivide into one basal-like, one {erbb2}-overexpressing, two luminal-like, and one normal breast tissue-like subgroup. the genes used for classification were selected based on their similar expression levels between pairs of consecutive samples taken from the same tumor separated by 15 weeks of neoadjuvant treatment. similar cluster analyses of two published, independent data sets representing different patient cohorts from different laboratories, uncovered some of the same breast cancer subtypes. in the one data set that included information on time to development of distant metastasis, subtypes were associated with significant differences in this clinical feature. by including a group of tumors from {brca1} carriers in the analysis, we found that this genotype predisposes to the basal tumor subtype. our results strongly support the idea that many of these breast tumor subtypes represent biologically distinct disease entities.\"\"germline {brca1} mutations predispose to breast cancer. to identify genetic modifiers of this risk, we performed a genome-wide association study in 1,193 individuals with {brca1} mutations who were diagnosed with invasive breast cancer under age 40 and 1,190 {brca1} carriers without breast cancer diagnosis over age 35. we took forward 96 {snps} for replication in another 5,986 {brca1} carriers (2,974 individuals with breast cancer and 3,012 unaffected individuals). five {snps} on 19p13 were associated with breast cancer risk (p(trend) = 2.3 × 10⁻⁹ to p(trend) = 3.9 × 10⁻⁷), two of which showed independent associations (rs8170, hazard ratio ({hr}) = 1.26, 95\\\\% {ci} 1.17-1.35; rs2363956 {hr} = 0.84, 95\\\\% {ci} 0.80-0.89). genotyping these {snps} in 6,800 population-based breast cancer cases and 6,613 controls identified a similar association with estrogen receptor-negative breast cancer (rs2363956 per-allele odds ratio ({or}) = 0.83, 95\\\\% {ci} 0.75-0.92, p(trend) = 0.0003) and an association with estrogen receptor-positive disease in the opposite direction ({or} = 1.07, 95\\\\% {ci} 1.01-1.14, p(trend) = 0.016). the five {snps} were also associated with triple-negative breast cancer in a separate study of 2,301 triple-negative cases and 3,949 controls (p(trend) = 1 × 10⁻⁷) to p(trend) = 8 × 10⁻⁵; rs2363956 per-allele {or} = 0.80, 95\\\\% {ci} 0.74-0.87, p(trend) = 1.1 × 10⁻⁷\"baltimore, md 21231, usa.\",the genomic landscapes of human breast and colorectal cancers.,\"human cancer is caused by the accumulation of mutations in oncogenes and tumor suppressor genes. to catalog the genetic changes that occur during tumorigenesis, we isolated {dna} from 11 breast and 11 colorectal tumors and determined the sequences of the genes in the reference sequence database in these samples. based on analysis of exons representing 20,857 transcripts from 18,191 genes, we conclude that the genomic landscapes of breast and colorectal cancers are composed of a handful of commonly mutated gene \"\"mountains\"\" and a much larger number of gene \"\"hills\"\" that are mutated at low frequency. we describe statistical and bioinformatic tools that may help identify mutations with a role in tumorigenesis. these results have implications for understanding the nature and heterogeneity of human cancers and for using personal genomics for tumor diagnosis and therapy.\"non-cell-autonomous driving of tumour growth supports sub-clonal heterogeneity,ibbmc, cnrs umr 8619, bat. 430, universit\\\\\\'{e} paris-sud, 91405-orsay, france.\",\"the third {capri} assessment meeting toronto, canada, april 20-21, 2007.\",\"{capri} is a community-wide experiment to test protein-protein docking methods in blind predictions. the toronto meeting assessed structure predictions made from 2005-2007 on nine target protein-protein complexes or homodimers, and reported new developments in functions used to score predicted interactions, in treatment of conformational flexibility, and in taking nonstructural information into account in the predictions.\"\"protein sequences evolve through random mutagenesis with selection for optimal fitness1. cooperative folding into a stable tertiary structure is one aspect of fitness, but evolutionary selection ultimately operates on function, not on structure. in the accompanying paper2, we proposed a model for the evolutionary constraint on a small protein interaction module (the {ww} domain) through application of the {sca}, a statistical analysis of multiple sequence alignments3, 4. construction of artificial protein sequences directed only by the {sca} showed that the information extracted by this analysis is sufficient to engineer the {ww} fold at atomic resolution. here, we demonstrate that these artificial {ww} sequences function like their natural counterparts, showing class-specific recognition of proline-containing target peptides5, 6, 7, 8. consistent with {sca} predictions, a distributed network of residues mediates functional specificity in {ww} domains. the ability to recapitulate natural-like function in designed sequences shows that a relatively small quantity of sequence information is sufficient to specify the global energetics of amino acid interactions.\"ramat-aviv, tel-aviv, 69978, israel. amos@tau.ac.il\",discovering statistically significant biclusters in gene expression data,\"in gene expression data, a bicluster is a subset of the genes exhibiting consistent patterns over a subset of the conditions. we propose a new method to detect significant biclusters in large expression datasets. our approach is graph theoretic coupled with statistical modelling of the data. under plausible assumptions, our algorithm is polynomial and is guaranteed to find the most significant biclusters. we tested our method on a collection of yeast expression profiles and on a human cancer dataset. cross validation results show high specificity in assigning function to genes based on their biclusters, and we are able to annotate in this way 196 uncharacterized yeast genes. we also demonstrate how the biclusters lead to detecting new concrete biological associations. in cancer data we are able to detect and relate finer tissue types than was previously possible. we also show that the method outperforms the biclustering algorithm of cheng and church (2000).\"\"multiple somatic rearrangements are often found in cancer genomes; however, the underlying processes of rearrangement and their contribution to cancer development are poorly characterized. here we use a paired-end sequencing strategy to identify somatic rearrangements in breast cancer genomes. there are more rearrangements in some breast cancers than previously appreciated. rearrangements are more frequent over gene footprints and most are intrachromosomal. multiple rearrangement architectures are present, but tandem duplications are particularly common in some cancers, perhaps reflecting a specific defect in {dna} maintenance. short overlapping sequences at most rearrangement junctions indicate that these have been mediated by non-homologous end-joining {dna} repair, although varying sequence patterns indicate that multiple processes of this type are operative. several expressed in-frame fusion genes were identified but none was recurrent. the study provides a new perspective on cancer genomes, highlighting the diversity of somatic rearrangements and their potential contribution to cancer development.\"\"prostate cancer is the second most common cause of male cancer deaths in the united states. however, the full range of prostate cancer genomic alterations is incompletely characterized. here we present the complete sequence of seven primary human prostate cancers and their paired normal counterparts. several tumours contained complex chains of balanced (that is, \\'copy-neutral\\') rearrangements that occurred within or adjacent to known cancer genes. rearrangement breakpoints were enriched near open chromatin, androgen receptor and {erg} {dna} binding sites in the setting of the {ets} gene fusion {tmprss2}-{erg}, but inversely correlated with these regions in tumours lacking {ets} fusions. this observation suggests a link between chromatin or transcriptional regulation and the genesis of genomic aberrations. three tumours contained rearrangements that disrupted {cadm2}, and four harboured events disrupting either {pten} (unbalanced events), a prostate tumour suppressor, or {magi2} (balanced events), a {pten} interacting protein not previously implicated in prostate tumorigenesis. thus, genomic rearrangements may arise from transcriptional or chromatin aberrancies and engage prostate tumorigenic mechanisms.\"\"{background}:assays of multiple tumor samples frequently reveal recurrent genomic aberrations, including point mutations and copy-number alterations, that affect individual genes. analyses that extend beyond single genes are often restricted to examining pathways, interactions and functional modules that are already {known.methods}:we present a method that identifies functional modules without any information other than patterns of recurrent and mutually exclusive aberrations ({rme} patterns) that arise due to positive selection for key cancer phenotypes. our algorithm efficiently constructs and searches networks of potential interactions and identifies significant modules ({rme} modules) by using the algorithmic significance {test.results}:we apply the method to the {tcga} collection of 145 glioblastoma samples, resulting in extension of known pathways and discovery of new functional modules. the method predicts a role for {ep300} that was previously unknown in glioblastoma. we demonstrate the clinical relevance of these results by validating that expression of {ep300} is prognostic, predicting survival independent of age at diagnosis and tumor {grade.conclusions}:we have developed a sensitive, simple, and fast method for automatically detecting functional modules in tumors based solely on patterns of recurrent genomic aberration. due to its ability to analyze very large amounts of diverse data, we expect it to be increasingly useful when applied to the many tumor panels scheduled to be assayed in the near future.\"saic-frederick, inc., nci-frederick, frederick, md 21702, usa. guna@ncifcrf.gov\",analysis of ordered and disordered protein complexes reveals structural features discriminating between stable and unstable monomers,\"most proteins exist in the cell as multi-component assemblies. however, which proteins need to be present simultaneously in order to perform a given function is frequently unknown. the first step toward this goal would be to predict proteins that can function only when in a complexed form. here, we propose a scheme to distinguish whether the protein components are ordered (stable) or disordered when separated from their complexed partners. we analyze structural characteristics of several types of complexes, such as natively unstructured proteins, ribosomal proteins, two-state and three-state complexes, and crystal-packing dimers. our analysis makes use of the fact that natively unstructured proteins, which undergo a disorder-to-order transition upon binding their partner, and stable monomeric proteins, which exist as dimers only in their crystal form, provide examples of two vastly different scenarios. we find that ordered monomers can be distinguished from disordered monomers on the basis of the per-residue surface and interface areas, which are significantly smaller for ordered proteins. with this scale, two-state dimers (where the monomers unfold upon dimer separation) and ribosomal proteins are shown to resemble disordered proteins. on the other hand, crystal-packing dimers, whose monomers are stable in solution, fall into the ordered protein category. while there should be a continuum in the distributions, nevertheless, the per-residue scale measures the confidence in the determination of whether a protein can exist as a stable monomer. further analysis, focusing on the chemical and contact preferences at the interface, interior and exposed surface areas, reveals that disordered proteins lack a strong hydrophobic core and are composed of highly polar surface area. we discuss the implication of our results for de novo design of stable monomeric proteins and peptides.\"\"this protocol explains how to discover functional signals in genomic sequences by detecting over- or under-represented oligonucleotides (words) or spaced pairs thereof (dyads) with the regulatory sequence analysis tools (http://rsat.ulb.ac.be/rsat/). two typical applications are presented: (i) predicting transcription factor-binding motifs in promoters of coregulated genes and (ii) discovering phylogenetic footprints in promoters of orthologous genes. the steps of this protocol include purging genomic sequences to discard redundant fragments, discovering over-represented patterns and assembling them to obtain degenerate motifs, scanning sequences and drawing feature maps. the main strength of the method is its statistical ground: the binomial significance provides an efficient control on the rate of false positives. in contrast with optimization-based pattern discovery algorithms, the method supports the detection of under- as well as over-represented motifs. computation times vary from seconds (gene clusters) to minutes (whole genomes). the execution of the whole protocol should take approximately 1 h.\"\"{\\\\^{a}€{\\\\oe}phylogenetic} profiling\\\\^{a}€� is based on the hypothesis that during evolution functionally or physically interacting genes are likely to be inherited or eliminated in a codependent manner. creating presence\\\\^{a}€ ” absence profiles of orthologous genes is now a common and powerful way of identifying functionally associated genes. in this approach, correctly determining orthology, as a means of identifying functional equivalence between two genes, is a critical and nontrivial step and largely explains why previous work in this area has mainly focused on using presence\\\\^{a}€ ” absence profiles in prokaryotic species. here, we demonstrate that eukaryotic genomes have a high proportion of multigene families whose phylogenetic profile distributions are poor in presence\\\\^{a}€ ” absence information content. this feature makes them prone to orthology mis-assignment and unsuited to standard profile-based prediction methods. using {cath} structural domain assignments from the {gene3d} database for 13 complete eukaryotic genomes, we have developed a novel modification of the phylogenetic profiling method that uses genome copy number of each domain superfamily to predict functional relationships. in our approach, superfamilies are subclustered at ten levels of sequence identity\\\\^{a}€”from 30\\\\% to 100\\\\%\\\\^{a}€”and phylogenetic profiles built at each level. all the profiles are compared using normalised euclidean distances to identify those with correlated changes in their domain copy number. we demonstrate that two protein families will \\\\^{a}€{\\\\oe}auto-tune\\\\^{a}€� with strong co-evolutionary signals when their profiles are compared at the similarity levels that capture their functional relationship. our method finds functional relationships that are not detectable by the conventional presence\\\\^{a}€ ” absence profile comparisons, and it does not require a priori any fixed criteria to define orthologous genes.\",algorithms for detecting significantly mutated pathways in cancer.,\"recent genome sequencing studies have shown that the somatic mutations that drive cancer development are distributed across a large number of genes. this mutational heterogeneity complicates efforts to distinguish functional mutations from sporadic, passenger mutations. since cancer mutations are hypothesized to target a relatively small number of cellular signaling and regulatory pathways, a common practice is to assess whether known pathways are enriched for mutated genes. we introduce an alternative approach that examines mutated genes in the context of a genome-scale gene interaction network. we present a computationally efficient strategy for de novo identification of subnetworks in an interaction network that are mutated in a statistically significant number of patients. this framework includes two major components. first, we use a diffusion process on the interaction network to define a local neighborhood of \"\"influence\"\" for each mutated gene in the network. second, we derive a two-stage multiple hypothesis test to bound the false discovery rate ({fdr}) associated with the identified subnetworks. we test these algorithms on a large human protein-protein interaction network using somatic mutation data from glioblastoma and lung adenocarcinoma samples. we successfully recover pathways that are known to be important in these cancers and also identify additional pathways that have been implicated in other cancers but not previously reported as mutated in these samples. we anticipate that our approach will find increasing use as cancer genome studies increase in size and scope.\"\"large-scale sequencing of cancer genomes has uncovered thousands of {dna} alterations, but the functional relevance of the majority of these mutations to tumorigenesis is unknown. we have developed a computational method, called cancer-specific high-throughput annotation of somatic mutations ({chasm}), to identify and prioritize those missense mutations most likely to generate functional changes that enhance tumor cell proliferation. the method has high sensitivity and specificity when discriminating between known driver missense mutations and randomly generated missense mutations (area under receiver operating characteristic curve, >0.91; area under {precision-recall} curve, >0.79). {chasm} substantially outperformed previously described missense mutation function prediction methods at discriminating known oncogenic mutations in p53 and the tyrosine kinase epidermal growth factor receptor. we applied the method to 607 missense mutations found in a recent glioblastoma multiforme sequencing study. based on a model that assumed the glioblastoma multiforme mutations are a mixture of drivers and passengers, we estimate that 8\\\\% of these mutations are drivers, causally contributing to tumorigenesis.\"600 n. wolfe street, baltimore, md 21287, usa.\",computational analysis of tissue-specific combinatorial gene regulation: predicting interaction between transcription factors in human tissues,\"tissue-specific gene expression is generally regulated by more than a single transcription factor ({tf}). multiple {tfs} work in concert to achieve tissue specificity. in order to explore these complex {tf} interaction networks, we performed a large-scale analysis of {tf} interactions for 30 human tissues. we first identified tissue-specific genes for 30 tissues based on gene expression databases. we then evaluated the relationships between {tfs} using the relative position and co-occurrence of their binding sites in the promoters of tissue-specific genes. the predicted {tf}–{tf} interactions were validated by both known protein–protein interactions and co-expression of their target genes. we found that our predictions are enriched in known protein–protein interactions (>80 times that of random expectation). in addition, we found that the target genes show the highest co-expression in the tissue of interest. our findings demonstrate that non-tissue specific {tfs} play a large role in regulation of tissue-specific genes. furthermore, they show that individual {tfs} can contribute to tissue specificity in different tissues by interacting with distinct {tf} partners. lastly, we identified several tissue-specific {tf} clusters that may play important roles in tissue-specific gene regulation.\"\"we present an integrated method called chromia for the genome-wide identification of functional target loci of transcription factors. designed to capture the characteristic patterns of transcription factor binding motif occurrences and the histone profiles associated with regulatory elements such as promoters and enhancers, chromia significantly outperforms other methods in the identification of 13 transcription factor binding sites in mouse embryonic stem cells, evaluated by both binding ({chip}-seq) and functional ({rna} interference knockdown) experiments.\"\"the formation of specific protein interactions plays a crucial role in most, if not all, biological processes, including signal transduction, cell regulation, the immune response and others. recent advances in our understanding of the molecular architecture of protein-protein binding sites, which facilitates such diversity in binding affinity and specificity, are enabling us to address key questions. what is the amino acid composition of binding sites? what are interface hotspots? how are binding sites organized? what are the differences between tight and weak interacting complexes? how does water contribute to binding? can the knowledge gained be translated into protein design? and does a universal code for binding exist, or is it the architecture and chemistry of the interface that enable diverse but specific binding solutions?\"\"substantial evidence supports the concept that cancers are organized in a cellular hierarchy with cancer stem cells ({csc}) at the apex. to date, the primary evidence for {cscs} derives from transplantation assays, which have known limitations. in particular, they are unable to report on the fate of cells within the original human tumor. because of the difficulty in measuring tumor characteristics in patients, cellular organization and other aspects of cancer dynamics have not been quantified directly, although they likely play a fundamental role in tumor progression and therapy response. as such, new approaches to study {cscs} in patient-derived tumor specimens are needed. in this study, we exploited ultradeep single-molecule genomic data derived from multiple microdissected colorectal cancer glands per tumor, along with a novel quantitative approach to measure tumor characteristics, define patient-specific tumor profiles, and infer tumor ancestral trees. we show that each cancer is unique in terms of its cellular organization, molecular heterogeneity, time from malignant transformation, and rate of mutation and apoptosis. importantly, we estimate {csc} fractions between 0.5\\\\% and 4\\\\%, indicative of a hierarchical organization responsible for long-lived {csc} lineages, with variable rates of symmetric cell division. we also observed extensive molecular heterogeneity, both between and within individual cancer glands, suggesting a complex hierarchy of mitotic clones. our framework enables the measurement of clinically relevant patient-specific characteristics in vivo, providing insight into the cellular organization and dynamics of tumor growth, with implications for personalized patient care. cancer res; 73(1); 41–49. {\\\\copyright}2012 {aacr}.\"1275 york avenue, new york, ny 10021, usa.\",a faster circular binary segmentation algorithm for the analysis of array {cgh} data,\"motivation: array {cgh} technologies enable the simultaneous measurement of {dna} copy number for thousands of sites on a genome. we developed the circular binary segmentation ({cbs}) algorithm to divide the genome into regions of equal copy number. the algorithm tests for change-points using a maximal t-statistic with a permutation reference distribution to obtain the corresponding p-value. the number of computations required for the maximal test statistic is {o(n2}), where n is the number of markers. this makes the full permutation approach computationally prohibitive for the newer arrays that contain tens of thousands markers and highlights the need for a faster algorithm.\"\"to characterize somatic alterations in colorectal carcinoma, we conducted a genome-scale analysis of 276 samples, analysing exome sequence, {dna} copy number, promoter methylation and messenger {rna} and {microrna} expression. a subset of these samples (97) underwent low-depth-of-coverage whole-genome sequencing. in total, 16\\\\% of colorectal carcinomas were found to be hypermutated: three-quarters of these had the expected high microsatellite instability, usually with hypermethylation and {mlh1} silencing, and one-quarter had somatic mismatch-repair gene and polymerase ε ({pole}) mutations. excluding the hypermutated cancers, colon and rectum cancers were found to have considerably similar patterns of genomic alteration. twenty-four genes were significantly mutated, and in addition to the expected {apc}, {tp53}, {smad4}, {pik3ca} and {kras} mutations, we found frequent mutations in {arid1a}, {sox9} and {fam123b}. recurrent copy-number alterations include potentially drug-targetable amplifications of {erbb2} and newly discovered amplification of {igf2}. recurrent chromosomal translocations include the fusion of {nav2} and {wnt} pathway member {tcf7l1}. integrative analyses suggest new markers for aggressive colorectal carcinoma and an important role for {myc}-directed transcriptional activation and repression.\"\"most proteins interact with only a few other proteins while a small number of proteins (hubs) have many interaction partners. hub proteins and non-hub proteins differ in several respects; however, understanding is not complete about what properties characterize the hubs and set them apart from proteins of low connectivity. therefore, we have investigated what differentiates hubs from non-hubs and static hubs (party hubs) from dynamic hubs (date hubs) in the protein-protein interaction network of saccharomyces cerevisiae. the many interactions of hub proteins can only partly be explained by bindings to similar proteins or domains. it is evident that domain repeats, which are associated with binding, are enriched in hubs. moreover, there is an over representation of multi-domain proteins and long proteins among the hubs. in addition, there are clear differences between party hubs and date hubs. fewer of the party hubs contain long disordered regions compared to date hubs, indicating that these regions are important for flexible binding but less so for static interactions. furthermore, party hubs interact to a large extent with each other, supporting the idea of party hubs as the cores of highly clustered functional modules. in addition, hub proteins, and in particular party hubs, are more often ancient. finally, the more recent paralogs of party hubs are underrepresented. our results indicate that multiple and repeated domains are enriched in hub proteins and, further, that long disordered regions, which are common in date hubs, are particularly important for flexible binding.\"{rhvdm}: an r package to predict the activity and targets of a transcription factor.,\"{summary}: highly parallel genomic platforms like microarrays often present researchers with long lists of differentially expressed genes but contain little or no information on how these genes are regulated. {rhvdm} is a novel r package which uses gene expression time course data to predict the activity and targets of a transcription factor. in the first step, {rhvdm} uses a small number of known targets to derive the activity profile of a given transcription factor. then, in a subsequent step, this activity profile is used to predict other putative targets of that transcription factor. a dynamic and mechanistic model of gene expression is at the heart of the technique. measurement error is taken into account during the process, which allows an objective assessment of the robustness of fit and, therefore, the quality of the predictions. the package relies on efficient algorithms and vectorization to accomplish potentially time consuming tasks including optimization and differential equation integration. we demonstrate the efficiency and accuracy of {rhvdm} by examining the activity of the tumour-suppressing transcription factor, p53. {availability}: the version of the package presented here (1.8.1) is freely available from the bioconductor web site ({http://bioconductor.org/packages/2.3/bioc/html/rhvdm}.html).\"\"we describe a bioinformatic tool, tumor aberration prediction suite ({taps}), for the identification of allele-specific copy numbers in tumor samples using data from affymetrix {snp} arrays. it includes detailed visualization of genomic segment characteristics and iterative pattern recognition for copy number identification, and does not require patient-matched normal samples. {taps} can be used to identify chromosomal aberrations with high sensitivity even when the proportion of tumor cells is as low as 30\\\\%. analysis of cancer samples indicates that {taps} is well suited to investigate samples with aneuploidy and tumor heterogeneity, which is commonly found in many types of solid tumors.\"\"we present a novel method that combines protein structure information with protein interaction data to identify residues that form part of an interaction interface. our prediction method can retrieve interaction hotspots with an accuracy of 60\\\\% (at a 20\\\\% false positive rate). the method was applied to all mutations in the online mendelian inheritance in man ({omim}) database, predicting 1,428 mutations to be related to an interaction defect. combining predicted and hand-curated sets, we discuss how mutations affect protein interactions in general.\"\"cancer results from genetic alterations that disturb the normal cooperative behavior of cells. recent high-throughput genomic studies of cancer cells have shown that the mutational landscape of cancer is complex and that individual cancers may evolve through mutations in as many as 20 different cancer-associated genes. we use data published by sj\\\\\"\"{o}blom et al. (2006) to develop a new mathematical model for the somatic evolution of colorectal cancers. we employ the {wright-fisher} process for exploring the basic parameters of this evolutionary process and derive an analytical approximation for the expected waiting time to the cancer phenotype. our results highlight the relative importance of selection over both the size of the cell population at risk and the mutation rate. the model predicts that the observed genetic diversity of cancer genomes can arise under a normal mutation rate if the average selective advantage per mutation is on the order of 1\\\\%. increased mutation rates due to genetic instability would allow even smaller selective advantages during tumorigenesis. the complexity of cancer progression can be understood as the result of multiple sequential mutations, each of which has a relatively small but positive effect on net cell growth. cancer is a disease of multicellular organisms that is characterized by a breakdown of cooperation between individual cells. the progression of cancer proceeds from a single genetically altered cell to billions of invasive cells through a series of clonal expansions. during tumorigenesis the cancer cells undergo replication and mutation, thereby increasing the size and invasiveness of the tumor. recent sequencing projects of cancer cells suggest that mutations in up to 20 different genes might be responsible for driving an individual tumor\\'s development. this insight contrasts with most mathematical models of cancer progression, which assume that the cancer phenotype is driven by mutations in only a few genes. we present a new mathematical model in which tumorigenesis is driven by mutations in many genes, most of which confer only a small selective advantage. specifically, the progression of a benign tumor of the colon (adenoma) to a malignant tumor (carcinoma) is described by a {wright-fisher} process with growing population size. we explore the basic parameters of the model that are consistent with observed data. we also derive an analytical formula for the expected waiting time for the progression from benign to maligant tumor in terms of the population size, the mutation rate, the selective advantage, and the number of susceptible genes.\"cnb-csic, cantoblanco, 28049, madrid, spain. valencia@cnb.uam.es\",computational methods for the prediction of protein interactions.,\"establishing protein interaction networks is crucial for understanding cellular operations. detailed knowledge of the \\'interactome\\', the full network of protein-protein interactions, in model cellular systems should provide new insights into the structure and properties of these systems. parallel to the first massive application of experimental techniques to the determination of protein interaction networks and protein complexes, the first computational methods, based on sequence and genomic information, have emerged.\"non-invasive analysis of acquired resistance to cancer therapy by sequencing of plasma {dna},philadelphia, pennsylvania 19104, usa;\",{penncnv}: an integrated hidden markov model designed for high-resolution copy number variation detection in whole-genome {snp} genotyping data.,\"comprehensive identification and cataloging of copy number variations ({cnvs}) is required to provide a complete view of human genetic variation. the resolution of {cnv} detection in previous experimental designs has been limited to tens or hundreds of kilobases. here we present {penncnv}, a hidden markov model ({hmm}) based approach, for kilobase-resolution detection of {cnvs} from illumina high-density {snp} genotyping data. this algorithm incorporates multiple sources of information, including total signal intensity and allelic intensity ratio at each {snp} marker, the distance between neighboring {snps}, the allele frequency of {snps}, and the pedigree information where available. we applied {penncnv} to genotyping data generated for 112 {hapmap} individuals; on average, we detected approximately 27 {cnvs} for each individual with a median size of approximately 12 kb. excluding common rearrangements in lymphoblastoid cell lines, the fraction of {cnvs} in offspring not detected in parents ({cnv}-{ndps}) was 3.3\\\\%. our results demonstrate the feasibility of whole-genome fine-mapping of {cnvs} via high-density {snp} genotyping.\"1, d-69117, heidelberg, germany. aloy@embl.de\",{interprets}: protein interaction prediction through tertiary structure,\"summary: {interprets} (interaction prediction through tertiary structure) is a web-based version of our method for predicting protein–protein interactions (aloy and russell, 2002, proc. natl acad. sci. {usa}, 99, 5896–5901). given a pair of query sequences, we first search for homologues in a database of interacting domains ({dbid}) of known three-dimensional complex structures. pairs of sequences homologous to a known interacting pair are scored for how well they preserve the atomic contacts at the interaction interface. {interprets} includes a useful interface for visualising molecular details of any predicted interaction.\"cambridge cb2 0qh, united kingdom. elevy@mrc-lmb.cam.ac.uk\",{piqsi}: protein quaternary structure investigation,\"the smooth particle mesh ewald summation method is widely used to efficiently compute long-range electrostatic force terms in molecular dynamics simulations, and there has been considerable work in developing optimized implementations for a variety of parallel computer architectures. we describe an implementation for nvidia graphical processing units ({gpus}) which are general purpose computing devices with a high degree of intrinsic parallelism and arithmetic performance. we find that, for typical biomolecular simulations (e.g., {dhfr}, {26k} atoms), a single {gpu} equipped workstation is able to provide sufficient performance to permit simulation rates of ≈50 ns/day when used in conjunction with the {acemd} molecular dynamics package(1) and exhibits an accuracy comparable to that of a reference double-precision {cpu} implementation.\"\"motivation: promoter prediction is an important task in genome annotation projects, and during the past years many new promoter prediction programs ({ppps}) have emerged. however, many of these programs are compared inadequately to other programs. in most cases, only a small portion of the genome is used to evaluate the program, which is not a realistic setting for whole genome annotation projects. in addition, a common evaluation design to properly compare {ppps} is still {lacking.results}: we present a large-scale benchmarking study of 17 state-of-the-art {ppps}. a multi-faceted evaluation strategy is proposed that can be used as a gold standard for promoter prediction evaluation, allowing authors of promoter prediction software to compare their method to existing methods in a proper way. this evaluation strategy is subsequently used to compare the chosen promoter predictors, and an in-depth analysis on predictive performance, promoter class specificity, overlap between predictors and positional bias of the predictions is {conducted.availability}: we provide the implementations of the four protocols, as well as the datasets required to perform the benchmarks to the academic community free of charge on {request.contact}: {yves.vandepeer@psb.ugent.besupplementary} information: supplementary data are available at bioinformatics online.\"\"the tumor suppressor p53 exerts antiproliferation effects through its ability to function as a sequence-specific {dna}-binding transcription factor. here, we demonstrate that p53 can be modified by acetylation both in vivo and in vitro. remarkably, the site of p53 that is acetylated by its coactivator, p300, resides in a c-terminal domain known to be critical for the regulation of p53 {dna} binding. furthermore, the acetylation of p53 can dramatically stimulate its sequence-specific {dna}-binding activity, possibly as a result of an acetylation-induced conformational change. these observations clearly indicate a novel pathway for p53 activation and, importantly, provide an example of an acetylation-mediated change in the function of a nonhistone regulatory protein. these results have significant implications regarding the molecular mechanisms of various acetyltransferase-containing transcriptional coactivators whose primary targets have been presumed to be histones.\"genetic variegation of clonal architecture and propagating cells in leukaemia.,\"little is known of the genetic architecture of cancer at the subclonal and single-cell level or in the cells responsible for cancer clone maintenance and propagation. here we have examined this issue in childhood acute lymphoblastic leukaemia in which the {etv6}-{runx1} gene fusion is an early or initiating genetic lesion followed by a modest number of recurrent or \\'driver\\' copy number alterations. by multiplexing fluorescence in situ hybridization probes for these mutations, up to eight genetic abnormalities can be detected in single cells, a genetic signature of subclones identified and a composite picture of subclonal architecture and putative ancestral trees assembled. subclones in acute lymphoblastic leukaemia have variegated genetics and complex, nonlinear or branching evolutionary histories. copy number alterations are independently and reiteratively acquired in subclones of individual patients, and in no preferential order. clonal architecture is dynamic and is subject to change in the lead-up to a diagnosis and in relapse. leukaemia propagating cells, assayed by serial transplantation in {nod}/{scid} {il2rγ}(null) mice, are also genetically variegated, mirroring subclonal patterns, and vary in competitive regenerative capacity in vivo. these data have implications for cancer genomics and for the targeted therapy of cancer.\"tel aviv 69978, israel.\",revealing modularity and organization in the yeast molecular network by integrated analysis of highly heterogeneous genomewide data,\"the dissection of complex biological systems is a challenging task, made difficult by the size of the underlying molecular network and the heterogeneous nature of the control mechanisms involved. novel high-throughput techniques are generating massive data sets on various aspects of such systems. here, we perform analysis of a highly diverse collection of genomewide data sets, including gene expression, protein interactions, growth phenotype data, and transcription factor binding, to reveal the modular organization of the yeast system. by integrating experimental data of heterogeneous sources and types, we are able to perform analysis on a much broader scope than previous studies. at the core of our methodology is the ability to identify modules, namely, groups of genes with statistically significant correlated behavior across diverse data sources. numerous biological processes are revealed through these modules, which also obey global hierarchical organization. we use the identified modules to study the yeast transcriptional network and predict the function of >800 uncharacterized genes. our analysis framework, samba ({statistical-algorithmic} method for bicluster analysis), enables the processing of current and future sources of biological information and is readily extendable to experimental techniques and higher organisms.\"\"colorectal cancer ({crc}) is a heterogeneous disease in which unique subtypes are characterized by distinct genetic and epigenetic alterations. here we performed comprehensive genome-scale {dna} methylation profiling of 125 colorectal tumors and 29 adjacent normal tissues. we identified four {dna} methylation–based subgroups of {crc} using model-based cluster analyses. each subtype shows characteristic genetic and clinical features, indicating that they represent biologically distinct subgroups. a {cimp}-high ({cimp}-h) subgroup, which exhibits an exceptionally high frequency of cancer-specific {dna} hypermethylation, is strongly associated with {mlh1} {dna} hypermethylation and the {brafv600e} mutation. a {cimp}-low ({cimp}-l) subgroup is enriched for {kras} mutations and characterized by {dna} hypermethylation of a subset of {cimp}-h-associated markers rather than a unique group of {cpg} islands. {non-cimp} tumors are separated into two distinct clusters. one {non-cimp} subgroup is distinguished by a significantly higher frequency of {tp53} mutations and frequent occurrence in the distal colon, while the tumors that belong to the fourth group exhibit a low frequency of both cancer-specific {dna} hypermethylation and gene mutations and are significantly enriched for rectal tumors. furthermore, we identified 112 genes that were down-regulated more than twofold in {cimp}-h tumors together with promoter {dna} hypermethylation. these represent ∼7\\\\% of genes that acquired promoter {dna} methylation in {cimp}-h tumors. intriguingly, 48/112 genes were also transcriptionally down-regulated in {non-cimp} subgroups, but this was not attributable to promoter {dna} hypermethylation. together, we identified four distinct {dna} methylation subgroups of {crc} and provided novel insight regarding the role of {cimp}-specific {dna} hypermethylation in gene silencing.\"{elife} - open access to the most promising advances in sciencecancer genomics: from discovery science to personalized medicine.,london, nw7 1aa, uk. cedric.notredame@europe.com\",t-coffee: a novel method for fast and accurate multiple sequence alignment,\"we describe a new method ({t-coffee}) for multiple sequence alignment that provides a dramatic improvement in accuracy with a modest sacrifice in speed as compared to the most commonly used alternatives. the method is broadly based on the popular progressive approach to multiple alignment but avoids the most serious pitfalls caused by the greedy nature of this algorithm. with {t-coffee} we pre-process a data set of all pair-wise alignments between the sequences. this provides us with a library of alignment information that can be used to guide the progressive alignment. intermediate alignments are then based not only on the sequences to be aligned next but also on how all of the sequences align with each other. this alignment information can be derived from heterogeneous sources such as a mixture of alignment programs and/or structure superposition. here, we illustrate the power of the approach by using a combination of local and global pair-wise alignments to generate the library. the resulting alignments are significantly more reliable, as determined by comparison with a set of 141 test cases, than any of the popular alternatives that we tried. the improvement, especially clear with the more difficult test cases, is always visible, regardless of the phylogenetic spread of the sequences in the tests.\"kanazawa university, kanazawa 920-0934, japan. titolab@kenroku.kanazawa-u.ac.jp\",a comprehensive two-hybrid analysis to explore the yeast protein interactome,\"protein–protein interactions play crucial roles in the execution of various biological functions. accordingly, their comprehensive description would contribute considerably to the functional interpretation of fully sequenced genomes, which are flooded with novel genes of unpredictable functions. we previously developed a system to examine two-hybrid interactions in all possible combinations between the ≈6,000 proteins of the budding yeast saccharomyces cerevisiae. here we have completed the comprehensive analysis using this system to identify 4,549 two-hybrid interactions among 3,278 proteins. unexpectedly, these data do not largely overlap with those obtained by the other project [uetz, p., et al. (2000) nature (london) 403, 623–627] and hence have substantially expanded our knowledge on the protein interaction space or interactome of the yeast. cumulative connection of these binary interactions generates a single huge network linking the vast majority of the proteins. bioinformatics-aided selection of biologically relevant interactions highlights various intriguing subnetworks. they include, for instance, the one that had successfully foreseen the involvement of a novel protein in spindle pole body function as well as the one that may uncover a hitherto unidentified multiprotein complex potentially participating in the process of vesicular transport. our data would thus significantly expand and improve the protein interaction map for the exploration of genome functions that eventually leads to thorough understanding of the cell as a molecular system.\"\"the ensembl (http://www.ensembl.org/) database project provides a bioinformatics framework to organise biology around the sequences of large genomes. it is a comprehensive source of stable automatic annotation of the human genome sequence, with confirmed gene predictions that have been integrated with external data sources, and is available as either an interactive web site or as flat files. it is also an open source software engineering project to develop a portable system able to handle very large genomes and associated requirements from sequence analysis to data storage and visualisation. the ensembl site is one of the leading sources of human genome sequence annotation and provided much of the analysis for publication by the international human genome project of the draft genome. the ensembl system is being installed around the world in both companies and academic sites on machines ranging from supercomputers to laptops.\"\"motivation: in this work, we aim to develop a computational approach for predicting {dna}-binding sites in proteins from amino acid sequences. to avoid over-fitting with this method, all available {dna}-binding proteins from the protein data bank ({pdb}) are used to construct the models. the random forest ({rf}) algorithm is used because it is fast and has robust performance for different parameter values. a novel hybrid feature is presented which incorporates evolutionary information of the amino acid sequence, secondary structure ({ss}) information and orthogonal binary vector ({obv}) information which reflects the characteristics of 20 kinds of amino acids for two physical-chemical properties (dipoles and volumes of the side chains). the numbers of binding and non-binding residues in proteins are highly unbalanced, so a novel scheme is proposed to deal with the problem of imbalanced datasets by downsizing the majority class.  results: the results show that the {rf} model achieves 91.41\\\\% overall accuracy with matthew\\'s correlation coefficient of 0.70 and an area under the receiver operating characteristic curve ({auc}) of 0.913. to our knowledge, the {rf} method using the hybrid feature is currently the computationally optimal approach for predicting {dna}-binding sites in proteins from amino acid sequences without using three-dimensional structural information. we have demonstrated that the prediction results are useful for understanding {protein-dna} interactions.  availability: {dbindr} web-server implementation is freely available at {http://www.cbi.seu.edu.cn/dbindr}/{dbindr}.htm.  contact: xsun@seu.edu.cn 10.1093/bioinformatics/btn583\"university college london, uk.\",{cath}--a hierarchic classification of protein domain structures.,\"protein evolution gives rise to families of structurally related proteins, within which sequence identities can be extremely low. as a result, structure-based classifications can be effective at identifying unanticipated relationships in known structures and in optimal cases function can also be assigned. the ever increasing number of known protein structures is too large to classify all proteins manually, therefore, automatic methods are needed for fast evaluation of protein structures. we present a semi-automatic procedure for deriving a novel hierarchical classification of protein domain structures ({cath}). the four main levels of our classification are protein class (c), architecture (a), topology (t) and homologous superfamily (h). class is the simplest level, and it essentially describes the secondary structure composition of each domain. in contrast, architecture summarises the shape revealed by the orientations of the secondary structure units, such as barrels and sandwiches. at the topology level, sequential connectivity is considered, such that members of the same architecture might have quite different topologies. when structures belonging to the same t-level have suitably high similarities combined with similar functions, the proteins are assumed to be evolutionarily related and put into the same homologous superfamily. analysis of the structural families generated by {cath} reveals the prominent features of protein structure space. we find that nearly a third of the homologous superfamilies (h-levels) belong to ten major t-levels, which we call superfolds, and furthermore that nearly two-thirds of these h-levels cluster into nine simple architectures. a database of well-characterised protein structure families, such as {cath}, will facilitate the assignment of structure-function/evolution relationships to both known and newly determined protein structures.\"national institutes of health, bethesda, md 20894.\",basic local alignment search tool.,\"a new approach to rapid sequence comparison, basic local alignment search tool ({blast}), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair ({msp}) score. recent mathematical results on the stochastic properties of {msp} scores allow an analysis of the performance of this method as well as the statistical significance of alignments it generates. the basic algorithm is simple and robust; it can be implemented in a number of ways and applied in a variety of contexts including straightforward {dna} and protein sequence database searches, motif searches, gene identification searches, and in the analysis of multiple regions of similarity in long {dna} sequences. in addition to its flexibility and tractability to mathematical analysis, {blast} is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity.\"rockville, maryland 20850, usa. johnq@tigr.org\",microarray data normalization and transformation,\"over the past decade, comprehensive sequencing efforts have revealed the genomic landscapes of common forms of human cancer. for most cancer types, this landscape consists of a small number of  ” mountains” (genes altered in a high percentage of tumors) and a much larger number of  ” hills” (genes altered infrequently). to date, these studies have revealed \\\\~{}140 genes that, when altered by intragenic mutations, can promote or  ” drive” tumorigenesis. a typical tumor contains two to eight of these  ” driver gene” mutations; the remaining mutations are passengers that confer no selective growth advantage. driver genes can be classified into 12 signaling pathways that regulate three core cellular processes: cell fate, cell survival, and genome maintenance. a better understanding of these pathways is one of the most pressing needs in basic cancer research. even now, however, our knowledge of cancer genomes is sufficient to guide the development of more effective approaches for reducing cancer morbidity and mortality.\"\"neoplasms are microcosms of evolution. within a neoplasm, a mosaic of mutant cells compete for space and resources, evade predation by the immune system and can even cooperate to disperse and colonize new organs. the evolution of neoplastic cells explains both why we get cancer and why it has been so difficult to cure. the tools of evolutionary biology and ecology are providing new insights into neoplastic progression and the clinical control of cancer.\"michael smith building, oxford road, manchester, m13 9pt, united kingdom.\",specificity in protein interactions and its relationship with sequence diversity and coevolution.,\"studies of interacting proteins have found correlated evolution of the sequences of binding partners, apparently as a result of compensating mutations to maintain specificity (i.e., molecular coevolution). here, we analyze the coevolution of interacting proteins in yeast and demonstrate correlated evolution of binding partners in eukaryotes. detailed investigation of this apparent coevolution, focusing on the proteins\\' surface and binding interface, surprisingly leads to no improvement in the correlation. we conclude that true coevolution, as characterized by compensatory mutations between binding partners, is unlikely to be chiefly responsible for the apparent correlated evolution. we postulate that the correlation between sequence alignments is simply due to interacting proteins being subject to similar constraints on their evolutionary rate. because gene expression has a strong influence on evolutionary rate, and interacting proteins will tend to have similar levels of expression, we investigated this particular constraint. we found that the absolute expression level outperformed correlated evolution for predicting interacting protein partners. a correlation between sequence alignments could also be identified not only between pairs of proteins that physically interact but also between those that are merely functionally related (i.e., within the same protein complex). this indicates that the observed correlated evolution of interacting proteins is due to similar constraints on evolutionary rate and not coevolution.\"center for computational biology and bioinformatics and college of engineering, rumelifeneri yolu, 34450 sariyer istanbul, turkey.\",similar binding sites and different partners: implications to shared proteins in cellular pathways.,transcripts and proteins\",\"{ncbi}\\'s reference sequence ({refseq}) database ({http://www.ncbi.nlm.nih.gov/refseq}/) is a curated non-redundant collection of sequences representing genomes, transcripts and proteins. the database includes 3774 organisms spanning prokaryotes, eukaryotes and viruses, and has records for 2 879 860 proteins ({refseq} release 19). {refseq} records integrate information from multiple sources, when additional data are available from those sources and therefore represent a current description of the sequence and its features. annotations include coding regions, conserved domains, {trnas}, sequence tagged sites ({sts}), variation, references, gene and protein product names, and database cross-references. sequence is reviewed and features are added using a combined approach of collaboration and other input from the scientific community, prediction, propagation from {genbank} and curation by {ncbi} staff. the format of all {refseq} records is validated, and an increasing number of tests are being applied to evaluate the quality of sequence and annotation, especially in the context of complete genomic sequence.\"saic frederick, nci-fcrdc, md 21702-1201, usa.\",hydrogen bonds and salt bridges across protein-protein interfaces.,\"to understand further, and to utilize, the interactions across protein-protein interfaces, we carried out an analysis of the hydrogen bonds and of the salt bridges in a collection of 319 non-redundant protein-protein interfaces derived from high-quality x-ray structures. we found that the geometry of the hydrogen bonds across protein interfaces is generally less optimal and has a wider distribution than typically observed within the chains. this difference originates from the more hydrophilic side chains buried in the binding interface than in the folded monomer interior. protein folding differs from protein binding. whereas in folding practically all degrees of freedom are available to the chain to attain its optimal configuration, this is not the case for rigid binding, where the protein molecules are already folded, with only six degrees of translational and rotational freedom available to the chains to achieve their most favorable bound configuration. these constraints enforce many polar/charged residues buried in the interface to form weak hydrogen bonds with protein atoms, rather than strongly hydrogen bonding to the solvent. since interfacial hydrogen bonds are weaker than the intra-chain ones to compete with the binding of water, more water molecules are involved in bridging hydrogen bond networks across the protein interface than in the protein interior. interfacial water molecules both mediate non-complementary donor-donor or acceptor-acceptor pairs, and connect non-optimally oriented donor-acceptor pairs. these differences between the interfacial hydrogen bonding patterns and the intra-chain ones further substantiate the notion that protein complexes formed by rigid binding may be far away from the global minimum conformations. moreover, we summarize the pattern of charge complementarity and of the conservation of hydrogen bond network across binding interfaces. we further illustrate the utility of this study in understanding the specificity of protein-protein associations, and hence in docking prediction and molecular (inhibitor) design.\"\"bioinformatics program, boston university, boston, massachusetts 02215; department of cell biology, harvard medical school, boston, massachusetts 02115; yeast structural genomics, ibbmc universit\\\\\\'{e} paris-sud, cnrs umr 8619, 91405-orsay, france; department of biomedical engineering, boston university, boston, massachusetts 02215; program in bioinformatics and integrative biology, university of massachusetts medical school, worcester, massachusetts 01605\",protein–protein docking benchmark version 3.0,\"we present version 3.0 of our publicly available protein–protein docking benchmark. this update includes 40 new test cases, representing a 48\\\\% increase from benchmark 2.0. for all of the new cases, the crystal structures of both binding partners are available. as with benchmark 2.0, structural classification of proteins (murzin et al., j mol biol 1995;247:536–540) was used to remove redundant test cases. the 124 unbound-unbound test cases in benchmark 3.0 are classified into 88 rigid-body cases, 19 medium-difficulty cases, and 17 difficult cases, based on the degree of conformational change at the interface upon complex formation. in addition to providing the community with more test cases for evaluating docking methods, the expansion of benchmark 3.0 will facilitate the development of new algorithms that require a large number of training examples. benchmark 3.0 is available to the public at http://zlab.bu.edu/benchmark. proteins 2008. {\\\\copyright} 2008 {wiley-liss}, inc.\"\"multiple sequence alignments are often used to reveal functionally important residues within a protein family. they can be particularly useful for the identification of key residues that determine functional differences between protein subfamilies. we present a new entropy-based method, sequence harmony ({sh}) that accurately detects subfamily-specific positions from a multiple sequence alignment. the {sh} algorithm implements a novel formula, able to score compositional differences between subfamilies, without imposing conservation, in a simple manner on an intuitive scale. we compare our method with the most important published methods, i.e. {amas}, {treedet} and {sdp}-pred, using three well-studied protein families: the receptor-binding domain ({mh2}) of the smad family of transcription factors, the ras-superfamily of small {gtpases} and the {mip}-family of integral membrane transporters. we demonstrate that {sh} accurately selects known functional sites with higher coverage than the other methods for these test-cases. this shows that compositional differences between protein subfamilies provide sufficient basis for identification of functional sites. in addition, {sh} selects a number of sites of unknown function that could be interesting candidates for further experimental investigation.\"\"why do proteins evolve at different rates? advances in systems biology and genomics have facilitated a move from studying individual proteins to characterizing global cellular factors. systematic surveys indicate that protein evolution is not determined exclusively by selection on protein structure and function, but is also affected by the genomic position of the encoding genes, their expression patterns, their position in biological networks and possibly their robustness to mistranslation. recent work has allowed insights into the relative importance of these factors. we discuss the status of a much-needed coherent view that integrates studies on protein evolution with biochemistry and functional and structural genomics.\"\"bayesian statistics allow scientists to easily incorporate prior knowledge into their data analysis. nonetheless, the sheer amount of computational power that is required for bayesian statistical analyses has previously limited their use in genetics. these computational constraints have now largely been overcome and the underlying advantages of bayesian approaches are putting them at the forefront of genetic data analysis in an increasing number of areas.\"kyoto university, uji, kyoto 611-0011, japan. kanehisa@kuicr.kyoto-u.ac.jp\",the {kegg} resource for deciphering the genome,\"a grand challenge in the post‐genomic era is a complete computer representation of the cell and the organism, which will enable computational prediction of higher‐level complexity of cellular processes and organism behavior from genomic information. toward this end we have been developing a knowledge‐based approach for network prediction, which is to predict, given a complete set of genes in the genome, the protein interaction networks that are responsible for various cellular processes. {kegg} at http://www.genome.ad.jp/kegg/ is the reference knowledge base that integrates current knowledge on molecular interaction networks such as pathways and complexes ({pathway} database), information about genes and proteins generated by genome projects ({genes}/{ssdb}/{ko} databases) and information about biochemical compounds and reactions ({compound}/{glycan}/{reaction} databases). these three types of database actually represent three graph objects, called the protein network, the gene universe and the chemical universe. new efforts are being made to abstract knowledge, both computationally and manually, about ortholog clusters in the {ko} ({kegg} orthology) database, and to collect and analyze carbohydrate structures in the {glycan} database.\"model-based clustering of array {cgh} data.,the ridgeway, mill hill, london nw7 1aa, uk. ffranca@nimr.mrc.ac.uk\",parameter optimized surfaces ({pops}): analysis of key interactions and conformational changes in the ribosome.,\"we present a new method for the calculation of solvent accessible surface areas at the atomic and residue levels, which we call parameter optimized surfaces ({pops}-a and {pops}-r ). atomic and residue areas (the latter simulated with a single sphere centered at the c(alpha)s atom for amino acids and at the p atom for nucleotides) have been optimized versus accurate all-atoms methods. we concentrated on an analytical formula for the approximation of solvent accessibilities. the formula is simple, easily derivable and fast to compute, therefore it is practical for use in molecular dynamics simulations as an approximation to the first solvation shell. the residue based approach {pops}-r has been derived as a useful tool for the analysis of large macromolecular assemblies like the ribosome, and is especially suited for use in refinement of low resolution structures. the structures of the {70s}, {50s} and {30s} ribosomes have been analyzed in detail and most of the interactions within the subunits and at their interfaces were clearly identified. some interesting differences between {30s} alone and within the {70s} have been highlighted. owing to the presence of the {p-trna} in the {70s} ribosome, localized conformational rearrangements occur within the subunits, exposing arg and lys residues to negatively charged binding sites of {p-trna}. {pops}-r also allows for estimates of the loss of free energy of solvation upon complex formation, particularly useful in designing new {protein-rna} complexes and in suggesting more focused experimental work.\"\"the diverse range of cellular functions is performed by a limited number of protein folds existing in nature. one may similarly expect that cellular functional diversity would be covered by a limited number of protein-protein interface architectures. here, we present 8205 interface clusters, each representing a unique interface architecture. this data set of protein-protein interfaces is analyzed and compared with older data sets. we observe that the number of both biological and crystal interfaces increases significantly compared to the number of protein data bank entries. furthermore, we find that the number of distinct interface architectures grows at a much faster rate than the number of folds and is yet to level off. we further analyze the growth trend of the functional coverage by constructing functional interaction networks from interfaces. the functional coverage is also found to steadily increase. interestingly, we also observe that despite the diversity of interface architectures, some are more favorable and frequently used, and of particular interest, are the ones that are also preferred in single chains.\"\"high-throughput genotyping microarrays assess both total {dna} copy number and allelic composition, which makes them a tool of choice for copy number studies in cancer, including total copy number and loss of heterozygosity ({loh}) analyses. even after state of the art preprocessing methods, allelic signal estimates from genotyping arrays still suffer from systematic effects that make them difficult to use effectively for such downstream analyses. we propose a method, {tumorboost}, for normalizing allelic estimates of one tumor sample based on estimates from a single matched normal. the method applies to any paired tumor-normal estimates from any microarray-based technology, combined with any preprocessing method. we demonstrate that it increases the signal-to-noise ratio of allelic signals, making it significantly easier to detect allelic imbalances. {tumorboost} increases the power to detect somatic copy-number events (including copy-neutral {loh}) in the tumor from allelic signals of affymetrix or illumina origin. we also conclude that high-precision allelic estimates can be obtained from a single pair of tumor-normal hybridizations, if {tumorboost} is combined with single-array preprocessing methods such as (allele-specific) {crma} v2 for affymetrix or {beadstudio}\\'s (proprietary) {xy}-normalization method for illumina. a bounded-memory implementation is available in the open-source and cross-platform r package aroma.cn, which is part of the aroma project (http://www.aroma-project.org/).\"\"the kyoto encyclopedia of genes and genomes ({kegg}) is the primary database resource of the japanese {genomenet} service (http://www.genome.ad.jp/) for understanding higher order functional meanings and utilities of the cell or the organism from its genome information. {kegg} consists of the {pathway} database for the computerized knowledge on molecular interaction networks such as pathways and complexes, the {genes} database for the information about genes and proteins generated by genome sequencing projects, and the {ligand} database for the information about chemical compounds and chemical reactions that are relevant to cellular processes. in addition to these three main databases, limited amounts of experimental data for microarray gene expression profiles and yeast two-hybrid systems are stored in the {expression} and {brite} databases, respectively. furthermore, a new database, named {ssdb}, is available for exploring the universe of all protein coding genes in the complete genomes and for identifying functional links and ortholog groups. the data objects in the {kegg} databases are all represented as graphs and various computational methods are developed to detect graph features that can be related to biological functions. for example, the correlated clusters are graph similarities which can be used to predict a set of genes coding for a pathway or a complex, as summarized in the ortholog group tables, and the cliques in the {ssdb} graph are used to annotate genes. the {kegg} databases are updated daily and made freely available (http://www.genome.ad.jp/kegg/).\"single-cell dissection of transcriptional heterogeneity in human colon tumors,\"the complementarity of gene expression and {protein-dna} interaction data led to several successful models of biological systems. however, recent studies in multiple species raise doubts about the relationship between these two datasets. these studies show that the overwhelming majority of genes bound by a particular transcription factor ({tf}) are not affected when that factor is knocked out. here, we show that this surprising result can be partially explained by considering the broader cellular context in which {tfs} operate. factors whose functions are not backed up by redundant paralogs show a fourfold increase in the agreement between their bound targets and the expression levels of those targets. in addition, we show that incorporating protein interaction networks provides physical explanations for knockout effects. new double knockout experiments support our conclusions. our results highlight the robustness provided by redundant {tfs} and indicate that in the context of diverse cellular systems, binding is still largely functional.\"p.o. box 19498, arlington, tx 76019, usa. michalak@uta.edu\",\"coexpression, coregulation, and cofunctionality of neighboring genes in eukaryotic genomes\",\"accumulating evidence indicates that gene order in eukaryotic genomes is not completely random and that genes with similar expression levels tend to be clustered within the same genomic neighborhoods. the mechanism behind these gene coexpression clusters is as yet unclear. in this article, plausible biochemical, genetic, evolutionary, and technological determinants of this pattern are briefly reviewed.\"\"all cancers carry somatic mutations. the patterns of mutation in cancer genomes reflect the {dna} damage and repair processes to which cancer cells and their precursors have been exposed. to explore these mechanisms further, we generated catalogs of somatic mutation from 21 breast cancers and applied mathematical methods to extract mutational signatures of the underlying processes. multiple distinct single- and double-nucleotide substitution signatures were discernible. cancers with {brca1} or {brca2} mutations exhibited a characteristic combination of substitution mutation signatures and a distinctive profile of deletions. complex relationships between somatic mutation prevalence and transcription were detected. a remarkable phenomenon of localized hypermutation, termed \"\"kataegis,\"\" was observed. regions of kataegis differed between cancers but usually colocalized with somatic rearrangements. base substitutions in these regions were almost exclusively of cytosine at {tpc} dinucleotides. the mechanisms underlying most of these mutational signatures are unknown. however, a role for the {apobec} family of cytidine deaminases is proposed. copyright {\\\\copyright} 2012 elsevier inc. all rights reserved.\"national institutes of health bethesda, md 20894, usa.\",predicting protein–protein interaction by searching evolutionary tree automorphism space,\"motivation: uncovering the protein–protein interaction network is a fundamental step in the quest to understand the molecular machinery of a cell. this motivates the search for efficient computational methods for predicting such interactions. among the available predictors are those that are based on the co-evolution hypothesis  ” evolutionary trees of protein families (that are known to interact) are expected to have similar topologies”. many of these methods are limited by the fact that they can handle only a small number of protein sequences. also, details on evolutionary tree topology are missing as they use similarity matrices in lieu of the {trees.results}: we introduce {morph}, a new algorithm for predicting protein interaction partners between members of two protein families that are known to interact. our approach can also be seen as a new method for searching the best superposition of the corresponding evolutionary trees based on tree automorphism group. we discuss relevant facts related to the predictability of protein–protein interaction based on their co-evolution. when compared with related computational approaches, our method reduces the search space by ∼3 × 105-fold and at the same time increases the accuracy of predicting correct binding {partners.contact}: przytyck@mail.nih.gov\"10550 north torrey pines road, la jolla, ca 92037, usa.\",comparative study of several algorithms for flexible ligand docking.,\"we have performed a comparative assessment of several programs for flexible molecular docking: {dock} 4.0, {flexx} 1.8, {autodock} 3.0, {gold} 1.2 and {icm} 2.8. this was accomplished using two different studies: docking experiments on a data set of 37 protein-ligand complexes and screening a library containing 10,037 entries against 11 different proteins. the docking accuracy of the methods was judged based on the corresponding rank-one solutions. we have found that the fraction of molecules docked with acceptable accuracy is 0.47, 0.31, 0.35, 0.52 and 0.93 for, respectively, {autodock}, {dock}, {flexx}, {gold} and {icm}. thus {icm} provided the highest accuracy in ligand docking against these receptors. the results from the other programs are found to be less accurate and of approximately the same quality. a speed comparison demonstrated that {flexx} was the fastest and {autodock} was the slowest among the tested docking programs. the database screening was performed using {dock}, {flexx} and {icm}. {icm} was able to identify the original ligands within the top 1\\\\% of the total library in 17 cases. the corresponding number for {dock} and {flexx} was 7 and 8, respectively. we have estimated that in virtual database screening, 50\\\\% of the potentially active compounds will be found among approximately 1.5\\\\% of the top scoring solutions found with {icm} and among approximately 9\\\\% of the top scoring solutions produced by {dock} and {flexx}.\"los angeles, ca, 90095-1570, usa.\",three-dimensional cluster analysis identifies interfaces and functional residue clusters in proteins.,national centre for biotechnology (cnb-csic), madrid, spain.\",prediction of protein interaction based on similarity of phylogenetic trees.,\"computational methods for predicting protein interaction partners are becoming increasingly popular. many of them are mature enough to be widely used by molecular biologists who can look for proteins related to the protein of interest in order to infer information about its context in the cell. in this chapter we describe the use of the mirrortree set of programs and related software for predicting protein interactions. they are all based on the idea that interacting or functionally related proteins tend to show similar phylogenetic trees due to coevolution. the basic mirrortree program can be used to calculate the similarity between the phylogenetic trees implicit in the multiple sequence alignments of two protein families. the {ecid} database contains protein interactions and relationships from different computational and experimental sources for the model organism escherichia coli, including the ones generated with mirrortree. finally, the {tsema} server uses the concept of tree similarity between interacting families to look for the best mapping between two families of interacting proteins: which member in one family interacts with which member in the other.\"inc., laboratory of experimental and computational biology, national cancer institute, frederick, md 21702, usa.\",protein–protein interactions: structurally conserved residues distinguish between binding sites and exposed protein surfaces,\"polar residue hot spots have been observed at protein–protein binding sites. here we show that hot spots occur predominantly at the interfaces of macromolecular complexes, distinguishing binding sites from the remainder of the surface. consequently, hot spots can be used to define binding epitopes. we further show a correspondence between energy hot spots and structurally conserved residues. the number of structurally conserved residues, particularly of high ranking energy hot spots, increases with the binding site contact size. this finding may suggest that effectively dispersing hot spots within a large contact area, rather than compactly clustering them, may be a strategy to sustain essential key interactions while still allowing certain protein flexibility at the interface. thus, most conserved polar residues at the binding interfaces confer rigidity to minimize the entropic cost on binding, whereas surrounding residues form a flexible cushion. furthermore, our finding that similar residue hot spots occur across different protein families suggests that affinity and specificity are not necessarily coupled: higher affinity does not directly imply greater specificity. conservation of trp on the protein surface indicates a highly likely binding site. to a lesser extent, conservation of phe and met also imply a binding site. for all three residues, there is a significant conservation in binding sites, whereas there is no conservation on the exposed surface. a hybrid strategy, mapping sequence alignment onto a single structure illustrates the possibility of binding site identification around these three residues.\"\"motivated by genetic expression data, we introduce plaid models.  these are a form of two-sided cluster analysis that allows clusters to  overlap. using these models we nd interpretable structure in some  yeast {dna} data, as well as in some nutrition data and some foreign  exchange data.  key words: clustering, {dna} microarray, information retrieval  1 introduction  this article introduces the plaid model, a tool for exploratory analysis of multivariate data. the motivating application is the search for interpretable biological structure in gene expression microarray data. eisen, spellman, brown \\\\&amp; botstein (1998) is an early and inuential paper advocating the use of cluster methods to identify groups of co-regulated genes from microarray data. we present the model and illustrate it on gene expression and other data. the plaid model allows a gene to be in more than one cluster, or in none at all. it also allows a cluster of genes to be dened with respect to only a subset of samples, no...\"\"the {sv40} transcriptional enhancer is composed of separate 15 to 20 base-pair-(bp)-long enhancer elements that cooperate with one another or duplicates of themselves to enhance transcription. these elements are bipartite, being composed of subunits, called enhansons, that can be duplicated or interchanged to create new enhancer elements. enhansons differ from the enhancer elements because they are very sensitive to changes in spacing. this prototypic enhancer, therefore, contains two distinct levels of organization, each of which requires redundancy to be effective.\"mutational heterogeneity in cancer and the search for new cancer-associated genes,\"major international projects are underway that are aimed at creating a comprehensive catalogue of all the genes responsible for the initiation and progression of cancer. these studies involve the sequencing of matched tumour-normal samples followed by mathematical analysis to identify those genes in which mutations occur more frequently than expected by random chance. here we describe a fundamental problem with cancer genome studies: as the sample size increases, the list of putatively significant genes produced by current analytical methods burgeons into the hundreds. the list includes many implausible genes (such as those encoding olfactory receptors and the muscle protein titin), suggesting extensive false-positive findings that overshadow true driver events. we show that this problem stems largely from mutational heterogeneity and provide a novel analytical methodology, {mutsigcv}, for resolving the problem. we apply {mutsigcv} to exome sequences from 3,083 tumour-normal pairs and discover extraordinary variation in mutation frequency and spectrum within cancer types, which sheds light on mutational processes and disease aetiology, and in mutation frequency across the genome, which is strongly correlated with {dna} replication timing and also with transcriptional activity. by incorporating mutational heterogeneity into the analyses, {mutsigcv} is able to eliminate most of the apparent artefactual findings and enable the identification of genes truly associated with cancer.\"\"tumorigenesis in humans is thought to be a multistep process where certain mutations confer a selective advantage, allowing lineages derived from the mutated cell to outcompete other cells. although molecular cell biology has substantially advanced cancer research, our understanding of the evolutionary dynamics that govern tumorigenesis is limited. this paper analyzes the computational implications of cancer progression presented by hanahan and weinberg in the hallmarks of cancer. we model the complexities of tumor progression as a small set of underlying rules that govern the transformation of normal cells to tumor cells. the rules are implemented in a stochastic multistep model. the model predicts that (i) early-onset cancers proceed through a different sequence of mutation acquisition than late-onset cancers; (ii) tumor heterogeneity varies with acquisition of genetic instability, mutation pathway, and selective pressures during tumorigenesis; (iii) there exists an optimal initial telomere length which lowers cancer incidence and raises time of cancer onset; and (iv) the ability to initiate angiogenesis is an important stage-setting mutation, which is often exploited by other cells. the model offers insight into how the sequence of acquired mutations affects the timing and cellular makeup of the resulting tumor and how the cellular-level population dynamics drive neoplastic evolution. cancer can be viewed as an ecological system in which cells with different mutations compete for survival. in this work, the authors present a three-dimensional stochastic model of these complex interactions. each cell is represented as an autonomous agent that follows simple rules governing its behavior, where behaviors change as cells gain cancerous mutations. the paper explores the timing of cancer onset, the order in which mutations are acquired, the diversity of tumors, and the competition and cooperation between cells in the tumor microenvironment. one key finding is that early-onset and late-onset tumors take different mutational paths to cancer. the paper provides insight into the early dynamics of tumorigenesis currently inaccessible to experimental investigation.\"modelling vemurafenib resistance in melanoma reveals a strategy to forestall drug resistance.,hinxton, cambridge, united kingdom.\",pfam: a comprehensive database of protein domain families based on seed alignments.,\"databases of multiple sequence alignments are a valuable aid to protein sequence classification and analysis. one of the main challenges when constructing such a database is to simultaneously satisfy the conflicting demands of completeness on the one hand and quality of alignment and domain definitions on the other. the latter properties are best dealt with by manual approaches, whereas completeness in practice is only amenable to automatic methods. herein we present a database based on hidden markov model profiles ({hmms}), which combines high quality and completeness. our database, pfam, consists of parts a and b. {pfam-a} is curated and contains well-characterized protein domain families with high quality alignments, which are maintained by using manually checked seed alignments and {hmms} to find and align all members. {pfam-b} contains sequence families that were generated automatically by applying the domainer algorithm to cluster and align the remaining protein sequences after removal of {pfam-a} domains. by using pfam, a large number of previously unannotated proteins from the caenorhabditis elegans genome project were classified. we have also identified many novel family memberships in known proteins, including new kazal, fibronectin type {iii}, and response regulator receiver domains. {pfam-a} families have permanent accession numbers and form a library of {hmms} available for searching and automatic annotation of new protein sequences.\"interaction-site prediction for protein complexes: a critical assessment.,\"progress in uncovering the protein interaction networks of several species has led to questions of what underlying principles might govern their organization. few studies have tried to determine the impact of protein interaction network evolution on the observed physiological differences between species. using comparative genomics and structural information, we show here that eukaryotic species have rewired their interactomes at a fast rate of approximately 10 \\\\^{a}ˆ\\'5 interactions changed per protein pair, per million years of divergence. for homo sapiens this corresponds to 10 3 interactions changed per million years. additionally we find that the specificity of binding strongly determines the interaction turnover and that different biological processes show significantly different link dynamics. in particular, human proteins involved in immune response, transport, and establishment of localization show signs of positive selection for change of interactions. our analysis suggests that a small degree of molecular divergence can give rise to important changes at the network level. we propose that the power law distribution observed in protein interaction networks could be partly explained by the cell\\'s requirement for different degrees of protein binding specificity.\"\"center for advanced research in biotechnology, university of maryland biotechnology institute, rockville, maryland 20850; genome center, university of california, davis, california 95616; department of biochemistry and mol biophysics, cubic, c2b2, nesg, columbia university, new york, new york 10032; istituto pasteur-fondazione cenci bolognetti, university of rome ldquola sapienzardquo, p.le aldo moro 5, rome 00185, italy\",critical assessment of methods of protein structure {prediction—round} {viii},\"abstract this article is an introduction to the special issue of the journal proteins, dedicated to the eighth {casp} experiment to assess the state of the art in protein structure prediction. the article describes the conduct of the experiment, the categories of prediction included, and outlines the evaluation and assessment procedures. highlights are the first blind assessment of model refinement methods showing that under some circumstances substantial model improvements are possible; improvements in the performance of methods for determining the accuracy of a model; and some progress in the accuracy of comparative models in regions not present in a principal template. against these advances must be stacked the fact that there is no detectable progress in model quality compared with {casp7} in either template-based or template free modeling, using the established {casp} measures. proteins 2009. {\\\\copyright} 2009 {wiley-liss}, inc.\"\"main text {introductionwe} have proposed that six hallmarks of cancer together constitute an organizing principle that provides a logical framework for understanding the remarkable diversity of neoplastic diseases (hanahan and weinberg, 2000). implicit in our discussion was the notion that as normal cells evolve progressively to a neoplastic state, they acquire a succession of these hallmark capabilities, and that the multistep process of human tumor pathogenesis could be rationalized by the need of incipient cancer cells to acquire the traits that enable them to become tumorigenic and ultimately malignant.\"\"bioinformatics program, boston university, boston, massachusetts 02215, usa.\",atomic contact vectors in protein-protein recognition,\"the ability to analyze and compare protein-protein interactions on the structural level is critical to our understanding of various aspects of molecular recognition and the functional interplay of components of biochemical networks. in this study, we introduce atomic contact vectors ({acvs}) as an intuitive way to represent the physico-chemical characteristics of a protein-protein interface as well as a way to compare interfaces to each other. we test the utility of {acvs} in classification by using them to distinguish between homodimers and crystal contacts. our results compare favorably with those reported by other authors. we then apply {acvs} to mine the {pdb} for all known protein-protein complexes and separate transient recognition complexes from permanent oligomeric ones. getting at the basis of this difference is important for our understanding of recognition and we achieved a success rate of 91\\\\% for distinguishing these two classes of complexes. although accessible surface area of the interface is a major discriminating feature, we also show that there are distinct differences in the contact preferences between the two kinds of complexes. illustrating the superiority of {acvs} as a basic comparison measure over a sequence-based approach, we derive a general rule of thumb to determine whether two protein-protein interfaces are redundant. with this method, we arrive at a nonredundant set of 209 recognition complexes—the largest set reported so far. proteins 2003. {\\\\copyright} 2003 {wiley-liss}, inc.\"\"protein-protein interaction networks provide a global picture of cellular function and biological processes. some proteins act as hub proteins, highly connected to others, whereas some others have few interactions. the dysfunction of some interactions causes many diseases, including cancer. proteins interact through their interfaces. therefore, studying the interface properties of cancer-related proteins will help explain their role in the interaction networks. similar or overlapping binding sites should be used repeatedly in single interface hub proteins, making them promiscuous. alternatively, multi-interface hub proteins make use of several distinct binding sites to bind to different partners. we propose a methodology to integrate protein interfaces into cancer interaction networks ({cispin}, cancer structural protein interface network). the interactions in the human protein interaction network are replaced by interfaces, coming from either known or predicted complexes. we provide a detailed analysis of cancer related human protein-protein interfaces and the topological properties of the cancer network. the results reveal that cancer-related proteins have smaller, more planar, more charged and less hydrophobic binding sites than non-cancer proteins, which may indicate low affinity and high specificity of the cancer-related interactions. we also classified the genes in {cispin} according to phenotypes. within phenotypes, for breast cancer, colorectal cancer and leukemia, interface properties were found to be discriminating from non-cancer interfaces with an accuracy of 71\\\\%, 67\\\\%, 61\\\\%, respectively. in addition, cancer-related proteins tend to interact with their partners through distinct interfaces, corresponding mostly to multi-interface hubs, which comprise 56\\\\% of cancer-related proteins, and constituting the nodes with higher essentiality in the network (76\\\\%). we illustrate the interface related affinity properties of two cancer-related hub proteins: erbb3, a multi interface, and raf1, a single interface hub. the results reveal that affinity of interactions of the multi-interface hub tends to be higher than that of the single-interface hub. these findings might be important in obtaining new targets in cancer as well as finding the details of specific binding regions of putative cancer drug candidates.\"d-38304 wolfenb\\\\\"\"{u}ttel, germany. vma@biobase.de\",\"{transfac}: transcriptional regulation, from patterns to profiles.\",an integrated genomic analysis of human glioblastoma multiforme.,\"glioblastoma multiforme ({gbm}) is the most common and lethal type of brain cancer. to identify the genetic alterations in {gbms}, we sequenced 20,661 protein coding genes, determined the presence of amplifications and deletions using high-density oligonucleotide arrays, and performed gene expression analyses using next-generation sequencing technologies in 22 human tumor samples. this comprehensive analysis led to the discovery of a variety of genes that were not known to be altered in {gbms}. most notably, we found recurrent mutations in the active site of isocitrate dehydrogenase 1 ({idh1}) in 12\\\\% of {gbm} patients. mutations in {idh1} occurred in a large fraction of young patients and in most patients with secondary {gbms} and were associated with an increase in overall survival. these studies demonstrate the value of unbiased genomic analyses in the characterization of human brain cancer and identify a potentially useful genetic alteration for the classification and targeted therapy of {gbms}.\"\"summary: phangorn is a package for phylogenetic reconstruction and analysis in the r language. previously it was only possible to estimate phylogenetic trees with distance methods in r. phangorn, now offers the possibility of reconstructing phylogenies with distance based methods, maximum parsimony or maximum likelihood ({ml}) and performing hadamard conjugation. extending the general {ml} framework, this package provides the possibility of estimating mixture and partition models. furthermore, phangorn offers several functions for comparing trees, phylogenetic models or splits, simulating character data and performing congruence analyses.\"\"boston university bioinformatics program, boston, massachusetts 02215, usa.\",protein–protein docking benchmark 2.0: an update,\"we present a new version of the {protein–protein} docking benchmark, reconstructed from the bottom up to include more complexes, particularly focusing on more unbound–unbound test cases. {scop} (structural classification of proteins) was used to assess redundancy between the complexes in this version. the new benchmark consists of 72 unbound–unbound cases, with 52 rigid-body cases, 13 medium-difficulty cases, and 7 high-difficulty cases with substantial conformational change. in addition, we retained 12 antibody–antigen test cases with the antibody structure in the bound form. the new benchmark provides a platform for evaluating the progress of docking methods on a wide variety of targets. the new version of the benchmark is available to the public at http://zlab.bu.edu/benchmark2. proteins 2005;60:214–216. {\\\\copyright} 2005 {wiley-liss}, inc.\"\"{background}:several entropy-based methods have been developed for scoring sequence conservation in protein multiple sequence alignments. high scoring amino acid positions may correlate with structurally or functionally important residues. however, amino acid background frequencies are usually not taken into account in these entropy-based scoring {schemes.results}:we demonstrate that using a relative entropy measure that incorporates amino acid background frequency results in improved performance in identifying functional sites from protein multiple sequence {alignments.conclusion}:our results suggest that the application of appropriate background frequency information may lead to more biologically relevant results in many areas of bioinformatics.\"\"larger organisms have more potentially carcinogenic cells, tend to live longer and require more ontogenic cell divisions. therefore, intuitively one might expect cancer incidence to scale with body size. evidence from mammals, however, suggests that the cancer risk does not correlate with body size. this observation defines  ” peto\\'s paradox.” here, we propose a novel hypothesis to resolve peto\\'s paradox. we suggest that malignant tumors are disadvantaged in larger hosts. in particular, we hypothesize that natural selection acting on competing phenotypes among the cancer cell population will tend to favor aggressive  ” cheaters” that then grow as a tumor on their parent tumor, creating a hypertumor that damages or destroys the original neoplasm. in larger organisms, tumors need more time to reach lethal size, so hypertumors have more time to evolve. so, in large organisms, cancer may be more common and less lethal. we illustrate this hypothesis in silico using a previously published hypertumor model. results from the model predict that malignant neoplasms in larger organisms should be disproportionately necrotic, aggressive, and vascularized than deadly tumors in small mammals. these predictions may serve as the basis on which to test the hypothesis, but to our knowledge, no one has yet performed a systematic investigation of comparative necrosis, histopathology, or vascularization among mammalian cancers.\"\"our results suggest that expression profiles can be used in the computational identification of functional {mirna}-target associations. one can expect a higher chance of finding negatively correlated expression profiles for {targetscan}-predicted interactions than for {mirbase}-predicted ones. with limited experimentally validated {mirna}-target interactions, expression profiles can only serve as a supplementary role in finding interactions between {mirnas} and {mrnas}.\"{peaks}: identification of regulatory motifs by their position in {dna} sequences.,\"many {dna} functional motifs tend to accumulate or cluster at specific gene locations. these locations can be detected, in a group of gene sequences, as high frequency \\'peaks\\' with respect to a reference position, such as the transcription start site ({tss}). we have developed a web tool for the identification of regions containing significant motif peaks. we show, by using different yeast gene datasets, that peak regions are strongly enriched in experimentally-validated motifs and contain potentially important novel motifs. http://genomics.imim.es/peaks\"katholieke universiteit leuven, leuven-heverlee, belgium. qizheng.sheng@esat.kuleuven.ac.be\",biclustering microarray data by gibbs sampling.,\"{motivation}: gibbs sampling has become a method of choice for the discovery of noisy patterns, known as motifs, in {dna} and protein sequences. because handling noise in microarray data presents similar challenges, we have adapted this strategy to the biclustering of discretized microarray data. {results}: in contrast with standard clustering that reveals genes that behave similarly over all the conditions, biclustering groups genes over only a subset of conditions for which those genes have a sharp probability distribution. we have opted for a simple probabilistic model of the biclusters because it has the key advantage of providing a transparent probabilistic interpretation of the biclusters in the form of an easily interpretable fingerprint. furthermore, gibbs sampling does not suffer from the problem of local minima that often characterizes {expectation-maximization}. we demonstrate the effectiveness of our approach on two synthetic data sets as well as a data set from leukemia patients.\"\"ovarian clear cell carcinoma ({occc}) is an aggressive human cancer that is generally resistant to therapy. to explore the genetic origin of {occc}, we determined the exomic sequences of eight tumors after immunoaffinity purification of cancer cells. through comparative analyses of normal cells from the same patients, we identified four genes that were mutated in at least two tumors. {pik3ca}, which encodes a subunit of phosphatidylinositol-3 kinase, and {kras}, which encodes a well-known oncoprotein, had previously been implicated in {occc}. the other two mutated genes were previously unknown to be involved in {occc}: {ppp2r1a} encodes a regulatory subunit of serine/threonine phosphatase 2, and {arid1a} encodes adenine-thymine ({at})–rich interactive domain–containing protein {1a}, which participates in chromatin remodeling. the nature and pattern of the mutations suggest that {ppp2r1a} functions as an oncogene and {arid1a} as a tumor-suppressor gene. in a total of 42 {occcs}, 7\\\\% had mutations in {ppp2r1a} and 57\\\\% had mutations in {arid1a}. these results suggest that aberrant chromatin remodeling contributes to the pathogenesis of {occc}.\"\"protein-protein interactions are critical for cellular functions. recently developed computational approaches for predicting protein-protein interactions utilize co-evolutionary information of the interacting partners, e.g., correlations between distance matrices, where each matrix stores the pairwise distances between a protein and its orthologs from a group of reference genomes. we proposed a novel, simple method to account for some of the intra-matrix correlations in improving the prediction accuracy. specifically, the phylogenetic species tree of the reference genomes is used as a guide tree for hierarchical clustering of the orthologous proteins. the distances between these clusters, derived from the original pairwise distance matrix using the neighbor joining algorithm, form intermediate distance matrices, which are then transformed and concatenated into a super phylogenetic vector. a support vector machine is trained and tested on pairs of proteins, represented as super phylogenetic vectors, whose interactions are known. the performance, measured as {roc} score in cross validation experiments, shows significant improvement of our method ({roc} score 0.8446) over that of using pearson correlations (0.6587). we have shown that the phylogenetic tree can be used as a guide to extract intra-matrix correlations in the distance matrices of orthologous proteins, where these correlations are represented as intermediate distance matrices of the ancestral orthologous proteins. both the unsupervised and supervised learning paradigms benefit from the explicit inclusion of these intermediate distance matrices, and particularly so in the latter case, which offers a better balance between sensitivity and specificity in the prediction of protein-protein interactions.\"frederick, maryland 21702, usa.\",studies of protein-protein interfaces: a statistical analysis of the hydrophobic effect,\"data sets of 362 structurally nonredundant protein-protein interfaces and of 57 symmetry-related oligomeric interfaces have been used to explore whether the hydrophobic effect that guides protein folding is also the main driving force for protein-protein associations. the buried nonpolar surface area has been used to measure the hydrophobic effect. our analysis indicates that, although the hydrophobic effect plays a dominant role in protein-protein binding, it is not as strong as that observed in the interior of protein monomers. comparison of the interiors of the monomers with those of the interfaces reveals that, in general, the hydrophobic amino acids are more frequent in the interior of the monomers than in the interior of the protein-protein interfaces. on the other hand, a higher proportion of charged and polar residues are buried at the interfaces, suggesting that hydrogen bonds and ion pairs contribute more to the stability of protein binding than to that of protein folding. moreover, comparison of the interior of the interfaces to protein surfaces indicates that the interfaces are poorer in polar/charged than the surfaces and are richer in hydrophobic residues. the interior of the interfaces appears to constitute a compromise between the stabilization contributed by the hydrophobic effect on the one hand and avoiding patches on the protein surfaces that are too hydrophobic on the other. such patches would be unfavorable for the unassociated monomers in solution. we conclude that, although the types of interactions are similar between protein-protein interfaces and single-chain proteins overall, the contribution of the hydrophobic effect to protein-protein associations is not as strong as to protein folding. this implies that packing patterns and interatom, or interresidue, pairwise potential functions, derived from monomers, are not ideally suited to predicting and assessing ligand associations or design. these would perform adequately only in cases where the hydrophobic effect at the binding site is substantial.\"germany.\",a structural perspective on protein-protein interactions.,{bivisu}: software tool for bicluster detection and visualization.,\"{bivisu} is an open-source software tool for detecting and visualizing biclusters embedded in a gene expression matrix. through the use of appropriate coherence relations, {bivisu} can detect constant, constant-row, constant-column, additive-related as well as multiplicative-related biclusters. the biclustering results are then visualized under a {2d} setting for easy inspection. in particular, parallel coordinate ({pc}) plots for each bicluster are displayed, from which objective and subjective cluster quality evaluation can be performed. availability: {bivisu} has been developed in matlab and is available at {http://www.eie.polyu.edu.hk/\\\\~{}nflaw/biclustering}/.\"swiss federal institute of technology, eth-h\\\\\"\"{o}nggerberg, 8093 z\\\\\"\"{u}rich, switzerland; groningen biomolecular sciences and biotechnology institute (gbb), department of biophysical chemistry, university of groningen, nijenborgh 4, 9747 ag groningen, the netherlands\",a biomolecular force field based on the free enthalpy of hydration and solvation: the {gromos} force-field parameter sets {53a5} and {53a6},\"successive parameterizations of the {gromos} force field have been used successfully to simulate biomolecular systems over a long period of time. the continuing expansion of computational power with time makes it possible to compute ever more properties for an increasing variety of molecular systems with greater precision. this has led to recurrent parameterizations of the {gromos} force field all aimed at achieving better agreement with experimental data. here we report the results of the latest, extensive reparameterization of the {gromos} force field. in contrast to the parameterization of other biomolecular force fields, this parameterization of the {gromos} force field is based primarily on reproducing the free enthalpies of hydration and apolar solvation for a range of compounds. this approach was chosen because the relative free enthalpy of solvation between polar and apolar environments is a key property in many biomolecular processes of interest, such as protein folding, biomolecular association, membrane formation, and transport over membranes. the newest parameter sets, {53a5} and {53a6}, were optimized by first fitting to reproduce the thermodynamic properties of pure liquids of a range of small polar molecules and the solvation free enthalpies of amino acid analogs in cyclohexane ({53a5}). the partial charges were then adjusted to reproduce the hydration free enthalpies in water ({53a6}). both parameter sets are fully documented, and the differences between these and previous parameter sets are discussed. {\\\\copyright} 2004 wiley periodicals, inc. j comput chem 25: 1656–1676, 2004\"comprehensive molecular portraits of human breast tumours.,\"we analysed primary breast cancers by genomic {dna} copy number arrays, {dna} methylation, exome sequencing, messenger {rna} arrays, {microrna} sequencing and reverse-phase protein arrays. our ability to integrate information across platforms provided key insights into previously defined gene expression subtypes and demonstrated the existence of four main breast cancer classes when combining data from five platforms, each of which shows significant molecular heterogeneity. somatic mutations in only three genes ({tp53}, {pik3ca} and {gata3}) occurred at >10\\\\% incidence across all breast cancers; however, there were numerous subtype-associated and novel gene mutations including the enrichment of specific mutations in {gata3}, {pik3ca} and {map3k1} with the luminal a subtype. we identified two novel protein-expression-defined subgroups, possibly produced by stromal/microenvironmental elements, and integrated analyses identified specific signalling pathways dominant in each molecular subtype including a {her2}/phosphorylated {her2}/{egfr}/phosphorylated {egfr} signature within the {her2}-enriched expression subtype. comparison of basal-like breast tumours with high-grade serous ovarian tumours showed many molecular commonalities, indicating a related aetiology and similar therapeutic opportunities. the biological finding of the four main breast cancer subtypes caused by different subsets of genetic and epigenetic abnormalities raises the hypothesis that much of the clinically observable plasticity and heterogeneity occurs within, and not across, these major biological subtypes of breast cancer.\"\"complex interactions between genes or proteins contribute substantially to phenotypic evolution. we present a probabilistic model and a maximum likelihood approach for cross-species clustering analysis and for identification of conserved as well as species-specific co-expression modules. this model enables a  ” soft” cross-species clustering ({scsc}) approach by encouraging but not enforcing orthologous genes to be grouped into the same cluster. {scsc} is therefore robust to obscure orthologous relationships and can reflect different functional roles of orthologous genes in different species. we generated a time-course gene expression dataset for differentiating mouse embryonic stem ({es}) cells, and compiled a dataset of published gene expression data on differentiating human {es} cells. applying {scsc} to analyze these datasets, we identified conserved and species-specific gene regulatory modules. together with {protein-dna} binding data, an {scsc} cluster specifically induced in murine {es} cells indicated that the {klf2}/4/5 transcription factors, although critical to maintaining the pluripotent phenotype in mouse {es} cells, were decoupled from the {oct4}/{sox2}/{nanog} regulatory module in human {es} cells. two of the target genes of murine {klf2}/4/5, {lin28} and {nodal}, were rewired to be targets of {oct4}/{sox2}/{nanog} in human {es} cells. moreover, by mapping {scsc} clusters onto {kegg} signaling pathways, we identified the signal transduction components that were induced in pluripotent {es} cells in either a conserved or a species-specific manner. these results suggest that the pluripotent cell identity can be established and maintained through more than one gene regulatory network. a major goal in biology is to understand the evolution of complex traits, such as the development of multicellular body plans. to a certain extent, complex traits are governed by regulated gene expression. the comparison expression data between species requires extra considerations than sequence comparison, because gene expression is not static and the level of expression is influenced by external conditions. considering that co-expression patterns are often comparable across species, we developed a statistical model for cross-species clustering analysis. the model allows each species to create its own clusters of the genes but also encourages the species to borrow strength from each others\\' clusters of orthologous genes. the result is a pairing of clusters, one from each species, where the paired clusters share many but not necessarily all orthologous genes. the model-based approach not only reduces subjective influence but also enables effective use of evolutionary dependence. applying this model to analyze human and mouse embryonic stem ({es}) cell data, we identified the transcription factors and the signaling proteins that are specifically expressed in either human or mouse {es} cells. these results suggest that the pluripotent cell identity can be established and maintained through more than one gene regulatory network.\"usa\",biclustering algorithms for biological data analysis: a survey,\"a large number of clustering approaches have been proposed for the analysis of gene expression data obtained from microarray experiments. however, the results from the application of standard clustering methods to genes are limited. this limitation is imposed by the existence of a number of experimental conditions where the activity of genes is uncorrelated. a similar limitation exists when clustering of conditions is performed. for this reason, a number of algorithms that perform simultaneous clustering on the row and column dimensions of the data matrix has been proposed. the goal is to find submatrices, that is, subgroups of genes and subgroups of conditions, where the genes exhibit highly correlated activities for every condition. in this paper, we refer to this class of algorithms as biclustering. biclustering is also referred in the literature as coclustering and direct clustering, among others names, and has also been used in fields such as information retrieval and data mining. in this comprehensive survey, we analyze a large number of existing approaches to biclustering, and classify them in accordance with the type of biclusters they can find, the patterns of biclusters that are discovered, the methods used to perform the search, the approaches used to evaluate the solution, and the target applications.\"\"biclustering extends the traditional clustering techniques by attempting to find (all) subgroups of genes with similar expression patterns under to-be-identified subsets of experimental conditions when applied to gene expression data. still the real power of this clustering strategy is yet to be fully realized due to the lack of effective and efficient algorithms for reliably solving the general biclustering problem. we report a {qualitative} {biclustering} algorithm ({qubic}) that can solve the biclustering problem in a more general form, compared to existing algorithms, through employing a combination of qualitative (or semi-quantitative) measures of gene expression data and a combinatorial optimization technique. one key unique feature of the {qubic} algorithm is that it can identify all statistically significant biclusters including biclusters with the so-called \\'scaling patterns\\', a problem considered to be rather challenging; another key unique feature is that the algorithm solves such general biclustering problems very efficiently, capable of solving biclustering problems with tens of thousands of genes under up to thousands of conditions in a few minutes of the {cpu} time on a desktop computer. we have demonstrated a considerably improved biclustering performance by our algorithm compared to the existing algorithms on various benchmark sets and data sets of our own. {qubic} was written in {ansi} c and tested using {gcc} (version 4.1.2) on linux. its source code is available at: http://csbl.bmb.uga.edu/∼maqin/bicluster. a server version of {qubic} is also available upon request.\"clonal evolution in breast cancer revealed by single nucleus genome sequencing,\"the presence of multiple subclones within tumors mandates understanding of longitudinal and spatial subclonal dynamics. resolving the spatial and temporal heterogeneity of subclones with cancer driver events may offer insight into therapy response, tumor evolutionary histories and clinical trial design.\"\"we used a novel method based on allele-specific quantitative polymerase chain reaction (intplex) for the analysis of circulating cell.free {dna} ({ccfdna}) to compare total {ccfdna} and {kras}- or {braf}-mutated {ccfdna} concentrations in blood samples from mice xenografted with the human {sw620} colorectal cancer ({crc}) cell line and from patients with {crc}. intplex enables single-copy detection of variant alleles down to a sensitivity of ≥0.005 mutant to wild-type ratio. the proportion of mutant allele corresponding to the percentage of tumor-derived {ccfdna} was elevated in xenografted mice with {kras} homozygous mutation and varied highly from 0.13\\\\% to 68.7\\\\% in samples from mutation-positive {crc} patients (n = 38). mutant {ccfdna} alleles were quantified in the plasma of every patient at stages {ii}/{iii} and {iv} with a mean of 8.4\\\\% (median, 8.4\\\\%) and 21.8\\\\% (median, 12.4\\\\%), respectively. twelve of 38 (31.6\\\\%) and 5 of 38 (13.2\\\\%) samples showed a mutation load higher than 25\\\\%and 50\\\\%, respectively. this suggests that an important part of {ccfdna} may originate from tumor cells. in addition, we observed that tumor-derived (mutant) {ccfdna} was more fragmented than {ccfdna} from normal tissues. this observation suggests that the form of tumor-derived and normal {ccfdna} could differ. our approach revealed that allelic dilution is much less pronounced than previously stated, considerably facilitating the noninvasive molecular analysis of tumors.\"baltimore, md 21205, usa. rafa@jhu.edu\",summaries of affymetrix {genechip} probe level data.,\"high density oligonucleotide array technology is widely used in many areas of biomedical research for quantitative and highly parallel measurements of gene expression. affymetrix {genechip} arrays are the most popular. in this technology each gene is typically represented by a set of 11-20 pairs of probes. in order to obtain expression measures it is necessary to summarize the probe level data. using two extensive spike-in studies and a dilution study, we developed a set of tools for assessing the effectiveness of expression measures. we found that the performance of the current version of the default expression measure provided by affymetrix microarray suite can be significantly improved by the use of probe level summaries derived from empirically motivated statistical models. in particular, improvements in the ability to detect differentially expressed genes are demonstrated.\"\"reactome, located at http://www.reactome.org is a curated, peer-reviewed resource of human biological processes. given the genetic makeup of an organism, the complete set of possible reactions constitutes its reactome. the basic unit of the reactome database is a reaction; reactions are then grouped into causal chains to form pathways. the reactome data model allows us to represent many diverse processes in the human system, including the pathways of intermediary metabolism, regulatory pathways, and signal transduction, and high-level processes, such as the cell cycle. reactome provides a qualitative framework, on which quantitative data can be superimposed. tools have been developed to facilitate custom data entry and annotation by expert biologists, and to allow visualization and exploration of the finished dataset as an interactive process map. although our primary curational domain is pathways from homo sapiens, we regularly create electronic projections of human pathways onto other organisms via putative orthologs, thus making reactome relevant to model organism research communities. the database is publicly available under open source terms, which allows both its content and its software infrastructure to be freely used and redistributed.\"\"biophysics department, max planck institute of medical research, 6900 heidelberg, federal republic of germany\",dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features.,\"for a successful analysis of the relation between amino acid sequence and protein structure, an unambiguous and physically meaningful definition of secondary structure is essential. we have developed a set of simple and physically motivated criteria for secondary structure, programmed as a pattern-recognition process of hydrogen-bonded and geometrical features extracted from x-ray coordinates. cooperative secondary structure is recognized as repeats of the elementary hydrogen-bonding patterns  ” turn” and  ” bridge.” repeating turns are  ” helices,” repeating bridges are  ” ladders,” connected ladders are  ” sheets.” geometric structure is defined in terms of the concepts torsion and curvature of differential geometry. local chain  ” chirality” is the torsional handedness of four consecutive cα positions and is positive for right-handed helices and negative for ideal twisted β-sheets. curved pieces are defined as  ” bends.” solvent  ” exposure” is given as the number of water molecules in possible contact with a residue. the end result is a compilation of the primary structure, including {ss} bonds, secondary structure, and solvent exposure of 62 different globular proteins. the presentation is in linear form: strip graphs for an overall view and strip tables for the details of each of 10.925 residues. the dictionary is also available in computer-readable form for protein structure prediction work.\"\"the university of california, santa cruz genome browser (http://genome.ucsc.edu) offers online access to a database of genomic sequence and annotation data for a wide variety of organisms. the browser also has many tools for visualizing, comparing and analyzing both publicly available and user-generated genomic data sets, aligning sequences and uploading user data. among the features released this year are a gene search tool and annotation track drag-reorder functionality as well as support for {bam} and {bigwig}/{bigbed} file formats. new display enhancements include overlay of multiple wiggle tracks through use of transparent coloring, options for displaying transformed wiggle data, a \\'mean+whiskers\\' windowing function for display of wiggle data at high zoom levels, and more color schemes for microarray data. new data highlights include seven new genome assemblies, a neandertal genome data portal, phenotype and disease association data, a human {rna} editing track, and a zebrafish conservation track. we also describe updates to existing tracks.\"national institutes of health, bethesda, md, 20894, usa. sherry@ncbi.nlm.nih.gov\",{dbsnp}: the {ncbi} database of genetic variation.,\"in response to a need for a general catalog of genome variation to address the large-scale sampling designs required by association studies, gene mapping and evolutionary biology, the national center for biotechnology information ({ncbi}) has established the {dbsnp} database [{s.t}.sherry, {m.ward} and k. sirotkin (1999) genome res., 9, 677-679]. submissions to {dbsnp} will be integrated with other sources of information at {ncbi} such as {genbank}, {pubmed}, {locuslink} and the human genome project data. the complete contents of {dbsnp} are available to the public at website: {http://www.ncbi.nlm.nih.gov/snp}. the complete contents of {dbsnp} can also be downloaded in multiple formats via anonymous {ftp} at ftp://ncbi.nlm.nih.gov/snp/.\"houston, texas 77030, usa.\",a switch from high-fidelity to error-prone {dna} double-strand break repair underlies stress-induced mutation.,\"it has been observed that the evolutionary distances of interacting proteins often display a higher level of similarity than those of noninteracting proteins. this finding indicates that interacting proteins are subject to common evolutionary constraints and constitutes the basis of a method to predict protein interactions known as mirrortree. it has been difficult, however, to identify the direct cause of the observed similarities between evolutionary trees. one possible explanation is the existence of compensatory mutations between partners\\' binding sites to maintain proper binding. this explanation, though, has been recently challenged, and it has been suggested that the signal of correlated evolution uncovered by the mirrortree method is unrelated to any correlated evolution between binding sites. we examine the contribution of binding sites to the correlation between evolutionary trees of interacting domains. we show that binding neighborhoods of interacting proteins have, on average, higher coevolutionary signal compared with the regions outside binding sites; however, when the binding neighborhood is removed, the remaining domain sequence still contains some coevolutionary signal. in conclusion, the correlation between evolutionary trees of interacting domains cannot exclusively be attributed to the correlated evolution of the binding sites or to common evolutionary pressure exerted on the whole protein domain sequence, each of which contributes to the signal measured by the mirrortree approach.\"london, england.\",principles of protein-protein interactions.,\"this review examines protein complexes in the brookhaven protein databank to gain a better understanding of the principles governing the interactions involved in protein-protein recognition. the factors that influence the formation of protein-protein complexes are explored in four different types of protein-protein complexes--homodimeric proteins, heterodimeric proteins, enzyme-inhibitor complexes, and antibody-protein complexes. the comparison between the complexes highlights differences that reflect their biological roles.\"kyoto university, kyoto 606-8502, japan.\",{mafft}: a novel method for rapid multiple sequence alignment based on fast fourier transform,\"a multiple sequence alignment program, {mafft}, has been developed. the {cpu} time is drastically reduced as compared with existing methods. {mafft} includes two novel techniques. (i) homo logous regions are rapidly identified by the fast fourier transform ({fft}), in which an amino acid sequence is converted to a sequence composed of volume and polarity values of each amino acid residue. (ii) we propose a simplified scoring system that performs well for reducing {cpu} time and increasing the accuracy of alignments even for sequences having large insertions or extensions as well as distantly related sequences of similar length. two different heuristics, the progressive method ({fft}‐{ns}‐2) and the iterative refinement method ({fft}‐{ns}‐i), are implemented in {mafft}. the performances of {fft}‐{ns}‐2 and {fft}‐{ns}‐i were compared with other methods by computer simulations and benchmark tests; the {cpu} time of {fft}‐{ns}‐2 is drastically reduced as compared with {clustalw} with comparable accuracy. {fft}‐{ns}‐i is over 100 times faster than {t‐coffee}, when the number of input sequences exceeds 60, without sacrificing the accuracy.\"campus box 8501, 4444 forest park avenue, st louis, missouri 63108, usa. waterston@gs.washington.edu\",initial sequencing and comparative analysis of the mouse genome,\"the sequence of the mouse genome is a key informational tool for understanding the contents of the human genome and a key experimental tool for biomedical research. here, we report the results of an international collaboration to produce a high-quality draft sequence of the mouse genome. we also present an initial comparative analysis of the mouse and human genomes, describing some of the insights that can be gleaned from the two sequences. we discuss topics including the analysis of the evolutionary forces shaping the size, structure and sequence of the genomes; the conservation of large-scale synteny across most of the genomes; the much lower extent of sequence orthology covering less than half of the genomes; the proportions of the genomes under selection; the number of protein-coding genes; the expansion of gene families related to reproduction and immunity; the evolution of proteins; and the identification of intraspecies polymorphism.\"rockefeller university, new york, ny 10021, usa.\",comparative protein structure modeling of genes and genomes,\"▪ abstract\\u2002comparative modeling predicts the three-dimensional structure of a given protein sequence (target) based primarily on its alignment to one or more proteins of known structure (templates). the prediction process consists of fold assignment, target–template alignment, model building, and model evaluation. the number of protein sequences that can be modeled and the accuracy of the predictions are increasing steadily because of the growth in the number of known protein structures and because of the improvements in the modeling software. further advances are necessary in recognizing weak sequence–structure similarities, aligning sequences with structures, modeling of rigid body shifts, distortions, loops and side chains, as well as detecting errors in a model. despite these problems, it is currently possible to model with useful accuracy significant parts of approximately one third of all known protein sequences. the use of individual comparative models in biology is already rewarding and increasingly widespread. a major new challenge for comparative modeling is the integration of it with the torrents of data from genome sequencing projects as well as from functional and structural genomics. in particular, there is a need to develop an automated, rapid, robust, sensitive, and accurate comparative modeling pipeline applicable to whole genomes. such large-scale modeling is likely to encourage new kinds of applications for the many resulting models, based on their large number and completeness at the level of the family, organism, or functional network.\"\"the identification of mutations that are present in a small fraction of {dna} templates is essential for progress in several areas of biomedical research. although massively parallel sequencing instruments are in principle well suited to this task, the error rates in such instruments are generally too high to allow confident identification of rare variants. we here describe an approach that can substantially increase the sensitivity of massively parallel sequencing instruments for this purpose. the keys to this approach, called the {safe-sequencing} system ( ” {safe-seqs}”), are (i) assignment of a unique identifier ({uid}) to each template molecule, (ii) amplification of each uniquely tagged template molecule to create {uid} families, and (iii) redundant sequencing of the amplification products. {pcr} fragments with the same {uid} are considered mutant ( ” supermutants”) only if ≥95\\\\% of them contain the identical mutation. we illustrate the utility of this approach for determining the fidelity of a polymerase, the accuracy of oligonucleotides synthesized in vitro, and the prevalence of mutations in the nuclear and mitochondrial genomes of normal cells.\"\"synonymous mutations change the sequence of a\\xa0gene without directly altering the sequence of the encoded protein. here, we present evidence that these ?silent? mutations frequently contribute to human cancer. selection on synonymous mutations in\\xa0oncogenes is cancer-type specific, and although the functional consequences of cancer-associated synonymous mutations may be diverse, they recurrently alter exonic motifs that regulate splicing and are associated with changes in oncogene splicing in tumors. the p53 tumor suppressor ({tp53}) also has recurrent synonymous mutations, but, in contrast to those in oncogenes, these are adjacent to splice sites and inactivate them. we estimate that between one in two and\\xa0one in five silent mutations in oncogenes have been selected, equating to ?6\\\\%? 8\\\\% of all selected single-nucleotide changes in these genes. in addition, our analyses suggest that dosage-sensitive oncogenes have selected mutations in their 3?\\xa0{utrs}.\"\"the recognition of specific {dna}-binding sites by transcription factors is a critical yet poorly understood step in the control of gene expression. members of the hox family of transcription factors bind {dna} by making nearly identical major groove contacts via the recognition helices of their homeodomains. in vivo specificity, however, often depends on extended and unstructured regions that link hox homeodomains to a {dna}-bound cofactor, extradenticle (exd). using a combination of structure determination, computational analysis, and in vitro and in vivo assays, we show that hox proteins recognize specific {hox-exd} binding sites via residues located in these extended regions that insert into the minor groove but only when presented with the correct {dna} sequence. our results suggest that these residues, which are conserved in a paralog-specific manner, confer specificity by recognizing a sequence-dependent {dna} structure instead of directly reading a specific {dna} sequence.\"\"motivation: most tumor samples are a heterogeneous mixture of cells, including admixture by normal (non-cancerous) cells and subpopulations of cancerous cells with different complements of somatic aberrations. this intra-tumor heterogeneity complicates the analysis of somatic aberrations in {dna} sequencing data from tumor samples.\"\"identifying interaction sites in proteins provides important clues to the function of a protein and is becoming increasingly relevant in topics such as systems biology and drug discovery. although there are numerous papers on the prediction of interaction sites using information derived from structure, there are only a few case reports on the prediction of interaction residues based solely on protein sequence. here, a sliding window approach is combined with the random forests method to predict protein interaction sites using (i) a combination of sequence- and structure-derived parameters and (ii) sequence information alone. for sequence-based prediction we achieved a precision of 84\\\\% with a 26\\\\% recall and an f-measure of 40\\\\%. when combined with structural information, the prediction performance increases to a precision of 76\\\\% and a recall of 38\\\\% with an f-measure of 51\\\\%. we also present an attempt to rationalize the sliding window size and demonstrate that a nine-residue window is the most suitable for predictor construction. finally, we demonstrate the applicability of our prediction methods by modeling the {ras-raf} complex using predicted interaction sites as target binding interfaces. our results suggest that it is possible to predict protein interaction sites with quite a high accuracy using only sequence information.\"\"the inference of transcriptional networks that regulate transitions into physiological or pathological cellular states remains a central challenge in systems biology. a mesenchymal phenotype is the hallmark of tumour aggressiveness in human malignant glioma, but the regulatory programs responsible for implementing the associated molecular signature are largely unknown. here we show that reverse-engineering and an unbiased interrogation of a glioma-specific regulatory network reveal the transcriptional module that activates expression of mesenchymal genes in malignant glioma. two transcription factors ({c/ebpβ} and {stat3}) emerge as synergistic initiators and master regulators of mesenchymal transformation. ectopic co-expression of {c/ebpβ} and {stat3} reprograms neural stem cells along the aberrant mesenchymal lineage, whereas elimination of the two factors in glioma cells leads to collapse of the mesenchymal signature and reduces tumour aggressiveness. in human glioma, expression of {c/ebpβ} and {stat3} correlates with mesenchymal differentiation and predicts poor clinical outcome. these results show that the activation of a small regulatory module is necessary and sufficient to initiate and maintain an aberrant phenotypic state in cancer cells.\"\"studies of epigenetic modifications would benefit from improved methods for high-throughput methylation profiling. we introduce two complementary approaches that use next-generation sequencing technology to detect cytosine methylation. in the first method, we designed 10,000 bisulfite padlock probes to profile 7,000 {cpg} locations distributed over the {encode} pilot project regions and applied them to human b-lymphocytes, fibroblasts and induced pluripotent stem cells. this unbiased choice of targets takes advantage of existing expression and chromatin immunoprecipitation data and enabled us to observe a pattern of low promoter methylation and high gene-body methylation in highly expressed genes. the second method, methyl-sensitive cut counting, generated nontargeted genome-scale data for 1.4 million {hpaii} sites in the {dna} of b-lymphocytes and confirmed that gene-body methylation in highly expressed genes is a consistent phenomenon throughout the human genome. our observations highlight the usefulness of techniques that are not inherently or intentionally biased towards particular subsets like {cpg} islands or promoter regions.\"tel aviv 69978, israel.\",conservation and evolvability in regulatory networks: the evolution of ribosomal regulation in yeast.,\"transcriptional modules of coregulated genes play a key role in regulatory networks. comparative studies show that modules of coexpressed genes are conserved across taxa. however, little is known about the mechanisms underlying the evolution of module regulation. here, we explore the evolution of cis-regulatory programs associated with conserved modules by integrating expression profiles for two yeast species and sequence data for a total of 17 fungal genomes. we show that although the cis-elements accompanying certain conserved modules are strictly conserved, those of other conserved modules are remarkably diverged. in particular, we infer the evolutionary history of the regulatory program governing ribosomal modules. we show how a cis-element emerged concurrently in dozens of promoters of ribosomal protein genes, followed by the loss of a more ancient cis-element. we suggest that this formation of an intermediate redundant regulatory program allows conserved transcriptional modules to gradually switch from one regulatory mechanism to another while maintaining their functionality. our work provides a general framework for the study of the dynamics of promoter evolution at the level of transcriptional modules and may help in understanding the evolvability and increased redundancy of transcriptional regulation in higher organisms.\"hinxton, cb10 1sd, cambridge, uk.\",structural characterisation and functional significance of transient protein-protein interactions.,\"protein-protein complexes that dissociate and associate readily, often depending on the physiological condition or environment, play an important role in many biological processes. in order to characterise these \"\"transient\"\" protein-protein interactions, two sets of complexes were collected and analysed. the first set consists of 16 experimentally validated \"\"weak\"\" transient homodimers, which are known to exist as monomers and dimers at physiological concentration, with dissociation constants in the micromolar range. a set of 23 functionally validated transient (i.e. intracellular signalling) heterodimers comprise the second set. this set includes complexes that are more stable, with nanomolar binding affinities, and require a molecular trigger to form and break the interaction. in comparison to more stable homodimeric complexes, the weak homodimers demonstrate smaller contact areas between protomers and the interfaces are more planar and polar on average. the physicochemical and geometrical properties of these weak homodimers more closely resemble those of non-obligate hetero-oligomeric complexes, whose components can exist either as monomers or as complexes in vivo. in contrast to the weak transient dimers, \"\"strong\"\" transient dimers often undergo large conformational changes upon association/dissociation and are characterised with larger, less planar and sometimes more hydrophobic interfaces. from sequence alignments we find that the interface residues of the weak transient homodimers are generally more conserved than surface residues, consistent with being constrained to maintain the protein-protein interaction during evolution. protein families that include members with different oligomeric states or structures are identified, and found to exhibit a lower sequence conservation at the interface. the results are discussed in terms of the physiological function and evolution of protein-protein interactions.\"washington university medical school, st. louis, mo 63110, usa. stormo@ural.wustl.edu\",{dna} binding sites: representation and discovery.,\"a public web-based facility to infer, analyse and graphically represent the likely modes of a protein motion, starting from a static structure, is presented. this facility is based on the use of {concoord} to generate an ensemble of feasible protein structures that are subsequently analysed by principal component analysis to identify probable concerted motions. the user is returned the ensemble of feasible structures, together with associated analyses, including animations and graphical representations of both the principal component of the ensemble covariance and indicators of strongly correlated pairwise atomic motions. whilst users are warned that completely reliable inferences about protein motion may be beyond even substantially more rigorous tools for exploring configurational space, it is hoped that the service will allow a much wider community to benefit from the insights that simple dynamic data may offer.\"p-1/12 cit, scheme viim, calcutta 700 054, india.\",a dissection of specific and non-specific protein-protein interfaces.,\"we compare the geometric and physical-chemical properties of interfaces involved in specific and non-specific protein-protein interactions in crystal structures reported in the protein data bank. specific interactions are illustrated by 70 protein-protein complexes and by subunit contacts in 122 homodimeric proteins; non-specific interactions are illustrated by 188 pairs of monomeric proteins making crystal-packing contacts selected to bury more than 800 a2 of protein surface. a majority of these pairs have 2-fold symmetry and form \"\"crystal dimers\"\" that cannot be distinguished from real dimers on the basis of the interface size or symmetry. the chemical and amino acid compositions of the large crystal-packing interfaces resemble the protein solvent-accessible surface. these interfaces are less hydrophobic than in homodimers and contain much fewer fully buried atoms. we develop a residue propensity score and a hydrophobic interaction score to assess preferences seen in the chemical and amino acid compositions of the different types of interfaces, and we derive indexes to evaluate the atomic packing, which we find to be less compact at non-specific than at specific interfaces. we test the capacity of these parameters to identify homodimeric proteins in crystal structures, and show that a simple combination of the non-polar interface area and the fraction of buried interface atoms assigns the quaternary structure of 88\\\\% of the homodimers and 77\\\\% of the monomers in our data set correctly. these success rates increase to 93-95\\\\% when the residue propensity score of the interfaces is taken into consideration.\"domains and functional sites\",\"motivation: {interpro} is a new integrated documentation resource for protein families, domains and functional sites, developed initially as a means of rationalising the complementary efforts of the {prosite}, {prints}, pfam and {prodom} database {projects.results}: merged annotations from {prints}, {prosite} and pfam form the {interpro} core. each combined {interpro} entry includes functional descriptions and literature references, and links are made back to the relevant parent database(s), allowing users to see at a glance whether a particular family or domain has associated patterns, profiles, fingerprints, etc. merged and individual entries (i.e. those that have no counterpart in the companion resources) are assigned unique accession numbers. release 1.2 of {interpro} (june 2000) contains over 3000 entries, representing families, domains, repeats and sites of post-translational modification ({ptms}) encoded by 6581 different regular expressions, profiles, fingerprints and hidden markov models ({hmms}). each {interpro} entry lists all the matches against {swiss}-{prot} and {trembl} (more than 1000000 hits from 264333 different proteins out of 384572 in {swiss}-{prot} and {trembl}).availability: the database is accessible for text- and sequence-based searches at {http://www.ebi.ac.uk/interpro/.contact}: interhelp@ebi.ac.uk\"integrated molecular analysis of clear-cell renal cell carcinoma,\"the performance of gene expression microarrays has been well characterized using controlled reference samples, but the performance on clinical samples remains less clear. we identified sources of technical bias affecting many genes in concert, thus causing spurious correlations in clinical data sets and false associations between genes and clinical variables. we developed a method to correct for technical bias in clinical microarray data, which increased concordance with known biological relationships in multiple data sets.\"\"cancer is a somatic evolutionary process characterized by the accumulation of mutations, which contribute to tumor growth, clinical progression, immune escape, and drug resistance development. evolutionary theory can be used to analyze the dynamics of tumor cell populations and to make inference about the evolutionary history of a tumor from molecular data. we review recent approaches to modeling the evolution of cancer, including population dynamics models of tumor initiation and progression, phylogenetic methods to model the evolutionary relationship between tumor subclones, and probabilistic graphical models to describe dependencies among mutations. evolutionary modeling helps to understand how tumors arise and will also play an increasingly important prognostic role in predicting disease progression and the outcome of medical interventions, such as targeted therapy. {\\\\copyright} the author(s) 2014. published by oxford university press on behalf of the society of systematic biologists.\"the molecular evolution of acquired resistance to targeted {egfr} blockade in colorectal cancers,sydney, new south wales 2006, australia.\",protein-protein interactions in human disease.,room 151, frederick, md 21702, usa.\",\"a new, structurally nonredundant, diverse data set of protein–protein interfaces and its implications\",\"here, we present a diverse, structurally nonredundant data set of two-chain protein–protein interfaces derived from the {pdb}. using a sequence order-independent structural comparison algorithm and hierarchical clustering, 3799 interface clusters are obtained. these yield 103 clusters with at least five nonhomologous members. we divide the clusters into three types. in type i clusters, the global structures of the chains from which the interfaces are derived are also similar. this cluster type is expected because, in general, related proteins associate in similar ways. in type {ii}, the interfaces are similar; however, remarkably, the overall structures and functions of the chains are different. the functional spectrum is broad, from enzymes/inhibitors to immunoglobulins and toxins. the fact that structurally different monomers associate in similar ways, suggests  ” good” binding architectures. this observation extends a paradigm in protein science: it has been well known that proteins with similar structures may have different functions. here, we show that it extends to interfaces. in type {iii} clusters, only one side of the interface is similar across the cluster. this structurally nonredundant data set provides rich data for studies of protein–protein interactions and recognition, cellular networks and drug design. in particular, it may be useful in addressing the difficult question of what are the favorable ways for proteins to interact. (the data set is available at http://protein3d.ncifcrf.gov/∼keskino/ and {http://home.ku.edu.tr/∼okeskin/interface}/{interfaces}.html.)\"university of wisconsin-madison, madison, wi 53706, usa.\",{kfc} server: interactive forecasting of protein interaction hot spots.,\"the {kfc} server is a web-based implementation of the {kfc} (knowledge-based {fade} and contacts) model-a machine learning approach for the prediction of binding hot spots, or the subset of residues that account for most of a protein interface\\'s; binding free energy. the server facilitates the automated analysis of a user submitted protein-protein or {protein-dna} interface and the visualization of its hot spot predictions. for each residue in the interface, the {kfc} server characterizes its local structural environment, compares that environment to the environments of experimentally determined hot spots and predicts if the interface residue is a hot spot. after the computational analysis, the user can visualize the results using an interactive job viewer able to quickly highlight predicted hot spots and surrounding structural features within the protein structure. the {kfc} server is accessible at http://kfc.mitchell-lab.org.\"269 campus drive, center for clinical sciences research 1115, stanford, ca 94305-5151, usa.\",significance analysis of microarrays applied to the ionizing radiation response.,\"microarrays can measure the expression of thousands of genes to identify changes in expression between different biological states. methods are needed to determine the significance of these changes while accounting for the enormous number of genes. we describe a method, significance analysis of microarrays ({sam}), that assigns a score to each gene on the basis of change in gene expression relative to the standard deviation of repeated measurements. for genes with scores greater than an adjustable threshold, {sam} uses permutations of the repeated measurements to estimate the percentage of genes identified by chance, the false discovery rate ({fdr}). when the transcriptional response of human cells to ionizing radiation was measured by microarrays, {sam} identified 34 genes that changed at least 1.5-fold with an estimated {fdr} of 12\\\\%, compared with {fdrs} of 60 and 84\\\\% by using conventional methods of analysis. of the 34 genes, 19 were involved in cell cycle regulation and 3 in apoptosis. surprisingly, four nucleotide excision repair genes were induced, suggesting that this repair pathway for {uv}-damaged {dna} might play a previously unrecognized role in repairing {dna} damaged by ionizing radiation.\"\"intratumor heterogeneity may foster tumor evolution and adaptation and hinder personalized-medicine strategies that depend on results from single tumor-biopsy samples. to examine intratumor heterogeneity, we performed exome sequencing, chromosome aberration analysis, and ploidy profiling on multiple spatially separated samples obtained from primary renal carcinomas and associated metastatic sites. we characterized the consequences of intratumor heterogeneity using immunohistochemical analysis, mutation functional analysis, and profiling of messenger {rna} expression. phylogenetic reconstruction revealed branched evolutionary tumor growth, with 63 to 69\\\\% of all somatic mutations not detectable across every tumor region. intratumor heterogeneity was observed for a mutation within an autoinhibitory domain of the mammalian target of rapamycin ({mtor}) kinase, correlating with s6 and {4ebp} phosphorylation in vivo and constitutive activation of {mtor} kinase activity in vitro. mutational intratumor heterogeneity was seen for multiple tumor-suppressor genes converging on loss of function; {setd2}, {pten}, and {kdm5c} underwent multiple distinct and spatially separated inactivating mutations within a single tumor, suggesting convergent phenotypic evolution. gene-expression signatures of good and poor prognosis were detected in different regions of the same tumor. allelic composition and ploidy profiling analysis revealed extensive intratumor heterogeneity, with 26 of 30 tumor samples from four tumors harboring divergent allelic-imbalance profiles and with ploidy heterogeneity in two of four tumors. intratumor heterogeneity can lead to underestimation of the tumor genomics landscape portrayed from single tumor-biopsy samples and may present major challenges to personalized-medicine and biomarker development. intratumor heterogeneity, associated with heterogeneous protein function, may foster tumor adaptation and therapeutic failure through darwinian selection. (funded by the medical research council and others.).\"\"development requires the establishment of precise patterns of gene expression, which are primarily controlled by transcription factors binding to cis-regulatory modules. although transcription factor occupancy can now be identified at genome-wide scales, decoding this regulatory landscape remains a daunting challenge. here we used a novel approach to predict spatio-temporal cis-regulatory activity based only on in vivo transcription factor binding and enhancer activity data. we generated a high-resolution atlas of cis-regulatory modules describing their temporal and combinatorial occupancy during drosophila mesoderm development. the binding profiles of cis-regulatory modules with characterized expression were used to train support vector machines to predict five spatio-temporal expression patterns. in vivo transgenic reporter assays demonstrate the high accuracy of these predictions and reveal an unanticipated plasticity in transcription factor binding leading to similar expression. this data-driven approach does not require previous knowledge of transcription factor sequence affinity, function or expression, making it widely applicable.\"expression and evolution\",\"transcription factors are key cellular components that control gene expression: their activities determine how cells function and respond to the environment. currently, there is great interest in research into human transcriptional regulation. however, surprisingly little is known about these regulators themselves. for example, how many transcription factors does the human genome contain? how are they expressed in different tissues? are they evolutionarily conserved? here, we present an analysis of 1,391 manually curated sequence-specific {dna}-binding transcription factors, their functions, genomic organization and evolutionary conservation. much remains to be explored, but this study provides a solid foundation for future investigations to elucidate regulatory mechanisms underlying diverse mammalian biological processes.\"evolution of the cancer genome.,\"the advent of massively parallel sequencing technologies has allowed the characterization of cancer genomes at an unprecedented resolution. investigation of the mutational landscape of tumours is providing new insights into cancer genome evolution, laying bare the interplay of somatic mutation, adaptation of clones to their environment and natural selection. these studies have demonstrated the extent of the heterogeneity of cancer genomes, have allowed inferences to be made about the forces that act on nascent cancer clones as they evolve and have shown insight into the mutational processes that generate genetic variation. here we review our emerging understanding of the dynamic evolution of the cancer genome and of the implications for basic cancer biology and the development of antitumour therapy.\"\"{micrornas} ({mirnas}) are endogenous not, vert, similar23 nt {rnas} that play important gene-regulatory roles in animals and plants by pairing to the {mrnas} of protein-coding genes to direct their posttranscriptional repression. this review outlines the current understanding of {mirna} target recognition in animals and discusses the widespread impact of {mirnas} on both the expression and evolution of protein-coding genes.\"vrije universiteit de boelelaan 1081a, 1081hv, amsterdam, the netherlands.\",{popscomp}: an automated interaction analysis of biomolecular complexes.,\"large-scale analysis of biomolecular complexes reveals the functional network within the cell. computational methods are required to extract the essential information from the available data. the {popscomp} server is designed to calculate the interaction surface between all components of a given complex structure consisting of proteins, {dna} or {rna} molecules. the server returns matrices and graphs of surface area burial that can be used to automatically annotate components and residues that are involved in complex formation, to pinpoint conformational changes and to estimate molecular interaction energies. the analysis can be performed on a per-atom level or alternatively on a per-residue level for low-resolution structures. here, we present an analysis of ribosomal structures in complex with various antibiotics to exemplify the potential and limitations of automated complex analysis. the {popscomp} server is accessible at http://ibivu.cs.vu.nl/programs/popscompwww/.\"91904, israel.\",{promateus}—an open research approach to protein-binding sites analysis,\"the development of bioinformatic tools by individual labs results in the abundance of parallel programs for the same task. for example, identification of binding site regions between interacting proteins is done using: {promate}, {whiscy}, {ppi}-pred, {pinup} and others. all servers first identify unique properties of binding sites and then incorporate them into a predictor. obviously, the resulting prediction would improve if the most suitable parameters from each of those predictors would be incorporated into one server. however, because of the variation in methods and databases, this is currently not feasible. here, the protein-binding site prediction server is extended into a general protein-binding sites research tool, {promateus}. this web tool, based on {promate}\\'s infrastructure enables the easy exploration and incorporation of new features and databases by the user, providing an evaluation of the benefit of individual features and their combination within a set framework. this transforms the individual research into a community exercise, bringing out the best from all users for optimized predictions. the analysis is demonstrated on a database of protein protein and {protein-dna} interactions. this approach is basically different from that used in generating meta-servers. the implications of the open-research approach are discussed. {promateus} is available at http://bip.weizmann.ac.il/promate.\"rehovot 76100, israel.\",thousands of samples are needed to generate a robust gene list for predicting outcome in cancer.,\"despite many studies of the likely survival outcome of individual patients with colorectal cancer, our knowledge of this subject remains poor. until recently, we had virtually no understanding of individual responses to therapy, but the discovery of the {kras} mutation as a marker of probable failure of epidermal growth factor receptor ({egfr})-targeted therapy is a first step in the tailoring of treatment to the individual. with the application of molecular analyses, as well as the ability to perform high-throughput screens, there has been an explosive increase in the number of markers thought to be associated with prognosis and treatment outcome in this disease. in this review, we attempt to summarize the sometimes confusing findings, and critically assess those markers already in the public domain.\"dynamics of genomic clones in breast cancer patient xenografts at single-cell resolution,high-accuracy detection of low-frequency variation.\",the clonal and mutational evolution spectrum of primary triple-negative breast cancers,\"primary triple-negative breast cancers ({tnbcs}), a tumour type defined by lack of oestrogen receptor, progesterone receptor and {erbb2} gene amplification, represent approximately 16\\\\% of all breast cancers. here we show in 104 {tnbc} cases that at the time of diagnosis these cancers exhibit a wide and continuous spectrum of genomic evolution, with some having only a handful of coding somatic aberrations in a few pathways, whereas others contain hundreds of coding somatic mutations. high-throughput {rna} sequencing ({rna}-seq) revealed that only approximately 36\\\\% of mutations are expressed. using deep re-sequencing measurements of allelic abundance for 2,414 somatic mutations, we determine for the first time-to our knowledge-in an epithelial tumour subtype, the relative abundance of clonal frequencies among cases representative of the population. we show that {tnbcs} vary widely in their clonal frequencies at the time of diagnosis, with the basal subtype of {tnbc} showing more variation than non-basal {tnbc}. although p53 (also known as {tp53}), {pik3ca} and {pten} somatic mutations seem to be clonally dominant compared to other genes, in some tumours their clonal frequencies are incompatible with founder status. mutations in cytoskeletal, cell shape and motility proteins occurred at lower clonal frequencies, suggesting that they occurred later during tumour progression. taken together, our results show that understanding the biology and therapeutic responses of patients with {tnbc} will require the determination of individual tumour clonal genotypes.\"\"lung cancer is the leading cause of cancer-related death, with non-small cell lung cancer ({nsclc}) being the predominant form of the disease. most lung cancer is caused by the accumulation of genomic alterations due to tobacco exposure. to uncover its mutational landscape, we performed whole-exome sequencing in 31 {nsclcs} and their matched normal tissue samples. we identified both common and unique mutation spectra and pathway activation in lung adenocarcinomas and squamous cell carcinomas, two major histologies in {nsclc}. in addition to identifying previously known lung cancer genes ({tp53}, {kras}, {egfr}, {cdkn2a} and {rb1}), the analysis revealed many genes not previously implicated in this malignancy. notably, a novel gene {csmd3} was identified as the second most frequently mutated gene (next to {tp53}) in lung cancer. we further demonstrated that loss of {csmd3} results in increased proliferation of airway epithelial cells. the study provides unprecedented insights into mutational processes, cellular pathways and gene networks associated with lung cancer. of potential immediate clinical relevance, several highly mutated genes identified in our study are promising druggable targets in cancer therapy including {alk}, {ctnna3}, {dcc}, {mll3}, {pcdhiix}, {pik3c2b}, {pik3cg} and {rock2}.\"\"{background}: the detection of conserved motifs in promoters of orthologous genes (phylogenetic footprints) has become a common strategy to predict cis-acting regulatory elements. several software tools are routinely used to raise hypotheses about regulation. however, these tools are generally used as black boxes, with default parameters. a systematic evaluation of optimal parameters for a footprint discovery strategy can bring a sizeable improvement to the predictions. {results}: we evaluate the performances of a footprint discovery approach based on the detection of over-represented spaced motifs. this method is particularly suitable for (but not restricted to) bacteria, since such motifs are typically bound by factors containing a {helix-turn}-helix domain. we evaluated footprint discovery in 368 escherichia coli k12 genes with annotated sites, under 40 different combinations of parameters (taxonomical level, background model, organism-specific filtering, operon inference). motifs are assessed both at the levels of correctness and significance. we further report a detailed analysis of 181 bacterial orthologs of the {lexa} repressor. distinct motifs are detected at various taxonomical levels, including the 7 previously characterized taxon-specific motifs. in addition, we highlight a significantly stronger conservation of half-motifs in actinobacteria, relative to firmicutes, suggesting an intermediate state in specificity switching between the two gram-positive phyla, and thereby revealing the on-going evolution of {lexa} auto-regulation. {conclusion}: the footprint discovery method proposed here shows excellent results with e. coli and can readily be extended to predict cis-acting regulatory signals and propose testable hypotheses in bacterial genomes for which nothing is known about regulation.\"\"circulating tumor cells ({ctcs}) enter peripheral blood from primary tumors and seed metastases. the genome sequencing of {ctcs} could offer noninvasive prognosis or even diagnosis, but has been hampered by low single-cell genome coverage of scarce {ctcs}. here, we report the use of the recently developed multiple annealing and looping-based amplification cycles for whole-genome amplification of single {ctcs} from lung cancer patients. we observed characteristic cancer-associated single-nucleotide variations and insertions/deletions in exomes of {ctcs}. these mutations provided information needed for individualized therapy, such as drug resistance and phenotypic transition, but were heterogeneous from cell to cell. in contrast, every {ctc} from an individual patient, regardless of the cancer subtypes, exhibited reproducible copy number variation ({cnv}) patterns, similar to those of the metastatic tumor of the same patient. interestingly, different patients with the same lung cancer adenocarcinoma ({adc}) shared similar {cnv} patterns in their {ctcs}. even more interestingly, patients of small-cell lung cancer have {cnv} patterns distinctly different from those of {adc} patients. our finding suggests that {cnvs} at certain genomic loci are selected for the metastasis of cancer. the reproducibility of cancer-specific {cnvs} offers potential for {ctc}-based cancer diagnostics.\"university of bergen, thorm{\\\\o}hlensgate 55, n-5008 bergen, norway, the bioinformatics centre, department of molecular biology \\\\& biotech research and innovation centre, university of copenhagen, ole maal{\\\\o}es vej 5, dk-2100 k{\\\\o}benhavn {\\\\o}, informatics and mathematical modeling, building 321, technical university of denmark, dk-2800 kgs. lyngby, center for comparative genomics, institute of biology, university of copenhagen, universitetsparken 15, dk-2100 copenhagen {\\\\o}, denmark and sars centre for marine molecular biology, university of bergen, thorm{\\\\o}hlensgate 55, n-5008 bergen, norway.\",\"{jaspar}, the open access database of transcription factor-binding profiles: new content and tools in the 2008 update.\",\"{jaspar} is a popular open-access database for matrix models describing {dna}-binding preferences for transcription factors and other {dna} patterns. with its third major release, {jaspar} has been expanded and equipped with additional functions aimed at both casual and power users. the heart of the {jaspar} database-the {jaspar} {core} sub-database-has increased by 12\\\\% in size, and three new specialized sub-databases have been added. new functions include clustering of matrix models by similarity, generation of random matrices by sampling from selected sets of existing models and a language-independent web service applications programming interface for matrix retrieval. {jaspar} is available at http://jaspar.genereg.net.\"emerging patterns of somatic mutations in cancer,\"{background}: site-specific transcription factors ({tfs}) are coordinators of developmental and physiological gene expression programs. their binding to cis-regulatory modules of target genes mediates the precise cell- and context-specific activation and repression of genes. the expression of {tfs} should therefore reflect the core expression program of each cell. {results}: we studied the expression dynamics of about 750 {tfs} using the available genomics resources in drosophila melanogaster. we find that 95\\\\% of these {tfs} are expressed at some point during embryonic development, with a peak roughly between 10 and 12 hours after egg laying, the core stages of organogenesis. we address the differential utilization of {dna}-binding domains in different developmental programs systematically in a spatio-temporal context, and show that the zinc finger class of {tfs} is predominantly early expressed, while homeobox {tfs} exhibit later expression in embryogenesis. {conclusions}: previous work, dissecting cis-regulatory modules during drosophila development, suggests that {tfs} are deployed in groups acting in a cooperative manner. in contrast, we find that there is rapid exchange of co-expressed partners amongst the fly {tfs}, at rates similar to the genome-wide dynamics of co-expression clusters. this suggests there may also be a high level of combinatorial complexity of {tfs} at cis-regulatory modules.\"\"{background}: applications of computational methods for predicting protein functional linkages are increasing. in recent years, several bacteria-specific methods for predicting linkages have been developed. the four major genomic context methods are: gene cluster, gene neighbor, rosetta stone, and phylogenetic profiles. these methods have been shown to be powerful tools and this paper provides guidelines for when each method is appropriate by exploring different features of each method and potential improvements offered by their combination. we also review many previous treatments of these prediction methods, use the latest available annotations, and offer a number of new observations. {results}: using escherichia coli k12 and bacillus subtilis, linkage predictions made by each of these methods were evaluated against three benchmarks: functional categories defined by {cog} and {kegg}, known pathways listed in {ecocyc}, and known operons listed in {regulondb}. each evaluated method had strengths and weaknesses, with no one method dominating all aspects of predictive ability studied. for functional categories, as previous studies have shown, the rosetta stone method was individually best at detecting linkages and predicting functions among proteins with shared {kegg} categories while the phylogenetic profile method was best for linkage detection and function prediction among proteins with common {cog} functions. differences in performance under {cog} versus {kegg} may be attributable to the presence of paralogs. better function prediction was observed when using a weighted combination of linkages based on reliability versus using a simple unweighted union of the linkage sets. for pathway reconstruction, 99 complete metabolic pathways in e. coli k12 (out of the 209 known, non-trivial pathways) and 193 pathways with 50\\\\% of their proteins were covered by linkages from at least one method. gene neighbor was most effective individually on pathway reconstruction, with 48 complete pathways reconstructed. for operon prediction, gene cluster predicted completely 59\\\\% of the known operons in e. coli k12 and 88\\\\% (333/418)in b. subtilis. comparing two versions of the e. coli k12 operon database, many of the unannotated predictions in the earlier version were updated to true predictions in the later version. using only linkages found by both gene cluster and gene neighbor improved the precision of operon predictions. additionally, as previous studies have shown, combining features based on intergenic region and protein function improved the specificity of operon prediction. {conclusion}: a common problem for computational methods is the generation of a large number of false positives that might be caused by an incomplete source of validation. by comparing two versions of a database, we demonstrated the dramatic differences on reported results. we used several benchmarks on which we have shown the comparative effectiveness of each prediction method, as well as provided guidelines as to which method is most appropriate for a given prediction task.\"\"department of molecular biology, scripps research institute, 10550 north torrey pines road, la jolla, california 92037, usa.\",optimal docking area: a new method for predicting protein–protein interaction sites,\"understanding energetics and mechanism of protein–protein association remains one of the biggest theoretical problems in structural biology. it is assumed that desolvation must play an essential role during the association process, and indeed protein–protein interfaces in obligate complexes have been found to be highly hydrophobic. however, the identification of protein interaction sites from surface analysis of proteins involved in non-obligate protein–protein complexes is more challenging. here we present optimal docking area ({oda}), a new fast and accurate method of analyzing a protein surface in search of areas with favorable energy change when buried upon protein–protein association. the method identifies continuous surface patches with optimal docking desolvation energy based on atomic solvation parameters adjusted for protein–protein docking. the procedure has been validated on the unbound structures of a total of 66 non-homologous proteins involved in non-obligate protein–protein hetero-complexes of known structure. optimal docking areas with significant low-docking surface energy were found in around half of the proteins. the \\'{oda} hot spots\\' detected in x-ray unbound structures were correctly located in the known protein–protein binding sites in 80\\\\% of the cases. the role of these low-surface-energy areas during complex formation is discussed. burial of these regions during protein–protein association may favor the complexed configurations with near-native interfaces but otherwise arbitrary orientations, thus driving the formation of an encounter complex. the patch prediction procedure is freely accessible at http://www.molsoft.com/oda and can be easily scaled up for predictions in structural proteomics. proteins 2005. {\\\\copyright} 2004 {wiley-liss}, inc.\"\"a computational method is proposed for inferring protein interactions from genome sequences on the basis of the observation that some pairs of interacting proteins have homologs in another organism fused into a single protein chain. searching sequences from many genomes revealed 6809 such putative protein-protein interactions {inescherichia} coli and 45,502 in yeast. many members of these pairs were confirmed as functionally related; computational filtering further enriches for interactions. some proteins have links to several other proteins; these coupled links appear to represent functional interactions such as complexes or pathways. experimentally confirmed interacting pairs are documented in a database of interacting proteins.\"\"motivation: there are two main areas of difficulty in homology modelling that are particularly important when sequence identity between target and template falls below 50\\\\%: sequence alignment and loop building. these problems become magnified with automatic modelling processes, as there is no human input to correct mistakes. as such we have benchmarked several stand-alone strategies that could be implemented in a workflow for automated high-throughput homology modelling. these include three new sequence-structure alignment programs: {3d}-coffee, staccato and {salign}, plus five homology modelling programs and their respective loop building methods: builder, nest, modeller, {segmod}/{encad} and {swiss-model}. the {sabmark} database provided 123 targets with at least five templates from the same {scop} family and sequence identities [\\\\&le;]50\\\\%. results: when using modeller as the common modelling program, {3d}-coffee outperforms staccato and {salign} using both multiple templates and the best single template, and across the sequence identity range 2050\\\\%. the mean model {rmsd} generated from {3d}-coffee using multiple templates is 15 and 28\\\\% (or using single templates, 3 and 13\\\\%) better than those generated by staccato and salign, respectively. {3d}-coffee gives equivalent modelling accuracy from multiple and single templates, but staccato and {salign} are more successful with single templates, their quality deteriorating as additional lower sequence identity templates are added. evaluating the different homology modelling programs, on average modeller performs marginally better in overall modelling than the others tested. however, on average nest produces the best loops with an 8\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\ improvement by mean {rmsd} compared to the loops generated by builder. contact: r.m.jackson@leeds.ac.uk. supplementary information: supplementary data are available at bioinformatics online. 10.1093/bioinformatics/btm262\"\"most of the existing mathematical models for tumour growth and tumour-induced angiogenesis neglect blood flow. this is an important factor on which both nutrient and metabolite supply depend. in this paper we aim to address this shortcoming by developing a mathematical model which shows how blood flow and red blood cell heterogeneity influence the growth of systems of normal and cancerous cells. the model is developed in two stages. first we determine the distribution of oxygen in a native vascular network, incorporating into our model features of blood flow and vascular dynamics such as structural adaptation, complex rheology and red blood cell circulation. once we have calculated the oxygen distribution, we then study the dynamics of a colony of normal and cancerous cells, placed in such a heterogeneous environment. during this second stage, we assume that the vascular network does not evolve and is independent of the dynamics of the surrounding tissue. the cells are considered as elements of a cellular automaton, whose evolution rules are inspired by the different behaviour of normal and cancer cells. our aim is to show that blood flow and red blood cell heterogeneity play major roles in the development of such colonies, even when the red blood cells are flowing through the vasculature of normal, healthy tissue.\"\"this year, 2011, marks the forty-year anniversary of the statistical analysis of retinoblastoma that provided the first evidence that tumorigenesis can be initiated by as few as two mutations. this work provided the foundation for the two-hit hypothesis that explained the role of recessive tumour suppressor genes ({tsgs}) in dominantly inherited cancer susceptibility syndromes. however, four decades later, it is now known that even partial inactivation of tumour suppressors can critically contribute to tumorigenesis. here we analyse this evidence and propose a continuum model of {tsg} function to explain the full range of {tsg} mutations found in cancer.\"\"the initiation of transcription is regulated by transcription factors ({tfs}) binding to {dna} response elements ({res}). how do {tfs} recognize specific binding sites among the many similar ones available in the genome? recent research has illustrated that even a single nucleotide substitution can alter the selective binding of {tfs} to coregulators, that prior binding events can lead to selective {dna} binding, and that selectivity is influenced by the availability of binding sites in the genome. here, we combine structural insights with recent genomics screens to address the problem of {tf}–{dna} interaction specificity. the emerging picture of selective binding site sequence recognition and {tf} activation involves three major factors: the cellular network, protein and {dna} as dynamic conformational ensembles and the tight packing of multiple {tfs} and coregulators on stretches of regulatory {dna}. the classification of {tf} recognition mechanisms based on these factors impacts our understanding of how transcription initiation is regulated.\"\"pancreatic cancer is a disease caused by the accumulation of genetic alterations in specific genes. elucidation of the human genome sequence, in conjunction with technical advances in the ability to perform whole exome sequencing, have provided new insight into the mutational spectra characteristic of this lethal tumour type. most recently, exomic sequencing has been used to clarify the clonal evolution of pancreatic cancer as well as provide time estimates of pancreatic carcinogenesis, indicating that a long window of opportunity may exist for early detection of this disease while in the curative stage. moving forward, these mutational analyses indicate potential targets for personalised diagnostic and therapeutic intervention as well as the optimal timing for intervention based on the natural history of pancreatic carcinogenesis and progression.\"predicting protein interaction sites: binding hot-spots in protein-protein and protein-ligand interfaces.,\"{motivation}: protein assemblies are currently poorly represented in structural databases and their structural elucidation is a key goal in biology. here we analyse clefts in protein surfaces, likely to correspond to binding \\'hot-spots\\', and rank them according to sequence conservation and simple measures of physical properties including hydrophobicity, desolvation, electrostatic and van der waals potentials, to predict which are involved in binding in the native complex. {results}: the resulting differences between predicting binding-sites at protein-protein and protein-ligand interfaces are striking. there is a high level of prediction accuracy (< or =93\\\\%) for protein-ligand interactions, based on the following attributes: van der waals potential, electrostatic potential, desolvation and surface conservation. generally, the prediction accuracy for protein-protein interactions is lower, with the exception of enzymes. our results show that the ease of cleft desolvation is strongly predictive of interfaces and strongly maintained across all classes of protein-binding interface.\"estimating the human mutation rate using autozygosity in a founder population,and college of engineering, rumelifeneri yolu, 34450 sariyer istanbul, turkey. basic research program, saic-frederick, inc., laboratory of experimental and computational biology, nci-frederick, nci-frederick building 469, room 151, frederick, md 21702, usa.\",\"protein–protein interactions: organization, cooperativity and mapping in a bottom-up systems biology approach\",\"understanding and ultimately predicting protein associations is immensely important for functional genomics and drug design. here, we propose that binding sites have preferred organizations. first, the hot spots cluster within densely packed \\'hot regions\\'. within these regions, they form networks of interactions. thus, hot spots located within a hot region contribute cooperatively to the stability of the complex. however, the contributions of separate, independent hot regions are additive. moreover, hot spots are often already pre-organized in the unbound (free) protein states. describing a binding site through independent local hot regions has implications for binding site definition, design and parametrization for prediction. the compactness and cooperativity emphasize the similarity between binding and folding. this proposition is grounded in computation and experiment. it explains why summation of the interactions may over-estimate the stability of the complex. furthermore, statistically, charge-charge coupling of the hot spots is disfavored. however, since within the highly packed regions the solvent is screened, the electrostatic contributions are strengthened. thus, we propose a new description of protein binding sites: a site consists of (one or a few) self-contained cooperative regions. since the residue hot spots are those conserved by evolution, proteins binding multiple partners at the same sites are expected to use all or some combination of these regions.\"\"we noticed that disease-related amyloidogenic proteins and especially cellular prion proteins have the highest proportion of incompletely desolvated backbone h bonds among soluble proteins. such bonds are vulnerable to water attack and thus represent structural weaknesses. we have measured the adsorption of proteins onto phospholipid bilayers and found a strong correlation between the extent of underwrapping of backbone h bonds in the native structure of a protein and its extent of deposition on the bilayer: the less the h bond wrapping, the higher the propensity for protein-bilayer binding. these observations support the proposition that soluble proteins with amyloidogenic propensity and membrane proteins share a pervasive building motif: the underwrapped h bonds. whereas in membrane proteins, this motif does not signal a structural vulnerability, in soluble proteins, it is responsible for their reactivity. 10.1073/pnas.0335642100\"\"computational identification of prognostic biomarkers capable of withstanding follow-up validation efforts is still an open challenge in cancer research. for instance, several gene expression profiles analysis methods have been developed to identify gene signatures that can classify cancer sub-phenotypes associated with poor prognosis. however, signatures originating from independent studies show only minimal overlap and perform poorly when classifying datasets other than the ones they were generated from. in this paper, we propose a computational systems biology approach that can infer robust prognostic markers by identifying upstream master regulators, causally related to the presentation of the phenotype of interest. such a strategy effectively extends and complements other existing methods and may help further elucidate the molecular mechanisms of the observed pathophysiological phenotype. results show that inferred regulators substantially outperform canonical gene signatures both on the original dataset and across distinct datasets.\"whole-genome sequencing of liver cancers identifies etiological influences on mutation patterns and recurrent mutations in chromatin regulators,the patterns and dynamics of genomic instability in metastatic pancreatic cancer,\"the analysis and usage of biological data is hindered by the spread of information across multiple repositories and the difficulties posed by different nomenclature systems and storage formats. in particular, there is an important need for data unification in the study and use of protein-protein interactions. without good integration strategies, it is difficult to analyze the whole set of available data and its properties. we introduce {biana} (biologic interactions and network analysis), a tool for biological information integration and network management. {biana} is a python framework designed to achieve two major goals: i) the integration of multiple sources of biological information, including biological entities and their relationships, and ii) the management of biological information as a network where entities are nodes and relationships are edges. moreover, {biana} uses properties of proteins and genes to infer latent biomolecular relationships by transferring edges to entities sharing similar properties. {biana} is also provided as a plugin for cytoscape, which allows users to visualize and interactively manage the data. a web interface to {biana} providing basic functionalities is also available. the software can be downloaded under {gnu} {gpl} license from {http://sbi.imim.es/web/biana}.php. {biana}\\'s approach to data unification solves many of the nomenclature issues common to systems dealing with biological data. {biana} can easily be extended to handle new specific data repositories and new specific data types. the unification protocol allows {biana} to be a flexible tool suitable for different user requirements: non-expert users can use a suggested unification protocol while expert users can define their own specific unification rules.\"\"previous studies have suggested that nature is restricted to about 1,000 protein folds to perform a great diversity of functions. here, we use protein interaction data from different sources and three-dimensional structures to suggest that the total number of interaction types is also limited, and estimate that most interactions in nature will conform to one of about 10,000 types. we currently know fewer than 2,000, and at the present rate of structure determination, it will be more than 20 years before we know a full representative set.\"england.\",{scop}: a structural classification of proteins database for the investigation of sequences and structures.,\"the 1000 genomes project aims to provide a deep characterization of human genome sequence variation as a foundation for investigating the relationship between genotype and phenotype. here we present results of the pilot phase of the project, designed to develop and compare different strategies for genome-wide sequencing with high-throughput platforms. we undertook three projects: low-coverage whole-genome sequencing of 179 individuals from four populations; high-coverage sequencing of two mother-father-child trios; and exon-targeted sequencing of 697 individuals from seven populations. we describe the location, allele frequency and local haplotype structure of approximately 15 million single nucleotide polymorphisms, 1 million short insertions and deletions, and 20,000 structural variants, most of which were previously undescribed. we show that, because we have catalogued the vast majority of common variation, over 95\\\\% of the currently accessible variants found in any individual are present in this data set. on average, each person is found to carry approximately 250 to 300 loss-of-function variants in annotated genes and 50 to 100 variants previously implicated in inherited disorders. we demonstrate how these results can be used to inform association and functional studies. from the two trios, we directly estimate the rate of de novo germline base substitution mutations to be approximately 10(-8) per base pair per generation. we explore the data with regard to signatures of natural selection, and identify a marked reduction of genetic variation in the neighbourhood of genes, due to selection at linked sites. these methods and public data will support the next phase of human genetic research.\"\"the gene expression omnibus ({geo}) project was initiated in response to the growing demand for a public repository for high-throughput gene expression data. {geo} provides a flexible and open design that facilitates submission, storage and retrieval of heterogeneous data sets from high-throughput gene expression and genomic hybridization experiments. {geo} is not intended to replace in house gene expression databases that benefit from coherent data sets, and which are constructed to facilitate a particular analytic method, but rather complement these by acting as a tertiary, central data distribution hub. the three central data entities of {geo} are platforms, samples and series, and were designed with gene expression and genomic hybridization experiments in mind. a platform is, essentially, a list of probes that define what set of molecules may be detected. a sample describes the set of molecules that are being probed and references a single platform used to generate its molecular abundance data. a series organizes samples into the meaningful data sets which make up an experiment. the {geo} repository is publicly accessible through the world wide web at http://www.ncbi.nlm.nih.gov/geo.\"\"interacting or functionally related protein families tend to have similar phylogenetic trees. based on this observation, techniques have been developed to predict interaction partners. the observed degree of similarity between the phylogenetic trees of two proteins is the result of many different factors besides the actual interaction or functional relationship between them. such factors influence the performance of interaction predictions. one aspect that can influence this similarity is related to the fact that a given protein interacts with many others, and hence it must adapt to all of them. accordingly, the interaction or coadaptation signal within its tree is a composite of the influence of all of the interactors. here, we introduce a new estimator of coevolution to overcome this and other problems. instead of relying on the individual value of tree similarity between two proteins, we use the whole network of similarities between all of the pairs of proteins within a genome to reassess the similarity of that pair, thereby taking into account its coevolutionary context. we show that this approach offers a substantial improvement in interaction prediction performance, providing a degree of accuracy/coverage comparable with, or in some cases better than, that of experimental techniques. moreover, important information on the structure, function, and evolution of macromolecular complexes can be inferred with this methodology.\"the clonal evolution of tumor cell populations.,\"it is proposed that most neoplasms arise from a single cell of origin, and tumor progression results from acquired genetic variability within the original clone allowing sequential selection of more aggressive sublines. tumor cell populations are apparently more genetically unstable than normal cells, perhaps from activation of specific gene loci in the neoplasm, continued presence of carcinogen, or even nutritional deficiencies within the tumor. the acquired genetic insta0ility and associated selection process, most readily recognized cytogenetically, results in advanced human malignancies being highly individual karyotypically and biologically. hence, each patient\\'s cancer may require individual specific therapy, and even this may be thwarted by emergence of a genetically variant subline resistant to the treatment. more research should be directed toward understanding and controlling the evolutionary process in tumors before it reaches the late stage usually seen in clinical cancer.\"faculty of medicine, university of oslo, po box 1112 blindern, 0317 oslo, norway. philippe.collas@medisin.uio.no\",\"chop it, {chip} it, check it: the current status of chromatin immunoprecipitation.\",\"our understanding of the significance of interactions of proteins with {dna} in the context of gene expression, cell differentiation or to some extent disease has immensely been enhanced by the advent of chromatin immunoprecipitation ({chip}). {chip} has been widely used to map the localization of post-translationally modified histones or histone variants on the genome or on a specific gene locus, or to map the association of transcription factors or chromatin modifying enzymes to the genome. in spite of its power, {chip} is a cumbersome procedure and typically requires large numbers of cells. this review outlines variations elaborated on the {chip} assay to shorten the procedure, make it suitable for small cell numbers and unravel the multiplicity of histone modifications on a single locus. in addition, the combination of {chip} assays with {dna} microarray and high-throughput sequencing technologies has in recent years enabled the profiling of histone modifications and transcription factor occupancy sites throughout the genome and in a high-resolution manner throughout a genomic region of interest. we also review applications of {chip} to the mapping of histone modifications or transcription factor binding at the genome-wide level. finally, we speculate on future perspectives opened by the combination of emerging {chip}-related technologies.\"meguro-ku, tokyo 152-8552, japan. ryota.suzuki@is.titech.ac.jp\",pvclust: an r package for assessing the uncertainty in hierarchical clustering,\"summary: pvclust is an add-on package for a statistical software r to assess the uncertainty in hierarchical cluster analysis. pvclust can be used easily for general statistical problems, such as {dna} microarray analysis, to perform the bootstrap analysis of clustering, which has been popular in phylogenetic analysis. pvclust calculates probability values (p-values) for each cluster using bootstrap resampling techniques. two types of p-values are available: approximately unbiased ({au}) p-value and bootstrap probability ({bp}) value. multiscale bootstrap resampling is used for the calculation of {au} p-value, which has superiority in bias over {bp} value calculated by the ordinary bootstrap resampling. in addition the computation time can be enormously decreased with parallel computing option.\"baltimore, maryland 21287, usa.\",development of human protein reference database as an initial platform for approaching systems biology in humans,\"an international, peer-reviewed genome sciences journal featuring outstanding original research that offers novel insights into the biology of all organisms\"via salvador allende, i-84081 baronissi (sa) italy.\",{pops}: a fast algorithm for solvent accessible surface areas at atomic and residue level,\"{pops} (parameter {optimsed} surfaces) is a new method to calculate solvent accessible surface areas, which is based on an empirically parameterisable analytical formula and fast to compute. atomic and residue areas (the latter represented by a single sphere centered on the cα atom of amino acids and at the p atom of nucleotides) have been optimised versus accurate all-atom methods. the parameterisation has been derived from a selected dataset of proteins and nucleic acids of different sizes and topologies. the residue based approach {pops}-r, has been devised as a useful tool for the analysis of large macromolecular assemblies like the ribosome and it is specially suited for the refinement of low resolution structures. {pops}-r also allows for estimates of the loss of free energy of solvation upon complex formation, which should be particularly useful for the design of new protein–protein and protein–nucleic acid complexes. the program {pops} is available at {http://mathbio.nimr.mrc.ac.uk/\\\\~{}ffranca/pops} and at the mirror site http://www.cs.vu.nl/\\\\~{}ibivu/programs/popswww.\"\"laboratoire d\\'enzymologie et biochimie structurales, cnrs, gif-sur-yvette, france; european bioinformatics institute (ebi-embl), wellcome trust genome campus, hinxton, cambridge, united kingdom; center for advanced research in biotechnology, university of maryland biotechnology institute, rockville, maryland; san diego supercomputer center, ucsd 0505, la jolla, california; structural bioinformatics, biochemistry building, department of biological sciences, imperial college, london, united kingdom; biomedical engineering, boston university, boston, massachusetts; bioinformatics laboratory, department of applied mathematics and statistics, suny at stony brook, stony brook, new york; service de conformation de macromol\\\\\\'{e}cules biologiques et bioinformatique, campus plaine-bc6, universit\\\\\\'{e} libre de bruxelles, brusssels, belgium\",{capri}: a critical assessment of {predicted} interactions,\"{capri} is a communitywide experiment to assess the capacity of protein-docking methods to predict protein–protein interactions. nineteen groups participated in rounds 1 and 2 of {capri} and submitted blind structure predictions for seven protein–protein complexes based on the known structure of the component proteins. the predictions were compared to the unpublished x-ray structures of the complexes. we describe here the motivations for launching {capri}, the rules that we applied to select targets and run the experiment, and some conclusions that can already be drawn. the results stress the need for new scoring functions and for methods handling the conformation changes that were observed in some of the target systems. {capri} has already been a powerful drive for the community of computational biologists who development docking algorithms. we hope that this issue of proteins will also be of interest to the community of structural biologists, which we call upon to provide new targets for future rounds of {capri}, and to all molecular biologists who view protein–protein recognition as an essential process. proteins 2003;52:2–9. {\\\\copyright} 2003 {wiley-liss}, inc.\"\"the physical and chemical properties of domain–domain interactions have been analysed in two-domain proteins selected from the protein classification, {cath}. the two-domain structures were divided into those derived from (i) monomeric proteins, or (ii) oligomeric or complexed proteins. the size, polarity, hydrogen bonding and packing of the intra-chain domain interface were calculated for both sets of two-domain structures. the results were compared with inter-chain interface parameters from permanent and non-obligate protein–protein complexes. in general, the intra-chain domain and inter-chain interfaces were remarkably similar. many of the intra-chain interface properties are intermediate between those calculated for permanent and non-obligate inter-chain complexes. residue interface propensities were also found to be very similar, with hydrophobic residues playing a major role, together with positively charged arginine residues. in addition, the residue composition of the domain interfaces were found to be more comparable with domain surfaces than domain cores. the implications of these results for domain swapping and protein folding are discussed.\"gower street, london, wc1e 6bt, england.\",analysis of protein-protein interaction sites using surface patches,\"protein-protein interaction sites in complexes of known structure are characterised using a series of parameters to evaluate what differentiates them from other sites on the protein surface. surface patches are defined in protomers from a data set of 28 homo-dimers, 20 different hetero-complexes (segregated into large and small protomers), and antigens from six antibody-antigen complexes. six parameters (solvation potential, residue interface propensity, hydrophobicity, planarity, protrusion and accessible surface area) are calculated for the observed interface patch and all other surface patches defined on each protein. a ranking of the observed interface, relative to all other possible patches, is calculated. with this approach it becomes possible to analyse the distribution of the rankings of all the observed patches, relative to all other surface patches, for each data set. for each type of complex, none of the parameters were definitive, but the majority showed trends for the observed interface to be distinguished from other surface patches.\"university of california at san francisco, 94143, usa.\",the hallmarks of cancer.,il 60637, usa.\",insufficiently dehydrated hydrogen bonds as determinants of protein interactions,\"the prediction of binding sites and the understanding of interfaces associated with protein complexation remains an open problem in molecular biophysics. this work shows that a crucial factor in predicting and rationalizing protein–protein interfaces can be inferred by assessing the extent of intramolecular desolvation of backbone hydrogen bonds in monomeric structures. our statistical analysis of native structures shows that, in the majority of soluble proteins, most backbone hydrogen bonds are thoroughly wrapped intramolecularly by nonpolar groups except for a few ones. these latter underwrapped hydrogen bonds may be dramatically stabilized by removal of water. this fact implies that packing defects are  ” sticky” in a way that decisively contributes to determining the binding sites for proteins, as an examination of numerous complexes demonstrates.\"urbana 61801, usa.\",{vmd: visual molecular dynamics.},\"{ vmd is a molecular graphics program designed for the display and analysis of molecular assemblies, in particular biopolymers such as proteins and nucleic acids. vmd can simultaneously display any number of structures using a wide variety of rendering styles and coloring methods. molecules are displayed as one or more \"\"representations,\"\" in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms. the atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes boolean operators and regular expressions. vmd provides a complete graphical user interface for program control, as well as a text interface using the tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls. full session logging is supported, which produces a vmd command script for later playback. high-resolution raster images of displayed molecules may be produced by generating input scripts for use by a number of photorealistic image-rendering applications. vmd has also been expressly designed with the ability to animate molecular dynamics (md) simulation trajectories, imported either from files or from a direct connection to a running md simulation. vmd is the visualization component of mdscope, a set of tools for interactive problem solving in structural biology, which also includes the parallel md program namd, and the mdcomm software used to connect the visualization and simulation programs. vmd is written in c++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the world wide web. }\"transcription as a source of genome instability,\"alterations in genome sequence and structure contribute to somatic disease, affect the fitness of subsequent generations and drive evolutionary processes. the crucial roles of highly accurate replication and efficient repair in maintaining overall genome integrity are well-known, but the more localized stability costs that are associated with transcribing {dna} into {rna} molecules are less appreciated. here we review the diverse ways in which the essential process of transcription alters the underlying {dna} template and thereby modifies the genetic landscape.\"\"both {dna} methylation and histone modification are involved in establishing patterns of gene repression during development. certain forms of histone methylation cause local formation of heterochromatin, which is readily reversible, whereas {dna} methylation leads to stable long-term repression. it has recently become apparent that {dna} methylation and histone modification pathways can be dependent on one another, and that this crosstalk can be mediated by biochemical interactions between {set} domain histone methyltransferases and {dna} methyltransferases. relationships between {dna} methylation and histone modification have implications for understanding normal development as well as somatic cell reprogramming and tumorigenesis.\"mapping copy number variation by population-scale genome sequencing.,\"genomic structural variants ({svs}) are abundant in humans, differing from other forms of variation in extent, origin and functional impact. despite progress in {sv} characterization, the nucleotide resolution architecture of most {svs} remains unknown. we constructed a map of unbalanced {svs} (that is, copy number variants) based on whole genome {dna} sequencing data from 185 human genomes, integrating evidence from complementary {sv} discovery approaches with extensive experimental validations. our map encompassed 22,025 deletions and 6,000 additional {svs}, including insertions and tandem duplications. most {svs} (53\\\\%) were mapped to nucleotide resolution, which facilitated analysing their origin and functional impact. we examined numerous whole and partial gene deletions with a genotyping approach and observed a depletion of gene disruptions amongst high frequency deletions. furthermore, we observed differences in the size spectra of {svs} originating from distinct formation mechanisms, and constructed a map of {sv} hotspots formed by common mechanisms. our analytical framework and {sv} map serves as a resource for sequencing-based association studies.\"\"systematic chromatin immunoprecipitation ({chip}-chip) experiments have become a central technique for mapping transcriptional interactions in model organisms and humans. however, measurement of chromatin binding does not necessarily imply regulation, and binding may be difficult to detect if it is condition or cofactor dependent. to address these challenges, we present an approach for reliably assigning transcription factors ({tfs}) to target genes that integrates many lines of direct and indirect evidence into a single probabilistic model. using this approach, we analyze publicly available {chip}-chip binding profiles measured for yeast {tfs} in standard conditions, showing that our model interprets these data with significantly higher accuracy than previous methods. pooling the high-confidence interactions reveals a large network containing 363 significant sets of factors ({tf} modules) that cooperate to regulate common target genes. in addition, the method predicts 980 novel binding interactions with high confidence that are likely to occur in so-far untested conditions. indeed, using new {chip}-chip experiments we show that predicted interactions for the factors rpn4p and pdr1p are observed only after treatment of cells with methyl-methanesulfonate, a {dna}-damaging agent. we outline the first approach for consistently integrating all available evidences for {tf}–target interactions and we comprehensively identify the resulting {tf} module hierarchy. prioritizing experimental conditions for each factor will be especially important as increasing numbers of {chip}-chip assays are performed in complex organisms such as humans, for which  ” standard conditions” are ill defined. transcription factors ({tfs}) bind close to their target genes for regulating transcript levels depending on cellular conditions. each gene may be regulated differently from others through the binding of specific groups of {tfs} ({tf} modules). recently, a wide variety of large-scale measurements about transcriptional networks has become available. here the authors present a framework for consistently integrating all of this evidence to systematically determine the precise set of genes directly regulated by each {tf} (i.e., {tf}–target interactions). the framework is applied to the yeast saccharomyces cerevisiae using seven distinct sources of evidences to score all possible {tf}–target interactions in this organism. subsequently, the authors employ another newly developed algorithm to reveal {tf} modules based on the top 5,000 {tf}–target interactions, yielding more than 300 {tf} modules. the new scoring scheme for {tf}–target interactions allows predicting the binding of {tfs} under so-far untested conditions, which is demonstrated by experimentally verifying interactions for two {tfs} (pdr1p, rpn4p). importantly, the new methods (scoring of {tf}–target interactions and {tf} module identification) are scalable to much larger datasets, making them applicable to future studies in humans, which are thought to have substantially larger numbers of {tf}–target interactions.\"pareto optimality, and the geometry of phenotype space\",\"biological systems that perform multiple tasks face a fundamental trade-off: a given phenotype cannot be optimal at all tasks. here we ask how trade-offs affect the range of phenotypes found in nature. using the pareto front concept from economics and engineering, we find that best-trade-off phenotypes are weighted averages of archetypes--phenotypes specialized for single tasks. for two tasks, phenotypes fall on the line connecting the two archetypes, which could explain linear trait correlations, allometric relationships, as well as bacterial gene-expression patterns. for three tasks, phenotypes fall within a triangle in phenotype space, whose vertices are the archetypes, as evident in morphological studies, including on darwin\\'s finches. tasks can be inferred from measured phenotypes based on the behavior of organisms nearest the archetypes.\"\"most of the studies characterizing {dna} methylation patterns have been restricted to particular genomic loci in a limited number of human samples and pathological conditions. herein, we present a compromise between an extremely comprehensive study of a human sample population with an intermediate level of resolution of {cpgs} at the genomic level. we obtained a {dna} methylation fingerprint of 1628 human samples in which we interrogated 1505 {cpg} sites. the {dna} methylation patterns revealed show this epigenetic mark to be critical in tissue-type definition and stemness, particularly around transcription start sites that are not within a {cpg} island. for disease, the generated {dna} methylation fingerprints show that, during tumorigenesis, human cancer cells underwent a progressive gain of promoter {cpg}-island hypermethylation and a loss of {cpg} methylation in {non-cpg}-island promoters. although transformed cells are those in which {dna} methylation disruption is more obvious, we observed that other common human diseases, such as neurological and autoimmune disorders, had their own distinct {dna} methylation profiles. most importantly, we provide proof of principle that the {dna} methylation fingerprints obtained might be useful for translational purposes by showing that we are able to identify the tumor type origin of cancers of unknown primary origin ({cups}). thus, the {dna} methylation patterns identified across the largest spectrum of samples, tissues, and diseases reported to date constitute a baseline for developing higher-resolution {dna} methylation maps and provide important clues concerning the contribution of {cpg} methylation to tissue identity and its changes in the most prevalent human diseases.\"\"department of biological chemistry, the weizmann institute of science, rehovot 76100, israel; department of physics of complex systems, the weizmann institute of science, rehovot 76100, israel\",cluster conservation as a novel tool for studying protein-protein interactions evolution,\"protein-protein interactions networks has come to be a buzzword associated with nets containing edges that represent a pair of interacting proteins (e.g. hormone-receptor, enzyme-inhibitor, antigen-antibody, and a subset of multichain biological machines). yet, each such interaction composes its own unique network, in which vertices represent amino acid residues, and edges represent atomic contacts. recent studies have shown that analyses of the data encapsulated in these detailed networks may impact predictions of structure-function correlation. here, we study homologous families of protein-protein interfaces, which share the same fold but vary in sequence. in this context, we address what properties of the network are shared among relatives with different sequences (and hence different atomic interactions) and which are not. herein, we develop the general mathematical framework needed to compare the modularity of homologous networks. we then apply this analysis to the structural data of a few interface families, including hemoglobin ?-?, growth hormone-receptor, and serine protease-inhibitor. our results suggest that interface modularity is an evolutionarily conserved property. hence, protein-protein interfaces can be clustered down to a few modules, with the boundaries being evolutionarily conserved along homologous complexes. this suggests that protein engineering of protein-protein binding sites may be simplified by varying each module, but retaining the overall modularity of the interface. proteins 2007. {\\\\copyright} 2007 {wiley-liss}, inc.\"germany.\",why do hubs in the yeast protein interaction network tend to be essential: reexamining the connection between the network topology and essentiality,\"the centrality-lethality rule, which notes that high-degree nodes in a protein interaction network tend to correspond to proteins that are essential, suggests that the topological prominence of a protein in a protein interaction network may be a good predictor of its biological importance. even though the correlation between degree and essentiality was confirmed by many independent studies, the reason for this correlation remains illusive. several hypotheses about putative connections between essentiality of hubs and the topology of protein–protein interaction networks have been proposed, but as we demonstrate, these explanations are not supported by the properties of protein interaction networks. to identify the main topological determinant of essentiality and to provide a biological explanation for the connection between the network topology and essentiality, we performed a rigorous analysis of six variants of the genomewide protein interaction network for saccharomyces cerevisiae obtained using different techniques. we demonstrated that the majority of hubs are essential due to their involvement in essential complex biological modules, a group of densely connected proteins with shared biological function that are enriched in essential proteins. moreover, we rejected two previously proposed explanations for the centrality-lethality rule, one relating the essentiality of hubs to their role in the overall network connectivity and another relying on the recently published essential protein interactions model. analysis of protein interaction networks in the budding yeast saccharomyces cerevisiae has revealed that a small number of proteins, the so-called hubs, interact with a disproportionately large number of other proteins. furthermore, many hub proteins have been shown to be essential for survival of the cell—that is, in optimal conditions, yeast cannot grow and multiply without them. this relation between essentiality and the number of neighbors in the protein–protein interaction network has been termed the centrality-lethality rule. however, why are such hubs essential? jeong and colleagues [1] suggested that overrepresentation of essential proteins among high-degree nodes can be attributed to the central role that hubs play in mediating interactions among numerous, less connected proteins. another view, proposed by he and zhang, suggested that that the majority of proteins are essential due to their involvement in one or more essential protein–protein interactions that are distributed uniformly at random along the network edges [2]. we find that none of the above reasons determines essentiality. instead, the majority of hubs are essential due to their involvement in essential complex biological modules, a group of densely connected proteins with shared biological function that are enriched in essential proteins. this study sheds new light on the topological complexity of protein interaction networks.\"and department of mathematics and computer science, freie universit\\\\\"\"{a}t berlin. sven.rahmann@cebitec.uni-bielefeld.de\",on the power of profiles for transcription factor binding site detection,\"{background}:high-throughput measurement of transcript intensities using affymetrix type oligonucleotide microarrays has produced a massive quantity of data during the last decade. different preprocessing techniques exist to convert the raw signal intensities measured by these chips into gene expression estimates. although these techniques have been widely benchmarked in the context of differential gene expression analysis, there are only few examples where their performance has been assessed in respect to coexpression-based studies such as sample {classification.results}:in the present paper we benchmark the three most used normalization procedures ({mas5}, {rma} and {gcrma}) in the context of inter-array correlation analysis, confirming and extending the finding that {rma} and {gcrma} consistently overestimate sample similarity upon normalization. we determine that median polish summarization is responsible for generating a large proportion of these over-similarity artifacts. furthermore, we show that most affected probesets show also internal signal disagreement, and tend to be composed by individual probes hitting different gene transcripts. we finally provide a correction to the {rma}/{gcrma} summarization procedure that massively reduces inter-array correlation artifacts, without affecting the detection of differentially expressed {genes.conclusions}:we propose {trma} as a modification of {rma} to normalize microarray experiments for correlation-based analysis.\"\"the regulatory sequence analysis tools ({rsat}, http://rsat.ulb.ac.be/rsat/) is a software suite that integrates a wide collection of modular tools for the detection of cis-regulatory elements in genome sequences. the suite includes programs for sequence retrieval, pattern discovery, phylogenetic footprint detection, pattern matching, genome scanning and feature map drawing. random controls can be performed with random gene selections or by generating random sequences according to a variety of background models (bernoulli, markov). beyond the original word-based pattern-discovery tools (oligo-analysis and dyad-analysis), we recently added a battery of tools for matrix-based detection of cis-acting elements, with some original features (adaptive background models, markov-chain estimation of p-values) that do not exist in other matrix-based scanning tools. the web server offers an intuitive interface, where each program can be accessed either separately or connected to the other tools. in addition, the tools are now available as web services, enabling their integration in programmatic workflows. genomes are regularly updated from various genome repositories ({ncbi} and {ensembl}) and 682 organisms are currently supported. since 1998, the tools have been used by several hundreds of researchers from all over the world. several predictions made with {rsat} were validated experimentally and published.\"\"therapeutic proteins such as antibodies constitute the most rapidly growing class of pharmaceuticals for use in diverse clinical settings including cancer, chronic inflammatory diseases, kidney transplantation, cardiovascular medicine, and infectious diseases. unfortunately, they tend to aggregate when stored under the concentrated conditions required in their usage. aggregation leads to a decrease in antibody activity and could elicit an immunological response. using full antibody atomistic molecular dynamics simulations, we identify the antibody regions prone to aggregation by using a technology that we developed called spatial aggregation propensity ({sap}). {sap} identifies the location and size of these aggregation prone regions, and allows us to perform target mutations of those regions to engineer antibodies for stability. we apply this method to therapeutic antibodies and demonstrate the significantly enhanced stability of our mutants compared with the wild type. the technology described here could be used to incorporate developability in a rational way during the screening of antibodies in the discovery phase for several diseases.\"whole-exome sequencing of circulating tumor cells provides a window into metastatic prostate cancer,cantoblanco, e-28049 madrid, spain.\",similarity of phylogenetic trees as indicator of protein-protein interaction.,\"deciphering the network of protein interactions that underlines cellular operations has become one of the main tasks of proteomics and computational biology. recently, a set of bioinformatics approaches has emerged for the prediction of possible interactions by combining sequence and genomic information. even though the initial results are very promising, the current methods are still far from perfect. we propose here a new way of discovering possible protein-protein interactions based on the comparison of the evolutionary distances between the sequences of the associated protein families, an idea based on previous observations of correspondence between the phylogenetic trees of associated proteins in systems such as ligands and receptors. here, we extend the approach to different test sets, including the statistical evaluation of their capacity to predict protein interactions. to demonstrate the possibilities of the system to perform large-scale predictions of interactions, we present the application to a collection of more than 67 000 pairs of e.coli proteins, of which 2742 are predicted to correspond to interacting proteins.\"san francisco 94143-1204.\",cooperativity in protein-folding kinetics.,\"how does a protein find its native state without a globally exhaustive search? we propose the \"\"{hz}\"\" (hydrophobic zipper) hypothesis: hydrophobic contacts act as constraints that bring other contacts into spatial proximity, which then further constrain and zip up the next contacts, etc. in contrast to helix-coil cooperativity, {hz}-heteropolymer collapse cooperativity is driven by nonlocal interactions, causes sheet and irregular conformations in addition to helices, leads to secondary structures concurrently with early hydrophobic core formation, is much more sequence dependent than helix-coil processes, and involves compact intermediate states that have much secondary--but little tertiary--structure. hydrophobic contacts in the 1992 protein data bank have the type of \"\"topological localness\"\" predicted by the hypothesis. the {hz} paths for amino acid sequences that mimic crambin and bovine pancreatic trypsin inhibitor are quickly found by computer; the best configurations thus reached have single hydrophobic cores that are within about 3 kcal/mol of the global minimum. this hypothesis shows how proteins could find globally optimal states without exhaustive search.\"\"transcription factor ({tf}) perturbation experiments give valuable insights into gene regulation. genome-scale evidence from microarray measurements may be used to identify regulatory interactions between {tfs} and targets. recently, hu and colleagues published a comprehensive study covering 269 {tf} knockout mutants for the yeast saccharomyces cerevisiae. however, the information that can be extracted from this valuable dataset is limited by the method employed to process the microarray data. here, we present a reanalysis of the original data using improved statistical techniques freely available from the {bioconductor} project. we identify over 100 000 differentially expressed genes—nine times the total reported by hu et al. we validate the biological significance of these genes by assessing their functions, the occurrence of upstream {tf}-binding sites, and the prevalence of protein–protein interactions. the reanalysed dataset outperforms the original across all measures, indicating that we have uncovered a vastly expanded list of relevant targets. in summary, this work presents a high-quality reanalysis that maximizes the information contained in the hu et al. compendium. the dataset is available from {arrayexpress} (accession: {e-mtab}-109) and it will be invaluable to any scientist interested in the yeast transcriptional regulatory system.\"\"even when there is agreement on what measure a protein multiple structure alignment should be optimizing, finding the optimal alignment is computationally prohibitive. one approach used by many previous methods is aligned fragment pair chaining, where short structural fragments from all the proteins are aligned against each other optimally, and the final alignment chains these together in geometrically consistent ways. ye and godzik have recently suggested that adding geometric flexibility may help better model protein structures in a variety of contexts. we introduce the program matt (multiple alignment with translations and twists), an aligned fragment pair chaining algorithm that, in intermediate steps, allows local flexibility between fragments: small translations and rotations are temporarily allowed to bring sets of aligned fragments closer, even if they are physically impossible under rigid body transformations. after a dynamic programming assembly guided by these \"\"bent\"\" alignments, geometric consistency is restored in the final step before the alignment is output. matt is tested against other recent multiple protein structure alignment programs on the popular homstrad and {sabmark} benchmark datasets. matt\\'s global performance is competitive with the other programs on homstrad, but outperforms the other programs on {sabmark}, a benchmark of multiple structure alignments of proteins with more distant homology. on both datasets, matt demonstrates an ability to better align the ends of alpha-helices and beta-strands, an important characteristic of any structure alignment program intended to help construct a structural template library for threading approaches to the inverse protein-folding problem. the related question of whether matt alignments can be used to distinguish distantly homologous structure pairs from pairs of proteins that are not homologous is also considered. for this purpose, a p-value score based on the length of the common core and average root mean squared deviation ({rmsd}) of matt alignments is shown to largely separate decoys from homologous protein structures in the {sabmark} benchmark dataset. we postulate that matt\\'s strong performance comes from its ability to model proteins in different conformational states and, perhaps even more important, its ability to model backbone distortions in more distantly related proteins.\"\"the evolution of multicellularity required the suppression of cancer. if every cell has some chance of becoming cancerous, large, long-lived organisms should have an increased risk of developing cancer compared with small, short-lived organisms. the lack of correlation between body size and cancer risk is known as peto\\'s paradox. animals with 1000 times more cells than humans do not exhibit an increased cancer risk, suggesting that natural mechanisms can suppress cancer 1000 times more effectively than is done in human cells. because cancer has proven difficult to cure, attention has turned to cancer prevention. in this review, similar to pharmaceutical companies mining natural products, we seek to understand how evolution has suppressed cancer to develop ultimately improved cancer prevention in humans.\"campus ring 1, 28759, bremen, germany, m.zacharias@jacobs-university.de.\",the interface of protein-protein complexes: analysis of contacts and prediction of interactions,\"specific protein-protein interactions are essential for cellular functions. experimentally determined three-dimensional structures of protein-protein complexes offer the possibility to characterize binding interfaces in terms of size, shape and packing density. comparison with crystal-packing interfaces representing nonspecific protein-protein contacts gives insight into how specific binding differs from nonspecific low-affinity binding. an overview is given on empirical structural rules for specific protein-protein recognition derived from known complex structures. although single parameters such as interface size, shape or surface complementary show clear trends for different interface types, each parameter alone is insufficient to fully distinguish between specific versus crystal-packing contacts. a combination of interface parameters is, however, well suited to characterize a specific interface. this knowledge provides us with the essential ingredients that make up a specific protein recognition site. it is also of great value for the prediction of protein binding sites and for the evaluation of predicted complex structures.\"\"center for bioinformatics, the university of kansas, lawrence, kansas; centre de biochimie structurale (cnrs umr 5048, inserm umr u554, umi), montpellier, france; department of molecular biosciences, the university of kansas, lawrence, kansas\",{dockground} system of databases for protein recognition studies: unbound structures for docking,\"computational docking approaches are important as a source of protein-protein complexes structures and as a means to understand the principles of protein association. a key element in designing better docking approaches, including search procedures, potentials, and scoring functions is their validation on experimentally determined structures. thus, the databases of such structures (benchmark sets) are important. the previous, first release of the {dockground} resource (douguet et al., bioinformatics 2006; 22:2612-2618) implemented a comprehensive database of cocrystallized (bound) protein-protein complexes in a relational database of annotated structures. the current release adds important features to the set of bound structures, such as regularly updated downloadable datasets: automatically generated nonredundant set, built according to most common criteria, and a manually curated set that includes only biological nonobligate complexes along with a number of additional useful characteristics. the main focus of the current release is unbound (experimental and simulated) protein-protein complexes. complexes from the bound dataset are used to identify crystallized unbound analogs. if such analogs do not exist, the unbound structures are simulated by rotamer library optimization. thus, the database contains comprehensive sets of complexes suitable for large scale benchmarking of docking algorithms. advanced methodologies for simulating unbound conformations are being explored for the next release. the future releases will include datasets of modeled protein-protein complexes, and systematic sets of docking decoys obtained by different docking algorithms. the growing {dockground} resource is designed to become a comprehensive public environment for developing and validating new docking methodologies. proteins 2007. {\\\\copyright} 2007 {wiley-liss}, inc.\"\"{background}:evolutionary conservation has been used successfully to help identify cis-acting {dna} regions that are important in regulating tissue-specific gene expression. motivated by increasing evidence that some {dna} regulatory regions are not evolutionary conserved, we have developed an approach for cis-regulatory region identification that does not rely upon evolutionary sequence {conservation.results}:the conservation-independent approach is based on an empirical potential energy between interacting transcription factors ({tfs}). in this analysis, the potential energy is defined as a function of the number of {tf} interactions in a genomic region and the strength of the interactions. by identifying sets of interacting {tfs}, the analysis locates regions enriched with the binding sites of these interacting {tfs}. we applied this approach to 30 human tissues and identified 6232 putative cis-regulatory modules ({crms}) regulating 2130 tissue-specific genes. interestingly, some genes appear to be regulated by different {crms} in different tissues. known regulatory regions are highly enriched in our predicted {crms}. in addition, {dnase} i hypersensitive sites, which tend to be associated with active regulatory regions, significantly overlap with the predicted {crms}, but not with more conserved regions. we also find that conserved and non-conserved {crms} regulate distinct gene groups. conserved {crms} control more essential genes and genes involved in fundamental cellular activities such as transcription. in contrast, non-conserved {crms}, in general, regulate more non-essential genes, such as genes related to neural {activity.conclusion}:these results demonstrate that identifying relevant sets of binding motifs can help in the mapping of {dna} regulatory regions, and suggest that non-conserved {crms} play an important role in gene regulation.\"\"in the united states, ovarian cancer ranks as the fifth deadliest cancer among women.1 of the several subtypes of epithelial ovarian cancer, high-grade serous carcinomas are the most common, accounting for approximately 70\\\\% of all cases of epithelial ovarian cancer in north america.2 although ovarian clear-cell carcinoma is the second most common subtype in north america (accounting for 12\\\\% of cases and an even higher percentage in japan3) and is the second leading cause of death from ovarian cancer,2 it is relatively understudied. ovarian clear-cell carcinoma is defined on the basis of histopathological findings, including a predominance of clear .\\xa0.\\xa0.\"\"identifying the interface between two interacting proteins provides important clues to the function of a protein, and is becoming increasing relevant to drug discovery. here, surface patch analysis was combined with a bayesian network to predict protein–protein binding sites with a success rate of 82\\\\% on a benchmark dataset of 180 proteins, improving by 6\\\\% on previous work and well above the 36\\\\% that would be achieved by a random method. a comparable success rate was achieved even when evolutionary information was missing, a further improvement on our previous method which was unable to handle incomplete data automatically. in a case study of the mog1p family, we showed that our bayesian network method can aid the prediction of previously uncharacterised binding sites and provide important clues to protein function. on mog1p itself a putative binding site involved in the {sln1}-{skn7} signal transduction pathway was detected, as was a ran binding site, previously characterized solely by conservation studies, even though our automated method operated without using homologous proteins. on the remaining members of the family (two structural genomics targets, and a protein involved in the photosystem {ii} complex in higher plants) we identified novel binding sites with little correspondence to those on mog1p. these results suggest that members of the mog1p family bind to different proteins and probably have different functions despite sharing the same overall fold. we also demonstrated the applicability of our method to drug discovery efforts by successfully locating a number of binding sites involved in the protein–protein interaction network of papilloma virus infection. in a separate study, we attempted to distinguish between the two types of binding site, obligate and non-obligate, within our dataset using a second bayesian network. this proved difficult although some separation was achieved on the basis of patch size, electrostatic potential and conservation. such was the similarity between the two interacting patch types, we were able to use obligate binding site properties to predict the location of non-obligate binding sites and vice versa.\"a single-array preprocessing method for estimating full-resolution raw copy numbers from all affymetrix genotyping arrays including {genomewidesnp} 5 \\\\& 6.,\"high-resolution copy-number ({cn}) analysis has in recent years gained much attention, not only for the purpose of identifying {cn} aberrations associated with a certain phenotype, but also for identifying {cn} polymorphisms. in order for such studies to be successful and cost effective, the statistical methods have to be optimized. we propose a single-array preprocessing method for estimating full-resolution total {cns}. it is applicable to all affymetrix genotyping arrays, including the recent ones that also contain non-polymorphic probes. a reference signal is only needed at the last step when calculating relative {cns}. as with our method for earlier generations of arrays, this one controls for allelic crosstalk, probe affinities and {pcr} fragment-length effects. additionally, it also corrects for probe sequence effects and co-hybridization of fragments digested by multiple enzymes that takes place on the latest chips. we compare our method with affymetrix\\'s {cn5} method and the {dchip} method by assessing how well they differentiate between various {cn} states at the full resolution and various amounts of smoothing. although {crma} v2 is a single-array method, we observe that it performs as well as or better than alternative methods that use data from all arrays for their preprocessing. this shows that it is possible to do online analysis in large-scale projects where additional arrays are introduced over time.\"66123 saarbr\\\\\"\"{u}cken, germany. hzhu@mpi-sb.mpg.de\",{noxclass}: prediction of protein-protein interaction types,\"{background}:structural models determined by x-ray crystallography play a central role in understanding protein-protein interactions at the molecular level. interpretation of these models requires the distinction between non-specific crystal packing contacts and biologically relevant interactions. this has been investigated previously and classification approaches have been proposed. however, less attention has been devoted to distinguishing different types of biological interactions. these interactions are classified as obligate and non-obligate according to the effect of the complex formation on the stability of the protomers. so far no automatic classification methods for distinguishing obligate, non-obligate and crystal packing interactions have been made {available.results}:six interface properties have been investigated on a dataset of 243 protein interactions. the six properties have been combined using a support vector machine algorithm, resulting in {noxclass}, a classifier for distinguishing obligate, non-obligate and crystal packing interactions. we achieve an accuracy of 91.8\\\\% for the classification of these three types of interactions using a leave-one-out cross-validation {procedure.conclusion}:{noxclass} allows the interpretation and analysis of protein quaternary structures. in particular, it generates testable hypotheses regarding the nature of protein-protein interactions, when experimental results are not available. we expect this server will benefit the users of protein structural models, as well as protein crystallographers and {nmr} spectroscopists. a web server based on the method and the datasets used in this study are available at http://noxclass.bioinf.mpi-inf.mpg.de/.\"brigham and women\\'s hospital and harvard medical school, new research building, 77 avenue louis pasteur, boston, ma 02115, usa. mlbulyk@rascal.med.harvard.edu\",computational prediction of transcription-factor binding site locations,\"identifying genomic locations of transcription-factor binding sites, particularly in higher eukaryotic genomes, has been an enormous challenge. various experimental and computational approaches have been used to detect these sites; methods involving computational comparisons of related genomes have been particularly successful.\"\"motivation: high-throughput data is providing a comprehensive view of the molecular changes in cancer tissues. new technologies allow for the simultaneous genome-wide assay of the state of genome copy number variation, gene expression, {dna} methylation and epigenetics of tumor samples and cancer cell lines.\"\"we developed a computational method to characterize aneuploidy in tumor samples based on coordinated aberrations in expression of genes localized to each chromosomal region. we summarized the total level of chromosomal aberration in a given tumor in a univariate measure termed total functional aneuploidy. we identified a signature of chromosomal instability from specific genes whose expression was consistently correlated with total functional aneuploidy in several cancer types. net overexpression of this signature was predictive of poor clinical outcome in 12 cancer data sets representing six cancer types. also, the signature of chromosomal instability was higher in metastasis samples than in primary tumors and was able to stratify grade 1 and grade 2 breast tumors according to clinical outcome. these results provide a means to assess the potential role of chromosomal instability in determining malignant potential over a broad range of tumors.\"1205 ave dr penfield, montreal, qc h3a 1b1, canada;  community and conservation ecology group, university of groningen, haren, the netherlands;  department of biology, university of oslo, oslo, norway;  department of ecology and evolutionary biology, university of michigan, ann arbor, mi, usa;  department of statistics, university of auckland, auckland, new zealand;  gatty marine laboratory, university of st andrews, fife, scotland;  department of ecology and evolutionary biology, university of arizona, tucson, az, usa;  school of natural sciences, university of california merced, merced, ca, usa;  department of renewable natural resources, university of alberta, edmonton, alberta, canada;  national center for ecological analysis and synthesis, university of california santa barbara, santa barbara, ca, usa;  center for advanced studies in ecology \\\\& biodiversity (caseb), departamento de ecologa, facultad de ciencias biolgicas, pontificia universidad catlica de chile, alameda 340, santiago, chile;  instituto de ecologa y biodiversidad (ieb), departamento de ciencias ecolgicas. facultad de ciencias, universidad de chile. casilla 653, santiago, chile;  department of fisheries and wildlife, michigan state university, east lansing, mi, usa;  school of life sciences, arizona state university, tempe, az, usa\",species abundance distributions: moving beyond single prediction theories to integration within an ecological framework,\"species abundance distributions ({sads}) follow one of ecology\\'s oldest and most universal laws – every community shows a hollow curve or hyperbolic shape on a histogram with many rare species and just a few common species. here, we review theoretical, empirical and statistical developments in the study of {sads}. several key points emerge. (i) literally dozens of models have been proposed to explain the hollow curve. unfortunately, very few models are ever rejected, primarily because few theories make any predictions beyond the hollow-curve {sad} itself. (ii) interesting work has been performed both empirically and theoretically, which goes beyond the hollow-curve prediction to provide a rich variety of information about how {sads} behave. these include the study of {sads} along environmental gradients and theories that integrate {sads} with other biodiversity patterns. central to this body of work is an effort to move beyond treating the {sad} in isolation and to integrate the {sad} into its ecological context to enable making many predictions. (iii) moving forward will entail understanding how sampling and scale affect {sads} and developing statistical tools for describing and comparing {sads}. we are optimistic that {sads} can provide significant insights into basic and applied ecological science.\"rehovot 76100, israel.\",the modular architecture of protein-protein binding interfaces.,\"the human genome is peppered with mobile repetitive elements called long interspersed nuclear element–1 (l1) retrotransposons. propagating through {rna} and {cdna} intermediates, these molecular parasites copy and insert themselves throughout the genome, with potentially disruptive effects on neighboring genes or regulatory sequences. in the germ line, unique sequence downstream of l1 elements can also be retrotransposed if transcription continues beyond the repeat, a process known as 3′ transduction. there has been growing interest in retrotransposition and 3′ transduction as a possible source of somatic mutations during tumorigenesis.\"\"multiple myeloma is an incurable malignancy of plasma cells, and its pathogenesis is poorly understood. here we report the massively parallel sequencing of 38 tumour genomes and their comparison to matched normal {dnas}. several new and unexpected oncogenic mechanisms were suggested by the pattern of somatic mutation across the data set. these include the mutation of genes involved in protein translation (seen in nearly half of the patients), genes involved in histone methylation, and genes involved in blood coagulation. in addition, a broader than anticipated role of {nf}-{κb} signalling was indicated by mutations in 11 members of the {nf}-{κb} pathway. of potential immediate clinical relevance, activating mutations of the kinase {braf} were observed in 4\\\\% of patients, suggesting the evaluation of {braf} inhibitors in multiple myeloma clinical trials. these results indicate that cancer genome sequencing of large collections of samples will yield new insights into cancer not anticipated by existing knowledge.\"\"we evaluate {3d} models of human nucleoside diphosphate kinase, mouse cellular retinoic acid binding protein i, and human eosinophil neurotoxin that were calculated by {modeller}, a program for comparative protein modeling by satisfaction of spatial restraints. the models have good stereochemistry and are at least as similar to the crystallographic structures as the closest template structures. the largest errors occur in the regions that were not aligned correctly or where the template structures are not similar to the correct structure. these regions correspond predominantly to exposed loops, insertions of any length, and non-conserved side chains. when a template structure with more than 40\\\\% sequence identity to the target protein is available, the model is likely to have about 90\\\\% of the mainchain atoms modeled with an rms deviation from the x-ray structure of approximately 1 a, in large part because the templates are likely to be that similar to the x-ray structure of the target. this rms deviation is comparable to the overall differences between refined {nmr} and x-ray crystallography structures of the same protein.\"massachusetts institute of technology, nine cambridge center, cambridge, ma 02142, usa.\",\"rictor, a novel binding partner of {mtor}, defines a {rapamycin-insensitive} and {raptor-independent} pathway that regulates the cytoskeleton\",\"the mammalian {tor} ({mtor}) pathway integrates nutrient- and growth factor-derived signals to regulate growth, the process whereby cells accumulate mass and increase in size. {mtor} is a large protein kinase and the target of rapamycin, an immunosuppressant that also blocks vessel restenosis and has potential anticancer applications. {mtor} interacts with the raptor and {gβl} proteins [1], [2] and [3] to form a complex that is the target of rapamycin. here, we demonstrate that {mtor} is also part of a distinct complex defined by the novel protein rictor (rapamycin-insensitive companion of {mtor}). rictor shares homology with the previously described pianissimo from d. discoidieum [4], {ste20p} from s. pombe [5], and {avo3p} from s. cerevisiae [6] and [7]. interestingly, {avo3p} is part of a rapamycin-insensitive {tor} complex that does not contain the yeast homolog of raptor and signals to the actin cytoskeleton through {pkc1} [6]. consistent with this finding, the rictor-containing {mtor} complex contains {gβl} but not raptor and it neither regulates the {mtor} effector {s6k1} nor is it bound by {fkbp12}-rapamycin. we find that the {rictor-mtor} complex modulates the phosphorylation of protein kinase c α ({pkcα}) and the actin cytoskeleton, suggesting that this aspect of {tor} signaling is conserved between yeast and mammals.\"usa\",a mathematical theory of communication,an abstract is not available.dense genotyping and resequencing data\",\"summary: the {protorp} server analyses protein-protein associations in {3d} structures. the server calculates a series of physical and chemical parameters of the protein interaction sites that contribute to the binding energy of the association. these parameters include, size and shape, intermolecular bonding, residue and atom composition and secondary structure contributions. the server is flexible, in that it allows users to analyse individual protein associations or large datasets of associations deposited in the {pdb}, or upload and analyse proprietary files. the properties calculated can be compared with parameter distributions for non-homologous datasets of different classes of protein associations provided on the server website. the server provides an efficient way of characterizing protein-protein associations of new or existing proteins, and a means of putting these values in the context of previously observed associations.  availability: http://www.bioinformatics.sussex.ac.uk/protorp  contact: s.jones@sussex.ac.uk 10.1093/bioinformatics/btn584\"')]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#function to concatenate list elements\n",
    "def concatListElements(x):\n",
    "    return ''.join(x)\n",
    "\n",
    "#Create new RDD containing only user_hash_id and abstract from the result of the join\n",
    "#Using groupByKey() and mapValues() to combine values for each key into a single list\n",
    "transJoinResultRDD = joinResultRDD.map(lambda x: (x[1][0], x[1][1].strip()))\\\n",
    "                                .groupByKey()\\\n",
    "                                .mapValues(list)\\\n",
    "                                .mapValues(concatListElements)\n",
    "joinResultRDD.unpersist()\n",
    "transJoinResultRDD.take(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('f05bcffe7951de9e5a32fff4a42eb088',\n",
       "  ['a',\n",
       "   'protein',\n",
       "   'domain',\n",
       "   'database',\n",
       "   'for',\n",
       "   'functional',\n",
       "   'characterization',\n",
       "   'and',\n",
       "   'annotation.\",\"{prosite}',\n",
       "   'consists',\n",
       "   'of',\n",
       "   'documentation',\n",
       "   'entries',\n",
       "   'describing',\n",
       "   'protein',\n",
       "   'domains,',\n",
       "   'families',\n",
       "   'and',\n",
       "   'functional',\n",
       "   'sites,',\n",
       "   'as',\n",
       "   'well',\n",
       "   'as',\n",
       "   'associated',\n",
       "   'patterns',\n",
       "   'and',\n",
       "   'profiles',\n",
       "   'to',\n",
       "   'identify',\n",
       "   'them.',\n",
       "   'it',\n",
       "   'is',\n",
       "   'complemented',\n",
       "   'by',\n",
       "   '{prorule},',\n",
       "   'a',\n",
       "   'collection',\n",
       "   'of',\n",
       "   'rules',\n",
       "   'based',\n",
       "   'on',\n",
       "   'profiles',\n",
       "   'and',\n",
       "   'patterns,',\n",
       "   'which',\n",
       "   'increases',\n",
       "   'the',\n",
       "   'discriminatory',\n",
       "   'power',\n",
       "   'of',\n",
       "   'these',\n",
       "   'profiles',\n",
       "   'and',\n",
       "   'patterns',\n",
       "   'by',\n",
       "   'providing',\n",
       "   'additional',\n",
       "   'information',\n",
       "   'about',\n",
       "   'functionally',\n",
       "   'and/or',\n",
       "   'structurally',\n",
       "   'critical',\n",
       "   'amino',\n",
       "   'acids.',\n",
       "   '{prosite}',\n",
       "   'is',\n",
       "   'largely',\n",
       "   'used',\n",
       "   'for',\n",
       "   'the',\n",
       "   'annotation',\n",
       "   'of',\n",
       "   'domain',\n",
       "   'features',\n",
       "   'of',\n",
       "   '{uniprotkb}/{swiss-prot}',\n",
       "   'entries.',\n",
       "   'among',\n",
       "   'the',\n",
       "   '983',\n",
       "   '({dna}-binding)',\n",
       "   'domains,',\n",
       "   'repeats',\n",
       "   'and',\n",
       "   'zinc',\n",
       "   'fingers',\n",
       "   'present',\n",
       "   'in',\n",
       "   '{swiss-prot}',\n",
       "   '(release',\n",
       "   '57.8',\n",
       "   'of',\n",
       "   '22',\n",
       "   'september',\n",
       "   '2009),',\n",
       "   '696',\n",
       "   '(',\n",
       "   'approximately',\n",
       "   '70\\\\%)',\n",
       "   'are',\n",
       "   'annotated',\n",
       "   'with',\n",
       "   '{prosite}',\n",
       "   'descriptors',\n",
       "   'using',\n",
       "   'information',\n",
       "   'from',\n",
       "   '{prorule}.',\n",
       "   'in',\n",
       "   'order',\n",
       "   'to',\n",
       "   'allow',\n",
       "   'better',\n",
       "   'functional',\n",
       "   'characterization',\n",
       "   'of',\n",
       "   'domains,',\n",
       "   '{prosite}',\n",
       "   'developments',\n",
       "   'focus',\n",
       "   'on',\n",
       "   'subfamily',\n",
       "   'specific',\n",
       "   'profiles',\n",
       "   'and',\n",
       "   'a',\n",
       "   'new',\n",
       "   'profile',\n",
       "   'building',\n",
       "   'method',\n",
       "   'giving',\n",
       "   'more',\n",
       "   'weight',\n",
       "   'to',\n",
       "   'functionally',\n",
       "   'important',\n",
       "   'residues.',\n",
       "   'here,',\n",
       "   'we',\n",
       "   'describe',\n",
       "   '{amsa},',\n",
       "   'an',\n",
       "   'annotated',\n",
       "   'multiple',\n",
       "   'sequence',\n",
       "   'alignment',\n",
       "   'format',\n",
       "   'used',\n",
       "   'to',\n",
       "   'build',\n",
       "   'a',\n",
       "   'new',\n",
       "   'generation',\n",
       "   'of',\n",
       "   'generalized',\n",
       "   'profiles,',\n",
       "   'the',\n",
       "   'migration',\n",
       "   'of',\n",
       "   '{scanprosite}',\n",
       "   'to',\n",
       "   '{vital-it},',\n",
       "   'a',\n",
       "   'cluster',\n",
       "   'of',\n",
       "   '633',\n",
       "   '{cpus},',\n",
       "   'and',\n",
       "   'the',\n",
       "   'adoption',\n",
       "   'of',\n",
       "   'the',\n",
       "   'distributed',\n",
       "   'annotation',\n",
       "   'system',\n",
       "   '({das})',\n",
       "   'to',\n",
       "   'facilitate',\n",
       "   '{prosite}',\n",
       "   'data',\n",
       "   'integration',\n",
       "   'and',\n",
       "   'interchange',\n",
       "   'with',\n",
       "   'other',\n",
       "   'sources.',\n",
       "   'the',\n",
       "   'latest',\n",
       "   'version',\n",
       "   'of',\n",
       "   '{prosite}',\n",
       "   '(release',\n",
       "   '20.54,',\n",
       "   'of',\n",
       "   '22',\n",
       "   'september',\n",
       "   '2009)',\n",
       "   'contains',\n",
       "   '1308',\n",
       "   'patterns,',\n",
       "   '863',\n",
       "   'profiles',\n",
       "   'and',\n",
       "   '869',\n",
       "   '{prorules}.',\n",
       "   '{prosite}',\n",
       "   'is',\n",
       "   'accessible',\n",
       "   'at:',\n",
       "   'http://www.expasy.org/prosite/.\"\"impressive',\n",
       "   'advances',\n",
       "   'in',\n",
       "   '{ngs}',\n",
       "   'have',\n",
       "   'enabled',\n",
       "   'an',\n",
       "   'immense',\n",
       "   'diversity',\n",
       "   'of',\n",
       "   'novel',\n",
       "   'applications.',\n",
       "   'the',\n",
       "   'barrier',\n",
       "   'of',\n",
       "   'the',\n",
       "   '\\\\$1000',\n",
       "   'genome',\n",
       "   'has',\n",
       "   'recently',\n",
       "   'been',\n",
       "   'broken.',\n",
       "   'important',\n",
       "   'novel',\n",
       "   'tools',\n",
       "   'for',\n",
       "   'clinical',\n",
       "   'diagnostics',\n",
       "   'based',\n",
       "   'on',\n",
       "   '{ngs}',\n",
       "   'are',\n",
       "   'appearing.',\n",
       "   'third-generation',\n",
       "   'technologies',\n",
       "   'may',\n",
       "   'further',\n",
       "   'revolutionize',\n",
       "   'genomics',\n",
       "   'research.',\n",
       "   'significant',\n",
       "   'challenges',\n",
       "   'for',\n",
       "   '{ngs}',\n",
       "   'remain,',\n",
       "   'in',\n",
       "   'particular',\n",
       "   'data',\n",
       "   'storage',\n",
       "   'and',\n",
       "   'processing.',\n",
       "   'ten',\n",
       "   'years',\n",
       "   'ago',\n",
       "   'next-generation',\n",
       "   'sequencing',\n",
       "   '({ngs})',\n",
       "   'technologies',\n",
       "   'appeared',\n",
       "   'on',\n",
       "   'the',\n",
       "   'market.',\n",
       "   'during',\n",
       "   'the',\n",
       "   'past',\n",
       "   'decade,',\n",
       "   'tremendous',\n",
       "   'progress',\n",
       "   'has',\n",
       "   'been',\n",
       "   'made',\n",
       "   'in',\n",
       "   'terms',\n",
       "   'of',\n",
       "   'speed,',\n",
       "   'read',\n",
       "   'length,',\n",
       "   'and',\n",
       "   'throughput,',\n",
       "   'along',\n",
       "   'with',\n",
       "   'a',\n",
       "   'sharp',\n",
       "   'reduction',\n",
       "   'in',\n",
       "   'per-base',\n",
       "   'cost.',\n",
       "   'together,',\n",
       "   'these',\n",
       "   'advances',\n",
       "   'democratized',\n",
       "   '{ngs}',\n",
       "   'and',\n",
       "   'paved',\n",
       "   'the',\n",
       "   'way',\n",
       "   'for',\n",
       "   'the',\n",
       "   'development',\n",
       "   'of',\n",
       "   'a',\n",
       "   'large',\n",
       "   'number',\n",
       "   'of',\n",
       "   'novel',\n",
       "   '{ngs}',\n",
       "   'applications',\n",
       "   'in',\n",
       "   'basic',\n",
       "   'science',\n",
       "   'as',\n",
       "   'well',\n",
       "   'as',\n",
       "   'in',\n",
       "   'translational',\n",
       "   'research',\n",
       "   'areas',\n",
       "   'such',\n",
       "   'as',\n",
       "   'clinical',\n",
       "   'diagnostics,',\n",
       "   'agrigenomics,',\n",
       "   'and',\n",
       "   'forensic',\n",
       "   'science.',\n",
       "   'here',\n",
       "   'we',\n",
       "   'provide',\n",
       "   'an',\n",
       "   'overview',\n",
       "   'of',\n",
       "   'the',\n",
       "   'evolution',\n",
       "   'of',\n",
       "   '{ngs}',\n",
       "   'and',\n",
       "   'discuss',\n",
       "   'the',\n",
       "   'most',\n",
       "   'significant',\n",
       "   'improvements',\n",
       "   'in',\n",
       "   'sequencing',\n",
       "   'technologies',\n",
       "   'and',\n",
       "   'library',\n",
       "   'preparation',\n",
       "   'protocols.',\n",
       "   'we',\n",
       "   'also',\n",
       "   'explore',\n",
       "   'the',\n",
       "   'current',\n",
       "   'landscape',\n",
       "   'of',\n",
       "   '{ngs}',\n",
       "   'applications',\n",
       "   'and',\n",
       "   'provide',\n",
       "   'a',\n",
       "   'perspective',\n",
       "   'for',\n",
       "   'future',\n",
       "   'developments.\"{dna}',\n",
       "   'methylation:',\n",
       "   'roles',\n",
       "   'in',\n",
       "   'mammalian',\n",
       "   'development,\"{dna}',\n",
       "   'methylation',\n",
       "   'is',\n",
       "   'among',\n",
       "   'the',\n",
       "   'best',\n",
       "   'studied',\n",
       "   'epigenetic',\n",
       "   'modifications',\n",
       "   'and',\n",
       "   'is',\n",
       "   'essential',\n",
       "   'to',\n",
       "   'mammalian',\n",
       "   'development.',\n",
       "   'although',\n",
       "   'the',\n",
       "   'methylation',\n",
       "   'status',\n",
       "   'of',\n",
       "   'most',\n",
       "   '{cpg}',\n",
       "   'dinucleotides',\n",
       "   'in',\n",
       "   'the',\n",
       "   'genome',\n",
       "   'is',\n",
       "   'stably',\n",
       "   'propagated',\n",
       "   'through',\n",
       "   'mitosis,',\n",
       "   'improvements',\n",
       "   'to',\n",
       "   'methods',\n",
       "   'for',\n",
       "   'measuring',\n",
       "   'methylation',\n",
       "   'have',\n",
       "   'identified',\n",
       "   'numerous',\n",
       "   'regions',\n",
       "   'in',\n",
       "   'which',\n",
       "   'it',\n",
       "   'is',\n",
       "   'dynamically',\n",
       "   'regulated.',\n",
       "   'in',\n",
       "   'this',\n",
       "   'review,',\n",
       "   'we',\n",
       "   'discuss',\n",
       "   'key',\n",
       "   'concepts',\n",
       "   'in',\n",
       "   'the',\n",
       "   'function',\n",
       "   'of',\n",
       "   '{dna}',\n",
       "   'methylation',\n",
       "   'in',\n",
       "   'mammals,',\n",
       "   'stemming',\n",
       "   'from',\n",
       "   'more',\n",
       "   'than',\n",
       "   'two',\n",
       "   'decades',\n",
       "   'of',\n",
       "   'research,',\n",
       "   'including',\n",
       "   'many',\n",
       "   'recent',\n",
       "   'studies',\n",
       "   'that',\n",
       "   'have',\n",
       "   'elucidated',\n",
       "   'when',\n",
       "   'and',\n",
       "   'where',\n",
       "   '{dna}',\n",
       "   'methylation',\n",
       "   'has',\n",
       "   'a',\n",
       "   'regulatory',\n",
       "   'role',\n",
       "   'in',\n",
       "   'the',\n",
       "   'genome.',\n",
       "   'we',\n",
       "   'include',\n",
       "   'insights',\n",
       "   'from',\n",
       "   'early',\n",
       "   'development,',\n",
       "   'embryonic',\n",
       "   'stem',\n",
       "   'cells',\n",
       "   'and',\n",
       "   'adult',\n",
       "   'lineages,',\n",
       "   'particularly',\n",
       "   'haematopoiesis,',\n",
       "   'to',\n",
       "   'highlight',\n",
       "   'the',\n",
       "   'general',\n",
       "   'features',\n",
       "   'of',\n",
       "   'this',\n",
       "   'modification',\n",
       "   'as',\n",
       "   'it',\n",
       "   'participates',\n",
       "   'in',\n",
       "   'both',\n",
       "   'global',\n",
       "   'and',\n",
       "   'localized',\n",
       "   'epigenetic',\n",
       "   'regulation.\"storing,',\n",
       "   'and',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathways.\",\"biological',\n",
       "   'pathways,',\n",
       "   'including',\n",
       "   'metabolic',\n",
       "   'pathways,',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'networks,',\n",
       "   'signal',\n",
       "   'transduction',\n",
       "   'pathways,',\n",
       "   'and',\n",
       "   'gene',\n",
       "   'regulatory',\n",
       "   'networks,',\n",
       "   'are',\n",
       "   'currently',\n",
       "   'represented',\n",
       "   'in',\n",
       "   'over',\n",
       "   '220',\n",
       "   'diverse',\n",
       "   'databases.',\n",
       "   'these',\n",
       "   'data',\n",
       "   'are',\n",
       "   'crucial',\n",
       "   'for',\n",
       "   'the',\n",
       "   'study',\n",
       "   'of',\n",
       "   'specific',\n",
       "   'biological',\n",
       "   'processes,',\n",
       "   'including',\n",
       "   'human',\n",
       "   'diseases.',\n",
       "   'standard',\n",
       "   'exchange',\n",
       "   'formats',\n",
       "   'for',\n",
       "   'pathway',\n",
       "   'information,',\n",
       "   'such',\n",
       "   'as',\n",
       "   '{biopax},',\n",
       "   '{cellml},',\n",
       "   '{sbml}',\n",
       "   'and',\n",
       "   '{psi}-{mi},',\n",
       "   'enable',\n",
       "   'convenient',\n",
       "   'collection',\n",
       "   'of',\n",
       "   'this',\n",
       "   'data',\n",
       "   'for',\n",
       "   'biological',\n",
       "   'research,',\n",
       "   'but',\n",
       "   'mechanisms',\n",
       "   'for',\n",
       "   'common',\n",
       "   'storage',\n",
       "   'and',\n",
       "   'communication',\n",
       "   'are',\n",
       "   'required.',\n",
       "   'we',\n",
       "   'have',\n",
       "   'developed',\n",
       "   '{cpath},',\n",
       "   'an',\n",
       "   'open',\n",
       "   'source',\n",
       "   'database',\n",
       "   'and',\n",
       "   'web',\n",
       "   'application',\n",
       "   'for',\n",
       "   'collecting,',\n",
       "   'storing,',\n",
       "   'and',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathway',\n",
       "   'data.',\n",
       "   '{cpath}',\n",
       "   'makes',\n",
       "   'it',\n",
       "   'easy',\n",
       "   'to',\n",
       "   'aggregate',\n",
       "   'custom',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'sets',\n",
       "   'available',\n",
       "   'in',\n",
       "   'standard',\n",
       "   'exchange',\n",
       "   'formats',\n",
       "   'from',\n",
       "   'multiple',\n",
       "   'databases,',\n",
       "   'present',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'to',\n",
       "   'biologists',\n",
       "   'via',\n",
       "   'a',\n",
       "   'customizable',\n",
       "   'web',\n",
       "   'interface,',\n",
       "   'and',\n",
       "   'export',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'via',\n",
       "   'a',\n",
       "   'web',\n",
       "   'service',\n",
       "   'to',\n",
       "   'third-party',\n",
       "   'software,',\n",
       "   'such',\n",
       "   'as',\n",
       "   'cytoscape,',\n",
       "   'for',\n",
       "   'visualization',\n",
       "   'and',\n",
       "   'analysis.',\n",
       "   '{cpath}',\n",
       "   'is',\n",
       "   'software',\n",
       "   'only,',\n",
       "   'and',\n",
       "   'does',\n",
       "   'not',\n",
       "   'include',\n",
       "   'new',\n",
       "   'pathway',\n",
       "   'information.',\n",
       "   'key',\n",
       "   'features',\n",
       "   'include:',\n",
       "   'a',\n",
       "   'built-in',\n",
       "   'identifier',\n",
       "   'mapping',\n",
       "   'service',\n",
       "   'for',\n",
       "   'linking',\n",
       "   'identical',\n",
       "   'interactors',\n",
       "   'and',\n",
       "   'linking',\n",
       "   'to',\n",
       "   'external',\n",
       "   'resources;',\n",
       "   'built-in',\n",
       "   'support',\n",
       "   'for',\n",
       "   '{psi}-{mi}',\n",
       "   'and',\n",
       "   '{biopax}',\n",
       "   'standard',\n",
       "   'pathway',\n",
       "   'exchange',\n",
       "   'formats;',\n",
       "   'a',\n",
       "   'web',\n",
       "   'service',\n",
       "   'interface',\n",
       "   'for',\n",
       "   'searching',\n",
       "   'and',\n",
       "   'retrieving',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'sets;',\n",
       "   'and',\n",
       "   'thorough',\n",
       "   'documentation.',\n",
       "   'the',\n",
       "   '{cpath}',\n",
       "   'software',\n",
       "   'is',\n",
       "   'freely',\n",
       "   'available',\n",
       "   'under',\n",
       "   'the',\n",
       "   '{lgpl}',\n",
       "   'open',\n",
       "   'source',\n",
       "   'license',\n",
       "   'for',\n",
       "   'academic',\n",
       "   'and',\n",
       "   'commercial',\n",
       "   'use.',\n",
       "   '{cpath}',\n",
       "   'is',\n",
       "   'a',\n",
       "   'robust,',\n",
       "   'scalable,',\n",
       "   'modular,',\n",
       "   'professional-grade',\n",
       "   'software',\n",
       "   'platform',\n",
       "   'for',\n",
       "   'collecting,',\n",
       "   'storing,',\n",
       "   'and',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathways.',\n",
       "   'it',\n",
       "   'can',\n",
       "   'serve',\n",
       "   'as',\n",
       "   'the',\n",
       "   'core',\n",
       "   'data',\n",
       "   'handling',\n",
       "   'component',\n",
       "   'in',\n",
       "   'information',\n",
       "   'systems',\n",
       "   'for',\n",
       "   'pathway',\n",
       "   'visualization,',\n",
       "   'analysis',\n",
       "   'and',\n",
       "   'modeling.\"new',\n",
       "   'haven,',\n",
       "   'connecticut',\n",
       "   '06520,',\n",
       "   'usa.\",annotation',\n",
       "   'transfer',\n",
       "   'between',\n",
       "   'genomes:',\n",
       "   'protein-protein',\n",
       "   'interologs',\n",
       "   'and',\n",
       "   '{protein-dna}',\n",
       "   'regulogs.,texts,',\n",
       "   'and',\n",
       "   'trees.\",\"{bionames}',\n",
       "   'is',\n",
       "   'a',\n",
       "   'web',\n",
       "   'database',\n",
       "   'of',\n",
       "   'taxonomic',\n",
       "   'names',\n",
       "   'for',\n",
       "   'animals,',\n",
       "   'linked',\n",
       "   'to',\n",
       "   'the',\n",
       "   'primary',\n",
       "   'literature',\n",
       "   'and,',\n",
       "   'wherever',\n",
       "   'possible,',\n",
       "   'to',\n",
       "   'phylogenetic',\n",
       "   'trees.',\n",
       "   'it',\n",
       "   'aims',\n",
       "   'to',\n",
       "   'provide',\n",
       "   'a',\n",
       "   'taxonomic',\n",
       "   '\"\"dashboard\"\"',\n",
       "   'where',\n",
       "   'at',\n",
       "   'a',\n",
       "   'glance',\n",
       "   'we',\n",
       "   'can',\n",
       "   'see',\n",
       "   'a',\n",
       "   'summary',\n",
       "   'of',\n",
       "   'the',\n",
       "   'taxonomic',\n",
       "   'and',\n",
       "   'phylogenetic',\n",
       "   'information',\n",
       "   'we',\n",
       "   'have',\n",
       "   'for',\n",
       "   'a',\n",
       "   'given',\n",
       "   'taxon',\n",
       "   'and',\n",
       "   'hence',\n",
       "   'provide',\n",
       "   'a',\n",
       "   'quick',\n",
       "   'answer',\n",
       "   'to',\n",
       "   'the',\n",
       "   'basic',\n",
       "   'question',\n",
       "   '\"\"what',\n",
       "   'is',\n",
       "   'this',\n",
       "   'taxon?\"\"',\n",
       "   '{bionames}',\n",
       "   'combines',\n",
       "   'classifications',\n",
       "   'from',\n",
       "   'the',\n",
       "   'global',\n",
       "   'biodiversity',\n",
       "   'information',\n",
       "   'facility',\n",
       "   '({gbif})',\n",
       "   'and',\n",
       "   '{genbank},',\n",
       "   'images',\n",
       "   'from',\n",
       "   'the',\n",
       "   'encyclopedia',\n",
       "   'of',\n",
       "   'life',\n",
       "   '({eol}),',\n",
       "   'animal',\n",
       "   'names',\n",
       "   'from',\n",
       "   'the',\n",
       "   'index',\n",
       "   'of',\n",
       "   'organism',\n",
       "   'names',\n",
       "   '({ion}),',\n",
       "   'and',\n",
       "   'bibliographic',\n",
       "   'data',\n",
       "   'from',\n",
       "   'multiple',\n",
       "   'sources',\n",
       "   'including',\n",
       "   'the',\n",
       "   'biodiversity',\n",
       "   'heritage',\n",
       "   'library',\n",
       "   '({bhl})',\n",
       "   'and',\n",
       "   '{crossref}.',\n",
       "   'the',\n",
       "   'user',\n",
       "   'interface',\n",
       "   'includes',\n",
       "   'display',\n",
       "   'of',\n",
       "   'full',\n",
       "   'text',\n",
       "   'articles,',\n",
       "   'interactive',\n",
       "   'timelines',\n",
       "   'of',\n",
       "   'taxonomic',\n",
       "   'publications,',\n",
       "   'and',\n",
       "   'zoomable',\n",
       "   'phylogenies.',\n",
       "   'it',\n",
       "   'is',\n",
       "   'available',\n",
       "   'at',\n",
       "   'http://bionames.org.\"\"the',\n",
       "   'prediction',\n",
       "   'of',\n",
       "   'regulatory',\n",
       "   'elements',\n",
       "   'is',\n",
       "   'a',\n",
       "   'problem',\n",
       "   'where',\n",
       "   'computational',\n",
       "   'methods',\n",
       "   'offer',\n",
       "   'great',\n",
       "   'hope.',\n",
       "   'over',\n",
       "   'the',\n",
       "   'past',\n",
       "   'few',\n",
       "   'years,',\n",
       "   'numerous',\n",
       "   'tools',\n",
       "   'have',\n",
       "   'become',\n",
       "   'available',\n",
       "   'for',\n",
       "   'this',\n",
       "   'task.',\n",
       "   'the',\n",
       "   'purpose',\n",
       "   'of',\n",
       "   'the',\n",
       "   'current',\n",
       "   'assessment',\n",
       "   'is',\n",
       "   'twofold:',\n",
       "   'to',\n",
       "   'provide',\n",
       "   'some',\n",
       "   'guidance',\n",
       "   'to',\n",
       "   'users',\n",
       "   'regarding',\n",
       "   'the',\n",
       "   'accuracy',\n",
       "   'of',\n",
       "   'currently',\n",
       "   'available',\n",
       "   'tools',\n",
       "   'in',\n",
       "   'various',\n",
       "   'settings,',\n",
       "   'and',\n",
       "   'to',\n",
       "   'provide',\n",
       "   'a',\n",
       "   'benchmark',\n",
       "   'of',\n",
       "   'data',\n",
       "   'sets',\n",
       "   'for',\n",
       "   'assessing',\n",
       "   'future',\n",
       "   'tools.\"\"unraveling',\n",
       "   'the',\n",
       "   'mechanisms',\n",
       "   'that',\n",
       "   'regulate',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'is',\n",
       "   'a',\n",
       "   'major',\n",
       "   'challenge',\n",
       "   'in',\n",
       "   'biology.',\n",
       "   'an',\n",
       "   'important',\n",
       "   'task',\n",
       "   'in',\n",
       "   'this',\n",
       "   'challenge',\n",
       "   'is',\n",
       "   'to',\n",
       "   'identify',\n",
       "   'regulatory',\n",
       "   'elements,',\n",
       "   'especially',\n",
       "   'the',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   'in',\n",
       "   'deoxyribonucleic',\n",
       "   'acid',\n",
       "   '({dna})',\n",
       "   'for',\n",
       "   'transcription',\n",
       "   'factors.',\n",
       "   'these',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   'are',\n",
       "   'short',\n",
       "   '{dna}',\n",
       "   'segments',\n",
       "   'that',\n",
       "   'are',\n",
       "   'called',\n",
       "   'motifs.',\n",
       "   'recent',\n",
       "   'advances',\n",
       "   'in',\n",
       "   'genome',\n",
       "   'sequence',\n",
       "   'availability',\n",
       "   'and',\n",
       "   'in',\n",
       "   'high-throughput',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'analysis',\n",
       "   'technologies',\n",
       "   'have',\n",
       "   'allowed',\n",
       "   ...]),\n",
       " ('beaca02b21b7cad6cb738c5e2682af8d',\n",
       "  ['new',\n",
       "   'haven,',\n",
       "   'connecticut',\n",
       "   '06520,',\n",
       "   'usa.\",annotation',\n",
       "   'transfer',\n",
       "   'between',\n",
       "   'genomes:',\n",
       "   'protein-protein',\n",
       "   'interologs',\n",
       "   'and',\n",
       "   '{protein-dna}',\n",
       "   'regulogs.,\"bioinformatics',\n",
       "   'program,',\n",
       "   'boston',\n",
       "   'university,',\n",
       "   'boston,',\n",
       "   'massachusetts;',\n",
       "   'department',\n",
       "   'of',\n",
       "   'biomedical',\n",
       "   'engineering,',\n",
       "   'boston',\n",
       "   'university,',\n",
       "   'boston,',\n",
       "   'massachusetts\",{zdock}:',\n",
       "   'an',\n",
       "   'initial-stage',\n",
       "   'protein-docking',\n",
       "   'algorithm,\"the',\n",
       "   'development',\n",
       "   'of',\n",
       "   'scoring',\n",
       "   'functions',\n",
       "   'is',\n",
       "   'of',\n",
       "   'great',\n",
       "   'importance',\n",
       "   'to',\n",
       "   'protein',\n",
       "   'docking.',\n",
       "   'here',\n",
       "   'we',\n",
       "   'present',\n",
       "   'a',\n",
       "   'new',\n",
       "   'scoring',\n",
       "   'function',\n",
       "   'for',\n",
       "   'the',\n",
       "   'initial',\n",
       "   'stage',\n",
       "   'of',\n",
       "   'unbound',\n",
       "   'docking.',\n",
       "   'it',\n",
       "   'combines',\n",
       "   'our',\n",
       "   'recently',\n",
       "   'developed',\n",
       "   'pairwise',\n",
       "   'shape',\n",
       "   'complementarity',\n",
       "   'with',\n",
       "   'desolvation',\n",
       "   'and',\n",
       "   'electrostatics.',\n",
       "   'we',\n",
       "   'compare',\n",
       "   'this',\n",
       "   'scoring',\n",
       "   'function',\n",
       "   'with',\n",
       "   'three',\n",
       "   'other',\n",
       "   'functions',\n",
       "   'on',\n",
       "   'a',\n",
       "   'large',\n",
       "   'benchmark',\n",
       "   'of',\n",
       "   '49',\n",
       "   'nonredundant',\n",
       "   'test',\n",
       "   'cases',\n",
       "   'and',\n",
       "   'show',\n",
       "   'its',\n",
       "   'superior',\n",
       "   'performance,',\n",
       "   'especially',\n",
       "   'for',\n",
       "   'the',\n",
       "   'antibody-antigen',\n",
       "   'category',\n",
       "   'of',\n",
       "   'test',\n",
       "   'cases.',\n",
       "   'for',\n",
       "   '44',\n",
       "   'test',\n",
       "   'cases',\n",
       "   '(90\\\\%',\n",
       "   'of',\n",
       "   'the',\n",
       "   'benchmark),',\n",
       "   'we',\n",
       "   'can',\n",
       "   'retain',\n",
       "   'at',\n",
       "   'least',\n",
       "   'one',\n",
       "   'near-native',\n",
       "   'structure',\n",
       "   'within',\n",
       "   'the',\n",
       "   'top',\n",
       "   '2000',\n",
       "   'predictions',\n",
       "   'at',\n",
       "   'the',\n",
       "   '6°',\n",
       "   'rotational',\n",
       "   'sampling',\n",
       "   'density,',\n",
       "   'with',\n",
       "   'an',\n",
       "   'average',\n",
       "   'of',\n",
       "   '52',\n",
       "   'near-native',\n",
       "   'structures',\n",
       "   'per',\n",
       "   'test',\n",
       "   'case.',\n",
       "   'the',\n",
       "   'remaining',\n",
       "   'five',\n",
       "   'difficult',\n",
       "   'test',\n",
       "   'cases',\n",
       "   'can',\n",
       "   'be',\n",
       "   'explained',\n",
       "   'by',\n",
       "   'a',\n",
       "   'combination',\n",
       "   'of',\n",
       "   'poor',\n",
       "   'binding',\n",
       "   'affinity,',\n",
       "   'large',\n",
       "   'backbone',\n",
       "   'conformational',\n",
       "   'changes,',\n",
       "   'and',\n",
       "   'our',\n",
       "   \"algorithm's\",\n",
       "   'strong',\n",
       "   'tendency',\n",
       "   'for',\n",
       "   'identifying',\n",
       "   'large',\n",
       "   'concave',\n",
       "   'binding',\n",
       "   'pockets.',\n",
       "   'all',\n",
       "   'four',\n",
       "   'scoring',\n",
       "   'functions',\n",
       "   'have',\n",
       "   'been',\n",
       "   'integrated',\n",
       "   'into',\n",
       "   'our',\n",
       "   'fast',\n",
       "   'fourier',\n",
       "   'transform',\n",
       "   'based',\n",
       "   'docking',\n",
       "   'algorithm',\n",
       "   '{zdock},',\n",
       "   'which',\n",
       "   'is',\n",
       "   'freely',\n",
       "   'available',\n",
       "   'to',\n",
       "   'academic',\n",
       "   'users',\n",
       "   'at',\n",
       "   'http://zlab.bu.edu/∼rong/dock.',\n",
       "   'proteins',\n",
       "   '2003;52:80–87.',\n",
       "   '{\\\\copyright}',\n",
       "   '2003',\n",
       "   '{wiley-liss},',\n",
       "   'inc.\"\"human',\n",
       "   'protein',\n",
       "   'reference',\n",
       "   'database',\n",
       "   '({hprd}--http://www.hprd.org/),',\n",
       "   'initially',\n",
       "   'described',\n",
       "   'in',\n",
       "   '2003,',\n",
       "   'is',\n",
       "   'a',\n",
       "   'database',\n",
       "   'of',\n",
       "   'curated',\n",
       "   'proteomic',\n",
       "   'information',\n",
       "   'pertaining',\n",
       "   'to',\n",
       "   'human',\n",
       "   'proteins.',\n",
       "   'we',\n",
       "   'have',\n",
       "   'recently',\n",
       "   'added',\n",
       "   'a',\n",
       "   'number',\n",
       "   'of',\n",
       "   'new',\n",
       "   'features',\n",
       "   'in',\n",
       "   '{hprd}.',\n",
       "   'these',\n",
       "   'include',\n",
       "   '{phosphomotif}',\n",
       "   'finder,',\n",
       "   'which',\n",
       "   'allows',\n",
       "   'users',\n",
       "   'to',\n",
       "   'find',\n",
       "   'the',\n",
       "   'presence',\n",
       "   'of',\n",
       "   'over',\n",
       "   '320',\n",
       "   'experimentally',\n",
       "   'verified',\n",
       "   'phosphorylation',\n",
       "   'motifs',\n",
       "   'in',\n",
       "   'proteins',\n",
       "   'of',\n",
       "   'interest.',\n",
       "   'another',\n",
       "   'new',\n",
       "   'feature',\n",
       "   'is',\n",
       "   'a',\n",
       "   'protein',\n",
       "   'distributed',\n",
       "   'annotation',\n",
       "   '{system--human}',\n",
       "   'proteinpedia',\n",
       "   '(http://www.humanproteinpedia.org/)--through',\n",
       "   'which',\n",
       "   'laboratories',\n",
       "   'can',\n",
       "   'submit',\n",
       "   'their',\n",
       "   'data,',\n",
       "   'which',\n",
       "   'is',\n",
       "   'mapped',\n",
       "   'onto',\n",
       "   'protein',\n",
       "   'entries',\n",
       "   'in',\n",
       "   '{hprd}.',\n",
       "   'over',\n",
       "   '75',\n",
       "   'laboratories',\n",
       "   'involved',\n",
       "   'in',\n",
       "   'proteomics',\n",
       "   'research',\n",
       "   'have',\n",
       "   'already',\n",
       "   'participated',\n",
       "   'in',\n",
       "   'this',\n",
       "   'effort',\n",
       "   'by',\n",
       "   'submitting',\n",
       "   'data',\n",
       "   'for',\n",
       "   'over',\n",
       "   '15,000',\n",
       "   'human',\n",
       "   'proteins.',\n",
       "   'the',\n",
       "   'submitted',\n",
       "   'data',\n",
       "   'includes',\n",
       "   'mass',\n",
       "   'spectrometry',\n",
       "   'and',\n",
       "   'protein',\n",
       "   'microarray-derived',\n",
       "   'data,',\n",
       "   'among',\n",
       "   'other',\n",
       "   'data',\n",
       "   'types.',\n",
       "   'finally,',\n",
       "   '{hprd}',\n",
       "   'is',\n",
       "   'also',\n",
       "   'linked',\n",
       "   'to',\n",
       "   'a',\n",
       "   'compendium',\n",
       "   'of',\n",
       "   'human',\n",
       "   'signaling',\n",
       "   'pathways',\n",
       "   'developed',\n",
       "   'by',\n",
       "   'our',\n",
       "   'group,',\n",
       "   '{netpath}',\n",
       "   '(http://www.netpath.org/),',\n",
       "   'which',\n",
       "   'currently',\n",
       "   'contains',\n",
       "   'annotations',\n",
       "   'for',\n",
       "   'several',\n",
       "   'cancer',\n",
       "   'and',\n",
       "   'immune',\n",
       "   'signaling',\n",
       "   'pathways.',\n",
       "   'since',\n",
       "   'the',\n",
       "   'last',\n",
       "   'update,',\n",
       "   'more',\n",
       "   'than',\n",
       "   '5500',\n",
       "   'new',\n",
       "   'protein',\n",
       "   'sequences',\n",
       "   'have',\n",
       "   'been',\n",
       "   'added,',\n",
       "   'making',\n",
       "   '{hprd}',\n",
       "   'a',\n",
       "   'comprehensive',\n",
       "   'resource',\n",
       "   'for',\n",
       "   'studying',\n",
       "   'the',\n",
       "   'human',\n",
       "   'proteome.\"\"{background}:microarrays',\n",
       "   'revolutionized',\n",
       "   'biological',\n",
       "   'research',\n",
       "   'by',\n",
       "   'enabling',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'comparisons',\n",
       "   'on',\n",
       "   'a',\n",
       "   'transcriptome-wide',\n",
       "   'scale.',\n",
       "   'microarrays,',\n",
       "   'however,',\n",
       "   'do',\n",
       "   'not',\n",
       "   'estimate',\n",
       "   'absolute',\n",
       "   'expression',\n",
       "   'level',\n",
       "   'accurately.',\n",
       "   'at',\n",
       "   'present,',\n",
       "   'high',\n",
       "   'throughput',\n",
       "   'sequencing',\n",
       "   'is',\n",
       "   'emerging',\n",
       "   'as',\n",
       "   'an',\n",
       "   'alternative',\n",
       "   'methodology',\n",
       "   'for',\n",
       "   'transcriptome',\n",
       "   'studies.',\n",
       "   'although',\n",
       "   'free',\n",
       "   'of',\n",
       "   'many',\n",
       "   'limitations',\n",
       "   'imposed',\n",
       "   'by',\n",
       "   'microarray',\n",
       "   'design,',\n",
       "   'its',\n",
       "   'potential',\n",
       "   'to',\n",
       "   'estimate',\n",
       "   'absolute',\n",
       "   'transcript',\n",
       "   'levels',\n",
       "   'is',\n",
       "   '{unknown.results}:in',\n",
       "   'this',\n",
       "   'study,',\n",
       "   'we',\n",
       "   'evaluate',\n",
       "   'relative',\n",
       "   'accuracy',\n",
       "   'of',\n",
       "   'microarrays',\n",
       "   'and',\n",
       "   'transcriptome',\n",
       "   'sequencing',\n",
       "   '({rna}-seq)',\n",
       "   'using',\n",
       "   'third',\n",
       "   'methodology:',\n",
       "   'proteomics.',\n",
       "   'we',\n",
       "   'find',\n",
       "   'that',\n",
       "   '{rna}-seq',\n",
       "   'provides',\n",
       "   'a',\n",
       "   'better',\n",
       "   'estimate',\n",
       "   'of',\n",
       "   'absolute',\n",
       "   'expression',\n",
       "   '{levels.conclusion}:our',\n",
       "   'result',\n",
       "   'shows',\n",
       "   'that',\n",
       "   'in',\n",
       "   'terms',\n",
       "   'of',\n",
       "   'overall',\n",
       "   'technical',\n",
       "   'performance,',\n",
       "   '{rna}-seq',\n",
       "   'is',\n",
       "   'the',\n",
       "   'technique',\n",
       "   'of',\n",
       "   'choice',\n",
       "   'for',\n",
       "   'studies',\n",
       "   'that',\n",
       "   'require',\n",
       "   'accurate',\n",
       "   'estimation',\n",
       "   'of',\n",
       "   'absolute',\n",
       "   'transcript',\n",
       "   'levels.\"\"cancers',\n",
       "   'evolve',\n",
       "   'by',\n",
       "   'a',\n",
       "   'reiterative',\n",
       "   'process',\n",
       "   'of',\n",
       "   'clonal',\n",
       "   'expansion,',\n",
       "   'genetic',\n",
       "   'diversification',\n",
       "   'and',\n",
       "   'clonal',\n",
       "   'selection',\n",
       "   'within',\n",
       "   'the',\n",
       "   'adaptive',\n",
       "   'landscapes',\n",
       "   'of',\n",
       "   'tissue',\n",
       "   'ecosystems.',\n",
       "   'the',\n",
       "   'dynamics',\n",
       "   'are',\n",
       "   'complex,',\n",
       "   'with',\n",
       "   'highly',\n",
       "   'variable',\n",
       "   'patterns',\n",
       "   'of',\n",
       "   'genetic',\n",
       "   'diversity',\n",
       "   'and',\n",
       "   'resulting',\n",
       "   'clonal',\n",
       "   'architecture.',\n",
       "   'therapeutic',\n",
       "   'intervention',\n",
       "   'may',\n",
       "   'destroy',\n",
       "   'cancer',\n",
       "   'clones',\n",
       "   'and',\n",
       "   'erode',\n",
       "   'their',\n",
       "   'habitats,',\n",
       "   'but',\n",
       "   'it',\n",
       "   'can',\n",
       "   'also',\n",
       "   'inadvertently',\n",
       "   'provide',\n",
       "   'a',\n",
       "   'potent',\n",
       "   'selective',\n",
       "   'pressure',\n",
       "   'for',\n",
       "   'the',\n",
       "   'expansion',\n",
       "   'of',\n",
       "   'resistant',\n",
       "   'variants.',\n",
       "   'the',\n",
       "   'inherently',\n",
       "   'darwinian',\n",
       "   'character',\n",
       "   'of',\n",
       "   'cancer',\n",
       "   'is',\n",
       "   'the',\n",
       "   'primary',\n",
       "   'reason',\n",
       "   'for',\n",
       "   'this',\n",
       "   'therapeutic',\n",
       "   'failure,',\n",
       "   'but',\n",
       "   'it',\n",
       "   'may',\n",
       "   'also',\n",
       "   'hold',\n",
       "   'the',\n",
       "   'key',\n",
       "   'to',\n",
       "   'more',\n",
       "   'effective',\n",
       "   'control.\"\"intratumoral',\n",
       "   'heterogeneity',\n",
       "   'arises',\n",
       "   'through',\n",
       "   'evolution',\n",
       "   'of',\n",
       "   'genetically',\n",
       "   'diverse',\n",
       "   'subclones',\n",
       "   'during',\n",
       "   'tumor',\n",
       "   'progression.',\n",
       "   'however,',\n",
       "   'whether',\n",
       "   'cells',\n",
       "   'within',\n",
       "   'single',\n",
       "   'genetic',\n",
       "   'clones',\n",
       "   'are',\n",
       "   'functionally',\n",
       "   'equivalent',\n",
       "   'remains',\n",
       "   'unknown.',\n",
       "   'by',\n",
       "   'combining',\n",
       "   '{dna}',\n",
       "   'copy',\n",
       "   'number',\n",
       "   'alteration',\n",
       "   '({cna})',\n",
       "   'profiling,',\n",
       "   'sequencing,',\n",
       "   'and',\n",
       "   'lentiviral',\n",
       "   'lineage',\n",
       "   'tracking,',\n",
       "   'we',\n",
       "   'followed',\n",
       "   'the',\n",
       "   'repopulation',\n",
       "   'dynamics',\n",
       "   'of',\n",
       "   '150',\n",
       "   'single',\n",
       "   'lentivirus-marked',\n",
       "   'lineages',\n",
       "   'from',\n",
       "   '10',\n",
       "   'human',\n",
       "   'colorectal',\n",
       "   'cancers',\n",
       "   'through',\n",
       "   'serial',\n",
       "   'xenograft',\n",
       "   'passages',\n",
       "   'in',\n",
       "   'mice.',\n",
       "   '{cna}',\n",
       "   'and',\n",
       "   'mutational',\n",
       "   'analysis',\n",
       "   'distinguished',\n",
       "   'individual',\n",
       "   'clones',\n",
       "   'and',\n",
       "   'showed',\n",
       "   'that',\n",
       "   'clones',\n",
       "   'remained',\n",
       "   'stable',\n",
       "   'on',\n",
       "   'serial',\n",
       "   'transplantation.',\n",
       "   'despite',\n",
       "   'this',\n",
       "   'stability,',\n",
       "   'the',\n",
       "   'proliferation,',\n",
       "   'persistence,',\n",
       "   'and',\n",
       "   'chemotherapy',\n",
       "   'tolerance',\n",
       "   'of',\n",
       "   'lentivirally',\n",
       "   'marked',\n",
       "   'lineages',\n",
       "   'were',\n",
       "   'variable',\n",
       "   'within',\n",
       "   'each',\n",
       "   'clone.',\n",
       "   'chemotherapy',\n",
       "   'promoted',\n",
       "   'dominance',\n",
       "   'of',\n",
       "   'previously',\n",
       "   'minor',\n",
       "   'or',\n",
       "   'dormant',\n",
       "   'lineages.',\n",
       "   'thus,',\n",
       "   'apart',\n",
       "   'from',\n",
       "   'genetic',\n",
       "   'diversity,',\n",
       "   'tumor',\n",
       "   'cells',\n",
       "   'display',\n",
       "   'inherent',\n",
       "   'functional',\n",
       "   'variability',\n",
       "   'in',\n",
       "   'tumor',\n",
       "   'propagation',\n",
       "   'potential,',\n",
       "   'a',\n",
       "   'mechanism',\n",
       "   'that',\n",
       "   'contributes',\n",
       "   'both',\n",
       "   'to',\n",
       "   'cancer',\n",
       "   'growth',\n",
       "   'and',\n",
       "   'therapy',\n",
       "   'tolerance.\"\"recent',\n",
       "   'advances',\n",
       "   'in',\n",
       "   'functional',\n",
       "   'genomics',\n",
       "   'have',\n",
       "   'helped',\n",
       "   'generate',\n",
       "   'large-scale',\n",
       "   'high-throughput',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'data.',\n",
       "   'such',\n",
       "   'networks,',\n",
       "   'though',\n",
       "   'extremely',\n",
       "   'valuable',\n",
       "   'towards',\n",
       "   'molecular',\n",
       "   'level',\n",
       "   'understanding',\n",
       "   'of',\n",
       "   'cells,',\n",
       "   'do',\n",
       "   'not',\n",
       "   'provide',\n",
       "   'any',\n",
       "   'direct',\n",
       "   'information',\n",
       "   'about',\n",
       "   'the',\n",
       "   'regions',\n",
       "   '(domains)',\n",
       "   'in',\n",
       "   'the',\n",
       "   'proteins',\n",
       "   'that',\n",
       "   'mediate',\n",
       "   'the',\n",
       "   'interaction.',\n",
       "   'here,',\n",
       "   'we',\n",
       "   'performed',\n",
       "   'co-evolutionary',\n",
       "   'analysis',\n",
       "   'of',\n",
       "   'domains',\n",
       "   'in',\n",
       "   'interacting',\n",
       "   'proteins',\n",
       "   'in',\n",
       "   'order',\n",
       "   'to',\n",
       "   'understand',\n",
       "   'the',\n",
       "   'degree',\n",
       "   'of',\n",
       "   'co-evolution',\n",
       "   'of',\n",
       "   'interacting',\n",
       "   'and',\n",
       "   'non-interacting',\n",
       "   'domains.',\n",
       "   'using',\n",
       "   'a',\n",
       "   'combination',\n",
       "   'of',\n",
       "   'sequence',\n",
       "   'and',\n",
       "   'structural',\n",
       "   'analysis,',\n",
       "   'we',\n",
       "   'analyzed',\n",
       "   'protein–protein',\n",
       "   'interactions',\n",
       "   'in',\n",
       "   '{f1-atpase},',\n",
       "   '{sec23p/sec24p},',\n",
       "   '{dna}-directed',\n",
       "   '{rna}',\n",
       "   'polymerase',\n",
       "   'and',\n",
       "   'nuclear',\n",
       "   'pore',\n",
       "   'complexes,',\n",
       "   'and',\n",
       "   'found',\n",
       "   'that',\n",
       "   'interacting',\n",
       "   'domain',\n",
       "   'pair(s)',\n",
       "   'for',\n",
       "   'a',\n",
       "   'given',\n",
       "   'interaction',\n",
       "   'exhibits',\n",
       "   'higher',\n",
       "   'level',\n",
       "   'of',\n",
       "   'co-evolution',\n",
       "   'than',\n",
       "   'the',\n",
       "   'non-interacting',\n",
       "   'domain',\n",
       "   'pairs.',\n",
       "   'motivated',\n",
       "   'by',\n",
       "   'this',\n",
       "   'finding,',\n",
       "   'we',\n",
       "   'developed',\n",
       "   'a',\n",
       "   'computational',\n",
       "   'method',\n",
       "   'to',\n",
       "   'test',\n",
       "   'the',\n",
       "   'generality',\n",
       "   'of',\n",
       "   'the',\n",
       "   'observed',\n",
       "   'trend,',\n",
       "   'and',\n",
       "   'to',\n",
       "   'predict',\n",
       "   'large-scale',\n",
       "   'domain–domain',\n",
       "   'interactions.',\n",
       "   'given',\n",
       "   'a',\n",
       "   'protein–protein',\n",
       "   'interaction,',\n",
       "   'the',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'predicts',\n",
       "   'the',\n",
       "   'domain',\n",
       "   'pair(s)',\n",
       "   'that',\n",
       "   'is',\n",
       "   'most',\n",
       "   'likely',\n",
       "   'to',\n",
       "   'mediate',\n",
       "   'the',\n",
       "   'protein',\n",
       "   'interaction.',\n",
       "   'we',\n",
       "   'applied',\n",
       "   'this',\n",
       "   'method',\n",
       "   'on',\n",
       "   'the',\n",
       "   'yeast',\n",
       "   'interactome',\n",
       "   'to',\n",
       "   'predict',\n",
       "   'domain–domain',\n",
       "   'interactions,',\n",
       "   'and',\n",
       "   'used',\n",
       "   'known',\n",
       "   'domain–domain',\n",
       "   'interactions',\n",
       "   'found',\n",
       "   'in',\n",
       "   '{pdb}',\n",
       "   'crystal',\n",
       "   'structures',\n",
       "   'to',\n",
       "   'validate',\n",
       "   'our',\n",
       "   'predictions.',\n",
       "   'our',\n",
       "   'results',\n",
       "   'show',\n",
       "   'that',\n",
       "   'the',\n",
       "   'prediction',\n",
       "   'accuracy',\n",
       "   'of',\n",
       "   'the',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'is',\n",
       "   'statistically',\n",
       "   'significant.',\n",
       "   'comparison',\n",
       "   'of',\n",
       "   'our',\n",
       "   'prediction',\n",
       "   'results',\n",
       "   'with',\n",
       "   'those',\n",
       "   'from',\n",
       "   'two',\n",
       "   'other',\n",
       "   'methods',\n",
       "   'reveals',\n",
       "   'that',\n",
       "   'only',\n",
       "   'a',\n",
       "   'fraction',\n",
       "   'of',\n",
       "   'predictions',\n",
       "   'are',\n",
       "   'shared',\n",
       "   'by',\n",
       "   'all',\n",
       "   'the',\n",
       "   'three',\n",
       "   'methods,',\n",
       "   'indicating',\n",
       "   'that',\n",
       "   'the',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'can',\n",
       "   'detect',\n",
       "   'known',\n",
       "   'interactions',\n",
       "   'missed',\n",
       "   'by',\n",
       "   'other',\n",
       "   'methods.',\n",
       "   'we',\n",
       "   'believe',\n",
       "   'that',\n",
       "   'the',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'can',\n",
       "   'be',\n",
       "   'used',\n",
       "   'with',\n",
       "   'other',\n",
       "   'methods',\n",
       "   'to',\n",
       "   'help',\n",
       "   'identify',\n",
       "   'previously',\n",
       "   'unrecognized',\n",
       "   'domain–domain',\n",
       "   'interactions',\n",
       "   'on',\n",
       "   'a',\n",
       "   'genome',\n",
       "   'scale,',\n",
       "   'and',\n",
       "   'could',\n",
       "   'potentially',\n",
       "   'help',\n",
       "   'reduce',\n",
       "   'the',\n",
       "   'search',\n",
       "   'space',\n",
       "   'for',\n",
       "   'identifying',\n",
       "   'interaction',\n",
       "   'sites.\"boston,',\n",
       "   'ma',\n",
       "   '02215.\",\"structure,',\n",
       "   'function,',\n",
       "   'and',\n",
       "   'evolution',\n",
       "   'of',\n",
       "   'transient',\n",
       "   'and',\n",
       "   'obligate',\n",
       "   'protein–protein',\n",
       "   'interactions\",\"recent',\n",
       "   'analyses',\n",
       "   'of',\n",
       "   'high-throughput',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'data',\n",
       "   'coupled',\n",
       "   'with',\n",
       "   'large-scale',\n",
       "   'investigations',\n",
       "   'of',\n",
       "   'evolutionary',\n",
       "   'properties',\n",
       "   'of',\n",
       "   'interaction',\n",
       "   'networks',\n",
       "   'have',\n",
       "   'left',\n",
       "   'some',\n",
       "   'unanswered',\n",
       "   'questions.',\n",
       "   'to',\n",
       "   'what',\n",
       "   'extent',\n",
       "   'do',\n",
       "   'protein',\n",
       "   'interactions',\n",
       "   'act',\n",
       "   'as',\n",
       "   'constraints',\n",
       "   'during',\n",
       "   'evolution',\n",
       "   'of',\n",
       "   'the',\n",
       "   'protein',\n",
       "   'sequence?',\n",
       "   'how',\n",
       "   'does',\n",
       "   'the',\n",
       "   'type',\n",
       "   'of',\n",
       "   'interaction,',\n",
       "   'specifically',\n",
       "   'transient',\n",
       "   'or',\n",
       "   'obligate,',\n",
       "   'play',\n",
       "   'into',\n",
       "   'these',\n",
       "   ...])]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#splitting the values of transJoinResultRDD every time a blank space is encountered\n",
    "splitRDD = transJoinResultRDD.map(lambda x: (x[0], x[1].split(' ')))\n",
    "transJoinResultRDD.unpersist()\n",
    "splitRDD.take(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('f05bcffe7951de9e5a32fff4a42eb088',\n",
       "  ['protein',\n",
       "   'domain',\n",
       "   'database',\n",
       "   'functional',\n",
       "   'characterization',\n",
       "   'annotation.\",\"{prosite}',\n",
       "   'consists',\n",
       "   'documentation',\n",
       "   'entries',\n",
       "   'describing',\n",
       "   'protein',\n",
       "   'domains,',\n",
       "   'families',\n",
       "   'functional',\n",
       "   'sites,',\n",
       "   'patterns',\n",
       "   'profiles',\n",
       "   'identify',\n",
       "   'them.',\n",
       "   'complemented',\n",
       "   '{prorule},',\n",
       "   'collection',\n",
       "   'rules',\n",
       "   'based',\n",
       "   'profiles',\n",
       "   'patterns,',\n",
       "   'increases',\n",
       "   'discriminatory',\n",
       "   'power',\n",
       "   'profiles',\n",
       "   'patterns',\n",
       "   'providing',\n",
       "   'additional',\n",
       "   'information',\n",
       "   'functionally',\n",
       "   'and/or',\n",
       "   'structurally',\n",
       "   'critical',\n",
       "   'amino',\n",
       "   'acids.',\n",
       "   '{prosite}',\n",
       "   'largely',\n",
       "   'annotation',\n",
       "   'domain',\n",
       "   'features',\n",
       "   '{uniprotkb}/{swiss-prot}',\n",
       "   'entries.',\n",
       "   '983',\n",
       "   '({dna}-binding)',\n",
       "   'domains,',\n",
       "   'repeats',\n",
       "   'zinc',\n",
       "   'fingers',\n",
       "   'present',\n",
       "   '{swiss-prot}',\n",
       "   '(release',\n",
       "   '57.8',\n",
       "   '22',\n",
       "   'september',\n",
       "   '2009),',\n",
       "   '696',\n",
       "   '(',\n",
       "   'approximately',\n",
       "   '70\\\\%)',\n",
       "   'annotated',\n",
       "   '{prosite}',\n",
       "   'descriptors',\n",
       "   'information',\n",
       "   '{prorule}.',\n",
       "   'order',\n",
       "   'functional',\n",
       "   'characterization',\n",
       "   'domains,',\n",
       "   '{prosite}',\n",
       "   'developments',\n",
       "   'focus',\n",
       "   'subfamily',\n",
       "   'specific',\n",
       "   'profiles',\n",
       "   'profile',\n",
       "   'building',\n",
       "   'method',\n",
       "   'giving',\n",
       "   'weight',\n",
       "   'functionally',\n",
       "   'important',\n",
       "   'residues.',\n",
       "   'here,',\n",
       "   'describe',\n",
       "   '{amsa},',\n",
       "   'annotated',\n",
       "   'multiple',\n",
       "   'sequence',\n",
       "   'alignment',\n",
       "   'format',\n",
       "   'build',\n",
       "   'generation',\n",
       "   'generalized',\n",
       "   'profiles,',\n",
       "   'migration',\n",
       "   '{scanprosite}',\n",
       "   '{vital-it},',\n",
       "   'cluster',\n",
       "   '633',\n",
       "   '{cpus},',\n",
       "   'adoption',\n",
       "   'distributed',\n",
       "   'annotation',\n",
       "   'system',\n",
       "   '({das})',\n",
       "   'facilitate',\n",
       "   '{prosite}',\n",
       "   'data',\n",
       "   'integration',\n",
       "   'interchange',\n",
       "   'sources.',\n",
       "   'latest',\n",
       "   'version',\n",
       "   '{prosite}',\n",
       "   '(release',\n",
       "   '20.54,',\n",
       "   '22',\n",
       "   'september',\n",
       "   '2009)',\n",
       "   '1308',\n",
       "   'patterns,',\n",
       "   '863',\n",
       "   'profiles',\n",
       "   '869',\n",
       "   '{prorules}.',\n",
       "   '{prosite}',\n",
       "   'accessible',\n",
       "   'at:',\n",
       "   'http://www.expasy.org/prosite/.\"\"impressive',\n",
       "   'advances',\n",
       "   '{ngs}',\n",
       "   'enabled',\n",
       "   'immense',\n",
       "   'diversity',\n",
       "   'applications.',\n",
       "   'barrier',\n",
       "   '\\\\$1000',\n",
       "   'genome',\n",
       "   'recently',\n",
       "   'broken.',\n",
       "   'important',\n",
       "   'tools',\n",
       "   'clinical',\n",
       "   'diagnostics',\n",
       "   'based',\n",
       "   '{ngs}',\n",
       "   'appearing.',\n",
       "   'third-generation',\n",
       "   'technologies',\n",
       "   'revolutionize',\n",
       "   'genomics',\n",
       "   'research.',\n",
       "   'significant',\n",
       "   'challenges',\n",
       "   '{ngs}',\n",
       "   'remain,',\n",
       "   'data',\n",
       "   'storage',\n",
       "   'processing.',\n",
       "   'ten',\n",
       "   'years',\n",
       "   'ago',\n",
       "   'next-generation',\n",
       "   'sequencing',\n",
       "   '({ngs})',\n",
       "   'technologies',\n",
       "   'appeared',\n",
       "   'market.',\n",
       "   'past',\n",
       "   'decade,',\n",
       "   'tremendous',\n",
       "   'progress',\n",
       "   'made',\n",
       "   'terms',\n",
       "   'speed,',\n",
       "   'read',\n",
       "   'length,',\n",
       "   'throughput,',\n",
       "   'sharp',\n",
       "   'reduction',\n",
       "   'per-base',\n",
       "   'cost.',\n",
       "   'together,',\n",
       "   'advances',\n",
       "   'democratized',\n",
       "   '{ngs}',\n",
       "   'paved',\n",
       "   'development',\n",
       "   'large',\n",
       "   'number',\n",
       "   '{ngs}',\n",
       "   'applications',\n",
       "   'basic',\n",
       "   'science',\n",
       "   'translational',\n",
       "   'research',\n",
       "   'areas',\n",
       "   'clinical',\n",
       "   'diagnostics,',\n",
       "   'agrigenomics,',\n",
       "   'forensic',\n",
       "   'science.',\n",
       "   'provide',\n",
       "   'overview',\n",
       "   'evolution',\n",
       "   '{ngs}',\n",
       "   'discuss',\n",
       "   'significant',\n",
       "   'improvements',\n",
       "   'sequencing',\n",
       "   'technologies',\n",
       "   'library',\n",
       "   'preparation',\n",
       "   'protocols.',\n",
       "   'explore',\n",
       "   'current',\n",
       "   'landscape',\n",
       "   '{ngs}',\n",
       "   'applications',\n",
       "   'provide',\n",
       "   'perspective',\n",
       "   'future',\n",
       "   'developments.\"{dna}',\n",
       "   'methylation:',\n",
       "   'roles',\n",
       "   'mammalian',\n",
       "   'development,\"{dna}',\n",
       "   'methylation',\n",
       "   'studied',\n",
       "   'epigenetic',\n",
       "   'modifications',\n",
       "   'essential',\n",
       "   'mammalian',\n",
       "   'development.',\n",
       "   'methylation',\n",
       "   'status',\n",
       "   '{cpg}',\n",
       "   'dinucleotides',\n",
       "   'genome',\n",
       "   'stably',\n",
       "   'propagated',\n",
       "   'mitosis,',\n",
       "   'improvements',\n",
       "   'methods',\n",
       "   'measuring',\n",
       "   'methylation',\n",
       "   'identified',\n",
       "   'numerous',\n",
       "   'regions',\n",
       "   'dynamically',\n",
       "   'regulated.',\n",
       "   'review,',\n",
       "   'discuss',\n",
       "   'key',\n",
       "   'concepts',\n",
       "   'function',\n",
       "   '{dna}',\n",
       "   'methylation',\n",
       "   'mammals,',\n",
       "   'stemming',\n",
       "   'decades',\n",
       "   'research,',\n",
       "   'including',\n",
       "   'recent',\n",
       "   'studies',\n",
       "   'elucidated',\n",
       "   '{dna}',\n",
       "   'methylation',\n",
       "   'regulatory',\n",
       "   'role',\n",
       "   'genome.',\n",
       "   'include',\n",
       "   'insights',\n",
       "   'early',\n",
       "   'development,',\n",
       "   'embryonic',\n",
       "   'stem',\n",
       "   'cells',\n",
       "   'adult',\n",
       "   'lineages,',\n",
       "   'haematopoiesis,',\n",
       "   'highlight',\n",
       "   'general',\n",
       "   'features',\n",
       "   'modification',\n",
       "   'participates',\n",
       "   'global',\n",
       "   'localized',\n",
       "   'epigenetic',\n",
       "   'regulation.\"storing,',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathways.\",\"biological',\n",
       "   'pathways,',\n",
       "   'including',\n",
       "   'metabolic',\n",
       "   'pathways,',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'networks,',\n",
       "   'signal',\n",
       "   'transduction',\n",
       "   'pathways,',\n",
       "   'gene',\n",
       "   'regulatory',\n",
       "   'networks,',\n",
       "   'represented',\n",
       "   '220',\n",
       "   'diverse',\n",
       "   'databases.',\n",
       "   'data',\n",
       "   'crucial',\n",
       "   'study',\n",
       "   'specific',\n",
       "   'biological',\n",
       "   'processes,',\n",
       "   'including',\n",
       "   'human',\n",
       "   'diseases.',\n",
       "   'standard',\n",
       "   'exchange',\n",
       "   'formats',\n",
       "   'pathway',\n",
       "   'information,',\n",
       "   '{biopax},',\n",
       "   '{cellml},',\n",
       "   '{sbml}',\n",
       "   '{psi}-{mi},',\n",
       "   'enable',\n",
       "   'convenient',\n",
       "   'collection',\n",
       "   'data',\n",
       "   'biological',\n",
       "   'research,',\n",
       "   'mechanisms',\n",
       "   'common',\n",
       "   'storage',\n",
       "   'communication',\n",
       "   'required.',\n",
       "   'developed',\n",
       "   '{cpath},',\n",
       "   'open',\n",
       "   'source',\n",
       "   'database',\n",
       "   'web',\n",
       "   'application',\n",
       "   'collecting,',\n",
       "   'storing,',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathway',\n",
       "   'data.',\n",
       "   '{cpath}',\n",
       "   'makes',\n",
       "   'easy',\n",
       "   'aggregate',\n",
       "   'custom',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'sets',\n",
       "   'standard',\n",
       "   'exchange',\n",
       "   'formats',\n",
       "   'multiple',\n",
       "   'databases,',\n",
       "   'present',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'biologists',\n",
       "   'customizable',\n",
       "   'web',\n",
       "   'interface,',\n",
       "   'export',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'web',\n",
       "   'service',\n",
       "   'third-party',\n",
       "   'software,',\n",
       "   'cytoscape,',\n",
       "   'visualization',\n",
       "   'analysis.',\n",
       "   '{cpath}',\n",
       "   'software',\n",
       "   'only,',\n",
       "   'include',\n",
       "   'pathway',\n",
       "   'information.',\n",
       "   'key',\n",
       "   'features',\n",
       "   'include:',\n",
       "   'built-in',\n",
       "   'identifier',\n",
       "   'mapping',\n",
       "   'service',\n",
       "   'linking',\n",
       "   'identical',\n",
       "   'interactors',\n",
       "   'linking',\n",
       "   'external',\n",
       "   'resources;',\n",
       "   'built-in',\n",
       "   'support',\n",
       "   '{psi}-{mi}',\n",
       "   '{biopax}',\n",
       "   'standard',\n",
       "   'pathway',\n",
       "   'exchange',\n",
       "   'formats;',\n",
       "   'web',\n",
       "   'service',\n",
       "   'interface',\n",
       "   'searching',\n",
       "   'retrieving',\n",
       "   'pathway',\n",
       "   'data',\n",
       "   'sets;',\n",
       "   'documentation.',\n",
       "   '{cpath}',\n",
       "   'software',\n",
       "   'freely',\n",
       "   '{lgpl}',\n",
       "   'open',\n",
       "   'source',\n",
       "   'license',\n",
       "   'academic',\n",
       "   'commercial',\n",
       "   'use.',\n",
       "   '{cpath}',\n",
       "   'robust,',\n",
       "   'scalable,',\n",
       "   'modular,',\n",
       "   'professional-grade',\n",
       "   'software',\n",
       "   'platform',\n",
       "   'collecting,',\n",
       "   'storing,',\n",
       "   'querying',\n",
       "   'biological',\n",
       "   'pathways.',\n",
       "   'serve',\n",
       "   'core',\n",
       "   'data',\n",
       "   'handling',\n",
       "   'component',\n",
       "   'information',\n",
       "   'systems',\n",
       "   'pathway',\n",
       "   'visualization,',\n",
       "   'analysis',\n",
       "   'modeling.\"new',\n",
       "   'haven,',\n",
       "   'connecticut',\n",
       "   '06520,',\n",
       "   'usa.\",annotation',\n",
       "   'transfer',\n",
       "   'genomes:',\n",
       "   'protein-protein',\n",
       "   'interologs',\n",
       "   '{protein-dna}',\n",
       "   'regulogs.,texts,',\n",
       "   'trees.\",\"{bionames}',\n",
       "   'web',\n",
       "   'database',\n",
       "   'taxonomic',\n",
       "   'names',\n",
       "   'animals,',\n",
       "   'linked',\n",
       "   'primary',\n",
       "   'literature',\n",
       "   'and,',\n",
       "   'possible,',\n",
       "   'phylogenetic',\n",
       "   'trees.',\n",
       "   'aims',\n",
       "   'provide',\n",
       "   'taxonomic',\n",
       "   '\"\"dashboard\"\"',\n",
       "   'glance',\n",
       "   'summary',\n",
       "   'taxonomic',\n",
       "   'phylogenetic',\n",
       "   'information',\n",
       "   'taxon',\n",
       "   'provide',\n",
       "   'quick',\n",
       "   'answer',\n",
       "   'basic',\n",
       "   'question',\n",
       "   '\"\"what',\n",
       "   'taxon?\"\"',\n",
       "   '{bionames}',\n",
       "   'combines',\n",
       "   'classifications',\n",
       "   'global',\n",
       "   'biodiversity',\n",
       "   'information',\n",
       "   'facility',\n",
       "   '({gbif})',\n",
       "   '{genbank},',\n",
       "   'images',\n",
       "   'encyclopedia',\n",
       "   'life',\n",
       "   '({eol}),',\n",
       "   'animal',\n",
       "   'names',\n",
       "   'index',\n",
       "   'organism',\n",
       "   'names',\n",
       "   '({ion}),',\n",
       "   'bibliographic',\n",
       "   'data',\n",
       "   'multiple',\n",
       "   'sources',\n",
       "   'including',\n",
       "   'biodiversity',\n",
       "   'heritage',\n",
       "   'library',\n",
       "   '({bhl})',\n",
       "   '{crossref}.',\n",
       "   'user',\n",
       "   'interface',\n",
       "   'includes',\n",
       "   'display',\n",
       "   'full',\n",
       "   'text',\n",
       "   'articles,',\n",
       "   'interactive',\n",
       "   'timelines',\n",
       "   'taxonomic',\n",
       "   'publications,',\n",
       "   'zoomable',\n",
       "   'phylogenies.',\n",
       "   'http://bionames.org.\"\"the',\n",
       "   'prediction',\n",
       "   'regulatory',\n",
       "   'elements',\n",
       "   'problem',\n",
       "   'computational',\n",
       "   'methods',\n",
       "   'offer',\n",
       "   'great',\n",
       "   'hope.',\n",
       "   'past',\n",
       "   'years,',\n",
       "   'numerous',\n",
       "   'tools',\n",
       "   'task.',\n",
       "   'purpose',\n",
       "   'current',\n",
       "   'assessment',\n",
       "   'twofold:',\n",
       "   'provide',\n",
       "   'guidance',\n",
       "   'users',\n",
       "   'accuracy',\n",
       "   'tools',\n",
       "   'settings,',\n",
       "   'provide',\n",
       "   'benchmark',\n",
       "   'data',\n",
       "   'sets',\n",
       "   'assessing',\n",
       "   'future',\n",
       "   'tools.\"\"unraveling',\n",
       "   'mechanisms',\n",
       "   'regulate',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'major',\n",
       "   'challenge',\n",
       "   'biology.',\n",
       "   'important',\n",
       "   'task',\n",
       "   'challenge',\n",
       "   'identify',\n",
       "   'regulatory',\n",
       "   'elements,',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   'deoxyribonucleic',\n",
       "   'acid',\n",
       "   '({dna})',\n",
       "   'transcription',\n",
       "   'factors.',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   'short',\n",
       "   '{dna}',\n",
       "   'segments',\n",
       "   'called',\n",
       "   'motifs.',\n",
       "   'recent',\n",
       "   'advances',\n",
       "   'genome',\n",
       "   'sequence',\n",
       "   'availability',\n",
       "   'high-throughput',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'analysis',\n",
       "   'technologies',\n",
       "   'allowed',\n",
       "   'development',\n",
       "   'computational',\n",
       "   'methods',\n",
       "   'motif',\n",
       "   'finding.',\n",
       "   'result,',\n",
       "   'large',\n",
       "   'number',\n",
       "   'motif',\n",
       "   'finding',\n",
       "   'algorithms',\n",
       "   'implemented',\n",
       "   'applied',\n",
       "   'motif',\n",
       "   'models',\n",
       "   'past',\n",
       "   'decade.',\n",
       "   'survey',\n",
       "   'reviews',\n",
       "   'latest',\n",
       "   'developments',\n",
       "   '{dna}',\n",
       "   'motif',\n",
       "   'finding',\n",
       "   'algorithms.',\n",
       "   'earlier',\n",
       "   'algorithms',\n",
       "   'promoter',\n",
       "   'sequences',\n",
       "   'coregulated',\n",
       "   'genes',\n",
       "   'single',\n",
       "   'genome',\n",
       "   'search',\n",
       "   'statistically',\n",
       "   'overrepresented',\n",
       "   'motifs.',\n",
       "   'recent',\n",
       "   'algorithms',\n",
       "   'designed',\n",
       "   'phylogenetic',\n",
       "   'footprinting',\n",
       "   'orthologous',\n",
       "   'sequences',\n",
       "   'integrated',\n",
       "   'approach',\n",
       "   'promoter',\n",
       "   'sequences',\n",
       "   'coregulated',\n",
       "   'genes',\n",
       "   'phylogenetic',\n",
       "   'footprinting',\n",
       "   'used.',\n",
       "   'algorithms',\n",
       "   'studied',\n",
       "   'reported',\n",
       "   'correctly',\n",
       "   'detect',\n",
       "   'motifs',\n",
       "   'previously',\n",
       "   'detected',\n",
       "   'laboratory',\n",
       "   'experimental',\n",
       "   'approaches,',\n",
       "   'algorithms',\n",
       "   'find',\n",
       "   'motifs.',\n",
       "   'however,',\n",
       "   'motif',\n",
       "   'finding',\n",
       "   'algorithms',\n",
       "   'shown',\n",
       "   'work',\n",
       "   'successfully',\n",
       "   'yeast',\n",
       "   'lower',\n",
       "   'organisms,',\n",
       "   'perform',\n",
       "   'significantly',\n",
       "   'worse',\n",
       "   'higher',\n",
       "   'organisms.',\n",
       "   'considerable',\n",
       "   'efforts',\n",
       "   'date,',\n",
       "   '{dna}',\n",
       "   'motif',\n",
       "   'finding',\n",
       "   'remains',\n",
       "   'complex',\n",
       "   'challenge',\n",
       "   'biologists',\n",
       "   'computer',\n",
       "   'scientists.',\n",
       "   'researchers',\n",
       "   'approaches',\n",
       "   'developing',\n",
       "   'motif',\n",
       "   'discovery',\n",
       "   'tools',\n",
       "   'progress',\n",
       "   'made',\n",
       "   'area',\n",
       "   'research',\n",
       "   'encouraging.',\n",
       "   'performance',\n",
       "   'comparison',\n",
       "   'motif',\n",
       "   'finding',\n",
       "   'tools',\n",
       "   'identification',\n",
       "   'tools',\n",
       "   'proven',\n",
       "   'difficult',\n",
       "   'task',\n",
       "   'tools',\n",
       "   'designed',\n",
       "   'based',\n",
       "   'algorithms',\n",
       "   'motif',\n",
       "   'models',\n",
       "   'diverse',\n",
       "   'complex',\n",
       "   'incomplete',\n",
       "   'understanding',\n",
       "   'biology',\n",
       "   'regulatory',\n",
       "   'mechanism',\n",
       "   'provide',\n",
       "   'adequate',\n",
       "   'evaluation',\n",
       "   'underlying',\n",
       "   'algorithms',\n",
       "   'motif',\n",
       "   'models.\"\"the',\n",
       "   'gibbs',\n",
       "   'motif',\n",
       "   'sampler',\n",
       "   'software',\n",
       "   'package',\n",
       "   'locating',\n",
       "   'common',\n",
       "   'elements',\n",
       "   'collections',\n",
       "   'biopolymer',\n",
       "   'sequences.',\n",
       "   'paper',\n",
       "   'describe',\n",
       "   'variation',\n",
       "   'gibbs',\n",
       "   'motif',\n",
       "   'sampler,',\n",
       "   'gibbs',\n",
       "   'recursive',\n",
       "   'sampler,',\n",
       "   'developed',\n",
       "   'specifically',\n",
       "   'locating',\n",
       "   'multiple',\n",
       "   'transcription',\n",
       "   'factor',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   'multiple',\n",
       "   'transcription',\n",
       "   'factors',\n",
       "   'simultaneously',\n",
       "   'unaligned',\n",
       "   '{dna}',\n",
       "   'sequences',\n",
       "   'heterogeneous',\n",
       "   '{dna}',\n",
       "   'composition.',\n",
       "   'describe',\n",
       "   'basic',\n",
       "   'operation',\n",
       "   'web-based',\n",
       "   'version',\n",
       "   'sampler.',\n",
       "   'sampler',\n",
       "   'acces-sed',\n",
       "   'http://bayesweb.wadsworth.org/gibbs/gibbs.html',\n",
       "   'http://www.bioinfo.rpi.edu/applications/bayesian/gibbs/gibbs.html.',\n",
       "   'online',\n",
       "   'user',\n",
       "   'guide',\n",
       "   'http://bayesweb.wadsworth.org/gibbs/bernoulli.html',\n",
       "   'http://www.bioinfo.rpi.edu/applications/bayesian/gibbs/manual/bernoulli.html.',\n",
       "   'solaris,',\n",
       "   'solaris.x86',\n",
       "   'linux',\n",
       "   'versions',\n",
       "   'sampler',\n",
       "   'stand-alone',\n",
       "   'programs',\n",
       "   'academic',\n",
       "   'not-for-profit',\n",
       "   'users.',\n",
       "   'commercial',\n",
       "   'licenses',\n",
       "   'available.',\n",
       "   'gibbs',\n",
       "   'recursive',\n",
       "   'sampler',\n",
       "   'distributed',\n",
       "   'accordance',\n",
       "   '{iscb}',\n",
       "   'level',\n",
       "   '0',\n",
       "   'guidelines',\n",
       "   'requirement',\n",
       "   'citation',\n",
       "   'scientific',\n",
       "   'publications.\"wellcome',\n",
       "   'trust',\n",
       "   'genome',\n",
       "   'campus,',\n",
       "   'hinxton,',\n",
       "   'cambridge,',\n",
       "   'cb10',\n",
       "   '1sd,',\n",
       "   'uk.\",the',\n",
       "   'ontology',\n",
       "   'lookup',\n",
       "   'service:',\n",
       "   'data',\n",
       "   'tools',\n",
       "   'controlled',\n",
       "   'vocabulary',\n",
       "   'queries,\"the',\n",
       "   'ontology',\n",
       "   'lookup',\n",
       "   'service',\n",
       "   '({ols})',\n",
       "   '(http://www.ebi.ac.uk/ols)',\n",
       "   'interactive',\n",
       "   'programmatic',\n",
       "   'interfaces',\n",
       "   'query,',\n",
       "   'browse',\n",
       "   'navigate',\n",
       "   'increasing',\n",
       "   'number',\n",
       "   'biomedical',\n",
       "   'ontologies',\n",
       "   'controlled',\n",
       "   'vocabularies.',\n",
       "   'volume',\n",
       "   'data',\n",
       "   'querying',\n",
       "   'quadrupled',\n",
       "   'production',\n",
       "   '{ols}',\n",
       "   'functionality',\n",
       "   'integrated',\n",
       "   'high-usage',\n",
       "   'databases',\n",
       "   'data',\n",
       "   'entry',\n",
       "   'tools.',\n",
       "   'improvements',\n",
       "   'made',\n",
       "   '{ols}',\n",
       "   'query',\n",
       "   'interfaces,',\n",
       "   'based',\n",
       "   'user',\n",
       "   'feedback',\n",
       "   'requirements,',\n",
       "   'improve',\n",
       "   'usability',\n",
       "   'service',\n",
       "   'interoperability',\n",
       "   'provide',\n",
       "   'ways',\n",
       "   'perform',\n",
       "   'queries.\"\"in',\n",
       "   'chromatin',\n",
       "   'immunoprecipitation',\n",
       "   'high-throughput',\n",
       "   'sequencing',\n",
       "   '({chip}-seq)',\n",
       "   'experiment,',\n",
       "   'important',\n",
       "   'consideration',\n",
       "   'experimental',\n",
       "   'design',\n",
       "   'minimum',\n",
       "   'number',\n",
       "   'sequenced',\n",
       "   'reads',\n",
       "   'required',\n",
       "   'obtain',\n",
       "   'statistically',\n",
       "   'significant',\n",
       "   'results.',\n",
       "   'present',\n",
       "   'extensive',\n",
       "   'evaluation',\n",
       "   'impact',\n",
       "   'sequencing',\n",
       "   'depth',\n",
       "   'identification',\n",
       "   'enriched',\n",
       "   'regions',\n",
       "   'key',\n",
       "   'histone',\n",
       "   'modifications',\n",
       "   '({h3k4me3},',\n",
       "   '{h3k36me3},',\n",
       "   '{h3k27me3}',\n",
       "   '{h3k9me2}/me3)',\n",
       "   'deep-sequenced',\n",
       "   'datasets',\n",
       "   'human',\n",
       "   'fly.',\n",
       "   'propose',\n",
       "   'define',\n",
       "   'sufficient',\n",
       "   'sequencing',\n",
       "   'depth',\n",
       "   'number',\n",
       "   'reads',\n",
       "   'detected',\n",
       "   'enrichment',\n",
       "   'regions',\n",
       "   'increase',\n",
       "   '<1\\\\%',\n",
       "   'additional',\n",
       "   'million',\n",
       "   'reads.',\n",
       "   'required',\n",
       "   'depth',\n",
       "   'depends',\n",
       "   'nature',\n",
       "   'mark',\n",
       "   'state',\n",
       "   'cell',\n",
       "   'experiment,',\n",
       "   'observe',\n",
       "   'sufficient',\n",
       "   'depth',\n",
       "   'reached',\n",
       "   '<20',\n",
       "   'million',\n",
       "   'reads',\n",
       "   'fly.',\n",
       "   'human,',\n",
       "   'clear',\n",
       "   'saturation',\n",
       "   'points',\n",
       "   'examined',\n",
       "   'datasets,',\n",
       "   'analysis',\n",
       "   'suggests',\n",
       "   '40–50',\n",
       "   'million',\n",
       "   'reads',\n",
       "   'practical',\n",
       "   'minimum',\n",
       "   'marks.',\n",
       "   'devise',\n",
       "   'mathematical',\n",
       "   'model',\n",
       "   'estimate',\n",
       "   'sufficient',\n",
       "   'depth',\n",
       "   'total',\n",
       "   'genomic',\n",
       "   'coverage',\n",
       "   'mark.',\n",
       "   'lastly,',\n",
       "   'find',\n",
       "   'algorithms',\n",
       "   'tested',\n",
       "   'agree',\n",
       "   'broad',\n",
       "   'enrichment',\n",
       "   'profiles,',\n",
       "   'lower',\n",
       "   'depths.',\n",
       "   'findings',\n",
       "   'suggest',\n",
       "   'sufficient',\n",
       "   'sequencing',\n",
       "   'depth',\n",
       "   'peak-calling',\n",
       "   'algorithm',\n",
       "   'essential',\n",
       "   'ensuring',\n",
       "   'robustness',\n",
       "   'conclusions',\n",
       "   ...]),\n",
       " ('beaca02b21b7cad6cb738c5e2682af8d',\n",
       "  ['haven,',\n",
       "   'connecticut',\n",
       "   '06520,',\n",
       "   'usa.\",annotation',\n",
       "   'transfer',\n",
       "   'genomes:',\n",
       "   'protein-protein',\n",
       "   'interologs',\n",
       "   '{protein-dna}',\n",
       "   'regulogs.,\"bioinformatics',\n",
       "   'program,',\n",
       "   'boston',\n",
       "   'university,',\n",
       "   'boston,',\n",
       "   'massachusetts;',\n",
       "   'department',\n",
       "   'biomedical',\n",
       "   'engineering,',\n",
       "   'boston',\n",
       "   'university,',\n",
       "   'boston,',\n",
       "   'massachusetts\",{zdock}:',\n",
       "   'initial-stage',\n",
       "   'protein-docking',\n",
       "   'algorithm,\"the',\n",
       "   'development',\n",
       "   'scoring',\n",
       "   'functions',\n",
       "   'great',\n",
       "   'importance',\n",
       "   'protein',\n",
       "   'docking.',\n",
       "   'present',\n",
       "   'scoring',\n",
       "   'function',\n",
       "   'initial',\n",
       "   'stage',\n",
       "   'unbound',\n",
       "   'docking.',\n",
       "   'combines',\n",
       "   'recently',\n",
       "   'developed',\n",
       "   'pairwise',\n",
       "   'shape',\n",
       "   'complementarity',\n",
       "   'desolvation',\n",
       "   'electrostatics.',\n",
       "   'compare',\n",
       "   'scoring',\n",
       "   'function',\n",
       "   'functions',\n",
       "   'large',\n",
       "   'benchmark',\n",
       "   '49',\n",
       "   'nonredundant',\n",
       "   'test',\n",
       "   'cases',\n",
       "   'show',\n",
       "   'superior',\n",
       "   'performance,',\n",
       "   'antibody-antigen',\n",
       "   'category',\n",
       "   'test',\n",
       "   'cases.',\n",
       "   '44',\n",
       "   'test',\n",
       "   'cases',\n",
       "   '(90\\\\%',\n",
       "   'benchmark),',\n",
       "   'retain',\n",
       "   'near-native',\n",
       "   'structure',\n",
       "   'top',\n",
       "   '2000',\n",
       "   'predictions',\n",
       "   '6°',\n",
       "   'rotational',\n",
       "   'sampling',\n",
       "   'density,',\n",
       "   'average',\n",
       "   '52',\n",
       "   'near-native',\n",
       "   'structures',\n",
       "   'test',\n",
       "   'case.',\n",
       "   'remaining',\n",
       "   'difficult',\n",
       "   'test',\n",
       "   'cases',\n",
       "   'explained',\n",
       "   'combination',\n",
       "   'poor',\n",
       "   'binding',\n",
       "   'affinity,',\n",
       "   'large',\n",
       "   'backbone',\n",
       "   'conformational',\n",
       "   'changes,',\n",
       "   \"algorithm's\",\n",
       "   'strong',\n",
       "   'tendency',\n",
       "   'identifying',\n",
       "   'large',\n",
       "   'concave',\n",
       "   'binding',\n",
       "   'pockets.',\n",
       "   'scoring',\n",
       "   'functions',\n",
       "   'integrated',\n",
       "   'fast',\n",
       "   'fourier',\n",
       "   'transform',\n",
       "   'based',\n",
       "   'docking',\n",
       "   'algorithm',\n",
       "   '{zdock},',\n",
       "   'freely',\n",
       "   'academic',\n",
       "   'users',\n",
       "   'http://zlab.bu.edu/∼rong/dock.',\n",
       "   'proteins',\n",
       "   '2003;52:80–87.',\n",
       "   '{\\\\copyright}',\n",
       "   '2003',\n",
       "   '{wiley-liss},',\n",
       "   'inc.\"\"human',\n",
       "   'protein',\n",
       "   'reference',\n",
       "   'database',\n",
       "   '({hprd}--http://www.hprd.org/),',\n",
       "   'initially',\n",
       "   '2003,',\n",
       "   'database',\n",
       "   'curated',\n",
       "   'proteomic',\n",
       "   'information',\n",
       "   'pertaining',\n",
       "   'human',\n",
       "   'proteins.',\n",
       "   'recently',\n",
       "   'added',\n",
       "   'number',\n",
       "   'features',\n",
       "   '{hprd}.',\n",
       "   'include',\n",
       "   '{phosphomotif}',\n",
       "   'finder,',\n",
       "   'users',\n",
       "   'find',\n",
       "   'presence',\n",
       "   '320',\n",
       "   'experimentally',\n",
       "   'verified',\n",
       "   'phosphorylation',\n",
       "   'motifs',\n",
       "   'proteins',\n",
       "   'interest.',\n",
       "   'feature',\n",
       "   'protein',\n",
       "   'distributed',\n",
       "   'annotation',\n",
       "   '{system--human}',\n",
       "   'proteinpedia',\n",
       "   '(http://www.humanproteinpedia.org/)--through',\n",
       "   'laboratories',\n",
       "   'submit',\n",
       "   'data,',\n",
       "   'mapped',\n",
       "   'protein',\n",
       "   'entries',\n",
       "   '{hprd}.',\n",
       "   '75',\n",
       "   'laboratories',\n",
       "   'involved',\n",
       "   'proteomics',\n",
       "   'research',\n",
       "   'participated',\n",
       "   'effort',\n",
       "   'submitting',\n",
       "   'data',\n",
       "   '15,000',\n",
       "   'human',\n",
       "   'proteins.',\n",
       "   'submitted',\n",
       "   'data',\n",
       "   'includes',\n",
       "   'mass',\n",
       "   'spectrometry',\n",
       "   'protein',\n",
       "   'microarray-derived',\n",
       "   'data,',\n",
       "   'data',\n",
       "   'types.',\n",
       "   'finally,',\n",
       "   '{hprd}',\n",
       "   'linked',\n",
       "   'compendium',\n",
       "   'human',\n",
       "   'signaling',\n",
       "   'pathways',\n",
       "   'developed',\n",
       "   'group,',\n",
       "   '{netpath}',\n",
       "   '(http://www.netpath.org/),',\n",
       "   'annotations',\n",
       "   'cancer',\n",
       "   'immune',\n",
       "   'signaling',\n",
       "   'pathways.',\n",
       "   'update,',\n",
       "   '5500',\n",
       "   'protein',\n",
       "   'sequences',\n",
       "   'added,',\n",
       "   'making',\n",
       "   '{hprd}',\n",
       "   'comprehensive',\n",
       "   'resource',\n",
       "   'studying',\n",
       "   'human',\n",
       "   'proteome.\"\"{background}:microarrays',\n",
       "   'revolutionized',\n",
       "   'biological',\n",
       "   'research',\n",
       "   'enabling',\n",
       "   'gene',\n",
       "   'expression',\n",
       "   'comparisons',\n",
       "   'transcriptome-wide',\n",
       "   'scale.',\n",
       "   'microarrays,',\n",
       "   'however,',\n",
       "   'estimate',\n",
       "   'absolute',\n",
       "   'expression',\n",
       "   'level',\n",
       "   'accurately.',\n",
       "   'present,',\n",
       "   'high',\n",
       "   'throughput',\n",
       "   'sequencing',\n",
       "   'emerging',\n",
       "   'alternative',\n",
       "   'methodology',\n",
       "   'transcriptome',\n",
       "   'studies.',\n",
       "   'free',\n",
       "   'limitations',\n",
       "   'imposed',\n",
       "   'microarray',\n",
       "   'design,',\n",
       "   'potential',\n",
       "   'estimate',\n",
       "   'absolute',\n",
       "   'transcript',\n",
       "   'levels',\n",
       "   '{unknown.results}:in',\n",
       "   'study,',\n",
       "   'evaluate',\n",
       "   'relative',\n",
       "   'accuracy',\n",
       "   'microarrays',\n",
       "   'transcriptome',\n",
       "   'sequencing',\n",
       "   '({rna}-seq)',\n",
       "   'methodology:',\n",
       "   'proteomics.',\n",
       "   'find',\n",
       "   '{rna}-seq',\n",
       "   'estimate',\n",
       "   'absolute',\n",
       "   'expression',\n",
       "   '{levels.conclusion}:our',\n",
       "   'result',\n",
       "   'shows',\n",
       "   'terms',\n",
       "   'technical',\n",
       "   'performance,',\n",
       "   '{rna}-seq',\n",
       "   'technique',\n",
       "   'choice',\n",
       "   'studies',\n",
       "   'require',\n",
       "   'accurate',\n",
       "   'estimation',\n",
       "   'absolute',\n",
       "   'transcript',\n",
       "   'levels.\"\"cancers',\n",
       "   'evolve',\n",
       "   'reiterative',\n",
       "   'process',\n",
       "   'clonal',\n",
       "   'expansion,',\n",
       "   'genetic',\n",
       "   'diversification',\n",
       "   'clonal',\n",
       "   'selection',\n",
       "   'adaptive',\n",
       "   'landscapes',\n",
       "   'tissue',\n",
       "   'ecosystems.',\n",
       "   'dynamics',\n",
       "   'complex,',\n",
       "   'highly',\n",
       "   'variable',\n",
       "   'patterns',\n",
       "   'genetic',\n",
       "   'diversity',\n",
       "   'resulting',\n",
       "   'clonal',\n",
       "   'architecture.',\n",
       "   'therapeutic',\n",
       "   'intervention',\n",
       "   'destroy',\n",
       "   'cancer',\n",
       "   'clones',\n",
       "   'erode',\n",
       "   'habitats,',\n",
       "   'inadvertently',\n",
       "   'provide',\n",
       "   'potent',\n",
       "   'selective',\n",
       "   'pressure',\n",
       "   'expansion',\n",
       "   'resistant',\n",
       "   'variants.',\n",
       "   'inherently',\n",
       "   'darwinian',\n",
       "   'character',\n",
       "   'cancer',\n",
       "   'primary',\n",
       "   'reason',\n",
       "   'therapeutic',\n",
       "   'failure,',\n",
       "   'hold',\n",
       "   'key',\n",
       "   'effective',\n",
       "   'control.\"\"intratumoral',\n",
       "   'heterogeneity',\n",
       "   'arises',\n",
       "   'evolution',\n",
       "   'genetically',\n",
       "   'diverse',\n",
       "   'subclones',\n",
       "   'tumor',\n",
       "   'progression.',\n",
       "   'however,',\n",
       "   'cells',\n",
       "   'single',\n",
       "   'genetic',\n",
       "   'clones',\n",
       "   'functionally',\n",
       "   'equivalent',\n",
       "   'remains',\n",
       "   'unknown.',\n",
       "   'combining',\n",
       "   '{dna}',\n",
       "   'copy',\n",
       "   'number',\n",
       "   'alteration',\n",
       "   '({cna})',\n",
       "   'profiling,',\n",
       "   'sequencing,',\n",
       "   'lentiviral',\n",
       "   'lineage',\n",
       "   'tracking,',\n",
       "   'repopulation',\n",
       "   'dynamics',\n",
       "   '150',\n",
       "   'single',\n",
       "   'lentivirus-marked',\n",
       "   'lineages',\n",
       "   '10',\n",
       "   'human',\n",
       "   'colorectal',\n",
       "   'cancers',\n",
       "   'serial',\n",
       "   'xenograft',\n",
       "   'passages',\n",
       "   'mice.',\n",
       "   '{cna}',\n",
       "   'mutational',\n",
       "   'analysis',\n",
       "   'distinguished',\n",
       "   'individual',\n",
       "   'clones',\n",
       "   'showed',\n",
       "   'clones',\n",
       "   'remained',\n",
       "   'stable',\n",
       "   'serial',\n",
       "   'transplantation.',\n",
       "   'stability,',\n",
       "   'proliferation,',\n",
       "   'persistence,',\n",
       "   'chemotherapy',\n",
       "   'tolerance',\n",
       "   'lentivirally',\n",
       "   'marked',\n",
       "   'lineages',\n",
       "   'variable',\n",
       "   'clone.',\n",
       "   'chemotherapy',\n",
       "   'promoted',\n",
       "   'dominance',\n",
       "   'previously',\n",
       "   'minor',\n",
       "   'dormant',\n",
       "   'lineages.',\n",
       "   'thus,',\n",
       "   'genetic',\n",
       "   'diversity,',\n",
       "   'tumor',\n",
       "   'cells',\n",
       "   'display',\n",
       "   'inherent',\n",
       "   'functional',\n",
       "   'variability',\n",
       "   'tumor',\n",
       "   'propagation',\n",
       "   'potential,',\n",
       "   'mechanism',\n",
       "   'contributes',\n",
       "   'cancer',\n",
       "   'growth',\n",
       "   'therapy',\n",
       "   'tolerance.\"\"recent',\n",
       "   'advances',\n",
       "   'functional',\n",
       "   'genomics',\n",
       "   'helped',\n",
       "   'generate',\n",
       "   'large-scale',\n",
       "   'high-throughput',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'data.',\n",
       "   'networks,',\n",
       "   'extremely',\n",
       "   'valuable',\n",
       "   'molecular',\n",
       "   'level',\n",
       "   'understanding',\n",
       "   'cells,',\n",
       "   'provide',\n",
       "   'direct',\n",
       "   'information',\n",
       "   'regions',\n",
       "   '(domains)',\n",
       "   'proteins',\n",
       "   'mediate',\n",
       "   'interaction.',\n",
       "   'here,',\n",
       "   'performed',\n",
       "   'co-evolutionary',\n",
       "   'analysis',\n",
       "   'domains',\n",
       "   'interacting',\n",
       "   'proteins',\n",
       "   'order',\n",
       "   'understand',\n",
       "   'degree',\n",
       "   'co-evolution',\n",
       "   'interacting',\n",
       "   'non-interacting',\n",
       "   'domains.',\n",
       "   'combination',\n",
       "   'sequence',\n",
       "   'structural',\n",
       "   'analysis,',\n",
       "   'analyzed',\n",
       "   'protein–protein',\n",
       "   'interactions',\n",
       "   '{f1-atpase},',\n",
       "   '{sec23p/sec24p},',\n",
       "   '{dna}-directed',\n",
       "   '{rna}',\n",
       "   'polymerase',\n",
       "   'nuclear',\n",
       "   'pore',\n",
       "   'complexes,',\n",
       "   'found',\n",
       "   'interacting',\n",
       "   'domain',\n",
       "   'pair(s)',\n",
       "   'interaction',\n",
       "   'exhibits',\n",
       "   'higher',\n",
       "   'level',\n",
       "   'co-evolution',\n",
       "   'non-interacting',\n",
       "   'domain',\n",
       "   'pairs.',\n",
       "   'motivated',\n",
       "   'finding,',\n",
       "   'developed',\n",
       "   'computational',\n",
       "   'method',\n",
       "   'test',\n",
       "   'generality',\n",
       "   'observed',\n",
       "   'trend,',\n",
       "   'predict',\n",
       "   'large-scale',\n",
       "   'domain–domain',\n",
       "   'interactions.',\n",
       "   'protein–protein',\n",
       "   'interaction,',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'predicts',\n",
       "   'domain',\n",
       "   'pair(s)',\n",
       "   'mediate',\n",
       "   'protein',\n",
       "   'interaction.',\n",
       "   'applied',\n",
       "   'method',\n",
       "   'yeast',\n",
       "   'interactome',\n",
       "   'predict',\n",
       "   'domain–domain',\n",
       "   'interactions,',\n",
       "   'domain–domain',\n",
       "   'interactions',\n",
       "   'found',\n",
       "   '{pdb}',\n",
       "   'crystal',\n",
       "   'structures',\n",
       "   'validate',\n",
       "   'predictions.',\n",
       "   'results',\n",
       "   'show',\n",
       "   'prediction',\n",
       "   'accuracy',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'statistically',\n",
       "   'significant.',\n",
       "   'comparison',\n",
       "   'prediction',\n",
       "   'results',\n",
       "   'methods',\n",
       "   'reveals',\n",
       "   'fraction',\n",
       "   'predictions',\n",
       "   'shared',\n",
       "   'methods,',\n",
       "   'indicating',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'detect',\n",
       "   'interactions',\n",
       "   'missed',\n",
       "   'methods.',\n",
       "   'proposed',\n",
       "   'method',\n",
       "   'methods',\n",
       "   'identify',\n",
       "   'previously',\n",
       "   'unrecognized',\n",
       "   'domain–domain',\n",
       "   'interactions',\n",
       "   'genome',\n",
       "   'scale,',\n",
       "   'potentially',\n",
       "   'reduce',\n",
       "   'search',\n",
       "   'space',\n",
       "   'identifying',\n",
       "   'interaction',\n",
       "   'sites.\"boston,',\n",
       "   'ma',\n",
       "   '02215.\",\"structure,',\n",
       "   'function,',\n",
       "   'evolution',\n",
       "   'transient',\n",
       "   'obligate',\n",
       "   'protein–protein',\n",
       "   'interactions\",\"recent',\n",
       "   'analyses',\n",
       "   'high-throughput',\n",
       "   'protein',\n",
       "   'interaction',\n",
       "   'data',\n",
       "   'coupled',\n",
       "   'large-scale',\n",
       "   'investigations',\n",
       "   'evolutionary',\n",
       "   'properties',\n",
       "   'interaction',\n",
       "   'networks',\n",
       "   'left',\n",
       "   'unanswered',\n",
       "   'questions.',\n",
       "   'extent',\n",
       "   'protein',\n",
       "   'interactions',\n",
       "   'act',\n",
       "   'constraints',\n",
       "   'evolution',\n",
       "   'protein',\n",
       "   'sequence?',\n",
       "   'type',\n",
       "   'interaction,',\n",
       "   'specifically',\n",
       "   'transient',\n",
       "   'obligate,',\n",
       "   'play',\n",
       "   'constraints?',\n",
       "   'mutations',\n",
       "   'binding',\n",
       "   'site',\n",
       "   'interacting',\n",
       "   'protein',\n",
       "   'correlated',\n",
       "   'mutations',\n",
       "   'binding',\n",
       "   'site',\n",
       "   'partner?',\n",
       "   'address',\n",
       "   'questions',\n",
       "   'relying',\n",
       "   'carefully',\n",
       "   'curated',\n",
       "   'dataset',\n",
       "   'protein',\n",
       "   'complex',\n",
       "   'structures.',\n",
       "   'results',\n",
       "   'point',\n",
       "   'importance',\n",
       "   'distinguishing',\n",
       "   'transient',\n",
       "   'obligate',\n",
       "   'interactions.',\n",
       "   'conclude',\n",
       "   'residues',\n",
       "   'interfaces',\n",
       "   'obligate',\n",
       "   'complexes',\n",
       "   'tend',\n",
       "   'evolve',\n",
       "   'slower',\n",
       "   'rate,',\n",
       "   'allowing',\n",
       "   'coevolve',\n",
       "   'interacting',\n",
       "   'partners.',\n",
       "   'contrast,',\n",
       "   'plasticity',\n",
       "   'inherent',\n",
       "   'transient',\n",
       "   'interactions',\n",
       "   'leads',\n",
       "   'increased',\n",
       "   'rate',\n",
       "   'substitution',\n",
       "   'interface',\n",
       "   'residues',\n",
       "   'leaves',\n",
       "   'evidence',\n",
       "   'correlated',\n",
       "   'mutations',\n",
       "   'interface.\"\"motivation:',\n",
       "   'structural',\n",
       "   'genomics',\n",
       "   'projects',\n",
       "   'beginning',\n",
       "   'produce',\n",
       "   'protein',\n",
       "   'structures',\n",
       "   'unknown',\n",
       "   'function,',\n",
       "   'therefore,',\n",
       "   'accurate,',\n",
       "   'automated',\n",
       "   'predictors',\n",
       "   'protein',\n",
       "   'function',\n",
       "   'required',\n",
       "   'structures',\n",
       "   'properly',\n",
       "   'annotated',\n",
       "   'reasonable',\n",
       "   'time.',\n",
       "   'identifying',\n",
       "   'interface',\n",
       "   'interacting',\n",
       "   'proteins',\n",
       "   'important',\n",
       "   'clues',\n",
       "   'function',\n",
       "   'protein',\n",
       "   'reduce',\n",
       "   'search',\n",
       "   'space',\n",
       "   'required',\n",
       "   'docking',\n",
       "   'algorithms',\n",
       "   'predict',\n",
       "   'structures',\n",
       "   'complexes.\"\"biomolecular',\n",
       "   'structure',\n",
       "   'modelling',\n",
       "   'unit,',\n",
       "   'department',\n",
       "   'biochemistry',\n",
       "   'molecular',\n",
       "   'biology,',\n",
       "   'university',\n",
       "   'college',\n",
       "   'london,',\n",
       "   'london,',\n",
       "   'united',\n",
       "   'kingdom\",scoring',\n",
       "   'residue',\n",
       "   'conservation,\"the',\n",
       "   'importance',\n",
       "   'residue',\n",
       "   'maintaining',\n",
       "   'structure',\n",
       "   'function',\n",
       "   'protein',\n",
       "   'inferred',\n",
       "   'conserved',\n",
       "   'appears',\n",
       "   'multiple',\n",
       "   'sequence',\n",
       "   'alignment',\n",
       "   'protein',\n",
       "   'homologues.',\n",
       "   'reliable',\n",
       "   'metric',\n",
       "   'quantifying',\n",
       "   'residue',\n",
       "   'conservation',\n",
       "   'desirable.',\n",
       "   'decades',\n",
       "   'scores',\n",
       "   'proposed,',\n",
       "   'emerged',\n",
       "   'generally',\n",
       "   'accepted',\n",
       "   'standard.',\n",
       "   'work',\n",
       "   'surveys',\n",
       "   'range',\n",
       "   'scores',\n",
       "   'biologists,',\n",
       "   'biochemists,',\n",
       "   'and,',\n",
       "   'recently,',\n",
       "   'bioinformatics',\n",
       "   'workers',\n",
       "   'developed,',\n",
       "   'reviews',\n",
       "   'intrinsic',\n",
       "   'problems',\n",
       "   'developing',\n",
       "   'evaluating',\n",
       "   'score.',\n",
       "   'general',\n",
       "   'formula',\n",
       "   'proposed',\n",
       "   'compare',\n",
       "   'properties',\n",
       "   'conservation',\n",
       "   'scores',\n",
       "   'measure',\n",
       "   'conservation',\n",
       "   'right.',\n",
       "   'proteins',\n",
       "   '2002;48:227–241.',\n",
       "   '{\\\\copyright}',\n",
       "   '2002',\n",
       "   '{wiley-liss},',\n",
       "   'inc.\"state',\n",
       "   'university',\n",
       "   'york',\n",
       "   'buffalo,',\n",
       "   '124',\n",
       "   'sherman',\n",
       "   'hall,',\n",
       "   'buffalo,',\n",
       "   'ny',\n",
       "   '14214,',\n",
       "   'usa.\",protein',\n",
       "   'binding',\n",
       "   'site',\n",
       "   'prediction',\n",
       "   'empirical',\n",
       "   'scoring',\n",
       "   'function,\"most',\n",
       "   'biological',\n",
       "   'processes',\n",
       "   'mediated',\n",
       "   'interactions',\n",
       "   'proteins',\n",
       "   'interacting',\n",
       "   'partners',\n",
       "   'including',\n",
       "   'proteins,',\n",
       "   'nucleic',\n",
       "   'acids',\n",
       "   'small',\n",
       "   'molecules.',\n",
       "   'work',\n",
       "   'establishes',\n",
       "   'method',\n",
       "   'called',\n",
       "   '{pinup}',\n",
       "   'binding',\n",
       "   'site',\n",
       "   'prediction',\n",
       "   'monomeric',\n",
       "   'proteins.',\n",
       "   'weight',\n",
       "   'parameters',\n",
       "   'optimize,',\n",
       "   '{pinup}',\n",
       "   'produces',\n",
       "   '42.2\\\\%',\n",
       "   'coverage',\n",
       "   'actual',\n",
       "   'interfaces',\n",
       "   '(percentage',\n",
       "   'correctly',\n",
       "   'predicted',\n",
       "   'interface',\n",
       "   'residues',\n",
       "   'actual',\n",
       "   'interface',\n",
       "   'residues)',\n",
       "   '44.5\\\\%',\n",
       "   'accuracy',\n",
       "   'predicted',\n",
       "   'interfaces',\n",
       "   '(percentage',\n",
       "   'correctly',\n",
       "   'predicted',\n",
       "   'interface',\n",
       "   'residues',\n",
       "   'predicted',\n",
       "   'interface',\n",
       "   'residues)',\n",
       "   'cross',\n",
       "   'validation',\n",
       "   '57-protein',\n",
       "   'dataset.',\n",
       "   'comparison,',\n",
       "   'expected',\n",
       "   'accuracy',\n",
       "   'random',\n",
       "   'prediction',\n",
       "   '(percentage',\n",
       "   'actual',\n",
       "   'interface',\n",
       "   'residues',\n",
       "   'surface',\n",
       "   'residues)',\n",
       "   '15\\\\%.',\n",
       "   'binding',\n",
       "   'sites',\n",
       "   '57-protein',\n",
       "   'set',\n",
       "   'found',\n",
       "   'easier',\n",
       "   'predict',\n",
       "   'independent',\n",
       "   'test',\n",
       "   'set',\n",
       "   '68',\n",
       "   'proteins.',\n",
       "   'average',\n",
       "   'coverage',\n",
       "   'accuracy',\n",
       "   'independent',\n",
       "   'test',\n",
       "   'set',\n",
       "   '30.5',\n",
       "   '29.4\\\\%,',\n",
       "   'respectively.',\n",
       "   'significant',\n",
       "   'gain',\n",
       "   '{pinup}',\n",
       "   'expected',\n",
       "   'random',\n",
       "   'prediction',\n",
       "   'attributed',\n",
       "   '(i)',\n",
       "   'effective',\n",
       "   'residue-energy',\n",
       "   'score',\n",
       "   'accessible-surface-area-dependent',\n",
       "   'interface-propensity,',\n",
       "   '(ii)',\n",
       "   'isolation',\n",
       "   'functional',\n",
       "   'constraints',\n",
       "   'contained',\n",
       "   'conservation',\n",
       "   'score',\n",
       "   'structural',\n",
       "   'constraints',\n",
       "   'combination',\n",
       "   'residue-energy',\n",
       "   'score',\n",
       "   '(for',\n",
       "   'structural',\n",
       "   'constraints)',\n",
       "   'conservation',\n",
       "   'score',\n",
       "   '(iii)',\n",
       "   'consensus',\n",
       "   'region',\n",
       "   'built',\n",
       "   'top-ranked',\n",
       "   'initial',\n",
       "   'patches.\"gower',\n",
       "   'street,',\n",
       "   'london,',\n",
       "   'wc1e',\n",
       "   '6bt,',\n",
       "   'england.\",prediction',\n",
       "   'protein-protein',\n",
       "   'interaction',\n",
       "   'sites',\n",
       "   'patch',\n",
       "   'analysis,\"a',\n",
       "   'method',\n",
       "   'defining',\n",
       "   'analysing',\n",
       "   'series',\n",
       "   'residue',\n",
       "   'patches',\n",
       "   'surface',\n",
       "   'protein',\n",
       "   'structures',\n",
       "   'predict',\n",
       "   'location',\n",
       "   'protein-protein',\n",
       "   'interaction',\n",
       "   'sites.',\n",
       "   'residue',\n",
       "   'patch',\n",
       "   'analysed',\n",
       "   'parameters;',\n",
       "   'solvation',\n",
       "   'potential,',\n",
       "   'residue',\n",
       "   'interface',\n",
       "   'propensity,',\n",
       "   'hydrophobicity,',\n",
       "   'planarity,',\n",
       "   'protrusion',\n",
       "   'accessible',\n",
       "   'surface',\n",
       "   'area.',\n",
       "   'method',\n",
       "   'involves',\n",
       "   'calculation',\n",
       "   'relative',\n",
       "   'combined',\n",
       "   'score',\n",
       "   'probability',\n",
       "   'surface',\n",
       "   'patch',\n",
       "   'forming',\n",
       "   'protein-protein',\n",
       "   'interactions.',\n",
       "   'predictions',\n",
       "   'made',\n",
       "   'structures',\n",
       "   'protomers',\n",
       "   '28',\n",
       "   'homo-dimers,',\n",
       "   'large',\n",
       "   'protomers',\n",
       "   '11',\n",
       "   'hetero-complexes,',\n",
       "   'small',\n",
       "   'protomers',\n",
       "   '14',\n",
       "   'hetero-complexes,',\n",
       "   'antigens',\n",
       "   'antibody-antigen',\n",
       "   'complexes.',\n",
       "   'predictions',\n",
       "   'successful',\n",
       "   '66\\\\%',\n",
       "   '(39/59)',\n",
       "   'structures',\n",
       "   'remainder',\n",
       "   'rationalized',\n",
       "   'terms',\n",
       "   'additional',\n",
       "   'interaction',\n",
       "   'sites.\"\"the',\n",
       "   'sequence',\n",
       "   ...])]"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#function to remove stop words from word list\n",
    "def excludeStopWords(wordList):\n",
    "    ls = list()\n",
    "    for word in wordList:\n",
    "#if word not in stop word list then add word in relevant words list\n",
    "        if word not in stopWordBrdcast.value:\n",
    "            ls.append(word)    \n",
    "    return(ls)\n",
    "\n",
    "#relevantWordRDD contains the user_hash_id and list of words not in the stopwords_en.txt\n",
    "relevantWordRDD = splitRDD.mapValues(excludeStopWords)\n",
    "splitRDD.unpersist()\n",
    "relevantWordRDD.take(2)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.3\n",
    "#counting the frequency of each individual words used by every user\n",
    "#rearranging the result as user_hash_id as key and the tuple (word, count) as value\n",
    "wordCountPerUserRDD = relevantWordRDD.flatMapValues(lambda x: x)\\\n",
    "                                        .map(lambda x: (x,1))\\\n",
    "                                        .reduceByKey(lambda a, b: a + b)\n",
    "relevantWordRDD.unpersist()\n",
    "wordCountPerUserRDD.take(10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('f05bcffe7951de9e5a32fff4a42eb088', ('database', 61)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('functional', 40)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('documentation', 4)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('families', 5)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('identify', 21)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('rules', 3)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('power', 8)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('{uniprotkb}/{swiss-prot}', 1)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('repeats', 1)),\n",
       " ('f05bcffe7951de9e5a32fff4a42eb088', ('70\\\\%)', 1))]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rearrangedRDD = wordCountPerUserRDD.map(lambda x: ((x[0][0], (x[0][1], x[1]))))\n",
    "wordCountPerUserRDD.unpersist()\n",
    "rearrangedRDD.take(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "ename": "Py4JJavaError",
     "evalue": "An error occurred while calling z:org.apache.spark.api.python.PythonRDD.runJob.\n: org.apache.spark.SparkException: Job aborted due to stage failure: Task 1 in stage 27.0 failed 1 times, most recent failure: Lost task 1.0 in stage 27.0 (TID 30, localhost, executor driver): com.esotericsoftware.kryo.KryoException: java.io.IOException: No space left on device\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:183)\n\tat com.esotericsoftware.kryo.io.Output.require(Output.java:160)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:246)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:232)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:54)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:43)\n\tat com.esotericsoftware.kryo.Kryo.writeClassAndObject(Kryo.java:628)\n\tat org.apache.spark.serializer.KryoSerializationStream.writeObject(KryoSerializer.scala:241)\n\tat org.apache.spark.serializer.SerializationStream.writeValue(Serializer.scala:134)\n\tat org.apache.spark.storage.DiskBlockObjectWriter.write(DiskBlockObjectWriter.scala:241)\n\tat org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:151)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:96)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:53)\n\tat org.apache.spark.scheduler.Task.run(Task.scala:109)\n\tat org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:345)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: java.io.IOException: No space left on device\n\tat java.io.FileOutputStream.writeBytes(Native Method)\n\tat java.io.FileOutputStream.write(FileOutputStream.java:326)\n\tat org.apache.spark.storage.TimeTrackingOutputStream.write(TimeTrackingOutputStream.java:58)\n\tat java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)\n\tat java.io.BufferedOutputStream.write(BufferedOutputStream.java:126)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.flushBufferedData(LZ4BlockOutputStream.java:220)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.write(LZ4BlockOutputStream.java:173)\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:181)\n\t... 17 more\n\nDriver stacktrace:\n\tat org.apache.spark.scheduler.DAGScheduler.org$apache$spark$scheduler$DAGScheduler$$failJobAndIndependentStages(DAGScheduler.scala:1602)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1590)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1589)\n\tat scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)\n\tat scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)\n\tat org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:1589)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:831)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:831)\n\tat scala.Option.foreach(Option.scala:257)\n\tat org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:831)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:1823)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1772)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1761)\n\tat org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:48)\n\tat org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:642)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2034)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2055)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2074)\n\tat org.apache.spark.api.python.PythonRDD$.runJob(PythonRDD.scala:149)\n\tat org.apache.spark.api.python.PythonRDD.runJob(PythonRDD.scala)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)\n\tat py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)\n\tat py4j.Gateway.invoke(Gateway.java:282)\n\tat py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)\n\tat py4j.commands.CallCommand.execute(CallCommand.java:79)\n\tat py4j.GatewayConnection.run(GatewayConnection.java:238)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: com.esotericsoftware.kryo.KryoException: java.io.IOException: No space left on device\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:183)\n\tat com.esotericsoftware.kryo.io.Output.require(Output.java:160)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:246)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:232)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:54)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:43)\n\tat com.esotericsoftware.kryo.Kryo.writeClassAndObject(Kryo.java:628)\n\tat org.apache.spark.serializer.KryoSerializationStream.writeObject(KryoSerializer.scala:241)\n\tat org.apache.spark.serializer.SerializationStream.writeValue(Serializer.scala:134)\n\tat org.apache.spark.storage.DiskBlockObjectWriter.write(DiskBlockObjectWriter.scala:241)\n\tat org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:151)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:96)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:53)\n\tat org.apache.spark.scheduler.Task.run(Task.scala:109)\n\tat org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:345)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\t... 1 more\nCaused by: java.io.IOException: No space left on device\n\tat java.io.FileOutputStream.writeBytes(Native Method)\n\tat java.io.FileOutputStream.write(FileOutputStream.java:326)\n\tat org.apache.spark.storage.TimeTrackingOutputStream.write(TimeTrackingOutputStream.java:58)\n\tat java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)\n\tat java.io.BufferedOutputStream.write(BufferedOutputStream.java:126)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.flushBufferedData(LZ4BlockOutputStream.java:220)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.write(LZ4BlockOutputStream.java:173)\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:181)\n\t... 17 more\n",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mPy4JJavaError\u001b[0m                             Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-13-a5e0b3b60649>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0mgroupedSortedRDD\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrearrangedRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgroupByKey\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m                                \u001b[0;34m.\u001b[0m\u001b[0mmap\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;32mlambda\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msorted\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlist\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mlambda\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mreverse\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m \u001b[0mrearrangedRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munpersist\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m \u001b[0mgroupedSortedRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtake\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m/usr/local/spark/python/pyspark/rdd.py\u001b[0m in \u001b[0;36mtake\u001b[0;34m(self, num)\u001b[0m\n\u001b[1;32m   1373\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1374\u001b[0m             \u001b[0mp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrange\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpartsScanned\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpartsScanned\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mnumPartsToTry\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtotalParts\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1375\u001b[0;31m             \u001b[0mres\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcontext\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrunJob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtakeUpToNumLeft\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mp\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1376\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1377\u001b[0m             \u001b[0mitems\u001b[0m \u001b[0;34m+=\u001b[0m \u001b[0mres\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/spark/python/pyspark/context.py\u001b[0m in \u001b[0;36mrunJob\u001b[0;34m(self, rdd, partitionFunc, partitions, allowLocal)\u001b[0m\n\u001b[1;32m   1011\u001b[0m         \u001b[0;31m# SparkContext#runJob.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1012\u001b[0m         \u001b[0mmappedRDD\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mrdd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmapPartitions\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpartitionFunc\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1013\u001b[0;31m         \u001b[0msock_info\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_jvm\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mPythonRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrunJob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_jsc\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msc\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmappedRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_jrdd\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpartitions\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1014\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mlist\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0m_load_from_socket\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msock_info\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmappedRDD\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_jrdd_deserializer\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1015\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/spark/python/lib/py4j-0.10.7-src.zip/py4j/java_gateway.py\u001b[0m in \u001b[0;36m__call__\u001b[0;34m(self, *args)\u001b[0m\n\u001b[1;32m   1255\u001b[0m         \u001b[0manswer\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgateway_client\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msend_command\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcommand\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1256\u001b[0m         return_value = get_return_value(\n\u001b[0;32m-> 1257\u001b[0;31m             answer, self.gateway_client, self.target_id, self.name)\n\u001b[0m\u001b[1;32m   1258\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1259\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mtemp_arg\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mtemp_args\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/spark/python/pyspark/sql/utils.py\u001b[0m in \u001b[0;36mdeco\u001b[0;34m(*a, **kw)\u001b[0m\n\u001b[1;32m     61\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mdeco\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkw\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     62\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 63\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkw\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     64\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mpy4j\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mprotocol\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mPy4JJavaError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m             \u001b[0ms\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0me\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjava_exception\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtoString\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/spark/python/lib/py4j-0.10.7-src.zip/py4j/protocol.py\u001b[0m in \u001b[0;36mget_return_value\u001b[0;34m(answer, gateway_client, target_id, name)\u001b[0m\n\u001b[1;32m    326\u001b[0m                 raise Py4JJavaError(\n\u001b[1;32m    327\u001b[0m                     \u001b[0;34m\"An error occurred while calling {0}{1}{2}.\\n\"\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 328\u001b[0;31m                     format(target_id, \".\", name), value)\n\u001b[0m\u001b[1;32m    329\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    330\u001b[0m                 raise Py4JError(\n",
      "\u001b[0;31mPy4JJavaError\u001b[0m: An error occurred while calling z:org.apache.spark.api.python.PythonRDD.runJob.\n: org.apache.spark.SparkException: Job aborted due to stage failure: Task 1 in stage 27.0 failed 1 times, most recent failure: Lost task 1.0 in stage 27.0 (TID 30, localhost, executor driver): com.esotericsoftware.kryo.KryoException: java.io.IOException: No space left on device\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:183)\n\tat com.esotericsoftware.kryo.io.Output.require(Output.java:160)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:246)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:232)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:54)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:43)\n\tat com.esotericsoftware.kryo.Kryo.writeClassAndObject(Kryo.java:628)\n\tat org.apache.spark.serializer.KryoSerializationStream.writeObject(KryoSerializer.scala:241)\n\tat org.apache.spark.serializer.SerializationStream.writeValue(Serializer.scala:134)\n\tat org.apache.spark.storage.DiskBlockObjectWriter.write(DiskBlockObjectWriter.scala:241)\n\tat org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:151)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:96)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:53)\n\tat org.apache.spark.scheduler.Task.run(Task.scala:109)\n\tat org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:345)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: java.io.IOException: No space left on device\n\tat java.io.FileOutputStream.writeBytes(Native Method)\n\tat java.io.FileOutputStream.write(FileOutputStream.java:326)\n\tat org.apache.spark.storage.TimeTrackingOutputStream.write(TimeTrackingOutputStream.java:58)\n\tat java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)\n\tat java.io.BufferedOutputStream.write(BufferedOutputStream.java:126)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.flushBufferedData(LZ4BlockOutputStream.java:220)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.write(LZ4BlockOutputStream.java:173)\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:181)\n\t... 17 more\n\nDriver stacktrace:\n\tat org.apache.spark.scheduler.DAGScheduler.org$apache$spark$scheduler$DAGScheduler$$failJobAndIndependentStages(DAGScheduler.scala:1602)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1590)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$abortStage$1.apply(DAGScheduler.scala:1589)\n\tat scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)\n\tat scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)\n\tat org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:1589)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:831)\n\tat org.apache.spark.scheduler.DAGScheduler$$anonfun$handleTaskSetFailed$1.apply(DAGScheduler.scala:831)\n\tat scala.Option.foreach(Option.scala:257)\n\tat org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:831)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:1823)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1772)\n\tat org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:1761)\n\tat org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:48)\n\tat org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:642)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2034)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2055)\n\tat org.apache.spark.SparkContext.runJob(SparkContext.scala:2074)\n\tat org.apache.spark.api.python.PythonRDD$.runJob(PythonRDD.scala:149)\n\tat org.apache.spark.api.python.PythonRDD.runJob(PythonRDD.scala)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)\n\tat py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)\n\tat py4j.Gateway.invoke(Gateway.java:282)\n\tat py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)\n\tat py4j.commands.CallCommand.execute(CallCommand.java:79)\n\tat py4j.GatewayConnection.run(GatewayConnection.java:238)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: com.esotericsoftware.kryo.KryoException: java.io.IOException: No space left on device\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:183)\n\tat com.esotericsoftware.kryo.io.Output.require(Output.java:160)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:246)\n\tat com.esotericsoftware.kryo.io.Output.writeBytes(Output.java:232)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:54)\n\tat com.esotericsoftware.kryo.serializers.DefaultArraySerializers$ByteArraySerializer.write(DefaultArraySerializers.java:43)\n\tat com.esotericsoftware.kryo.Kryo.writeClassAndObject(Kryo.java:628)\n\tat org.apache.spark.serializer.KryoSerializationStream.writeObject(KryoSerializer.scala:241)\n\tat org.apache.spark.serializer.SerializationStream.writeValue(Serializer.scala:134)\n\tat org.apache.spark.storage.DiskBlockObjectWriter.write(DiskBlockObjectWriter.scala:241)\n\tat org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:151)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:96)\n\tat org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:53)\n\tat org.apache.spark.scheduler.Task.run(Task.scala:109)\n\tat org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:345)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\t... 1 more\nCaused by: java.io.IOException: No space left on device\n\tat java.io.FileOutputStream.writeBytes(Native Method)\n\tat java.io.FileOutputStream.write(FileOutputStream.java:326)\n\tat org.apache.spark.storage.TimeTrackingOutputStream.write(TimeTrackingOutputStream.java:58)\n\tat java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)\n\tat java.io.BufferedOutputStream.write(BufferedOutputStream.java:126)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.flushBufferedData(LZ4BlockOutputStream.java:220)\n\tat net.jpountz.lz4.LZ4BlockOutputStream.write(LZ4BlockOutputStream.java:173)\n\tat com.esotericsoftware.kryo.io.Output.flush(Output.java:181)\n\t... 17 more\n"
     ]
    }
   ],
   "source": [
    "#Exercise 1.3\n",
    "#Grouping by key and sorting the words in according to their respective word count in descending order\n",
    "groupedSortedRDD = rearrangedRDD.groupByKey()\\\n",
    "                                .map(lambda x: (x[0], sorted(list(x[1]), key=lambda x: x[1], reverse=True)))\n",
    "rearrangedRDD.unpersist()\n",
    "groupedSortedRDD.take(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.3\n",
    "#Choosing the top ten words used by each user\n",
    "topTenGroupRDD = groupedSortedRDD.map(lambda x: ([x[0]] ,x[1][0:10]))\n",
    "groupedSortedRDD.unpersist()\n",
    "topTenGroupRDD.take(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.3\n",
    "#Displaying the top ten words used by each user without the associated word count\n",
    "finalResultRDD = topTenGroupRDD.mapValues(lambda x: [t[0] for t in x])\n",
    "topTenGroupRDD.unpersist()\n",
    "finalResultRDD.take(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.3\n",
    "#Saving the results as text file\n",
    "#finalResultRDD.saveAsTextFile(\"output_files\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.4 (Basic Analysis for Recommender Systems)\n",
    "Basic analysis to get an idea of the characteristics of the dataset."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 a)\n",
    "#Number of distinct users\n",
    "#userLibRDD.count()\n",
    "userLibRDD.map(lambda line: line.split(';')[0]).distinct().count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 a)\n",
    "#Number of distinct items\n",
    "papersRDD.map(lambda line: line.split(',')[0]).distinct().count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 a)\n",
    "#Number of total ratings by users\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: x[1].split(','))\\\n",
    "            .flatMap(lambda x: x)\\\n",
    "            .count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 b)\n",
    "#Min number of ratings a user has given\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: (x[0], x[1].split(',')))\\\n",
    "            .mapValues(len)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .min()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 c)\n",
    "#Max number of ratings a user has given\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: (x[0], x[1].split(',')))\\\n",
    "            .mapValues(len)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .max()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 d)\n",
    "#Average number of ratings of users\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: (x[0], x[1].split(',')))\\\n",
    "            .mapValues(len)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .mean()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 e)\n",
    "#Standard deviation for ratings of users\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: (x[0], x[1].split(',')))\\\n",
    "            .mapValues(len)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .stdev()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 f)\n",
    "#Min number of ratings an item has received\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: x[1].split(','))\\\n",
    "            .flatMap(lambda x: x)\\\n",
    "            .map(lambda x: (x, 1))\\\n",
    "            .reduceByKey(lambda a, b: a + b)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .min()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 g)\n",
    "#Max number of ratings an item has received\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: x[1].split(','))\\\n",
    "            .flatMap(lambda x: x)\\\n",
    "            .map(lambda x: (x, 1))\\\n",
    "            .reduceByKey(lambda a, b: a + b)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .max()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 h)\n",
    "#Average number of ratings of items\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: x[1].split(','))\\\n",
    "            .flatMap(lambda x: x)\\\n",
    "            .map(lambda x: (x, 1))\\\n",
    "            .reduceByKey(lambda a, b: a + b)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .mean()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.4 i)\n",
    "#Standard deviation for ratings of items\n",
    "userLibRDD.map(lambda line: (line.split(';')[0], line.split(';')[1]))\\\n",
    "            .map(lambda x: x[1].split(','))\\\n",
    "            .flatMap(lambda x: x)\\\n",
    "            .map(lambda x: (x, 1))\\\n",
    "            .reduceByKey(lambda a, b: a + b)\\\n",
    "            .map(lambda x: x[1])\\\n",
    "            .stdev()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.5 (Loading the dataset into Dataframes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.5\n",
    "#loading data from users_libraries.txt into a dataframe\n",
    "df = sparkSession.read.csv(path=\"./mod_users_libraries.txt\", sep=';')\n",
    "df.show(5, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.5\n",
    "#loading data from users_libraries.txt into a dataframe\n",
    "#Loading the data in and RDD and splitting the paper_id id values\n",
    "userLibraryRDD = sparkSession.sparkContext.textFile(\"./mod_users_libraries.txt\")\\\n",
    "                .map(lambda x: (x.split(';')[0], x.split(';')[1]))\\\n",
    "                .map(lambda x: (x[0], x[1].split(',')))\\\n",
    "                .flatMapValues(lambda x: x)\n",
    "\n",
    "#Creating the schema for the dataframe\n",
    "schemaString = \"user_hash_id paper_id\"\n",
    "fields = [StructField(field_name, StringType(), True) for field_name in schemaString.split()]\n",
    "schema = StructType(fields)\n",
    "#Creating the dataframe\n",
    "userLibDF = sparkSession.createDataFrame(userLibRDD, schema)\n",
    "userLibDF.show(5, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.5\n",
    "#Loading papers.csv data into a dataframe\n",
    "#Defining the column names\n",
    "columns = ['paper_id', 'type', 'journal', 'book_title', \\\n",
    "           'series', 'publisher', 'pages', 'volume', \\\n",
    "           'number', 'year', 'month', 'postedate',\\\n",
    "           'address', 'title', 'abstract']\n",
    "papersDF = sparkSession.read\\\n",
    "            .load(\"./papers.csv\", format=\"csv\", sep=\",\", inferSchema=\"true\", quote='\"', header=\"false\")\\\n",
    "            .toDF(*columns)\n",
    "papersDF.show(2, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.5\n",
    "#Loading stopwords_en.txt data into a dataframe\n",
    "stopWordsDF = sparkSession.read\\\n",
    "                .load(\"./stopwords_en.txt\", format=\"text\", sep=\" \", inferSchema=\"true\", header=\"false\")\\\n",
    "                .toDF('stop_word')\n",
    "stopWordsDF.show(5, truncate=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.6 (Tasks on top of Dataframes)\n",
    "### Exercise 1.4 (Basic Analysis for Recommender Systems)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 a) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Number of distinct users\n",
    "userLibDF.select(\"user_hash_id\").distinct().count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 a) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Number of distinct items\n",
    "papersDF.select(\"paper_id\").distinct().count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 a) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Number of ratings\n",
    "userLibDF.select(\"paper_id\").count()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 b) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Min number of ratings a user has given\n",
    "userLibDF.select(\"user_hash_id\").groupBy(\"user_hash_id\").count().agg(Funct.min(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 c) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Max number of ratings a user has given\n",
    "userLibDF.select(\"user_hash_id\").groupBy(\"user_hash_id\").count().agg(Funct.max(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 d) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Average number of ratings of users\n",
    "userLibDF.select(\"user_hash_id\").groupBy(\"user_hash_id\").count().agg(Funct.mean(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 e) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Standard deviation for ratings of users\n",
    "userLibDF.select(\"user_hash_id\").groupBy(\"user_hash_id\").count().agg(Funct.stddev(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 f) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Min number of ratings an item has received\n",
    "userLibDF.select(\"paper_id\").groupBy(\"paper_id\").count().agg(Funct.min(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 g) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Max number of ratings an item has received\n",
    "userLibDF.select(\"paper_id\").groupBy(\"paper_id\").count().agg(Funct.max(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 h) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Average number of ratings of items\n",
    "userLibDF.select(\"paper_id\").groupBy(\"paper_id\").count().agg(Funct.mean(\"count\")).show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.4 i) Using DataFrames (Basic Analysis for Recommender Systems)\n",
    "#Average number of ratings of items\n",
    "userLibDF.select(\"paper_id\").groupBy(\"paper_id\").count().agg(Funct.stddev(\"count\")).show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Exercise 1.6 (Tasks on top of Dataframes)\n",
    "### Exercise 1.3 (Joining Collections)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Joining the userLib dataframe with the papersDF dataframe over paper_id\n",
    "#Dropping any row that has null value in the abstract column\n",
    "joinedDF = userLibDF.join(papersDF, userLibDF.paper_id == papersDF.paper_id, \"inner\")\\\n",
    "                        .select(userLibDF.user_hash_id,\\\n",
    "                                Funct.explode(Funct.split(papersDF.abstract,\" \"))\\\n",
    "                                .alias(\"word\"))\\\n",
    "                        .na.drop(\"any\")\n",
    "\n",
    "joinedDF.show(20, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Subtracting the stop words from the user and abstract words dataframe\n",
    "withoutStpWrdDF = joinedDF.join(stopWordsDF, joinedDF.word==stopWordsDF.stop_word, how=\"left_anti\")\n",
    "withoutStpWrdDF.show(20, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Grouping the joinedDf dataframe over 'user_hash_id' and 'word' columns\n",
    "#Sorting in ascending order the result w.r.t the 'user_hash_id' column\n",
    "groupedDF = withoutStpWrdDF.groupBy(\"user_hash_id\", \"word\")\\\n",
    "                    .agg(Funct.count(\"word\").alias(\"word_count\"))\\\n",
    "                    .sort(Funct.asc(\"user_hash_id\"))\n",
    "\n",
    "groupedDF.show(10, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Creating a window and using 'rank' function over the window to give a rank to the words per user and according \n",
    "#to the word count\n",
    "window = Window.partitionBy(\"user_hash_id\").orderBy(Funct.desc(\"word_count\"))\n",
    "rankedResultDF = groupedDF.withColumn(\"ranking\", Funct.rank().over(window))\n",
    "filteredRankedResDF = rankedResultDF.filter(rankedResultDF.ranking < 10).select(\"user_hash_id\", \"word\", \"ranking\")\n",
    "filteredRankedResDF.show(30, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Transforming the dataframe so that the top 10 most used words are combined into a list for eash user \n",
    "finalResultDF = filteredRankedResDF.groupBy(\"user_hash_id\").agg(Funct.collect_list(\"word\").alias(\"word\"))\n",
    "finalResultDF.show(10, truncate=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Exercise 1.6\n",
    "#Exercise 1.3 Using Dataframes (Joining collections)\n",
    "#Saving the result as a text file\n",
    "#finalResultDF.select(finalResultDF.user_hash_id, finalResultDF.word.cast(\"string\")).write.csv(path='dataframe_result',mode='overwrite',sep=' ')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
